{
  "cells": [
    {
      "cell_type": "markdown",
      "source": [
        "# Overview\n",
        "\n",
        "We're encountering an bug where the sentence chunks and their corresponding labels are not aligning. This is messing up our intrinsic evaluation results. Misalignment is ikely due to the 'nan' values of some labels in a few datasets not being parsed."
      ],
      "metadata": {
        "id": "zJtBPPG_7u06"
      }
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "eAcY_Hnk4TUT"
      },
      "source": [
        "# Env Setup"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "X5f-2VCub2fX",
        "outputId": "fa9518b1-12b9-4e8e-eae7-5adb5c98ea15"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Collecting openai\n",
            "  Downloading openai-1.3.7-py3-none-any.whl (221 kB)\n",
            "\u001b[?25l     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m0.0/221.4 kB\u001b[0m \u001b[31m?\u001b[0m eta \u001b[36m-:--:--\u001b[0m\r\u001b[2K     \u001b[91m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m\u001b[91m╸\u001b[0m\u001b[90m━━━━━━━━━━\u001b[0m \u001b[32m163.8/221.4 kB\u001b[0m \u001b[31m5.3 MB/s\u001b[0m eta \u001b[36m0:00:01\u001b[0m\r\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m221.4/221.4 kB\u001b[0m \u001b[31m5.4 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hRequirement already satisfied: anyio<4,>=3.5.0 in /usr/local/lib/python3.10/dist-packages (from openai) (3.7.1)\n",
            "Requirement already satisfied: distro<2,>=1.7.0 in /usr/lib/python3/dist-packages (from openai) (1.7.0)\n",
            "Collecting httpx<1,>=0.23.0 (from openai)\n",
            "  Downloading httpx-0.25.2-py3-none-any.whl (74 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m75.0/75.0 kB\u001b[0m \u001b[31m10.2 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hRequirement already satisfied: pydantic<3,>=1.9.0 in /usr/local/lib/python3.10/dist-packages (from openai) (1.10.13)\n",
            "Requirement already satisfied: sniffio in /usr/local/lib/python3.10/dist-packages (from openai) (1.3.0)\n",
            "Requirement already satisfied: tqdm>4 in /usr/local/lib/python3.10/dist-packages (from openai) (4.66.1)\n",
            "Requirement already satisfied: typing-extensions<5,>=4.5 in /usr/local/lib/python3.10/dist-packages (from openai) (4.5.0)\n",
            "Requirement already satisfied: idna>=2.8 in /usr/local/lib/python3.10/dist-packages (from anyio<4,>=3.5.0->openai) (3.6)\n",
            "Requirement already satisfied: exceptiongroup in /usr/local/lib/python3.10/dist-packages (from anyio<4,>=3.5.0->openai) (1.2.0)\n",
            "Requirement already satisfied: certifi in /usr/local/lib/python3.10/dist-packages (from httpx<1,>=0.23.0->openai) (2023.11.17)\n",
            "Collecting httpcore==1.* (from httpx<1,>=0.23.0->openai)\n",
            "  Downloading httpcore-1.0.2-py3-none-any.whl (76 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m76.9/76.9 kB\u001b[0m \u001b[31m6.8 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting h11<0.15,>=0.13 (from httpcore==1.*->httpx<1,>=0.23.0->openai)\n",
            "  Downloading h11-0.14.0-py3-none-any.whl (58 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m58.3/58.3 kB\u001b[0m \u001b[31m5.6 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hInstalling collected packages: h11, httpcore, httpx, openai\n",
            "\u001b[31mERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.\n",
            "llmx 0.0.15a0 requires cohere, which is not installed.\n",
            "llmx 0.0.15a0 requires tiktoken, which is not installed.\u001b[0m\u001b[31m\n",
            "\u001b[0mSuccessfully installed h11-0.14.0 httpcore-1.0.2 httpx-0.25.2 openai-1.3.7\n"
          ]
        }
      ],
      "source": [
        "pip install openai"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "jBKcciX3z249",
        "outputId": "1d940f3a-47ee-4c34-ee79-81b5e7eccb30"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Requirement already satisfied: nest_asyncio in /usr/local/lib/python3.10/dist-packages (1.5.8)\n"
          ]
        }
      ],
      "source": [
        "pip install nest_asyncio"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "ysb7oa_FvlJq"
      },
      "outputs": [],
      "source": [
        "import csv\n",
        "import json\n",
        "import pandas as pd\n",
        "import numpy as np\n",
        "import re\n",
        "import string\n",
        "from collections import Counter\n",
        "import openai\n",
        "import os\n",
        "from google.colab import drive\n",
        "import time\n",
        "import ast"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "wPo08KiRcXsx",
        "outputId": "fb3ad30c-74ac-41d1-e1c2-4a9e2b19d42d"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Mounted at /content/drive\n"
          ]
        }
      ],
      "source": [
        "drive.mount('/content/drive')"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "MOOGViRbM1on",
        "outputId": "70485d92-2787-4ec3-f4ce-8b8e6e2467b7"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "/content/drive/MyDrive/6.8611 Research Project/Colab Notebooks\n"
          ]
        }
      ],
      "source": [
        "cd drive/MyDrive/'6.8611 Research Project'/'Colab Notebooks'"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "78oLc-uw6qb1",
        "outputId": "18e61744-c850-4831-a789-a3ea652c70d1"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            " BC5CDR-D_devel_1.csv               ' NER with BERT.ipynb'\n",
            " BC5CDR-D_devel_2.csv                openai-test.ipynb\n",
            " Data-cleaning.ipynb                 retry_prompts.gsheet\n",
            " \u001b[0m\u001b[01;34mdevel_gpt_generated_datasets\u001b[0m/       RW-Fine-Tuning-Human-Annotated.ipynb\n",
            " Fine-Tuning-Human-Annotated.ipynb   TEST_LABEL_BUGS.ipynb\n",
            " Fine-Tuning-One-Shot.ipynb          tokens_labels.csv\n",
            " Fine-Tuning-Zero-Shot.ipynb         zero-shot-bc5cdr-chem.pynb\n",
            " intrinsic_eval_entity.ipynb        'zero_shot[FASTER].ipynb'\n",
            " intrinsic_eval.ipynb                zero-shot.pynb\n",
            " \u001b[01;34mllm-annotations\u001b[0m/\n"
          ]
        }
      ],
      "source": [
        "ls"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "70AoigM1u6Ct"
      },
      "source": [
        "# Data Preprocessing"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "QD23yqLLb5-X"
      },
      "outputs": [],
      "source": [
        "# load the datasets into dataframes\n",
        "\n",
        "def load_tsv_dataset(file_path):\n",
        "  \"\"\"\n",
        "  Loads a tsv dataset. Renames thne columns to 'token' and 'label'.\n",
        "  Note that renaming the columns will overwrite the first row of the dataframe\n",
        "  \"\"\"\n",
        "  df = pd.read_csv(file_path, delimiter='\\t', header=None, engine='python')\n",
        "  df.columns = ['token', 'label']\n",
        "  print(df.head())\n",
        "  return df\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "jAyxxu1wp1Lw"
      },
      "outputs": [],
      "source": [
        "# load the datasets into dataframes\n",
        "\n",
        "def load_csv_dataset(file_path):\n",
        "  \"\"\"\n",
        "  Loads a csv dataset. Renames thne columns to 'token' and 'label'.\n",
        "  Note that renaming the columns will overwrite the first row of the dataframe\n",
        "  \"\"\"\n",
        "  df = pd.read_csv(file_path, header=None, engine='python')\n",
        "  df.columns = ['token', 'label']\n",
        "  print(df.head())\n",
        "  return df\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "0eHrO5iXiYW5"
      },
      "outputs": [],
      "source": [
        "def split_by_sentence(list_of_strings):\n",
        "  sentences = []\n",
        "  current_sentence = []\n",
        "\n",
        "  for word in list_of_strings:\n",
        "      current_sentence.append(word)\n",
        "      if type(word) is str and word.endswith('.'):\n",
        "          sentence_str = ' '.join(map(str, current_sentence))\n",
        "          sentences.append(sentence_str)\n",
        "          current_sentence = []\n",
        "\n",
        "  return sentences"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "z5rzPsANcKA-"
      },
      "outputs": [],
      "source": [
        "def get_filtered_entities(df, target_label):\n",
        "  \"\"\"\n",
        "  df (pandas dataframe): has two columns 'token' and 'label'\n",
        "  target_label: 'B', 'I', or 'O' (see description above for what these signify)\n",
        "\n",
        "  Filtering involves: removing blanks, and filtering out entities that consist\n",
        "  only of punctuation, numbers, or single letters.\n",
        "\n",
        "  Return a frequency of all filtered entities with label 'target_label'.\n",
        "  \"\"\"\n",
        "  filtered_df = df[df['label'] == target_label]\n",
        "  target_entities = filtered_df['token'].tolist() # a set of all the entities with the target label\n",
        "\n",
        "  # regex for filtering out nonsense strings\n",
        "  punctuation = re.escape(string.punctuation)\n",
        "  pattern = re.compile(rf'^(?![a-zA-Z]?$)(?!\\d+$)(?!^[{punctuation}]+$).+')\n",
        "  target_entities = [ent for ent in target_entities if pattern.match(ent)]\n",
        "  return Counter(target_entities)"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "_LHyMze2Rwx0"
      },
      "source": [
        "# Bugs"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "t5TC5GQk8o0S"
      },
      "outputs": [],
      "source": [
        "dataset = \"BC5CDR-chem\" #NEED TO SET"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "q5RadoCORx0M",
        "outputId": "2067fb1b-06ce-475f-a2c9-d8b56be781d8"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "           token label\n",
            "0             22     B\n",
            "1              -     I\n",
            "2  oxacalcitriol     I\n",
            "3     suppresses     O\n",
            "4      secondary     O\n",
            "         token label\n",
            "0      Torsade     O\n",
            "1           de     O\n",
            "2      pointes     O\n",
            "3  ventricular     O\n",
            "4  tachycardia     O\n"
          ]
        }
      ],
      "source": [
        "devel = f'llm-annotations/datasets/{dataset}/devel.tsv'\n",
        "devel_df = load_tsv_dataset(devel)\n",
        "\n",
        "test = f'llm-annotations/datasets/{dataset}/test.tsv'\n",
        "test_df = load_tsv_dataset(test)"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "yitqFHkSR1iL",
        "outputId": "e000d78d-cc81-4dc7-d20c-d2a9cb36f57e"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "14135 Torsade\n",
            "99756 Torsade\n",
            "99785 Torsade\n",
            "NUM CHUNKS 1750\n",
            "['Torsade', 'de', 'pointes', 'ventricular', 'tachycardia', 'during', 'low', 'dose', 'intermittent', 'dobutamine', 'treatment', 'in', 'a', 'patient', 'with', 'dilated', 'cardiomyopathy', 'and', 'congestive', 'heart', 'failure', '.', 'The', 'authors', 'describe', 'the', 'case', 'of', 'a', '56', '-', 'year', '-', 'old', 'woman', 'with', 'chronic', ',', 'severe', 'heart', 'failure', 'secondary', 'to', 'dilated', 'cardiomyopathy', 'and', 'absence', 'of', 'significant', 'ventricular', 'arrhythmias', 'who', 'developed', 'QT', 'prolongation', 'and', 'torsade', 'de', 'pointes', 'ventricular', 'tachycardia', 'during', 'one', 'cycle', 'of', 'intermittent', 'low', 'dose', '(', '2', '.', '5', 'mcg', '/', 'kg', 'per', 'min', ')', 'dobutamine', '.', 'This', 'report', 'of', 'torsade', 'de', 'pointes', 'ventricular', 'tachycardia', 'during', 'intermittent', 'dobutamine', 'supports', 'the', 'hypothesis', 'that', 'unpredictable', 'fatal', 'arrhythmias', 'may', 'occur']\n",
            "0 Torsade\n",
            "75487 Torsades\n",
            "79450 Torsade\n",
            "['polyuria', '.', 'PKC', '-', 'alpha', 'null', 'mice', '(', 'PKCa', 'KO']\n",
            "['and', 'turnover', 'of', 'noradrenaline', '(', 'null', ')', 'were', 'also', 'performed']\n",
            "['that', 'three', 'drugs', 'stimulating', 'central', 'null', 'receptors', 'failed', 'to', 'affect']\n",
            "['.', 'The', 'brain', 'concentration', 'of', 'null', 'was', 'not', 'changed', 'by']\n",
            "['decreased', 'the', 'brain', 'level', 'of', 'null', '.', 'The', 'rate', 'of']\n",
            "['null', '.', 'The', 'rate', 'of', 'null', 'turnover', 'was', 'not', 'altered']\n",
            "['following', 'which', 'the', 'disappearance', 'of', 'null', 'from', 'the', 'brain', 'was']\n",
            "['that', 'in', 'rats', 'the', 'brain', 'null', 'plays', 'a', 'less', 'important']\n",
            "119879 Torsades\n",
            "NUM TEST CHUNKS 1852\n"
          ]
        }
      ],
      "source": [
        "def get_chunks(all_tokens, CHUNK_SIZE = 300):\n",
        "  for i in range(len(all_tokens)):\n",
        "    token = all_tokens[i]\n",
        "    if isinstance(token, str) and 'Torsade' in token:\n",
        "      print(i, token)\n",
        "    if isinstance(token,float) and str(token)==\"nan\":\n",
        "      print(all_tokens[i-5: i+5])\n",
        "      all_tokens[i]=\"null\"\n",
        "\n",
        "  sentences = split_by_sentence(all_tokens)\n",
        "\n",
        "  SENTENCE_CHUNKS = []\n",
        "\n",
        "  curr_chunk, curr_chunk_len = [], 0\n",
        "  for sent in sentences:\n",
        "    curr_chunk.append(sent)\n",
        "    curr_chunk_len += len(sent)\n",
        "    if curr_chunk_len >= CHUNK_SIZE:\n",
        "      SENTENCE_CHUNKS.append(' '.join(curr_chunk))\n",
        "      curr_chunk = []\n",
        "      curr_chunk_len = 0\n",
        "\n",
        "  SENTENCE_CHUNKS.append(' '.join(curr_chunk))\n",
        "\n",
        "  return SENTENCE_CHUNKS\n",
        "\n",
        "\n",
        "all_tokens = devel_df['token'].tolist()\n",
        "CHUNK_SIZE = 300 # string length of the chunk\n",
        "SENTENCE_CHUNKS = get_chunks(all_tokens)\n",
        "\n",
        "print('NUM CHUNKS', len(SENTENCE_CHUNKS))\n",
        "\n",
        "all_test_tokens = test_df['token'].tolist()\n",
        "print(all_test_tokens[0:100])\n",
        "CHUNK_SIZE = 300 # string length of the chunk\n",
        "TEST_SENTENCE_CHUNKS = get_chunks(all_test_tokens)\n",
        "\n",
        "print('NUM TEST CHUNKS', len(TEST_SENTENCE_CHUNKS))"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "sVcaP0TEj3tP",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 1000
        },
        "outputId": "ec1dead1-34ac-49da-cfce-45221ab4946d"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\u001b[1;30;43mStreaming output truncated to the last 5000 lines.\u001b[0m\n",
            "42  13096              ,     O\n",
            "43  13097             an     O\n",
            "44  13098       amoeboid     O\n",
            "45  13099      protozoan     O\n",
            "46  13100              .     O\n",
            "47  13101             An     O\n",
            "48  13102     evaluation     O\n",
            "was intraperitoneally administered to the guinea pigs 30 minutes before the onset of local anoxia . Mutations that extend the specificity of the endonuclease activity of lambda terminase . The gene set in this mitochondrial genome is a subset of that of Reclinomonas americana , an amoeboid protozoan .\n",
            "['was', 'intraperitoneally', 'administered', 'to', 'the', 'guinea', 'pigs', '30', 'minutes', 'before', 'the', 'onset', 'of', 'local', 'anoxia', '.', 'Mutations', 'that', 'extend', 'the', 'specificity', 'of', 'the', 'endonuclease', 'activity', 'of', 'lambda', 'terminase', '.', 'The', 'gene', 'set', 'in', 'this', 'mitochondrial', 'genome', 'is', 'a', 'subset', 'of', 'that', 'of', 'Reclinomonas', 'americana', ',', 'an', 'amoeboid', 'protozoan', '.']\n",
            "49 49\n",
            "['administered_0', 'to_1', 'the_2', 'guinea_3', 'pigs_4', '30_5', 'minutes_6', 'before_7', 'the_8', 'onset_9', 'of_10', 'local_11', 'anoxia_12', '._13', 'Mutations_14', 'that_15', 'extend_16', 'the_17', 'specificity_18', 'of_19', 'the_20', 'endonuclease_21', 'activity_22', 'of_23', 'lambda_24', 'terminase_25', '._26', 'The_27', 'gene_28', 'set_29', 'in_30', 'this_31', 'mitochondrial_32', 'genome_33', 'is_34', 'a_35', 'subset_36', 'of_37', 'that_38', 'of_39', 'Reclinomonas_40', 'americana_41', ',_42', 'an_43', 'amoeboid_44', 'protozoan_45', '._46', 'An_47', 'evaluation_48']\n",
            "BAD:  13054\n",
            "rpws:      index     token label\n",
            "0   13103        of     O\n",
            "1   13104         a     O\n",
            "2   13105  positive     O\n",
            "3   13106   control     O\n",
            "4   13107       for     O\n",
            "..    ...       ...   ...\n",
            "91  13194       the     O\n",
            "92  13195         3     O\n",
            "93  13196         .     O\n",
            "94  13197         6     O\n",
            "95  13198         -     O\n",
            "\n",
            "[96 rows x 3 columns]\n",
            "An evaluation of a positive control for platelet neutralization procedure testing with seven commercial activated partial thromboplastin time reagents . The prosomal RNA - binding protein p27K is a member of the alpha - type human prosomal gene family . A single clone , which was conserved and had near - perfect homology to eight human / rodent expressed sequence tags , was used as template for 5 ' and 3 ' rapid amplification of cDNA ends and SPICE ( system for polymerase chain reaction amplification of cDNA ends ) reactions to obtain the 3 .\n",
            "['An', 'evaluation', 'of', 'a', 'positive', 'control', 'for', 'platelet', 'neutralization', 'procedure', 'testing', 'with', 'seven', 'commercial', 'activated', 'partial', 'thromboplastin', 'time', 'reagents', '.', 'The', 'prosomal', 'RNA', '-', 'binding', 'protein', 'p27K', 'is', 'a', 'member', 'of', 'the', 'alpha', '-', 'type', 'human', 'prosomal', 'gene', 'family', '.', 'A', 'single', 'clone', ',', 'which', 'was', 'conserved', 'and', 'had', 'near', '-', 'perfect', 'homology', 'to', 'eight', 'human', '/', 'rodent', 'expressed', 'sequence', 'tags', ',', 'was', 'used', 'as', 'template', 'for', '5', \"'\", 'and', '3', \"'\", 'rapid', 'amplification', 'of', 'cDNA', 'ends', 'and', 'SPICE', '(', 'system', 'for', 'polymerase', 'chain', 'reaction', 'amplification', 'of', 'cDNA', 'ends', ')', 'reactions', 'to', 'obtain', 'the', '3', '.']\n",
            "96 96\n",
            "['of_0', 'a_1', 'positive_2', 'control_3', 'for_4', 'platelet_5', 'neutralization_6', 'procedure_7', 'testing_8', 'with_9', 'seven_10', 'commercial_11', 'activated_12', 'partial_13', 'thromboplastin_14', 'time_15', 'reagents_16', '._17', 'The_18', 'prosomal_19', 'RNA_20', '-_21', 'binding_22', 'protein_23', 'p27K_24', 'is_25', 'a_26', 'member_27', 'of_28', 'the_29', 'alpha_30', '-_31', 'type_32', 'human_33', 'prosomal_34', 'gene_35', 'family_36', '._37', 'A_38', 'single_39', 'clone_40', ',_41', 'which_42', 'was_43', 'conserved_44', 'and_45', 'had_46', 'near_47', '-_48', 'perfect_49', 'homology_50', 'to_51', 'eight_52', 'human_53', '/_54', 'rodent_55', 'expressed_56', 'sequence_57', 'tags_58', ',_59', 'was_60', 'used_61', 'as_62', 'template_63', 'for_64', '5_65', \"'_66\", 'and_67', '3_68', \"'_69\", 'rapid_70', 'amplification_71', 'of_72', 'cDNA_73', 'ends_74', 'and_75', 'SPICE_76', '(_77', 'system_78', 'for_79', 'polymerase_80', 'chain_81', 'reaction_82', 'amplification_83', 'of_84', 'cDNA_85', 'ends_86', ')_87', 'reactions_88', 'to_89', 'obtain_90', 'the_91', '3_92', '._93', '6_94', '-_95']\n",
            "BAD:  13103\n",
            "rpws:      index token label\n",
            "0   13199    kb     O\n",
            "1   13200  cDNA     O\n",
            "2   13201     ,     O\n",
            "3   13202  LGL2     B\n",
            "4   13203     (     O\n",
            "..    ...   ...   ...\n",
            "65  13264  exon     O\n",
            "66  13265     3     O\n",
            "67  13266     .     O\n",
            "68  13267    It     O\n",
            "69  13268    is     O\n",
            "\n",
            "[70 rows x 3 columns]\n",
            "6 - kb cDNA , LGL2 ( Genbank , AF 110195 ) encoding a deduced polypeptide ( lgl2 ) of 963 amino acids . These tiny premature infants also received PN for significantly longer periods of time , and the longer the infusions were administered the greater was the risk of cholestasis developing . The SM - specific selection of exon 2 results from the inhibition of exon 3 .\n",
            "['6', '-', 'kb', 'cDNA', ',', 'LGL2', '(', 'Genbank', ',', 'AF', '110195', ')', 'encoding', 'a', 'deduced', 'polypeptide', '(', 'lgl2', ')', 'of', '963', 'amino', 'acids', '.', 'These', 'tiny', 'premature', 'infants', 'also', 'received', 'PN', 'for', 'significantly', 'longer', 'periods', 'of', 'time', ',', 'and', 'the', 'longer', 'the', 'infusions', 'were', 'administered', 'the', 'greater', 'was', 'the', 'risk', 'of', 'cholestasis', 'developing', '.', 'The', 'SM', '-', 'specific', 'selection', 'of', 'exon', '2', 'results', 'from', 'the', 'inhibition', 'of', 'exon', '3', '.']\n",
            "70 70\n",
            "['kb_0', 'cDNA_1', ',_2', 'LGL2_3', '(_4', 'Genbank_5', ',_6', 'AF_7', '110195_8', ')_9', 'encoding_10', 'a_11', 'deduced_12', 'polypeptide_13', '(_14', 'lgl2_15', ')_16', 'of_17', '963_18', 'amino_19', 'acids_20', '._21', 'These_22', 'tiny_23', 'premature_24', 'infants_25', 'also_26', 'received_27', 'PN_28', 'for_29', 'significantly_30', 'longer_31', 'periods_32', 'of_33', 'time_34', ',_35', 'and_36', 'the_37', 'longer_38', 'the_39', 'infusions_40', 'were_41', 'administered_42', 'the_43', 'greater_44', 'was_45', 'the_46', 'risk_47', 'of_48', 'cholestasis_49', 'developing_50', '._51', 'The_52', 'SM_53', '-_54', 'specific_55', 'selection_56', 'of_57', 'exon_58', '2_59', 'results_60', 'from_61', 'the_62', 'inhibition_63', 'of_64', 'exon_65', '3_66', '._67', 'It_68', 'is_69']\n",
            "BAD:  13199\n",
            "rpws:      index    token label\n",
            "0   13269   always     O\n",
            "1   13270  present     O\n",
            "2   13271      and     O\n",
            "3   13272      has     O\n",
            "4   13273        a     O\n",
            "..    ...      ...   ...\n",
            "62  13331        (     O\n",
            "63  13332       22     O\n",
            "64  13333        .     O\n",
            "65  13334        2     O\n",
            "66  13335        -     O\n",
            "\n",
            "[67 rows x 3 columns]\n",
            "It is always present and has a discrete origin and insertion . Medical management of early - stage breast cancer . Five girls with Turner ' s syndrome aged 12 to 17 years showed LH values ( 5 . 0 - 14 . 5 IU / 12 h ) at the upper end of or slightly above the normal range and pathologically high values ( 22 .\n",
            "['It', 'is', 'always', 'present', 'and', 'has', 'a', 'discrete', 'origin', 'and', 'insertion', '.', 'Medical', 'management', 'of', 'early', '-', 'stage', 'breast', 'cancer', '.', 'Five', 'girls', 'with', 'Turner', \"'\", 's', 'syndrome', 'aged', '12', 'to', '17', 'years', 'showed', 'LH', 'values', '(', '5', '.', '0', '-', '14', '.', '5', 'IU', '/', '12', 'h', ')', 'at', 'the', 'upper', 'end', 'of', 'or', 'slightly', 'above', 'the', 'normal', 'range', 'and', 'pathologically', 'high', 'values', '(', '22', '.']\n",
            "67 67\n",
            "['always_0', 'present_1', 'and_2', 'has_3', 'a_4', 'discrete_5', 'origin_6', 'and_7', 'insertion_8', '._9', 'Medical_10', 'management_11', 'of_12', 'early_13', '-_14', 'stage_15', 'breast_16', 'cancer_17', '._18', 'Five_19', 'girls_20', 'with_21', 'Turner_22', \"'_23\", 's_24', 'syndrome_25', 'aged_26', '12_27', 'to_28', '17_29', 'years_30', 'showed_31', 'LH_32', 'values_33', '(_34', '5_35', '._36', '0_37', '-_38', '14_39', '._40', '5_41', 'IU_42', '/_43', '12_44', 'h_45', ')_46', 'at_47', 'the_48', 'upper_49', 'end_50', 'of_51', 'or_52', 'slightly_53', 'above_54', 'the_55', 'normal_56', 'range_57', 'and_58', 'pathologically_59', 'high_60', 'values_61', '(_62', '22_63', '._64', '2_65', '-_66']\n",
            "BAD:  13269\n",
            "rpws:       index   token label\n",
            "0    13336      43     O\n",
            "1    13337       .     O\n",
            "2    13338       5     O\n",
            "3    13339      IU     O\n",
            "4    13340       /     O\n",
            "..     ...     ...   ...\n",
            "128  13464      of     O\n",
            "129  13465  DEFCAP     B\n",
            "130  13466       .     O\n",
            "131  13467      In     O\n",
            "132  13468       6     O\n",
            "\n",
            "[133 rows x 3 columns]\n",
            "2 - 43 . 5 IU / 12 h ) for FSH . In vitro death assays with transient overexpression of deletion constructs of both isoforms using beta - galactosidase as a reporter gene in MCF7 cells suggest the following : 1 ) the nucleotide binding domain may act as a negative regulator of the killing activity of DEFCAP ; 2 ) the LRR / CARD represents a putative constitutively active inducer of apoptosis ; 3 ) the killing activity of LRR / CARD is inhibitable by benzyloxycarbonyl - Val - Ala - Asp ( OMe ) - fluoromethyl ketone and to a lesser extent by Asp - Glu - Val - Asp ( OMe ) - fluoromethyl ketone ; and 4 ) the CARD is critical for killing activity of DEFCAP .\n",
            "['2', '-', '43', '.', '5', 'IU', '/', '12', 'h', ')', 'for', 'FSH', '.', 'In', 'vitro', 'death', 'assays', 'with', 'transient', 'overexpression', 'of', 'deletion', 'constructs', 'of', 'both', 'isoforms', 'using', 'beta', '-', 'galactosidase', 'as', 'a', 'reporter', 'gene', 'in', 'MCF7', 'cells', 'suggest', 'the', 'following', ':', '1', ')', 'the', 'nucleotide', 'binding', 'domain', 'may', 'act', 'as', 'a', 'negative', 'regulator', 'of', 'the', 'killing', 'activity', 'of', 'DEFCAP', ';', '2', ')', 'the', 'LRR', '/', 'CARD', 'represents', 'a', 'putative', 'constitutively', 'active', 'inducer', 'of', 'apoptosis', ';', '3', ')', 'the', 'killing', 'activity', 'of', 'LRR', '/', 'CARD', 'is', 'inhibitable', 'by', 'benzyloxycarbonyl', '-', 'Val', '-', 'Ala', '-', 'Asp', '(', 'OMe', ')', '-', 'fluoromethyl', 'ketone', 'and', 'to', 'a', 'lesser', 'extent', 'by', 'Asp', '-', 'Glu', '-', 'Val', '-', 'Asp', '(', 'OMe', ')', '-', 'fluoromethyl', 'ketone', ';', 'and', '4', ')', 'the', 'CARD', 'is', 'critical', 'for', 'killing', 'activity', 'of', 'DEFCAP', '.']\n",
            "133 133\n",
            "['43_0', '._1', '5_2', 'IU_3', '/_4', '12_5', 'h_6', ')_7', 'for_8', 'FSH_9', '._10', 'In_11', 'vitro_12', 'death_13', 'assays_14', 'with_15', 'transient_16', 'overexpression_17', 'of_18', 'deletion_19', 'constructs_20', 'of_21', 'both_22', 'isoforms_23', 'using_24', 'beta_25', '-_26', 'galactosidase_27', 'as_28', 'a_29', 'reporter_30', 'gene_31', 'in_32', 'MCF7_33', 'cells_34', 'suggest_35', 'the_36', 'following_37', ':_38', '1_39', ')_40', 'the_41', 'nucleotide_42', 'binding_43', 'domain_44', 'may_45', 'act_46', 'as_47', 'a_48', 'negative_49', 'regulator_50', 'of_51', 'the_52', 'killing_53', 'activity_54', 'of_55', 'DEFCAP_56', ';_57', '2_58', ')_59', 'the_60', 'LRR_61', '/_62', 'CARD_63', 'represents_64', 'a_65', 'putative_66', 'constitutively_67', 'active_68', 'inducer_69', 'of_70', 'apoptosis_71', ';_72', '3_73', ')_74', 'the_75', 'killing_76', 'activity_77', 'of_78', 'LRR_79', '/_80', 'CARD_81', 'is_82', 'inhibitable_83', 'by_84', 'benzyloxycarbonyl_85', '-_86', 'Val_87', '-_88', 'Ala_89', '-_90', 'Asp_91', '(_92', 'OMe_93', ')_94', '-_95', 'fluoromethyl_96', 'ketone_97', 'and_98', 'to_99', 'a_100', 'lesser_101', 'extent_102', 'by_103', 'Asp_104', '-_105', 'Glu_106', '-_107', 'Val_108', '-_109', 'Asp_110', '(_111', 'OMe_112', ')_113', '-_114', 'fluoromethyl_115', 'ketone_116', ';_117', 'and_118', '4_119', ')_120', 'the_121', 'CARD_122', 'is_123', 'critical_124', 'for_125', 'killing_126', 'activity_127', 'of_128', 'DEFCAP_129', '._130', 'In_131', '6_132']\n",
            "BAD:  13336\n",
            "rpws:      index          token label\n",
            "0   13469       patients     O\n",
            "1   13470              (     O\n",
            "2   13471              9     O\n",
            "3   13472        kidneys     O\n",
            "4   13473              )     O\n",
            "5   13474     irrigation     O\n",
            "6   13475           with     O\n",
            "7   13476       Solution     O\n",
            "8   13477              G     O\n",
            "9   13478             in     O\n",
            "10  13479            the     O\n",
            "11  13480          renal     O\n",
            "12  13481         pelvis     O\n",
            "13  13482            was     O\n",
            "14  13483      performed     O\n",
            "15  13484            for     O\n",
            "16  13485            the     O\n",
            "17  13486    dissolution     O\n",
            "18  13487             of     O\n",
            "19  13488          their     O\n",
            "20  13489     infectious     O\n",
            "21  13490         stones     O\n",
            "22  13491              .     O\n",
            "23  13492            The     O\n",
            "24  13493            use     O\n",
            "25  13494             of     O\n",
            "26  13495          sigma     B\n",
            "27  13496             54     I\n",
            "28  13497      promoters     I\n",
            "29  13498              ,     O\n",
            "30  13499          known     O\n",
            "31  13500             to     O\n",
            "32  13501        require     O\n",
            "33  13502        cognate     O\n",
            "34  13503        binding     O\n",
            "35  13504       proteins     O\n",
            "36  13505              ,     O\n",
            "37  13506          could     O\n",
            "38  13507          allow     O\n",
            "39  13508            the     O\n",
            "40  13509           fine     O\n",
            "41  13510              -     O\n",
            "42  13511         tuning     O\n",
            "43  13512           that     O\n",
            "44  13513       provides     O\n",
            "45  13514            the     O\n",
            "46  13515       temporal     O\n",
            "47  13516       ordering     O\n",
            "48  13517             of     O\n",
            "49  13518      flagellar     O\n",
            "50  13519           gene     O\n",
            "51  13520  transcription     O\n",
            "52  13521              .     O\n",
            "53  13522            The     O\n",
            "54  13523    association     O\n",
            "In 6 patients ( 9 kidneys ) irrigation with Solution G in the renal pelvis was performed for the dissolution of their infectious stones . The use of sigma 54 promoters , known to require cognate binding proteins , could allow the fine - tuning that provides the temporal ordering of flagellar gene transcription .\n",
            "['In', '6', 'patients', '(', '9', 'kidneys', ')', 'irrigation', 'with', 'Solution', 'G', 'in', 'the', 'renal', 'pelvis', 'was', 'performed', 'for', 'the', 'dissolution', 'of', 'their', 'infectious', 'stones', '.', 'The', 'use', 'of', 'sigma', '54', 'promoters', ',', 'known', 'to', 'require', 'cognate', 'binding', 'proteins', ',', 'could', 'allow', 'the', 'fine', '-', 'tuning', 'that', 'provides', 'the', 'temporal', 'ordering', 'of', 'flagellar', 'gene', 'transcription', '.']\n",
            "55 55\n",
            "['patients_0', '(_1', '9_2', 'kidneys_3', ')_4', 'irrigation_5', 'with_6', 'Solution_7', 'G_8', 'in_9', 'the_10', 'renal_11', 'pelvis_12', 'was_13', 'performed_14', 'for_15', 'the_16', 'dissolution_17', 'of_18', 'their_19', 'infectious_20', 'stones_21', '._22', 'The_23', 'use_24', 'of_25', 'sigma_26', '54_27', 'promoters_28', ',_29', 'known_30', 'to_31', 'require_32', 'cognate_33', 'binding_34', 'proteins_35', ',_36', 'could_37', 'allow_38', 'the_39', 'fine_40', '-_41', 'tuning_42', 'that_43', 'provides_44', 'the_45', 'temporal_46', 'ordering_47', 'of_48', 'flagellar_49', 'gene_50', 'transcription_51', '._52', 'The_53', 'association_54']\n",
            "BAD:  13469\n",
            "rpws:      index       token label\n",
            "0   13524          of     O\n",
            "1   13525           I     B\n",
            "2   13526           -     I\n",
            "3   13527          92     I\n",
            "4   13528        with     O\n",
            "..    ...         ...   ...\n",
            "86  13610  motionless     O\n",
            "87  13611     staring     O\n",
            "88  13612           .     O\n",
            "89  13613         The     O\n",
            "90  13614        lack     O\n",
            "\n",
            "[91 rows x 3 columns]\n",
            "The association of I - 92 with p92 , p84 , p75 , p73 , p69 , and p57 was completely reversible after treatment with the detergent deoxycholate ( DOC ) . To our knowledge no transcription factors have previously been identified that exhibit androgen - dependent expression in the epididymis . Focal or regional 8 - to 20 - Hz low - voltage epileptiform paroxysms in either hippocampus ( 10 patients ) , amygdala ( 1 patient ) , or both ( 1 patient ) preceded initial motionless staring .\n",
            "['The', 'association', 'of', 'I', '-', '92', 'with', 'p92', ',', 'p84', ',', 'p75', ',', 'p73', ',', 'p69', ',', 'and', 'p57', 'was', 'completely', 'reversible', 'after', 'treatment', 'with', 'the', 'detergent', 'deoxycholate', '(', 'DOC', ')', '.', 'To', 'our', 'knowledge', 'no', 'transcription', 'factors', 'have', 'previously', 'been', 'identified', 'that', 'exhibit', 'androgen', '-', 'dependent', 'expression', 'in', 'the', 'epididymis', '.', 'Focal', 'or', 'regional', '8', '-', 'to', '20', '-', 'Hz', 'low', '-', 'voltage', 'epileptiform', 'paroxysms', 'in', 'either', 'hippocampus', '(', '10', 'patients', ')', ',', 'amygdala', '(', '1', 'patient', ')', ',', 'or', 'both', '(', '1', 'patient', ')', 'preceded', 'initial', 'motionless', 'staring', '.']\n",
            "91 91\n",
            "['of_0', 'I_1', '-_2', '92_3', 'with_4', 'p92_5', ',_6', 'p84_7', ',_8', 'p75_9', ',_10', 'p73_11', ',_12', 'p69_13', ',_14', 'and_15', 'p57_16', 'was_17', 'completely_18', 'reversible_19', 'after_20', 'treatment_21', 'with_22', 'the_23', 'detergent_24', 'deoxycholate_25', '(_26', 'DOC_27', ')_28', '._29', 'To_30', 'our_31', 'knowledge_32', 'no_33', 'transcription_34', 'factors_35', 'have_36', 'previously_37', 'been_38', 'identified_39', 'that_40', 'exhibit_41', 'androgen_42', '-_43', 'dependent_44', 'expression_45', 'in_46', 'the_47', 'epididymis_48', '._49', 'Focal_50', 'or_51', 'regional_52', '8_53', '-_54', 'to_55', '20_56', '-_57', 'Hz_58', 'low_59', '-_60', 'voltage_61', 'epileptiform_62', 'paroxysms_63', 'in_64', 'either_65', 'hippocampus_66', '(_67', '10_68', 'patients_69', ')_70', ',_71', 'amygdala_72', '(_73', '1_74', 'patient_75', ')_76', ',_77', 'or_78', 'both_79', '(_80', '1_81', 'patient_82', ')_83', 'preceded_84', 'initial_85', 'motionless_86', 'staring_87', '._88', 'The_89', 'lack_90']\n",
            "BAD:  13524\n",
            "rpws:      index            token label\n",
            "0   13615               of     O\n",
            "1   13616     heterozygous     O\n",
            "2   13617        positions     O\n",
            "3   13618      essentially     O\n",
            "4   13619      facilitates     O\n",
            "..    ...              ...   ...\n",
            "61  13676        erythroid     O\n",
            "62  13677  differentiation     O\n",
            "63  13678                .     O\n",
            "64  13679              The     O\n",
            "65  13680    effectiveness     O\n",
            "\n",
            "[66 rows x 3 columns]\n",
            "The lack of heterozygous positions essentially facilitates on the one hand the data analysis and on the other hand the detection of new alleles . Epithelial cell - specific activation is achieved by the cooperative interaction of apparently ubiquitous transcriptional factors . Human parvovirus B19 gene expression from the viral p6 promoter ( B19p6 ) is restricted to primary human hematopoietic cells undergoing erythroid differentiation .\n",
            "['The', 'lack', 'of', 'heterozygous', 'positions', 'essentially', 'facilitates', 'on', 'the', 'one', 'hand', 'the', 'data', 'analysis', 'and', 'on', 'the', 'other', 'hand', 'the', 'detection', 'of', 'new', 'alleles', '.', 'Epithelial', 'cell', '-', 'specific', 'activation', 'is', 'achieved', 'by', 'the', 'cooperative', 'interaction', 'of', 'apparently', 'ubiquitous', 'transcriptional', 'factors', '.', 'Human', 'parvovirus', 'B19', 'gene', 'expression', 'from', 'the', 'viral', 'p6', 'promoter', '(', 'B19p6', ')', 'is', 'restricted', 'to', 'primary', 'human', 'hematopoietic', 'cells', 'undergoing', 'erythroid', 'differentiation', '.']\n",
            "66 66\n",
            "['of_0', 'heterozygous_1', 'positions_2', 'essentially_3', 'facilitates_4', 'on_5', 'the_6', 'one_7', 'hand_8', 'the_9', 'data_10', 'analysis_11', 'and_12', 'on_13', 'the_14', 'other_15', 'hand_16', 'the_17', 'detection_18', 'of_19', 'new_20', 'alleles_21', '._22', 'Epithelial_23', 'cell_24', '-_25', 'specific_26', 'activation_27', 'is_28', 'achieved_29', 'by_30', 'the_31', 'cooperative_32', 'interaction_33', 'of_34', 'apparently_35', 'ubiquitous_36', 'transcriptional_37', 'factors_38', '._39', 'Human_40', 'parvovirus_41', 'B19_42', 'gene_43', 'expression_44', 'from_45', 'the_46', 'viral_47', 'p6_48', 'promoter_49', '(_50', 'B19p6_51', ')_52', 'is_53', 'restricted_54', 'to_55', 'primary_56', 'human_57', 'hematopoietic_58', 'cells_59', 'undergoing_60', 'erythroid_61', 'differentiation_62', '._63', 'The_64', 'effectiveness_65']\n",
            "BAD:  13615\n",
            "rpws:      index            token label\n",
            "0   13681              and     O\n",
            "1   13682     pathogenetic     O\n",
            "2   13683       expediency     O\n",
            "3   13684               of     O\n",
            "4   13685       correcting     O\n",
            "5   13686        disorders     O\n",
            "6   13687               of     O\n",
            "7   13688            lipid     O\n",
            "8   13689       metabolism     O\n",
            "9   13690               in     O\n",
            "10  13691  atherosclerosis     O\n",
            "11  13692             with     O\n",
            "12  13693   enterosorbents     O\n",
            "13  13694               is     O\n",
            "14  13695    substantiated     O\n",
            "15  13696                .     O\n",
            "16  13697               To     O\n",
            "17  13698          prevent     O\n",
            "18  13699          section     O\n",
            "19  13700         wrinkles     O\n",
            "20  13701          usually     O\n",
            "21  13702      encountered     O\n",
            "22  13703             with     O\n",
            "23  13704              the     O\n",
            "24  13705              use     O\n",
            "25  13706               of     O\n",
            "26  13707           coated     O\n",
            "27  13708           single     O\n",
            "28  13709                -     O\n",
            "29  13710             hole     O\n",
            "30  13711            grids     O\n",
            "31  13712                ,     O\n",
            "32  13713                a     O\n",
            "33  13714           simple     O\n",
            "34  13715           method     O\n",
            "35  13716              was     O\n",
            "36  13717        developed     O\n",
            "37  13718                .     O\n",
            "38  13719              The     O\n",
            "39  13720      Cryotherapy     O\n",
            "40  13721              for     O\n",
            "41  13722      Retinopathy     O\n",
            "42  13723               of     O\n",
            "43  13724      Prematurity     O\n",
            "44  13725            Study     O\n",
            "45  13726              has     O\n",
            "46  13727            shown     O\n",
            "47  13728             that     O\n",
            "48  13729             this     O\n",
            "49  13730           number     O\n",
            "50  13731              can     O\n",
            "51  13732               be     O\n",
            "52  13733           halved     O\n",
            "53  13734             with     O\n",
            "54  13735        treatment     O\n",
            "55  13736                .     O\n",
            "56  13737       Ultrasound     O\n",
            "57  13738                -     O\n",
            "The effectiveness and pathogenetic expediency of correcting disorders of lipid metabolism in atherosclerosis with enterosorbents is substantiated . To prevent section wrinkles usually encountered with the use of coated single - hole grids , a simple method was developed . The Cryotherapy for Retinopathy of Prematurity Study has shown that this number can be halved with treatment .\n",
            "['The', 'effectiveness', 'and', 'pathogenetic', 'expediency', 'of', 'correcting', 'disorders', 'of', 'lipid', 'metabolism', 'in', 'atherosclerosis', 'with', 'enterosorbents', 'is', 'substantiated', '.', 'To', 'prevent', 'section', 'wrinkles', 'usually', 'encountered', 'with', 'the', 'use', 'of', 'coated', 'single', '-', 'hole', 'grids', ',', 'a', 'simple', 'method', 'was', 'developed', '.', 'The', 'Cryotherapy', 'for', 'Retinopathy', 'of', 'Prematurity', 'Study', 'has', 'shown', 'that', 'this', 'number', 'can', 'be', 'halved', 'with', 'treatment', '.']\n",
            "58 58\n",
            "['and_0', 'pathogenetic_1', 'expediency_2', 'of_3', 'correcting_4', 'disorders_5', 'of_6', 'lipid_7', 'metabolism_8', 'in_9', 'atherosclerosis_10', 'with_11', 'enterosorbents_12', 'is_13', 'substantiated_14', '._15', 'To_16', 'prevent_17', 'section_18', 'wrinkles_19', 'usually_20', 'encountered_21', 'with_22', 'the_23', 'use_24', 'of_25', 'coated_26', 'single_27', '-_28', 'hole_29', 'grids_30', ',_31', 'a_32', 'simple_33', 'method_34', 'was_35', 'developed_36', '._37', 'The_38', 'Cryotherapy_39', 'for_40', 'Retinopathy_41', 'of_42', 'Prematurity_43', 'Study_44', 'has_45', 'shown_46', 'that_47', 'this_48', 'number_49', 'can_50', 'be_51', 'halved_52', 'with_53', 'treatment_54', '._55', 'Ultrasound_56', '-_57']\n",
            "BAD:  13681\n",
            "rpws:      index             token label\n",
            "0   13739            guided     O\n",
            "1   13740              fine     O\n",
            "2   13741                 -     O\n",
            "3   13742            needle     O\n",
            "4   13743        aspiration     O\n",
            "5   13744            biopsy     O\n",
            "6   13745                 (     O\n",
            "7   13746                US     O\n",
            "8   13747                 -     O\n",
            "9   13748              FNAB     O\n",
            "10  13749                 )     O\n",
            "11  13750          revealed     O\n",
            "12  13751            normal     O\n",
            "13  13752                 -     O\n",
            "14  13753           looking     O\n",
            "15  13754          squamous     O\n",
            "16  13755             cells     O\n",
            "17  13756                 .     O\n",
            "18  13757          Bleeding     O\n",
            "19  13758               was     O\n",
            "20  13759      successfully     O\n",
            "21  13760         prevented     O\n",
            "22  13761                in     O\n",
            "23  13762                28     O\n",
            "24  13763          patients     O\n",
            "25  13764              with     O\n",
            "26  13765               the     O\n",
            "27  13766               use     O\n",
            "28  13767                of     O\n",
            "29  13768                 a     O\n",
            "30  13769          combined     O\n",
            "31  13770         treatment     O\n",
            "32  13771     incorporating     O\n",
            "33  13772                IV     O\n",
            "34  13773      desmopressin     O\n",
            "35  13774                 ,     O\n",
            "36  13775                an     O\n",
            "37  13776  antifibrinolytic     O\n",
            "38  13777             agent     O\n",
            "39  13778                 (     O\n",
            "40  13779        tranexamic     O\n",
            "41  13780              acid     O\n",
            "42  13781                 )     O\n",
            "43  13782                 ,     O\n",
            "44  13783               and     O\n",
            "45  13784             local     O\n",
            "46  13785           methods     O\n",
            "47  13786                 (     O\n",
            "48  13787          surgical     O\n",
            "49  13788              glue     O\n",
            "50  13789               and     O\n",
            "51  13790       compression     O\n",
            "52  13791        techniques     O\n",
            "53  13792                 )     O\n",
            "54  13793                 .     O\n",
            "55  13794            Hetero     O\n",
            "56  13795                 -     O\n",
            "Ultrasound - guided fine - needle aspiration biopsy ( US - FNAB ) revealed normal - looking squamous cells . Bleeding was successfully prevented in 28 patients with the use of a combined treatment incorporating IV desmopressin , an antifibrinolytic agent ( tranexamic acid ) , and local methods ( surgical glue and compression techniques ) .\n",
            "['Ultrasound', '-', 'guided', 'fine', '-', 'needle', 'aspiration', 'biopsy', '(', 'US', '-', 'FNAB', ')', 'revealed', 'normal', '-', 'looking', 'squamous', 'cells', '.', 'Bleeding', 'was', 'successfully', 'prevented', 'in', '28', 'patients', 'with', 'the', 'use', 'of', 'a', 'combined', 'treatment', 'incorporating', 'IV', 'desmopressin', ',', 'an', 'antifibrinolytic', 'agent', '(', 'tranexamic', 'acid', ')', ',', 'and', 'local', 'methods', '(', 'surgical', 'glue', 'and', 'compression', 'techniques', ')', '.']\n",
            "57 57\n",
            "['guided_0', 'fine_1', '-_2', 'needle_3', 'aspiration_4', 'biopsy_5', '(_6', 'US_7', '-_8', 'FNAB_9', ')_10', 'revealed_11', 'normal_12', '-_13', 'looking_14', 'squamous_15', 'cells_16', '._17', 'Bleeding_18', 'was_19', 'successfully_20', 'prevented_21', 'in_22', '28_23', 'patients_24', 'with_25', 'the_26', 'use_27', 'of_28', 'a_29', 'combined_30', 'treatment_31', 'incorporating_32', 'IV_33', 'desmopressin_34', ',_35', 'an_36', 'antifibrinolytic_37', 'agent_38', '(_39', 'tranexamic_40', 'acid_41', ')_42', ',_43', 'and_44', 'local_45', 'methods_46', '(_47', 'surgical_48', 'glue_49', 'and_50', 'compression_51', 'techniques_52', ')_53', '._54', 'Hetero_55', '-_56']\n",
            "BAD:  13739\n",
            "rpws:      index            token label\n",
            "0   13796  oligomerization     O\n",
            "1   13797               of     O\n",
            "2   13798            Smad4     B\n",
            "3   13799             with     O\n",
            "4   13800              the     O\n",
            "..    ...              ...   ...\n",
            "58  13854         cerebral     O\n",
            "59  13855           cortex     O\n",
            "60  13856                .     O\n",
            "61  13857         Analysis     O\n",
            "62  13858               of     O\n",
            "\n",
            "[63 rows x 3 columns]\n",
            "Hetero - oligomerization of Smad4 with the pathway - restricted SMAD proteins is essential for Smad4 - mediated transcription . This fragment contains two complete endo - beta - 1 , 4 - glucanase - encoding genes , designated celCCC and celCCG . One of them is its limited sensitivity to weak interactions , which are common in the mammalian cerebral cortex .\n",
            "['Hetero', '-', 'oligomerization', 'of', 'Smad4', 'with', 'the', 'pathway', '-', 'restricted', 'SMAD', 'proteins', 'is', 'essential', 'for', 'Smad4', '-', 'mediated', 'transcription', '.', 'This', 'fragment', 'contains', 'two', 'complete', 'endo', '-', 'beta', '-', '1', ',', '4', '-', 'glucanase', '-', 'encoding', 'genes', ',', 'designated', 'celCCC', 'and', 'celCCG', '.', 'One', 'of', 'them', 'is', 'its', 'limited', 'sensitivity', 'to', 'weak', 'interactions', ',', 'which', 'are', 'common', 'in', 'the', 'mammalian', 'cerebral', 'cortex', '.']\n",
            "63 63\n",
            "['oligomerization_0', 'of_1', 'Smad4_2', 'with_3', 'the_4', 'pathway_5', '-_6', 'restricted_7', 'SMAD_8', 'proteins_9', 'is_10', 'essential_11', 'for_12', 'Smad4_13', '-_14', 'mediated_15', 'transcription_16', '._17', 'This_18', 'fragment_19', 'contains_20', 'two_21', 'complete_22', 'endo_23', '-_24', 'beta_25', '-_26', '1_27', ',_28', '4_29', '-_30', 'glucanase_31', '-_32', 'encoding_33', 'genes_34', ',_35', 'designated_36', 'celCCC_37', 'and_38', 'celCCG_39', '._40', 'One_41', 'of_42', 'them_43', 'is_44', 'its_45', 'limited_46', 'sensitivity_47', 'to_48', 'weak_49', 'interactions_50', ',_51', 'which_52', 'are_53', 'common_54', 'in_55', 'the_56', 'mammalian_57', 'cerebral_58', 'cortex_59', '._60', 'Analysis_61', 'of_62']\n",
            "BAD:  13796\n",
            "rpws:      index        token label\n",
            "0   13859          the     O\n",
            "1   13860            5     O\n",
            "2   13861            '     O\n",
            "3   13862          and     O\n",
            "4   13863            3     O\n",
            "..    ...          ...   ...\n",
            "64  13923         type     O\n",
            "65  13924            R     O\n",
            "66  13925            .     O\n",
            "67  13926  sphaeroides     O\n",
            "68  13927            2     O\n",
            "\n",
            "[69 rows x 3 columns]\n",
            "Analysis of the 5 ' and 3 ' termini of the transcripts produced in vivo from the puf operon of R . sphaeroides PUF delta 348 - 420 ( three transcripts ; 0 . 59 , 0 . 64 , and 2 . 63 kilobases ) lacking the puf - intercistronic terminator structure were identical to those of the corresponding puf transcripts derived from wild - type R .\n",
            "['Analysis', 'of', 'the', '5', \"'\", 'and', '3', \"'\", 'termini', 'of', 'the', 'transcripts', 'produced', 'in', 'vivo', 'from', 'the', 'puf', 'operon', 'of', 'R', '.', 'sphaeroides', 'PUF', 'delta', '348', '-', '420', '(', 'three', 'transcripts', ';', '0', '.', '59', ',', '0', '.', '64', ',', 'and', '2', '.', '63', 'kilobases', ')', 'lacking', 'the', 'puf', '-', 'intercistronic', 'terminator', 'structure', 'were', 'identical', 'to', 'those', 'of', 'the', 'corresponding', 'puf', 'transcripts', 'derived', 'from', 'wild', '-', 'type', 'R', '.']\n",
            "69 69\n",
            "['the_0', '5_1', \"'_2\", 'and_3', '3_4', \"'_5\", 'termini_6', 'of_7', 'the_8', 'transcripts_9', 'produced_10', 'in_11', 'vivo_12', 'from_13', 'the_14', 'puf_15', 'operon_16', 'of_17', 'R_18', '._19', 'sphaeroides_20', 'PUF_21', 'delta_22', '348_23', '-_24', '420_25', '(_26', 'three_27', 'transcripts_28', ';_29', '0_30', '._31', '59_32', ',_33', '0_34', '._35', '64_36', ',_37', 'and_38', '2_39', '._40', '63_41', 'kilobases_42', ')_43', 'lacking_44', 'the_45', 'puf_46', '-_47', 'intercistronic_48', 'terminator_49', 'structure_50', 'were_51', 'identical_52', 'to_53', 'those_54', 'of_55', 'the_56', 'corresponding_57', 'puf_58', 'transcripts_59', 'derived_60', 'from_61', 'wild_62', '-_63', 'type_64', 'R_65', '._66', 'sphaeroides_67', '2_68']\n",
            "BAD:  13859\n",
            "rpws:      index       token label\n",
            "0   13928           .     O\n",
            "1   13929           4     O\n",
            "2   13930           .     O\n",
            "3   13931           1     O\n",
            "4   13932           (     O\n",
            "..    ...         ...   ...\n",
            "84  14012         LN2     O\n",
            "85  14013           )     O\n",
            "86  14014           .     O\n",
            "87  14015         The     O\n",
            "88  14016  cerebellar     O\n",
            "\n",
            "[89 rows x 3 columns]\n",
            "sphaeroides 2 . 4 . 1 ( four transcripts ; 0 . 50 , 0 . 66 , 0 . 71 , and 2 . 7 kilobases ) showing that the transcripts begin and end at the same sites . The optimum cooling rate from RT to 5 degrees C resulting in maximum survival of human spermatozoa was found to be 0 . 5 to 1 degree per minute when cooled from RT to 5 degrees C and subsequently frozen - thawed in liquid nitrogen ( LN2 ) .\n",
            "['sphaeroides', '2', '.', '4', '.', '1', '(', 'four', 'transcripts', ';', '0', '.', '50', ',', '0', '.', '66', ',', '0', '.', '71', ',', 'and', '2', '.', '7', 'kilobases', ')', 'showing', 'that', 'the', 'transcripts', 'begin', 'and', 'end', 'at', 'the', 'same', 'sites', '.', 'The', 'optimum', 'cooling', 'rate', 'from', 'RT', 'to', '5', 'degrees', 'C', 'resulting', 'in', 'maximum', 'survival', 'of', 'human', 'spermatozoa', 'was', 'found', 'to', 'be', '0', '.', '5', 'to', '1', 'degree', 'per', 'minute', 'when', 'cooled', 'from', 'RT', 'to', '5', 'degrees', 'C', 'and', 'subsequently', 'frozen', '-', 'thawed', 'in', 'liquid', 'nitrogen', '(', 'LN2', ')', '.']\n",
            "89 89\n",
            "['._0', '4_1', '._2', '1_3', '(_4', 'four_5', 'transcripts_6', ';_7', '0_8', '._9', '50_10', ',_11', '0_12', '._13', '66_14', ',_15', '0_16', '._17', '71_18', ',_19', 'and_20', '2_21', '._22', '7_23', 'kilobases_24', ')_25', 'showing_26', 'that_27', 'the_28', 'transcripts_29', 'begin_30', 'and_31', 'end_32', 'at_33', 'the_34', 'same_35', 'sites_36', '._37', 'The_38', 'optimum_39', 'cooling_40', 'rate_41', 'from_42', 'RT_43', 'to_44', '5_45', 'degrees_46', 'C_47', 'resulting_48', 'in_49', 'maximum_50', 'survival_51', 'of_52', 'human_53', 'spermatozoa_54', 'was_55', 'found_56', 'to_57', 'be_58', '0_59', '._60', '5_61', 'to_62', '1_63', 'degree_64', 'per_65', 'minute_66', 'when_67', 'cooled_68', 'from_69', 'RT_70', 'to_71', '5_72', 'degrees_73', 'C_74', 'and_75', 'subsequently_76', 'frozen_77', '-_78', 'thawed_79', 'in_80', 'liquid_81', 'nitrogen_82', '(_83', 'LN2_84', ')_85', '._86', 'The_87', 'cerebellar_88']\n",
            "BAD:  13928\n",
            "rpws:      index            token label\n",
            "0   14017         cortical     O\n",
            "1   14018          vessels     O\n",
            "2   14019                :     O\n",
            "3   14020             they     O\n",
            "4   14021              are     O\n",
            "..    ...              ...   ...\n",
            "67  14084             wild     O\n",
            "68  14085             type     O\n",
            "69  14086                .     O\n",
            "70  14087         Neonatal     O\n",
            "71  14088  hyperthyroidism     O\n",
            "\n",
            "[72 rows x 3 columns]\n",
            "The cerebellar cortical vessels : they are compared to the cerebral vessels . Transesophageal echocardiography risk factors for stroke in nonvalvular atrial fibrillation . 22 - Veneto Region ) Preventive health counseling tailored to the needs of this group may be most beneficial . Strains lacking a functional RPL16A gene grow as rapidly as wild type , whereas those containing a null allele of RPL16B grow more slowly than wild type .\n",
            "['The', 'cerebellar', 'cortical', 'vessels', ':', 'they', 'are', 'compared', 'to', 'the', 'cerebral', 'vessels', '.', 'Transesophageal', 'echocardiography', 'risk', 'factors', 'for', 'stroke', 'in', 'nonvalvular', 'atrial', 'fibrillation', '.', '22', '-', 'Veneto', 'Region', ')', 'Preventive', 'health', 'counseling', 'tailored', 'to', 'the', 'needs', 'of', 'this', 'group', 'may', 'be', 'most', 'beneficial', '.', 'Strains', 'lacking', 'a', 'functional', 'RPL16A', 'gene', 'grow', 'as', 'rapidly', 'as', 'wild', 'type', ',', 'whereas', 'those', 'containing', 'a', 'null', 'allele', 'of', 'RPL16B', 'grow', 'more', 'slowly', 'than', 'wild', 'type', '.']\n",
            "72 72\n",
            "['cortical_0', 'vessels_1', ':_2', 'they_3', 'are_4', 'compared_5', 'to_6', 'the_7', 'cerebral_8', 'vessels_9', '._10', 'Transesophageal_11', 'echocardiography_12', 'risk_13', 'factors_14', 'for_15', 'stroke_16', 'in_17', 'nonvalvular_18', 'atrial_19', 'fibrillation_20', '._21', '22_22', '-_23', 'Veneto_24', 'Region_25', ')_26', 'Preventive_27', 'health_28', 'counseling_29', 'tailored_30', 'to_31', 'the_32', 'needs_33', 'of_34', 'this_35', 'group_36', 'may_37', 'be_38', 'most_39', 'beneficial_40', '._41', 'Strains_42', 'lacking_43', 'a_44', 'functional_45', 'RPL16A_46', 'gene_47', 'grow_48', 'as_49', 'rapidly_50', 'as_51', 'wild_52', 'type_53', ',_54', 'whereas_55', 'those_56', 'containing_57', 'a_58', 'nan_59', 'allele_60', 'of_61', 'RPL16B_62', 'grow_63', 'more_64', 'slowly_65', 'than_66', 'wild_67', 'type_68', '._69', 'Neonatal_70', 'hyperthyroidism_71']\n",
            "BAD:  14017\n",
            "rpws:      index          token label\n",
            "0   14089          After     O\n",
            "1   14090       Northern     O\n",
            "2   14091           blot     O\n",
            "3   14092  hybridization     O\n",
            "4   14093              ,     O\n",
            "..    ...            ...   ...\n",
            "68  14157              ;     O\n",
            "69  14158            Shc     B\n",
            "70  14159              .     O\n",
            "71  14160           GRB2     B\n",
            "72  14161              .     O\n",
            "\n",
            "[73 rows x 3 columns]\n",
            "Neonatal hyperthyroidism After Northern blot hybridization , two Cx31 transcripts of 2 . 2 and 1 . 8 kb were detected in total RNA of the human keratinocyte cell line HaCaT and two transcripts of 2 . 2 and 1 . 9 kb in total RNA of E6 / E7 transfected human keratinocytes ( HEK cells ) . In summary , FGFR3 signaling pathway utilizes two GRB2 - containing complexes ; Shc .\n",
            "['Neonatal', 'hyperthyroidism', 'After', 'Northern', 'blot', 'hybridization', ',', 'two', 'Cx31', 'transcripts', 'of', '2', '.', '2', 'and', '1', '.', '8', 'kb', 'were', 'detected', 'in', 'total', 'RNA', 'of', 'the', 'human', 'keratinocyte', 'cell', 'line', 'HaCaT', 'and', 'two', 'transcripts', 'of', '2', '.', '2', 'and', '1', '.', '9', 'kb', 'in', 'total', 'RNA', 'of', 'E6', '/', 'E7', 'transfected', 'human', 'keratinocytes', '(', 'HEK', 'cells', ')', '.', 'In', 'summary', ',', 'FGFR3', 'signaling', 'pathway', 'utilizes', 'two', 'GRB2', '-', 'containing', 'complexes', ';', 'Shc', '.']\n",
            "73 73\n",
            "['After_0', 'Northern_1', 'blot_2', 'hybridization_3', ',_4', 'two_5', 'Cx31_6', 'transcripts_7', 'of_8', '2_9', '._10', '2_11', 'and_12', '1_13', '._14', '8_15', 'kb_16', 'were_17', 'detected_18', 'in_19', 'total_20', 'RNA_21', 'of_22', 'the_23', 'human_24', 'keratinocyte_25', 'cell_26', 'line_27', 'HaCaT_28', 'and_29', 'two_30', 'transcripts_31', 'of_32', '2_33', '._34', '2_35', 'and_36', '1_37', '._38', '9_39', 'kb_40', 'in_41', 'total_42', 'RNA_43', 'of_44', 'E6_45', '/_46', 'E7_47', 'transfected_48', 'human_49', 'keratinocytes_50', '(_51', 'HEK_52', 'cells_53', ')_54', '._55', 'In_56', 'summary_57', ',_58', 'FGFR3_59', 'signaling_60', 'pathway_61', 'utilizes_62', 'two_63', 'GRB2_64', '-_65', 'containing_66', 'complexes_67', ';_68', 'Shc_69', '._70', 'GRB2_71', '._72']\n",
            "BAD:  14089\n",
            "rpws:      index     token label\n",
            "0   14162       Sos     B\n",
            "1   14163       and     O\n",
            "2   14164       80K     B\n",
            "3   14165         -     I\n",
            "4   14166         H     I\n",
            "..    ...       ...   ...\n",
            "56  14218  promoter     O\n",
            "57  14219    region     O\n",
            "58  14220         .     O\n",
            "59  14221     Their     O\n",
            "60  14222      main     O\n",
            "\n",
            "[61 rows x 3 columns]\n",
            "GRB2 . Sos and 80K - H . pp66 . GRB2 . Sos ; these two complexes may alternatively link FGFG3 to mitogen - activated protein kinase . In an effort to identify factors involved in the expression of this important erythroid - specific regulatory protein , we have isolated the mouse EKLF gene and systematically analyzed the promoter region .\n",
            "['GRB2', '.', 'Sos', 'and', '80K', '-', 'H', '.', 'pp66', '.', 'GRB2', '.', 'Sos', ';', 'these', 'two', 'complexes', 'may', 'alternatively', 'link', 'FGFG3', 'to', 'mitogen', '-', 'activated', 'protein', 'kinase', '.', 'In', 'an', 'effort', 'to', 'identify', 'factors', 'involved', 'in', 'the', 'expression', 'of', 'this', 'important', 'erythroid', '-', 'specific', 'regulatory', 'protein', ',', 'we', 'have', 'isolated', 'the', 'mouse', 'EKLF', 'gene', 'and', 'systematically', 'analyzed', 'the', 'promoter', 'region', '.']\n",
            "61 61\n",
            "['Sos_0', 'and_1', '80K_2', '-_3', 'H_4', '._5', 'pp66_6', '._7', 'GRB2_8', '._9', 'Sos_10', ';_11', 'these_12', 'two_13', 'complexes_14', 'may_15', 'alternatively_16', 'link_17', 'FGFG3_18', 'to_19', 'mitogen_20', '-_21', 'activated_22', 'protein_23', 'kinase_24', '._25', 'In_26', 'an_27', 'effort_28', 'to_29', 'identify_30', 'factors_31', 'involved_32', 'in_33', 'the_34', 'expression_35', 'of_36', 'this_37', 'important_38', 'erythroid_39', '-_40', 'specific_41', 'regulatory_42', 'protein_43', ',_44', 'we_45', 'have_46', 'isolated_47', 'the_48', 'mouse_49', 'EKLF_50', 'gene_51', 'and_52', 'systematically_53', 'analyzed_54', 'the_55', 'promoter_56', 'region_57', '._58', 'Their_59', 'main_60']\n",
            "BAD:  14162\n",
            "rpws:      index            token label\n",
            "0   14223  representatives     O\n",
            "1   14224              are     O\n",
            "2   14225              the     O\n",
            "3   14226         American     O\n",
            "4   14227       diagnostic     O\n",
            "..    ...              ...   ...\n",
            "83  14306              900     O\n",
            "84  14307              kHz     O\n",
            "85  14308                .     O\n",
            "86  14309               In     O\n",
            "87  14310         contrast     O\n",
            "\n",
            "[88 rows x 3 columns]\n",
            "Their main representatives are the American diagnostic and statistical manual of mental disorders , DSM - III , its revised version DSM - III - R and the new international classification of mental disorders iCD - 10 . Any opacity on a chest radiograph has a wide differential diagnosis . At the same acoustic power , the rate of chloride formation with 20 kHz ultrasound was greater when a probe with a larger tip area was used , but significantly less than the rate with 900 kHz .\n",
            "['Their', 'main', 'representatives', 'are', 'the', 'American', 'diagnostic', 'and', 'statistical', 'manual', 'of', 'mental', 'disorders', ',', 'DSM', '-', 'III', ',', 'its', 'revised', 'version', 'DSM', '-', 'III', '-', 'R', 'and', 'the', 'new', 'international', 'classification', 'of', 'mental', 'disorders', 'iCD', '-', '10', '.', 'Any', 'opacity', 'on', 'a', 'chest', 'radiograph', 'has', 'a', 'wide', 'differential', 'diagnosis', '.', 'At', 'the', 'same', 'acoustic', 'power', ',', 'the', 'rate', 'of', 'chloride', 'formation', 'with', '20', 'kHz', 'ultrasound', 'was', 'greater', 'when', 'a', 'probe', 'with', 'a', 'larger', 'tip', 'area', 'was', 'used', ',', 'but', 'significantly', 'less', 'than', 'the', 'rate', 'with', '900', 'kHz', '.']\n",
            "88 88\n",
            "['representatives_0', 'are_1', 'the_2', 'American_3', 'diagnostic_4', 'and_5', 'statistical_6', 'manual_7', 'of_8', 'mental_9', 'disorders_10', ',_11', 'DSM_12', '-_13', 'III_14', ',_15', 'its_16', 'revised_17', 'version_18', 'DSM_19', '-_20', 'III_21', '-_22', 'R_23', 'and_24', 'the_25', 'new_26', 'international_27', 'classification_28', 'of_29', 'mental_30', 'disorders_31', 'iCD_32', '-_33', '10_34', '._35', 'Any_36', 'opacity_37', 'on_38', 'a_39', 'chest_40', 'radiograph_41', 'has_42', 'a_43', 'wide_44', 'differential_45', 'diagnosis_46', '._47', 'At_48', 'the_49', 'same_50', 'acoustic_51', 'power_52', ',_53', 'the_54', 'rate_55', 'of_56', 'chloride_57', 'formation_58', 'with_59', '20_60', 'kHz_61', 'ultrasound_62', 'was_63', 'greater_64', 'when_65', 'a_66', 'probe_67', 'with_68', 'a_69', 'larger_70', 'tip_71', 'area_72', 'was_73', 'used_74', ',_75', 'but_76', 'significantly_77', 'less_78', 'than_79', 'the_80', 'rate_81', 'with_82', '900_83', 'kHz_84', '._85', 'In_86', 'contrast_87']\n",
            "BAD:  14223\n",
            "rpws:      index    token label\n",
            "0   14311       to     O\n",
            "1   14312  Torpedo     O\n",
            "2   14313        ,     O\n",
            "3   14314    where     O\n",
            "4   14315     only     O\n",
            "..    ...      ...   ...\n",
            "67  14378    tract     O\n",
            "68  14379  neurons     O\n",
            "69  14380        .     O\n",
            "70  14381      The     O\n",
            "71  14382     GvpE     B\n",
            "\n",
            "[72 rows x 3 columns]\n",
            "In contrast to Torpedo , where only a single transcript is seen , the mouse expresses several mRNAs encoding different isoforms . Four families of homing endonucleases have been identified , including the LAGLIDADG , the His - Cys box , the GIY - YIG and the H - N - H endonucleases . Convergence of trigeminal input with visceral and phrenic inputs on primate C1 - C2 spinothalamic tract neurons .\n",
            "['In', 'contrast', 'to', 'Torpedo', ',', 'where', 'only', 'a', 'single', 'transcript', 'is', 'seen', ',', 'the', 'mouse', 'expresses', 'several', 'mRNAs', 'encoding', 'different', 'isoforms', '.', 'Four', 'families', 'of', 'homing', 'endonucleases', 'have', 'been', 'identified', ',', 'including', 'the', 'LAGLIDADG', ',', 'the', 'His', '-', 'Cys', 'box', ',', 'the', 'GIY', '-', 'YIG', 'and', 'the', 'H', '-', 'N', '-', 'H', 'endonucleases', '.', 'Convergence', 'of', 'trigeminal', 'input', 'with', 'visceral', 'and', 'phrenic', 'inputs', 'on', 'primate', 'C1', '-', 'C2', 'spinothalamic', 'tract', 'neurons', '.']\n",
            "72 72\n",
            "['to_0', 'Torpedo_1', ',_2', 'where_3', 'only_4', 'a_5', 'single_6', 'transcript_7', 'is_8', 'seen_9', ',_10', 'the_11', 'mouse_12', 'expresses_13', 'several_14', 'mRNAs_15', 'encoding_16', 'different_17', 'isoforms_18', '._19', 'Four_20', 'families_21', 'of_22', 'homing_23', 'endonucleases_24', 'have_25', 'been_26', 'identified_27', ',_28', 'including_29', 'the_30', 'LAGLIDADG_31', ',_32', 'the_33', 'His_34', '-_35', 'Cys_36', 'box_37', ',_38', 'the_39', 'GIY_40', '-_41', 'YIG_42', 'and_43', 'the_44', 'H_45', '-_46', 'N_47', '-_48', 'H_49', 'endonucleases_50', '._51', 'Convergence_52', 'of_53', 'trigeminal_54', 'input_55', 'with_56', 'visceral_57', 'and_58', 'phrenic_59', 'inputs_60', 'on_61', 'primate_62', 'C1_63', '-_64', 'C2_65', 'spinothalamic_66', 'tract_67', 'neurons_68', '._69', 'The_70', 'GvpE_71']\n",
            "BAD:  14311\n",
            "rpws:      index            token label\n",
            "0   14383          protein     I\n",
            "1   14384         involved     O\n",
            "2   14385               in     O\n",
            "3   14386              the     O\n",
            "4   14387       regulation     O\n",
            "5   14388               of     O\n",
            "6   14389              gas     O\n",
            "7   14390         vesicles     O\n",
            "8   14391        synthesis     O\n",
            "9   14392               in     O\n",
            "10  14393       halophilic     O\n",
            "11  14394          archaea     O\n",
            "12  14395              has     O\n",
            "13  14396             been     O\n",
            "14  14397       identified     O\n",
            "15  14398               as     O\n",
            "16  14399              the     O\n",
            "17  14400  transcriptional     O\n",
            "18  14401        activator     O\n",
            "19  14402              for     O\n",
            "20  14403              the     O\n",
            "21  14404         promoter     O\n",
            "22  14405          located     O\n",
            "23  14406         upstream     O\n",
            "24  14407               of     O\n",
            "25  14408              the     O\n",
            "26  14409             gvpA     B\n",
            "27  14410             gene     I\n",
            "28  14411         encoding     O\n",
            "29  14412              the     O\n",
            "30  14413            major     O\n",
            "31  14414              gas     O\n",
            "32  14415          vesicle     O\n",
            "33  14416       structural     O\n",
            "34  14417          protein     O\n",
            "35  14418             GvpA     B\n",
            "36  14419                .     O\n",
            "37  14420      Nosographic     O\n",
            "38  14421         problems     O\n",
            "39  14422               In     O\n",
            "40  14423            early     O\n",
            "41  14424     midgestation     O\n",
            "42  14425          embryos     O\n",
            "43  14426               it     O\n",
            "44  14427               is     O\n",
            "45  14428        expressed     O\n",
            "46  14429               in     O\n",
            "47  14430    telencephalic     O\n",
            "48  14431                ,     O\n",
            "49  14432     diencephalic     O\n",
            "50  14433              and     O\n",
            "51  14434    mesencephalic     O\n",
            "52  14435          regions     O\n",
            "53  14436              but     O\n",
            "54  14437             from     O\n",
            "55  14438              day     O\n",
            "56  14439               11     O\n",
            "57  14440                .     O\n",
            "58  14441               75     O\n",
            "59  14442               of     O\n",
            "The GvpE protein involved in the regulation of gas vesicles synthesis in halophilic archaea has been identified as the transcriptional activator for the promoter located upstream of the gvpA gene encoding the major gas vesicle structural protein GvpA . Nosographic problems In early midgestation embryos it is expressed in telencephalic , diencephalic and mesencephalic regions but from day 11 .\n",
            "['The', 'GvpE', 'protein', 'involved', 'in', 'the', 'regulation', 'of', 'gas', 'vesicles', 'synthesis', 'in', 'halophilic', 'archaea', 'has', 'been', 'identified', 'as', 'the', 'transcriptional', 'activator', 'for', 'the', 'promoter', 'located', 'upstream', 'of', 'the', 'gvpA', 'gene', 'encoding', 'the', 'major', 'gas', 'vesicle', 'structural', 'protein', 'GvpA', '.', 'Nosographic', 'problems', 'In', 'early', 'midgestation', 'embryos', 'it', 'is', 'expressed', 'in', 'telencephalic', ',', 'diencephalic', 'and', 'mesencephalic', 'regions', 'but', 'from', 'day', '11', '.']\n",
            "60 60\n",
            "['protein_0', 'involved_1', 'in_2', 'the_3', 'regulation_4', 'of_5', 'gas_6', 'vesicles_7', 'synthesis_8', 'in_9', 'halophilic_10', 'archaea_11', 'has_12', 'been_13', 'identified_14', 'as_15', 'the_16', 'transcriptional_17', 'activator_18', 'for_19', 'the_20', 'promoter_21', 'located_22', 'upstream_23', 'of_24', 'the_25', 'gvpA_26', 'gene_27', 'encoding_28', 'the_29', 'major_30', 'gas_31', 'vesicle_32', 'structural_33', 'protein_34', 'GvpA_35', '._36', 'Nosographic_37', 'problems_38', 'In_39', 'early_40', 'midgestation_41', 'embryos_42', 'it_43', 'is_44', 'expressed_45', 'in_46', 'telencephalic_47', ',_48', 'diencephalic_49', 'and_50', 'mesencephalic_51', 'regions_52', 'but_53', 'from_54', 'day_55', '11_56', '._57', '75_58', 'of_59']\n",
            "BAD:  14383\n",
            "rpws:      index       token label\n",
            "0   14443   gestation     O\n",
            "1   14444         its     O\n",
            "2   14445  expression     O\n",
            "3   14446  disappears     O\n",
            "4   14447        from     O\n",
            "..    ...         ...   ...\n",
            "65  14508         dog     O\n",
            "66  14509   sequences     O\n",
            "67  14510           .     O\n",
            "68  14511       Using     O\n",
            "69  14512      probes     O\n",
            "\n",
            "[70 rows x 3 columns]\n",
            "75 of gestation its expression disappears from dorsal telencephalon and is confined to diencephalic and mesencephalic regions . The training programs had a duration of 30 days , 5 days / week , and was performed on a motor - driven treadmill . Phylogenetic analysis using a neighbor - joining algorithm showed closest similarity of the horse mu chain - encoding constant region gene to human and dog sequences .\n",
            "['75', 'of', 'gestation', 'its', 'expression', 'disappears', 'from', 'dorsal', 'telencephalon', 'and', 'is', 'confined', 'to', 'diencephalic', 'and', 'mesencephalic', 'regions', '.', 'The', 'training', 'programs', 'had', 'a', 'duration', 'of', '30', 'days', ',', '5', 'days', '/', 'week', ',', 'and', 'was', 'performed', 'on', 'a', 'motor', '-', 'driven', 'treadmill', '.', 'Phylogenetic', 'analysis', 'using', 'a', 'neighbor', '-', 'joining', 'algorithm', 'showed', 'closest', 'similarity', 'of', 'the', 'horse', 'mu', 'chain', '-', 'encoding', 'constant', 'region', 'gene', 'to', 'human', 'and', 'dog', 'sequences', '.']\n",
            "70 70\n",
            "['gestation_0', 'its_1', 'expression_2', 'disappears_3', 'from_4', 'dorsal_5', 'telencephalon_6', 'and_7', 'is_8', 'confined_9', 'to_10', 'diencephalic_11', 'and_12', 'mesencephalic_13', 'regions_14', '._15', 'The_16', 'training_17', 'programs_18', 'had_19', 'a_20', 'duration_21', 'of_22', '30_23', 'days_24', ',_25', '5_26', 'days_27', '/_28', 'week_29', ',_30', 'and_31', 'was_32', 'performed_33', 'on_34', 'a_35', 'motor_36', '-_37', 'driven_38', 'treadmill_39', '._40', 'Phylogenetic_41', 'analysis_42', 'using_43', 'a_44', 'neighbor_45', '-_46', 'joining_47', 'algorithm_48', 'showed_49', 'closest_50', 'similarity_51', 'of_52', 'the_53', 'horse_54', 'mu_55', 'chain_56', '-_57', 'encoding_58', 'constant_59', 'region_60', 'gene_61', 'to_62', 'human_63', 'and_64', 'dog_65', 'sequences_66', '._67', 'Using_68', 'probes_69']\n",
            "BAD:  14443\n",
            "rpws:      index           token label\n",
            "0   14513            from     O\n",
            "1   14514           these     O\n",
            "2   14515         regions     O\n",
            "3   14516               ,     O\n",
            "4   14517  rearrangements     O\n",
            "..    ...             ...   ...\n",
            "67  14580          finger     O\n",
            "68  14581          region     O\n",
            "69  14582               .     O\n",
            "70  14583         Earlier     O\n",
            "71  14584       published     O\n",
            "\n",
            "[72 rows x 3 columns]\n",
            "Using probes from these regions , rearrangements have been identified in each of nine cases of t ( 8 ; 21 ) AML examined . Cloning and nucleotide sequence analysis reveals that BUR6 encodes a homolog of DRAP1 ( also called NC2alpha ) , a mammalian repressor of basal transcription . BRAP2 also shares significant homology with a hypothetical protein from yeast Saccharomyces cerevisiae , especially in the zinc finger region .\n",
            "['Using', 'probes', 'from', 'these', 'regions', ',', 'rearrangements', 'have', 'been', 'identified', 'in', 'each', 'of', 'nine', 'cases', 'of', 't', '(', '8', ';', '21', ')', 'AML', 'examined', '.', 'Cloning', 'and', 'nucleotide', 'sequence', 'analysis', 'reveals', 'that', 'BUR6', 'encodes', 'a', 'homolog', 'of', 'DRAP1', '(', 'also', 'called', 'NC2alpha', ')', ',', 'a', 'mammalian', 'repressor', 'of', 'basal', 'transcription', '.', 'BRAP2', 'also', 'shares', 'significant', 'homology', 'with', 'a', 'hypothetical', 'protein', 'from', 'yeast', 'Saccharomyces', 'cerevisiae', ',', 'especially', 'in', 'the', 'zinc', 'finger', 'region', '.']\n",
            "72 72\n",
            "['from_0', 'these_1', 'regions_2', ',_3', 'rearrangements_4', 'have_5', 'been_6', 'identified_7', 'in_8', 'each_9', 'of_10', 'nine_11', 'cases_12', 'of_13', 't_14', '(_15', '8_16', ';_17', '21_18', ')_19', 'AML_20', 'examined_21', '._22', 'Cloning_23', 'and_24', 'nucleotide_25', 'sequence_26', 'analysis_27', 'reveals_28', 'that_29', 'BUR6_30', 'encodes_31', 'a_32', 'homolog_33', 'of_34', 'DRAP1_35', '(_36', 'also_37', 'called_38', 'NC2alpha_39', ')_40', ',_41', 'a_42', 'mammalian_43', 'repressor_44', 'of_45', 'basal_46', 'transcription_47', '._48', 'BRAP2_49', 'also_50', 'shares_51', 'significant_52', 'homology_53', 'with_54', 'a_55', 'hypothetical_56', 'protein_57', 'from_58', 'yeast_59', 'Saccharomyces_60', 'cerevisiae_61', ',_62', 'especially_63', 'in_64', 'the_65', 'zinc_66', 'finger_67', 'region_68', '._69', 'Earlier_70', 'published_71']\n",
            "BAD:  14513\n",
            "rpws:      index         token label\n",
            "0   14585          data     O\n",
            "1   14586             ,     O\n",
            "2   14587    indicating     O\n",
            "3   14588           Sp1     B\n",
            "4   14589       binding     O\n",
            "5   14590            to     O\n",
            "6   14591           the     O\n",
            "7   14592            R1     B\n",
            "8   14593         alpha     I\n",
            "9   14594             /     I\n",
            "10  14595          beta     I\n",
            "11  14596       regions     I\n",
            "12  14597             ,     O\n",
            "13  14598         could     O\n",
            "14  14599           not     O\n",
            "15  14600            be     O\n",
            "16  14601     confirmed     O\n",
            "17  14602             ,     O\n",
            "18  14603    suggesting     O\n",
            "19  14604          that     O\n",
            "20  14605           the     O\n",
            "21  14606            R1     O\n",
            "22  14607     initiator     O\n",
            "23  14608       element     O\n",
            "24  14609           may     O\n",
            "25  14610      function     O\n",
            "26  14611   independent     O\n",
            "27  14612            of     O\n",
            "28  14613           Sp1     B\n",
            "29  14614             .     O\n",
            "30  14615            In     O\n",
            "31  14616      addition     O\n",
            "32  14617             ,     O\n",
            "33  14618       reduced     O\n",
            "34  14619    metabolism     O\n",
            "35  14620            to     O\n",
            "36  14621      cotinine     O\n",
            "37  14622          from     O\n",
            "38  14623           the     O\n",
            "39  14624  proliposomes     O\n",
            "40  14625           and     O\n",
            "41  14626            MP     O\n",
            "42  14627           was     O\n",
            "43  14628    apparently     O\n",
            "44  14629   responsible     O\n",
            "45  14630           for     O\n",
            "46  14631           the     O\n",
            "47  14632     sustained     O\n",
            "48  14633        plasma     O\n",
            "49  14634      nicotine     O\n",
            "50  14635        levels     O\n",
            "51  14636             .     O\n",
            "52  14637    Activation     O\n",
            "53  14638            of     O\n",
            "Earlier published data , indicating Sp1 binding to the R1 alpha / beta regions , could not be confirmed , suggesting that the R1 initiator element may function independent of Sp1 . In addition , reduced metabolism to cotinine from the proliposomes and MP was apparently responsible for the sustained plasma nicotine levels .\n",
            "['Earlier', 'published', 'data', ',', 'indicating', 'Sp1', 'binding', 'to', 'the', 'R1', 'alpha', '/', 'beta', 'regions', ',', 'could', 'not', 'be', 'confirmed', ',', 'suggesting', 'that', 'the', 'R1', 'initiator', 'element', 'may', 'function', 'independent', 'of', 'Sp1', '.', 'In', 'addition', ',', 'reduced', 'metabolism', 'to', 'cotinine', 'from', 'the', 'proliposomes', 'and', 'MP', 'was', 'apparently', 'responsible', 'for', 'the', 'sustained', 'plasma', 'nicotine', 'levels', '.']\n",
            "54 54\n",
            "['data_0', ',_1', 'indicating_2', 'Sp1_3', 'binding_4', 'to_5', 'the_6', 'R1_7', 'alpha_8', '/_9', 'beta_10', 'regions_11', ',_12', 'could_13', 'not_14', 'be_15', 'confirmed_16', ',_17', 'suggesting_18', 'that_19', 'the_20', 'R1_21', 'initiator_22', 'element_23', 'may_24', 'function_25', 'independent_26', 'of_27', 'Sp1_28', '._29', 'In_30', 'addition_31', ',_32', 'reduced_33', 'metabolism_34', 'to_35', 'cotinine_36', 'from_37', 'the_38', 'proliposomes_39', 'and_40', 'MP_41', 'was_42', 'apparently_43', 'responsible_44', 'for_45', 'the_46', 'sustained_47', 'plasma_48', 'nicotine_49', 'levels_50', '._51', 'Activation_52', 'of_53']\n",
            "BAD:  14585\n",
            "rpws:      index     token label\n",
            "0   14639       HIV     O\n",
            "1   14640         -     O\n",
            "2   14641         1     O\n",
            "3   14642  requires     O\n",
            "4   14643       the     O\n",
            "..    ...       ...   ...\n",
            "77  14716         1     I\n",
            "78  14717  proteins     I\n",
            "79  14718         .     O\n",
            "80  14719       The     O\n",
            "81  14720      rice     O\n",
            "\n",
            "[82 rows x 3 columns]\n",
            "Activation of HIV - 1 requires the binding of host cell transcription factors to cis elements in the proviral long terminal repeat ( LTR ) . The yeast mitochondrial Hsp70 , Ssc1p , functions as a molecular chaperone with its partner proteins , Mdj1p ( DnaJ homologue ) and Yge1p ( GrpE homologue ) . Increased TRE DNA binding failed to lead to increased transactivation correlating with the inability of SB 203580 to increase phosphorylation of these AP - 1 proteins .\n",
            "['Activation', 'of', 'HIV', '-', '1', 'requires', 'the', 'binding', 'of', 'host', 'cell', 'transcription', 'factors', 'to', 'cis', 'elements', 'in', 'the', 'proviral', 'long', 'terminal', 'repeat', '(', 'LTR', ')', '.', 'The', 'yeast', 'mitochondrial', 'Hsp70', ',', 'Ssc1p', ',', 'functions', 'as', 'a', 'molecular', 'chaperone', 'with', 'its', 'partner', 'proteins', ',', 'Mdj1p', '(', 'DnaJ', 'homologue', ')', 'and', 'Yge1p', '(', 'GrpE', 'homologue', ')', '.', 'Increased', 'TRE', 'DNA', 'binding', 'failed', 'to', 'lead', 'to', 'increased', 'transactivation', 'correlating', 'with', 'the', 'inability', 'of', 'SB', '203580', 'to', 'increase', 'phosphorylation', 'of', 'these', 'AP', '-', '1', 'proteins', '.']\n",
            "82 82\n",
            "['HIV_0', '-_1', '1_2', 'requires_3', 'the_4', 'binding_5', 'of_6', 'host_7', 'cell_8', 'transcription_9', 'factors_10', 'to_11', 'cis_12', 'elements_13', 'in_14', 'the_15', 'proviral_16', 'long_17', 'terminal_18', 'repeat_19', '(_20', 'LTR_21', ')_22', '._23', 'The_24', 'yeast_25', 'mitochondrial_26', 'Hsp70_27', ',_28', 'Ssc1p_29', ',_30', 'functions_31', 'as_32', 'a_33', 'molecular_34', 'chaperone_35', 'with_36', 'its_37', 'partner_38', 'proteins_39', ',_40', 'Mdj1p_41', '(_42', 'DnaJ_43', 'homologue_44', ')_45', 'and_46', 'Yge1p_47', '(_48', 'GrpE_49', 'homologue_50', ')_51', '._52', 'Increased_53', 'TRE_54', 'DNA_55', 'binding_56', 'failed_57', 'to_58', 'lead_59', 'to_60', 'increased_61', 'transactivation_62', 'correlating_63', 'with_64', 'the_65', 'inability_66', 'of_67', 'SB_68', '203580_69', 'to_70', 'increase_71', 'phosphorylation_72', 'of_73', 'these_74', 'AP_75', '-_76', '1_77', 'proteins_78', '._79', 'The_80', 'rice_81']\n",
            "BAD:  14639\n",
            "rpws:      index     token label\n",
            "0   14721      gene     O\n",
            "1   14722  consists     O\n",
            "2   14723        of     O\n",
            "3   14724      four     O\n",
            "4   14725     exons     O\n",
            "..    ...       ...   ...\n",
            "64  14785       and     O\n",
            "65  14786        Tb     O\n",
            "66  14787         .     O\n",
            "67  14788        Wi     O\n",
            "68  14789  appeared     O\n",
            "\n",
            "[69 rows x 3 columns]\n",
            "The rice gene consists of four exons . However , and in contrast to other MMP genes , no significative synergistic effect on CAT activity between the AP - 1 and PEA - 3 elements found in the collagenase - 3 gene promoter was found . The equation for mean trabecular number passed the test , whereas the validity of the equations for mean trabecular separation and Tb .\n",
            "['The', 'rice', 'gene', 'consists', 'of', 'four', 'exons', '.', 'However', ',', 'and', 'in', 'contrast', 'to', 'other', 'MMP', 'genes', ',', 'no', 'significative', 'synergistic', 'effect', 'on', 'CAT', 'activity', 'between', 'the', 'AP', '-', '1', 'and', 'PEA', '-', '3', 'elements', 'found', 'in', 'the', 'collagenase', '-', '3', 'gene', 'promoter', 'was', 'found', '.', 'The', 'equation', 'for', 'mean', 'trabecular', 'number', 'passed', 'the', 'test', ',', 'whereas', 'the', 'validity', 'of', 'the', 'equations', 'for', 'mean', 'trabecular', 'separation', 'and', 'Tb', '.']\n",
            "69 69\n",
            "['gene_0', 'consists_1', 'of_2', 'four_3', 'exons_4', '._5', 'However_6', ',_7', 'and_8', 'in_9', 'contrast_10', 'to_11', 'other_12', 'MMP_13', 'genes_14', ',_15', 'no_16', 'significative_17', 'synergistic_18', 'effect_19', 'on_20', 'CAT_21', 'activity_22', 'between_23', 'the_24', 'AP_25', '-_26', '1_27', 'and_28', 'PEA_29', '-_30', '3_31', 'elements_32', 'found_33', 'in_34', 'the_35', 'collagenase_36', '-_37', '3_38', 'gene_39', 'promoter_40', 'was_41', 'found_42', '._43', 'The_44', 'equation_45', 'for_46', 'mean_47', 'trabecular_48', 'number_49', 'passed_50', 'the_51', 'test_52', ',_53', 'whereas_54', 'the_55', 'validity_56', 'of_57', 'the_58', 'equations_59', 'for_60', 'mean_61', 'trabecular_62', 'separation_63', 'and_64', 'Tb_65', '._66', 'Wi_67', 'appeared_68']\n",
            "BAD:  14721\n",
            "rpws:      index           token label\n",
            "0   14790         limited     O\n",
            "1   14791               .     O\n",
            "2   14792  Methodological     O\n",
            "3   14793  considerations     O\n",
            "4   14794        limiting     O\n",
            "..    ...             ...   ...\n",
            "57  14847             the     O\n",
            "58  14848        duodenum     O\n",
            "59  14849               .     O\n",
            "60  14850        Reversal     O\n",
            "61  14851              of     O\n",
            "\n",
            "[62 rows x 3 columns]\n",
            "Wi appeared limited . Methodological considerations limiting causal assertions permissible with nonexperimental data are discussed . In contrast , postmitotic neurons possessed extremely low levels of E2F1 protein as assessed by the electrophoretic mobility shift assay and Western blotting . Recurrence of the hepatic cyst after partial excision and drainage was complicated by fistula formation between the cyst and the duodenum .\n",
            "['Wi', 'appeared', 'limited', '.', 'Methodological', 'considerations', 'limiting', 'causal', 'assertions', 'permissible', 'with', 'nonexperimental', 'data', 'are', 'discussed', '.', 'In', 'contrast', ',', 'postmitotic', 'neurons', 'possessed', 'extremely', 'low', 'levels', 'of', 'E2F1', 'protein', 'as', 'assessed', 'by', 'the', 'electrophoretic', 'mobility', 'shift', 'assay', 'and', 'Western', 'blotting', '.', 'Recurrence', 'of', 'the', 'hepatic', 'cyst', 'after', 'partial', 'excision', 'and', 'drainage', 'was', 'complicated', 'by', 'fistula', 'formation', 'between', 'the', 'cyst', 'and', 'the', 'duodenum', '.']\n",
            "62 62\n",
            "['limited_0', '._1', 'Methodological_2', 'considerations_3', 'limiting_4', 'causal_5', 'assertions_6', 'permissible_7', 'with_8', 'nonexperimental_9', 'data_10', 'are_11', 'discussed_12', '._13', 'In_14', 'contrast_15', ',_16', 'postmitotic_17', 'neurons_18', 'possessed_19', 'extremely_20', 'low_21', 'levels_22', 'of_23', 'E2F1_24', 'protein_25', 'as_26', 'assessed_27', 'by_28', 'the_29', 'electrophoretic_30', 'mobility_31', 'shift_32', 'assay_33', 'and_34', 'Western_35', 'blotting_36', '._37', 'Recurrence_38', 'of_39', 'the_40', 'hepatic_41', 'cyst_42', 'after_43', 'partial_44', 'excision_45', 'and_46', 'drainage_47', 'was_48', 'complicated_49', 'by_50', 'fistula_51', 'formation_52', 'between_53', 'the_54', 'cyst_55', 'and_56', 'the_57', 'duodenum_58', '._59', 'Reversal_60', 'of_61']\n",
            "BAD:  14790\n",
            "rpws:      index         token label\n",
            "0   14852    mefloquine     O\n",
            "1   14853    resistance     O\n",
            "2   14854            in     O\n",
            "3   14855        rodent     O\n",
            "4   14856    Plasmodium     O\n",
            "..    ...           ...   ...\n",
            "67  14919          beta     I\n",
            "68  14920          gene     I\n",
            "69  14921             .     O\n",
            "70  14922           The     O\n",
            "71  14923  experimental     O\n",
            "\n",
            "[72 rows x 3 columns]\n",
            "Reversal of mefloquine resistance in rodent Plasmodium . Possible consequences of such binder distribution through powder aggregates are discussed . The human parvalbumin mRNA contains the putative polyadenylation signal AATAAA 13 nucleotides upstream from the polyadenylation site . The delta Mo + SV tumor DNAs from B - lineage tumors were typically rearranged at the immunoglobulin gene loci and contained germ line configurations of the T - cell receptor beta gene .\n",
            "['Reversal', 'of', 'mefloquine', 'resistance', 'in', 'rodent', 'Plasmodium', '.', 'Possible', 'consequences', 'of', 'such', 'binder', 'distribution', 'through', 'powder', 'aggregates', 'are', 'discussed', '.', 'The', 'human', 'parvalbumin', 'mRNA', 'contains', 'the', 'putative', 'polyadenylation', 'signal', 'AATAAA', '13', 'nucleotides', 'upstream', 'from', 'the', 'polyadenylation', 'site', '.', 'The', 'delta', 'Mo', '+', 'SV', 'tumor', 'DNAs', 'from', 'B', '-', 'lineage', 'tumors', 'were', 'typically', 'rearranged', 'at', 'the', 'immunoglobulin', 'gene', 'loci', 'and', 'contained', 'germ', 'line', 'configurations', 'of', 'the', 'T', '-', 'cell', 'receptor', 'beta', 'gene', '.']\n",
            "72 72\n",
            "['mefloquine_0', 'resistance_1', 'in_2', 'rodent_3', 'Plasmodium_4', '._5', 'Possible_6', 'consequences_7', 'of_8', 'such_9', 'binder_10', 'distribution_11', 'through_12', 'powder_13', 'aggregates_14', 'are_15', 'discussed_16', '._17', 'The_18', 'human_19', 'parvalbumin_20', 'mRNA_21', 'contains_22', 'the_23', 'putative_24', 'polyadenylation_25', 'signal_26', 'AATAAA_27', '13_28', 'nucleotides_29', 'upstream_30', 'from_31', 'the_32', 'polyadenylation_33', 'site_34', '._35', 'The_36', 'delta_37', 'Mo_38', '+_39', 'SV_40', 'tumor_41', 'DNAs_42', 'from_43', 'B_44', '-_45', 'lineage_46', 'tumors_47', 'were_48', 'typically_49', 'rearranged_50', 'at_51', 'the_52', 'immunoglobulin_53', 'gene_54', 'loci_55', 'and_56', 'contained_57', 'germ_58', 'line_59', 'configurations_60', 'of_61', 'the_62', 'T_63', '-_64', 'cell_65', 'receptor_66', 'beta_67', 'gene_68', '._69', 'The_70', 'experimental_71']\n",
            "BAD:  14852\n",
            "rpws:      index        token label\n",
            "0   14924      results     O\n",
            "1   14925      suggest     O\n",
            "2   14926  equilibrium     O\n",
            "3   14927      complex     O\n",
            "4   14928    formation     O\n",
            "..    ...          ...   ...\n",
            "77  15001           to     O\n",
            "78  15002  vaccination     O\n",
            "79  15003            .     O\n",
            "80  15004           To     O\n",
            "81  15005      examine     O\n",
            "\n",
            "[82 rows x 3 columns]\n",
            "The experimental results suggest equilibrium complex formation between OTC and PVP , it being mainly the complex which is i . v . injected when both products are combined in an injection formulation . We conclude that the MMF lesion is secondary to intramuscular injection of aluminium hydroxide - containing vaccines , shows both long - term persistence of aluminium hydroxide and an ongoing local immune reaction , and is detected in patients with systemic symptoms which appeared subsequently to vaccination .\n",
            "['The', 'experimental', 'results', 'suggest', 'equilibrium', 'complex', 'formation', 'between', 'OTC', 'and', 'PVP', ',', 'it', 'being', 'mainly', 'the', 'complex', 'which', 'is', 'i', '.', 'v', '.', 'injected', 'when', 'both', 'products', 'are', 'combined', 'in', 'an', 'injection', 'formulation', '.', 'We', 'conclude', 'that', 'the', 'MMF', 'lesion', 'is', 'secondary', 'to', 'intramuscular', 'injection', 'of', 'aluminium', 'hydroxide', '-', 'containing', 'vaccines', ',', 'shows', 'both', 'long', '-', 'term', 'persistence', 'of', 'aluminium', 'hydroxide', 'and', 'an', 'ongoing', 'local', 'immune', 'reaction', ',', 'and', 'is', 'detected', 'in', 'patients', 'with', 'systemic', 'symptoms', 'which', 'appeared', 'subsequently', 'to', 'vaccination', '.']\n",
            "82 82\n",
            "['results_0', 'suggest_1', 'equilibrium_2', 'complex_3', 'formation_4', 'between_5', 'OTC_6', 'and_7', 'PVP_8', ',_9', 'it_10', 'being_11', 'mainly_12', 'the_13', 'complex_14', 'which_15', 'is_16', 'i_17', '._18', 'v_19', '._20', 'injected_21', 'when_22', 'both_23', 'products_24', 'are_25', 'combined_26', 'in_27', 'an_28', 'injection_29', 'formulation_30', '._31', 'We_32', 'conclude_33', 'that_34', 'the_35', 'MMF_36', 'lesion_37', 'is_38', 'secondary_39', 'to_40', 'intramuscular_41', 'injection_42', 'of_43', 'aluminium_44', 'hydroxide_45', '-_46', 'containing_47', 'vaccines_48', ',_49', 'shows_50', 'both_51', 'long_52', '-_53', 'term_54', 'persistence_55', 'of_56', 'aluminium_57', 'hydroxide_58', 'and_59', 'an_60', 'ongoing_61', 'local_62', 'immune_63', 'reaction_64', ',_65', 'and_66', 'is_67', 'detected_68', 'in_69', 'patients_70', 'with_71', 'systemic_72', 'symptoms_73', 'which_74', 'appeared_75', 'subsequently_76', 'to_77', 'vaccination_78', '._79', 'To_80', 'examine_81']\n",
            "BAD:  14924\n",
            "rpws:      index              token label\n",
            "0   15006                the     O\n",
            "1   15007             effect     O\n",
            "2   15008                 of     O\n",
            "3   15009                the     O\n",
            "4   15010               cAMP     O\n",
            "5   15011             signal     O\n",
            "6   15012       transduction     O\n",
            "7   15013            pathway     O\n",
            "8   15014                 on     O\n",
            "9   15015      transcription     O\n",
            "10  15016                 of     O\n",
            "11  15017                the     O\n",
            "12  15018               gene     O\n",
            "13  15019           encoding     O\n",
            "14  15020                the     O\n",
            "15  15021          catalytic     O\n",
            "16  15022            subunit     O\n",
            "17  15023                 of     O\n",
            "18  15024            glucose     B\n",
            "19  15025                  -     I\n",
            "20  15026                  6     I\n",
            "21  15027                  -     I\n",
            "22  15028        phosphatase     I\n",
            "23  15029                  (     O\n",
            "24  15030             G6Pase     B\n",
            "25  15031                  )     O\n",
            "26  15032                  ,     O\n",
            "27  15033             G6Pase     B\n",
            "28  15034                  -     O\n",
            "29  15035    chloramphenicol     B\n",
            "30  15036  acetyltransferase     I\n",
            "31  15037                  (     O\n",
            "32  15038                CAT     B\n",
            "33  15039                  )     O\n",
            "34  15040             fusion     O\n",
            "35  15041              genes     O\n",
            "36  15042               were     O\n",
            "37  15043        transiently     O\n",
            "38  15044        transfected     O\n",
            "39  15045               into     O\n",
            "40  15046             either     O\n",
            "41  15047                the     O\n",
            "42  15048              liver     O\n",
            "43  15049                  -     O\n",
            "44  15050            derived     O\n",
            "45  15051              HepG2     O\n",
            "46  15052                 or     O\n",
            "47  15053             kidney     O\n",
            "48  15054                  -     O\n",
            "49  15055            derived     O\n",
            "50  15056                LLC     O\n",
            "51  15057                  -     O\n",
            "52  15058                 PK     O\n",
            "53  15059               cell     O\n",
            "54  15060               line     O\n",
            "55  15061                  .     O\n",
            "56  15062            Whereas     O\n",
            "57  15063                the     O\n",
            "To examine the effect of the cAMP signal transduction pathway on transcription of the gene encoding the catalytic subunit of glucose - 6 - phosphatase ( G6Pase ) , G6Pase - chloramphenicol acetyltransferase ( CAT ) fusion genes were transiently transfected into either the liver - derived HepG2 or kidney - derived LLC - PK cell line .\n",
            "['To', 'examine', 'the', 'effect', 'of', 'the', 'cAMP', 'signal', 'transduction', 'pathway', 'on', 'transcription', 'of', 'the', 'gene', 'encoding', 'the', 'catalytic', 'subunit', 'of', 'glucose', '-', '6', '-', 'phosphatase', '(', 'G6Pase', ')', ',', 'G6Pase', '-', 'chloramphenicol', 'acetyltransferase', '(', 'CAT', ')', 'fusion', 'genes', 'were', 'transiently', 'transfected', 'into', 'either', 'the', 'liver', '-', 'derived', 'HepG2', 'or', 'kidney', '-', 'derived', 'LLC', '-', 'PK', 'cell', 'line', '.']\n",
            "58 58\n",
            "['the_0', 'effect_1', 'of_2', 'the_3', 'cAMP_4', 'signal_5', 'transduction_6', 'pathway_7', 'on_8', 'transcription_9', 'of_10', 'the_11', 'gene_12', 'encoding_13', 'the_14', 'catalytic_15', 'subunit_16', 'of_17', 'glucose_18', '-_19', '6_20', '-_21', 'phosphatase_22', '(_23', 'G6Pase_24', ')_25', ',_26', 'G6Pase_27', '-_28', 'chloramphenicol_29', 'acetyltransferase_30', '(_31', 'CAT_32', ')_33', 'fusion_34', 'genes_35', 'were_36', 'transiently_37', 'transfected_38', 'into_39', 'either_40', 'the_41', 'liver_42', '-_43', 'derived_44', 'HepG2_45', 'or_46', 'kidney_47', '-_48', 'derived_49', 'LLC_50', '-_51', 'PK_52', 'cell_53', 'line_54', '._55', 'Whereas_56', 'the_57']\n",
            "BAD:  15006\n",
            "rpws:      index        token label\n",
            "0   15064     sequence     O\n",
            "1   15065           of     O\n",
            "2   15066        alpha     B\n",
            "3   15067            -     I\n",
            "4   15068      actinin     I\n",
            "..    ...          ...   ...\n",
            "71  15135  cyclosporin     O\n",
            "72  15136            A     O\n",
            "73  15137            .     O\n",
            "74  15138            (     O\n",
            "75  15139           We     O\n",
            "\n",
            "[76 rows x 3 columns]\n",
            "Whereas the sequence of alpha - actinin ( Noegel , A . , W . By contrast , clonidine ( 1 micrograms / kg ) elicited an immediate and prolonged fall in blood pressure and heart rate when given into the vertebral artery , but not intravenously . The shift from complex I to complex II seen only in SCM - 1 - producer T cell lines upon activation was completely suppressed by cyclosporin A .\n",
            "['Whereas', 'the', 'sequence', 'of', 'alpha', '-', 'actinin', '(', 'Noegel', ',', 'A', '.', ',', 'W', '.', 'By', 'contrast', ',', 'clonidine', '(', '1', 'micrograms', '/', 'kg', ')', 'elicited', 'an', 'immediate', 'and', 'prolonged', 'fall', 'in', 'blood', 'pressure', 'and', 'heart', 'rate', 'when', 'given', 'into', 'the', 'vertebral', 'artery', ',', 'but', 'not', 'intravenously', '.', 'The', 'shift', 'from', 'complex', 'I', 'to', 'complex', 'II', 'seen', 'only', 'in', 'SCM', '-', '1', '-', 'producer', 'T', 'cell', 'lines', 'upon', 'activation', 'was', 'completely', 'suppressed', 'by', 'cyclosporin', 'A', '.']\n",
            "76 76\n",
            "['sequence_0', 'of_1', 'alpha_2', '-_3', 'actinin_4', '(_5', 'Noegel_6', ',_7', 'A_8', '._9', ',_10', 'W_11', '._12', 'By_13', 'contrast_14', ',_15', 'clonidine_16', '(_17', '1_18', 'micrograms_19', '/_20', 'kg_21', ')_22', 'elicited_23', 'an_24', 'immediate_25', 'and_26', 'prolonged_27', 'fall_28', 'in_29', 'blood_30', 'pressure_31', 'and_32', 'heart_33', 'rate_34', 'when_35', 'given_36', 'into_37', 'the_38', 'vertebral_39', 'artery_40', ',_41', 'but_42', 'not_43', 'intravenously_44', '._45', 'The_46', 'shift_47', 'from_48', 'complex_49', 'I_50', 'to_51', 'complex_52', 'II_53', 'seen_54', 'only_55', 'in_56', 'SCM_57', '-_58', '1_59', '-_60', 'producer_61', 'T_62', 'cell_63', 'lines_64', 'upon_65', 'activation_66', 'was_67', 'completely_68', 'suppressed_69', 'by_70', 'cyclosporin_71', 'A_72', '._73', '(_74', 'We_75']\n",
            "BAD:  15064\n",
            "rpws:      index           token label\n",
            "0   15140        observed     O\n",
            "1   15141               a     O\n",
            "2   15142        ferritin     B\n",
            "3   15143           value     O\n",
            "4   15144              as     O\n",
            "5   15145            high     O\n",
            "6   15146              as     O\n",
            "7   15147              47     O\n",
            "8   15148              mg     O\n",
            "9   15149               /     O\n",
            "10  15150               L     O\n",
            "11  15151              in     O\n",
            "12  15152             one     O\n",
            "13  15153         patient     O\n",
            "14  15154               .     O\n",
            "15  15155               )     O\n",
            "16  15156         Results     O\n",
            "17  15157            with     O\n",
            "18  15158             all     O\n",
            "19  15159           these     O\n",
            "20  15160            kits     O\n",
            "21  15161             did     O\n",
            "22  15162             not     O\n",
            "23  15163           inter     O\n",
            "24  15164               -     O\n",
            "25  15165         compare     O\n",
            "26  15166            well     O\n",
            "27  15167             for     O\n",
            "28  15168        ferritin     B\n",
            "29  15169  concentrations     O\n",
            "30  15170         greater     O\n",
            "31  15171            than     O\n",
            "32  15172             300     O\n",
            "33  15173      micrograms     O\n",
            "34  15174               /     O\n",
            "35  15175               L     O\n",
            "36  15176               ,     O\n",
            "37  15177               a     O\n",
            "38  15178         finding     O\n",
            "39  15179            that     O\n",
            "40  15180           casts     O\n",
            "41  15181         further     O\n",
            "42  15182           doubt     O\n",
            "43  15183              on     O\n",
            "44  15184             the     O\n",
            "45  15185   controversial     O\n",
            "46  15186             use     O\n",
            "47  15187              of     O\n",
            "48  15188           serum     B\n",
            "49  15189        ferritin     I\n",
            "50  15190     measurement     O\n",
            "51  15191              in     O\n",
            "52  15192           cases     O\n",
            "53  15193              of     O\n",
            "54  15194            iron     O\n",
            "55  15195        overload     O\n",
            "56  15196               .     O\n",
            "57  15197            Post     O\n",
            "58  15198               -     O\n",
            "( We observed a ferritin value as high as 47 mg / L in one patient . ) Results with all these kits did not inter - compare well for ferritin concentrations greater than 300 micrograms / L , a finding that casts further doubt on the controversial use of serum ferritin measurement in cases of iron overload .\n",
            "['(', 'We', 'observed', 'a', 'ferritin', 'value', 'as', 'high', 'as', '47', 'mg', '/', 'L', 'in', 'one', 'patient', '.', ')', 'Results', 'with', 'all', 'these', 'kits', 'did', 'not', 'inter', '-', 'compare', 'well', 'for', 'ferritin', 'concentrations', 'greater', 'than', '300', 'micrograms', '/', 'L', ',', 'a', 'finding', 'that', 'casts', 'further', 'doubt', 'on', 'the', 'controversial', 'use', 'of', 'serum', 'ferritin', 'measurement', 'in', 'cases', 'of', 'iron', 'overload', '.']\n",
            "59 59\n",
            "['observed_0', 'a_1', 'ferritin_2', 'value_3', 'as_4', 'high_5', 'as_6', '47_7', 'mg_8', '/_9', 'L_10', 'in_11', 'one_12', 'patient_13', '._14', ')_15', 'Results_16', 'with_17', 'all_18', 'these_19', 'kits_20', 'did_21', 'not_22', 'inter_23', '-_24', 'compare_25', 'well_26', 'for_27', 'ferritin_28', 'concentrations_29', 'greater_30', 'than_31', '300_32', 'micrograms_33', '/_34', 'L_35', ',_36', 'a_37', 'finding_38', 'that_39', 'casts_40', 'further_41', 'doubt_42', 'on_43', 'the_44', 'controversial_45', 'use_46', 'of_47', 'serum_48', 'ferritin_49', 'measurement_50', 'in_51', 'cases_52', 'of_53', 'iron_54', 'overload_55', '._56', 'Post_57', '-_58']\n",
            "BAD:  15140\n",
            "rpws:      index         token label\n",
            "0   15199           CPB     O\n",
            "1   15200       cardiac     O\n",
            "2   15201         index     O\n",
            "3   15202           was     O\n",
            "4   15203      superior     O\n",
            "..    ...           ...   ...\n",
            "76  15275  craniofacial     O\n",
            "77  15276    mesenchyme     O\n",
            "78  15277             .     O\n",
            "79  15278    Disruption     O\n",
            "80  15279            of     O\n",
            "\n",
            "[81 rows x 3 columns]\n",
            "Post - CPB cardiac index was superior in group B ( 3 . 9 + / - 0 . 3 vs . ER beta was less potent than ER alpha in activating E2 - stimulated ERELuc activity ( 4 - vs . The expression pattern of LjEmx changed dramatically during embryogenesis ; expression was seen initially in the entire neural tube and mesoderm , which were secondarily downregulated , and secondarily in cranial nerve ganglia and in the craniofacial mesenchyme .\n",
            "['Post', '-', 'CPB', 'cardiac', 'index', 'was', 'superior', 'in', 'group', 'B', '(', '3', '.', '9', '+', '/', '-', '0', '.', '3', 'vs', '.', 'ER', 'beta', 'was', 'less', 'potent', 'than', 'ER', 'alpha', 'in', 'activating', 'E2', '-', 'stimulated', 'ERELuc', 'activity', '(', '4', '-', 'vs', '.', 'The', 'expression', 'pattern', 'of', 'LjEmx', 'changed', 'dramatically', 'during', 'embryogenesis', ';', 'expression', 'was', 'seen', 'initially', 'in', 'the', 'entire', 'neural', 'tube', 'and', 'mesoderm', ',', 'which', 'were', 'secondarily', 'downregulated', ',', 'and', 'secondarily', 'in', 'cranial', 'nerve', 'ganglia', 'and', 'in', 'the', 'craniofacial', 'mesenchyme', '.']\n",
            "81 81\n",
            "['CPB_0', 'cardiac_1', 'index_2', 'was_3', 'superior_4', 'in_5', 'group_6', 'B_7', '(_8', '3_9', '._10', '9_11', '+_12', '/_13', '-_14', '0_15', '._16', '3_17', 'vs_18', '._19', 'ER_20', 'beta_21', 'was_22', 'less_23', 'potent_24', 'than_25', 'ER_26', 'alpha_27', 'in_28', 'activating_29', 'E2_30', '-_31', 'stimulated_32', 'ERELuc_33', 'activity_34', '(_35', '4_36', '-_37', 'vs_38', '._39', 'The_40', 'expression_41', 'pattern_42', 'of_43', 'LjEmx_44', 'changed_45', 'dramatically_46', 'during_47', 'embryogenesis_48', ';_49', 'expression_50', 'was_51', 'seen_52', 'initially_53', 'in_54', 'the_55', 'entire_56', 'neural_57', 'tube_58', 'and_59', 'mesoderm_60', ',_61', 'which_62', 'were_63', 'secondarily_64', 'downregulated_65', ',_66', 'and_67', 'secondarily_68', 'in_69', 'cranial_70', 'nerve_71', 'ganglia_72', 'and_73', 'in_74', 'the_75', 'craniofacial_76', 'mesenchyme_77', '._78', 'Disruption_79', 'of_80']\n",
            "BAD:  15199\n",
            "rpws:      index            token label\n",
            "0   15280              PML     B\n",
            "1   15281       subnuclear     I\n",
            "2   15282          domains     I\n",
            "3   15283               by     O\n",
            "4   15284              the     O\n",
            "5   15285           acidic     B\n",
            "6   15286              IE1     I\n",
            "7   15287          protein     I\n",
            "8   15288               of     O\n",
            "9   15289            human     O\n",
            "10  15290  cytomegalovirus     O\n",
            "11  15291               is     O\n",
            "12  15292         mediated     O\n",
            "13  15293          through     O\n",
            "14  15294      interaction     O\n",
            "15  15295             with     O\n",
            "16  15296              PML     B\n",
            "17  15297              and     O\n",
            "18  15298              may     O\n",
            "19  15299         modulate     O\n",
            "20  15300                a     O\n",
            "21  15301             RING     O\n",
            "22  15302           finger     O\n",
            "23  15303                -     O\n",
            "24  15304        dependent     O\n",
            "25  15305          cryptic     O\n",
            "26  15306   transactivator     O\n",
            "27  15307         function     O\n",
            "28  15308               of     O\n",
            "29  15309              PML     B\n",
            "30  15310                .     O\n",
            "31  15311      Furthermore     O\n",
            "32  15312                ,     O\n",
            "33  15313              the     O\n",
            "34  15314               GC     O\n",
            "35  15315                -     O\n",
            "36  15316             rich     O\n",
            "37  15317        sequences     O\n",
            "38  15318            could     O\n",
            "39  15319           confer     O\n",
            "40  15320              Sp1     B\n",
            "41  15321                -     O\n",
            "42  15322        dependent     O\n",
            "43  15323  transactivation     O\n",
            "44  15324               to     O\n",
            "45  15325                a     O\n",
            "46  15326     heterologous     O\n",
            "47  15327        prolactin     B\n",
            "48  15328          minimal     I\n",
            "49  15329         promoter     I\n",
            "50  15330                .     O\n",
            "51  15331           Twelve     O\n",
            "52  15332         patients     O\n",
            "Disruption of PML subnuclear domains by the acidic IE1 protein of human cytomegalovirus is mediated through interaction with PML and may modulate a RING finger - dependent cryptic transactivator function of PML . Furthermore , the GC - rich sequences could confer Sp1 - dependent transactivation to a heterologous prolactin minimal promoter .\n",
            "['Disruption', 'of', 'PML', 'subnuclear', 'domains', 'by', 'the', 'acidic', 'IE1', 'protein', 'of', 'human', 'cytomegalovirus', 'is', 'mediated', 'through', 'interaction', 'with', 'PML', 'and', 'may', 'modulate', 'a', 'RING', 'finger', '-', 'dependent', 'cryptic', 'transactivator', 'function', 'of', 'PML', '.', 'Furthermore', ',', 'the', 'GC', '-', 'rich', 'sequences', 'could', 'confer', 'Sp1', '-', 'dependent', 'transactivation', 'to', 'a', 'heterologous', 'prolactin', 'minimal', 'promoter', '.']\n",
            "53 53\n",
            "['PML_0', 'subnuclear_1', 'domains_2', 'by_3', 'the_4', 'acidic_5', 'IE1_6', 'protein_7', 'of_8', 'human_9', 'cytomegalovirus_10', 'is_11', 'mediated_12', 'through_13', 'interaction_14', 'with_15', 'PML_16', 'and_17', 'may_18', 'modulate_19', 'a_20', 'RING_21', 'finger_22', '-_23', 'dependent_24', 'cryptic_25', 'transactivator_26', 'function_27', 'of_28', 'PML_29', '._30', 'Furthermore_31', ',_32', 'the_33', 'GC_34', '-_35', 'rich_36', 'sequences_37', 'could_38', 'confer_39', 'Sp1_40', '-_41', 'dependent_42', 'transactivation_43', 'to_44', 'a_45', 'heterologous_46', 'prolactin_47', 'minimal_48', 'promoter_49', '._50', 'Twelve_51', 'patients_52']\n",
            "BAD:  15280\n",
            "rpws:      index      token label\n",
            "0   15333         of     O\n",
            "1   15334    leprosy     O\n",
            "2   15335       with     O\n",
            "3   15336  arthritis     O\n",
            "4   15337        and     O\n",
            "..    ...        ...   ...\n",
            "59  15392       OHSD     I\n",
            "60  15393          )     O\n",
            "61  15394          .     O\n",
            "62  15395        The     O\n",
            "63  15396       NrfC     B\n",
            "\n",
            "[64 rows x 3 columns]\n",
            "Twelve patients of leprosy with arthritis and 161 patients without arthritis were studied for immunological parameters like immunoglobulins ( IgG , IgM , IgA ) , C - reactive proteins and rheumatoid factor . Several reports assert that prolactin affects the delta 5 and delta 4 pathways through its effect on the activity of 3beta - hydroxysteroid dehydrogenase ( 3beta - OHSD ) .\n",
            "['Twelve', 'patients', 'of', 'leprosy', 'with', 'arthritis', 'and', '161', 'patients', 'without', 'arthritis', 'were', 'studied', 'for', 'immunological', 'parameters', 'like', 'immunoglobulins', '(', 'IgG', ',', 'IgM', ',', 'IgA', ')', ',', 'C', '-', 'reactive', 'proteins', 'and', 'rheumatoid', 'factor', '.', 'Several', 'reports', 'assert', 'that', 'prolactin', 'affects', 'the', 'delta', '5', 'and', 'delta', '4', 'pathways', 'through', 'its', 'effect', 'on', 'the', 'activity', 'of', '3beta', '-', 'hydroxysteroid', 'dehydrogenase', '(', '3beta', '-', 'OHSD', ')', '.']\n",
            "64 64\n",
            "['of_0', 'leprosy_1', 'with_2', 'arthritis_3', 'and_4', '161_5', 'patients_6', 'without_7', 'arthritis_8', 'were_9', 'studied_10', 'for_11', 'immunological_12', 'parameters_13', 'like_14', 'immunoglobulins_15', '(_16', 'IgG_17', ',_18', 'IgM_19', ',_20', 'IgA_21', ')_22', ',_23', 'C_24', '-_25', 'reactive_26', 'proteins_27', 'and_28', 'rheumatoid_29', 'factor_30', '._31', 'Several_32', 'reports_33', 'assert_34', 'that_35', 'prolactin_36', 'affects_37', 'the_38', 'delta_39', '5_40', 'and_41', 'delta_42', '4_43', 'pathways_44', 'through_45', 'its_46', 'effect_47', 'on_48', 'the_49', 'activity_50', 'of_51', '3beta_52', '-_53', 'hydroxysteroid_54', 'dehydrogenase_55', '(_56', '3beta_57', '-_58', 'OHSD_59', ')_60', '._61', 'The_62', 'NrfC_63']\n",
            "BAD:  15333\n",
            "rpws:      index        token label\n",
            "0   15397  polypeptide     I\n",
            "1   15398            ,     O\n",
            "2   15399            M     B\n",
            "3   15400            (     I\n",
            "4   15401            r     I\n",
            "..    ...          ...   ...\n",
            "86  15483            (     O\n",
            "87  15484            R     O\n",
            "88  15485            .     O\n",
            "89  15486       Stress     O\n",
            "90  15487     thallium     O\n",
            "\n",
            "[91 rows x 3 columns]\n",
            "The NrfC polypeptide , M ( r ) 24 , 567 , contains 16 cysteine residues arranged in four clusters typical of the CooF super - family of non - haem iron - sulphur proteins . Zonography in urology ( author ' s transl ) ( i ) p60 bound fast - migrating , underprocessed wild - type ICP22 and ICP22 lacking the carboxyl - terminal 24 amino acids but not ICP22 lacking the carboxyl - terminal 40 amino acids , whereas the previously identified cellular protein p78 ( R .\n",
            "['The', 'NrfC', 'polypeptide', ',', 'M', '(', 'r', ')', '24', ',', '567', ',', 'contains', '16', 'cysteine', 'residues', 'arranged', 'in', 'four', 'clusters', 'typical', 'of', 'the', 'CooF', 'super', '-', 'family', 'of', 'non', '-', 'haem', 'iron', '-', 'sulphur', 'proteins', '.', 'Zonography', 'in', 'urology', '(', 'author', \"'\", 's', 'transl', ')', '(', 'i', ')', 'p60', 'bound', 'fast', '-', 'migrating', ',', 'underprocessed', 'wild', '-', 'type', 'ICP22', 'and', 'ICP22', 'lacking', 'the', 'carboxyl', '-', 'terminal', '24', 'amino', 'acids', 'but', 'not', 'ICP22', 'lacking', 'the', 'carboxyl', '-', 'terminal', '40', 'amino', 'acids', ',', 'whereas', 'the', 'previously', 'identified', 'cellular', 'protein', 'p78', '(', 'R', '.']\n",
            "91 91\n",
            "['polypeptide_0', ',_1', 'M_2', '(_3', 'r_4', ')_5', '24_6', ',_7', '567_8', ',_9', 'contains_10', '16_11', 'cysteine_12', 'residues_13', 'arranged_14', 'in_15', 'four_16', 'clusters_17', 'typical_18', 'of_19', 'the_20', 'CooF_21', 'super_22', '-_23', 'family_24', 'of_25', 'non_26', '-_27', 'haem_28', 'iron_29', '-_30', 'sulphur_31', 'proteins_32', '._33', 'Zonography_34', 'in_35', 'urology_36', '(_37', 'author_38', \"'_39\", 's_40', 'transl_41', ')_42', '(_43', 'i_44', ')_45', 'p60_46', 'bound_47', 'fast_48', '-_49', 'migrating_50', ',_51', 'underprocessed_52', 'wild_53', '-_54', 'type_55', 'ICP22_56', 'and_57', 'ICP22_58', 'lacking_59', 'the_60', 'carboxyl_61', '-_62', 'terminal_63', '24_64', 'amino_65', 'acids_66', 'but_67', 'not_68', 'ICP22_69', 'lacking_70', 'the_71', 'carboxyl_72', '-_73', 'terminal_74', '40_75', 'amino_76', 'acids_77', ',_78', 'whereas_79', 'the_80', 'previously_81', 'identified_82', 'cellular_83', 'protein_84', 'p78_85', '(_86', 'R_87', '._88', 'Stress_89', 'thallium_90']\n",
            "BAD:  15397\n",
            "rpws:      index        token label\n",
            "0   15488            -     O\n",
            "1   15489          201     O\n",
            "2   15490   myocardial     O\n",
            "3   15491      imaging     O\n",
            "4   15492          was     O\n",
            "5   15493         used     O\n",
            "6   15494           in     O\n",
            "7   15495          two     O\n",
            "8   15496       angina     O\n",
            "9   15497            -     O\n",
            "10  15498         free     O\n",
            "11  15499     patients     O\n",
            "12  15500         with     O\n",
            "13  15501       severe     O\n",
            "14  15502   congestive     O\n",
            "15  15503        heart     O\n",
            "16  15504      failure     O\n",
            "17  15505           to     O\n",
            "18  15506     identify     O\n",
            "19  15507   clinically     O\n",
            "20  15508       silent     O\n",
            "21  15509        areas     O\n",
            "22  15510           of     O\n",
            "23  15511     ischemic     O\n",
            "24  15512   myocardium     O\n",
            "25  15513          and     O\n",
            "26  15514           to     O\n",
            "27  15515  distinguish     O\n",
            "28  15516      between     O\n",
            "29  15517         scar     O\n",
            "30  15518          and     O\n",
            "31  15519   reversibly     O\n",
            "32  15520     ischemic     O\n",
            "33  15521   myocardium     O\n",
            "34  15522           as     O\n",
            "35  15523            a     O\n",
            "36  15524        cause     O\n",
            "37  15525          for     O\n",
            "38  15526     akinesia     O\n",
            "39  15527           of     O\n",
            "40  15528         left     O\n",
            "41  15529  ventricular     O\n",
            "42  15530         wall     O\n",
            "43  15531     segments     O\n",
            "44  15532            .     O\n",
            "45  15533          The     O\n",
            "46  15534    community     O\n",
            "Stress thallium - 201 myocardial imaging was used in two angina - free patients with severe congestive heart failure to identify clinically silent areas of ischemic myocardium and to distinguish between scar and reversibly ischemic myocardium as a cause for akinesia of left ventricular wall segments .\n",
            "['Stress', 'thallium', '-', '201', 'myocardial', 'imaging', 'was', 'used', 'in', 'two', 'angina', '-', 'free', 'patients', 'with', 'severe', 'congestive', 'heart', 'failure', 'to', 'identify', 'clinically', 'silent', 'areas', 'of', 'ischemic', 'myocardium', 'and', 'to', 'distinguish', 'between', 'scar', 'and', 'reversibly', 'ischemic', 'myocardium', 'as', 'a', 'cause', 'for', 'akinesia', 'of', 'left', 'ventricular', 'wall', 'segments', '.']\n",
            "47 47\n",
            "['-_0', '201_1', 'myocardial_2', 'imaging_3', 'was_4', 'used_5', 'in_6', 'two_7', 'angina_8', '-_9', 'free_10', 'patients_11', 'with_12', 'severe_13', 'congestive_14', 'heart_15', 'failure_16', 'to_17', 'identify_18', 'clinically_19', 'silent_20', 'areas_21', 'of_22', 'ischemic_23', 'myocardium_24', 'and_25', 'to_26', 'distinguish_27', 'between_28', 'scar_29', 'and_30', 'reversibly_31', 'ischemic_32', 'myocardium_33', 'as_34', 'a_35', 'cause_36', 'for_37', 'akinesia_38', 'of_39', 'left_40', 'ventricular_41', 'wall_42', 'segments_43', '._44', 'The_45', 'community_46']\n",
            "BAD:  15488\n",
            "rpws:      index       token label\n",
            "0   15535           -     O\n",
            "1   15536    acquired     O\n",
            "2   15537  bacteremia     O\n",
            "3   15538         was     O\n",
            "4   15539      mainly     O\n",
            "..    ...         ...   ...\n",
            "81  15616        gene     O\n",
            "82  15617  expression     O\n",
            "83  15618           .     O\n",
            "84  15619  Procedures     O\n",
            "85  15620        with     O\n",
            "\n",
            "[86 rows x 3 columns]\n",
            "The community - acquired bacteremia was mainly due to E . coli . Y1 , Y2 , and Y4 / PP1 . Low plasma glucose concentrations that may or may not be sufficiently low to result in symptoms can be observed as a concomitant of several diverse diseases . Developmentally - regulated interaction of a transcription factor complex containing CDP / cut with the early histone H3 gene promoter of the sea urchin Tetrapygus niger is associated with changes in chromatin structure and gene expression .\n",
            "['The', 'community', '-', 'acquired', 'bacteremia', 'was', 'mainly', 'due', 'to', 'E', '.', 'coli', '.', 'Y1', ',', 'Y2', ',', 'and', 'Y4', '/', 'PP1', '.', 'Low', 'plasma', 'glucose', 'concentrations', 'that', 'may', 'or', 'may', 'not', 'be', 'sufficiently', 'low', 'to', 'result', 'in', 'symptoms', 'can', 'be', 'observed', 'as', 'a', 'concomitant', 'of', 'several', 'diverse', 'diseases', '.', 'Developmentally', '-', 'regulated', 'interaction', 'of', 'a', 'transcription', 'factor', 'complex', 'containing', 'CDP', '/', 'cut', 'with', 'the', 'early', 'histone', 'H3', 'gene', 'promoter', 'of', 'the', 'sea', 'urchin', 'Tetrapygus', 'niger', 'is', 'associated', 'with', 'changes', 'in', 'chromatin', 'structure', 'and', 'gene', 'expression', '.']\n",
            "86 86\n",
            "['-_0', 'acquired_1', 'bacteremia_2', 'was_3', 'mainly_4', 'due_5', 'to_6', 'E_7', '._8', 'coli_9', '._10', 'Y1_11', ',_12', 'Y2_13', ',_14', 'and_15', 'Y4_16', '/_17', 'PP1_18', '._19', 'Low_20', 'plasma_21', 'glucose_22', 'concentrations_23', 'that_24', 'may_25', 'or_26', 'may_27', 'not_28', 'be_29', 'sufficiently_30', 'low_31', 'to_32', 'result_33', 'in_34', 'symptoms_35', 'can_36', 'be_37', 'observed_38', 'as_39', 'a_40', 'concomitant_41', 'of_42', 'several_43', 'diverse_44', 'diseases_45', '._46', 'Developmentally_47', '-_48', 'regulated_49', 'interaction_50', 'of_51', 'a_52', 'transcription_53', 'factor_54', 'complex_55', 'containing_56', 'CDP_57', '/_58', 'cut_59', 'with_60', 'the_61', 'early_62', 'histone_63', 'H3_64', 'gene_65', 'promoter_66', 'of_67', 'the_68', 'sea_69', 'urchin_70', 'Tetrapygus_71', 'niger_72', 'is_73', 'associated_74', 'with_75', 'changes_76', 'in_77', 'chromatin_78', 'structure_79', 'and_80', 'gene_81', 'expression_82', '._83', 'Procedures_84', 'with_85']\n",
            "BAD:  15535\n",
            "rpws:      index      token label\n",
            "0   15621  perfected     O\n",
            "1   15622     method     O\n",
            "2   15623      After     O\n",
            "3   15624          a     O\n",
            "4   15625      shift     O\n",
            "..    ...        ...   ...\n",
            "72  15693   Upstream     I\n",
            "73  15694       Gene     I\n",
            "74  15695          .     O\n",
            "75  15696        The     O\n",
            "76  15697        STS     B\n",
            "\n",
            "[77 rows x 3 columns]\n",
            "Procedures with perfected method After a shift to 37 degrees C , the mutant Rat7 - 1p / Nup159 - 1p is lost from the nuclear rim of rat7 - 1 cells and NPCs , which are clustered together in these cells grown under permissive conditions become substantially less clustered . The mRNA for this protein is expressed in the T - ALL cell line Jurkat and has been designated HUG1 , for HOX11 Upstream Gene .\n",
            "['Procedures', 'with', 'perfected', 'method', 'After', 'a', 'shift', 'to', '37', 'degrees', 'C', ',', 'the', 'mutant', 'Rat7', '-', '1p', '/', 'Nup159', '-', '1p', 'is', 'lost', 'from', 'the', 'nuclear', 'rim', 'of', 'rat7', '-', '1', 'cells', 'and', 'NPCs', ',', 'which', 'are', 'clustered', 'together', 'in', 'these', 'cells', 'grown', 'under', 'permissive', 'conditions', 'become', 'substantially', 'less', 'clustered', '.', 'The', 'mRNA', 'for', 'this', 'protein', 'is', 'expressed', 'in', 'the', 'T', '-', 'ALL', 'cell', 'line', 'Jurkat', 'and', 'has', 'been', 'designated', 'HUG1', ',', 'for', 'HOX11', 'Upstream', 'Gene', '.']\n",
            "77 77\n",
            "['perfected_0', 'method_1', 'After_2', 'a_3', 'shift_4', 'to_5', '37_6', 'degrees_7', 'C_8', ',_9', 'the_10', 'mutant_11', 'Rat7_12', '-_13', '1p_14', '/_15', 'Nup159_16', '-_17', '1p_18', 'is_19', 'lost_20', 'from_21', 'the_22', 'nuclear_23', 'rim_24', 'of_25', 'rat7_26', '-_27', '1_28', 'cells_29', 'and_30', 'NPCs_31', ',_32', 'which_33', 'are_34', 'clustered_35', 'together_36', 'in_37', 'these_38', 'cells_39', 'grown_40', 'under_41', 'permissive_42', 'conditions_43', 'become_44', 'substantially_45', 'less_46', 'clustered_47', '._48', 'The_49', 'mRNA_50', 'for_51', 'this_52', 'protein_53', 'is_54', 'expressed_55', 'in_56', 'the_57', 'T_58', '-_59', 'ALL_60', 'cell_61', 'line_62', 'Jurkat_63', 'and_64', 'has_65', 'been_66', 'designated_67', 'HUG1_68', ',_69', 'for_70', 'HOX11_71', 'Upstream_72', 'Gene_73', '._74', 'The_75', 'STS_76']\n",
            "BAD:  15621\n",
            "rpws:      index     token label\n",
            "0   15698  sequence     I\n",
            "1   15699        WI     I\n",
            "2   15700         -     I\n",
            "3   15701     14920     I\n",
            "4   15702        is     O\n",
            "..    ...       ...   ...\n",
            "63  15761       LTR     I\n",
            "64  15762  enhancer     I\n",
            "65  15763         .     O\n",
            "66  15764      ORF3     O\n",
            "67  15765   appears     O\n",
            "\n",
            "[68 rows x 3 columns]\n",
            "The STS sequence WI - 14920 is in fact derived from the 3 ' - untranslated region of the human PUNC gene . Its activity was twice that of a construct where the CAT gene was driven by the H - 2Kb 5 ' enhancer region ( H2TF1 / KBF1 site ) and comparable to that of pRSVCAT construct carrying the strong Rous sarcoma virus LTR enhancer .\n",
            "['The', 'STS', 'sequence', 'WI', '-', '14920', 'is', 'in', 'fact', 'derived', 'from', 'the', '3', \"'\", '-', 'untranslated', 'region', 'of', 'the', 'human', 'PUNC', 'gene', '.', 'Its', 'activity', 'was', 'twice', 'that', 'of', 'a', 'construct', 'where', 'the', 'CAT', 'gene', 'was', 'driven', 'by', 'the', 'H', '-', '2Kb', '5', \"'\", 'enhancer', 'region', '(', 'H2TF1', '/', 'KBF1', 'site', ')', 'and', 'comparable', 'to', 'that', 'of', 'pRSVCAT', 'construct', 'carrying', 'the', 'strong', 'Rous', 'sarcoma', 'virus', 'LTR', 'enhancer', '.']\n",
            "68 68\n",
            "['sequence_0', 'WI_1', '-_2', '14920_3', 'is_4', 'in_5', 'fact_6', 'derived_7', 'from_8', 'the_9', '3_10', \"'_11\", '-_12', 'untranslated_13', 'region_14', 'of_15', 'the_16', 'human_17', 'PUNC_18', 'gene_19', '._20', 'Its_21', 'activity_22', 'was_23', 'twice_24', 'that_25', 'of_26', 'a_27', 'construct_28', 'where_29', 'the_30', 'CAT_31', 'gene_32', 'was_33', 'driven_34', 'by_35', 'the_36', 'H_37', '-_38', '2Kb_39', '5_40', \"'_41\", 'enhancer_42', 'region_43', '(_44', 'H2TF1_45', '/_46', 'KBF1_47', 'site_48', ')_49', 'and_50', 'comparable_51', 'to_52', 'that_53', 'of_54', 'pRSVCAT_55', 'construct_56', 'carrying_57', 'the_58', 'strong_59', 'Rous_60', 'sarcoma_61', 'virus_62', 'LTR_63', 'enhancer_64', '._65', 'ORF3_66', 'appears_67']\n",
            "BAD:  15698\n",
            "rpws:      index            token label\n",
            "0   15766               to     O\n",
            "1   15767           encode     O\n",
            "2   15768              the     O\n",
            "3   15769        homologue     O\n",
            "4   15770               of     O\n",
            "5   15771              the     O\n",
            "6   15772             well     O\n",
            "7   15773                -     O\n",
            "8   15774        conserved     O\n",
            "9   15775      proteasomal     B\n",
            "10  15776              26S     I\n",
            "11  15777       regulatory     I\n",
            "12  15778          subunit     I\n",
            "13  15779                .     O\n",
            "14  15780              The     O\n",
            "15  15781              tam     B\n",
            "16  15782                A     I\n",
            "17  15783             gene     I\n",
            "18  15784               of     I\n",
            "19  15785      Aspergillus     I\n",
            "20  15786         nidulans     I\n",
            "21  15787          encodes     O\n",
            "22  15788                a     O\n",
            "23  15789              739     O\n",
            "24  15790                -     O\n",
            "25  15791            amino     O\n",
            "26  15792             acid     O\n",
            "27  15793          protein     O\n",
            "28  15794             with     O\n",
            "29  15795       similarity     O\n",
            "30  15796               to     O\n",
            "31  15797           Uga35p     B\n",
            "32  15798                /     O\n",
            "33  15799           Dal81p     B\n",
            "34  15800                /     O\n",
            "35  15801            DurLp     B\n",
            "36  15802               of     O\n",
            "37  15803    Saccharomyces     O\n",
            "38  15804       cerevisiae     O\n",
            "39  15805                .     O\n",
            "40  15806           Little     O\n",
            "41  15807         honoured     O\n",
            "42  15808               in     O\n",
            "43  15809              his     O\n",
            "44  15810              own     O\n",
            "45  15811          country     O\n",
            "46  15812                :     O\n",
            "47  15813          statues     O\n",
            "48  15814               in     O\n",
            "49  15815      recognition     O\n",
            "50  15816               of     O\n",
            "51  15817           Edward     O\n",
            "52  15818           Jenner     O\n",
            "53  15819               MD     O\n",
            "54  15820              FRS     O\n",
            "55  15821                .     O\n",
            "56  15822                A     O\n",
            "57  15823  pharmacokinetic     O\n",
            "ORF3 appears to encode the homologue of the well - conserved proteasomal 26S regulatory subunit . The tam A gene of Aspergillus nidulans encodes a 739 - amino acid protein with similarity to Uga35p / Dal81p / DurLp of Saccharomyces cerevisiae . Little honoured in his own country : statues in recognition of Edward Jenner MD FRS .\n",
            "['ORF3', 'appears', 'to', 'encode', 'the', 'homologue', 'of', 'the', 'well', '-', 'conserved', 'proteasomal', '26S', 'regulatory', 'subunit', '.', 'The', 'tam', 'A', 'gene', 'of', 'Aspergillus', 'nidulans', 'encodes', 'a', '739', '-', 'amino', 'acid', 'protein', 'with', 'similarity', 'to', 'Uga35p', '/', 'Dal81p', '/', 'DurLp', 'of', 'Saccharomyces', 'cerevisiae', '.', 'Little', 'honoured', 'in', 'his', 'own', 'country', ':', 'statues', 'in', 'recognition', 'of', 'Edward', 'Jenner', 'MD', 'FRS', '.']\n",
            "58 58\n",
            "['to_0', 'encode_1', 'the_2', 'homologue_3', 'of_4', 'the_5', 'well_6', '-_7', 'conserved_8', 'proteasomal_9', '26S_10', 'regulatory_11', 'subunit_12', '._13', 'The_14', 'tam_15', 'A_16', 'gene_17', 'of_18', 'Aspergillus_19', 'nidulans_20', 'encodes_21', 'a_22', '739_23', '-_24', 'amino_25', 'acid_26', 'protein_27', 'with_28', 'similarity_29', 'to_30', 'Uga35p_31', '/_32', 'Dal81p_33', '/_34', 'DurLp_35', 'of_36', 'Saccharomyces_37', 'cerevisiae_38', '._39', 'Little_40', 'honoured_41', 'in_42', 'his_43', 'own_44', 'country_45', ':_46', 'statues_47', 'in_48', 'recognition_49', 'of_50', 'Edward_51', 'Jenner_52', 'MD_53', 'FRS_54', '._55', 'A_56', 'pharmacokinetic_57']\n",
            "BAD:  15766\n",
            "rpws:      index        token label\n",
            "0   15824  interaction     O\n",
            "1   15825      between     O\n",
            "2   15826     LY354740     O\n",
            "3   15827          and     O\n",
            "4   15828     diazepam     O\n",
            "..    ...          ...   ...\n",
            "60  15884        first     O\n",
            "61  15885       intron     O\n",
            "62  15886            .     O\n",
            "63  15887           No     O\n",
            "64  15888     promoter     O\n",
            "\n",
            "[65 rows x 3 columns]\n",
            "A pharmacokinetic interaction between LY354740 and diazepam , leading to the lowering of the plasma level of free diazepam , was also demonstrated . The promoter of this methyltransferase gene lacks an identifiable TATA box but is characterized by a CpG island which begins approximately 723 nucleotides upstream of the major transcriptional start site and extends through exon 1 and into the first intron .\n",
            "['A', 'pharmacokinetic', 'interaction', 'between', 'LY354740', 'and', 'diazepam', ',', 'leading', 'to', 'the', 'lowering', 'of', 'the', 'plasma', 'level', 'of', 'free', 'diazepam', ',', 'was', 'also', 'demonstrated', '.', 'The', 'promoter', 'of', 'this', 'methyltransferase', 'gene', 'lacks', 'an', 'identifiable', 'TATA', 'box', 'but', 'is', 'characterized', 'by', 'a', 'CpG', 'island', 'which', 'begins', 'approximately', '723', 'nucleotides', 'upstream', 'of', 'the', 'major', 'transcriptional', 'start', 'site', 'and', 'extends', 'through', 'exon', '1', 'and', 'into', 'the', 'first', 'intron', '.']\n",
            "65 65\n",
            "['interaction_0', 'between_1', 'LY354740_2', 'and_3', 'diazepam_4', ',_5', 'leading_6', 'to_7', 'the_8', 'lowering_9', 'of_10', 'the_11', 'plasma_12', 'level_13', 'of_14', 'free_15', 'diazepam_16', ',_17', 'was_18', 'also_19', 'demonstrated_20', '._21', 'The_22', 'promoter_23', 'of_24', 'this_25', 'methyltransferase_26', 'gene_27', 'lacks_28', 'an_29', 'identifiable_30', 'TATA_31', 'box_32', 'but_33', 'is_34', 'characterized_35', 'by_36', 'a_37', 'CpG_38', 'island_39', 'which_40', 'begins_41', 'approximately_42', '723_43', 'nucleotides_44', 'upstream_45', 'of_46', 'the_47', 'major_48', 'transcriptional_49', 'start_50', 'site_51', 'and_52', 'extends_53', 'through_54', 'exon_55', '1_56', 'and_57', 'into_58', 'the_59', 'first_60', 'intron_61', '._62', 'No_63', 'promoter_64']\n",
            "BAD:  15824\n",
            "rpws:      index     token label\n",
            "0   15889  activity     O\n",
            "1   15890     could     O\n",
            "2   15891        be     O\n",
            "3   15892  detected     O\n",
            "4   15893      with     O\n",
            "..    ...       ...   ...\n",
            "75  15964       Rho     B\n",
            "76  15965  proteins     I\n",
            "77  15966         .     O\n",
            "78  15967        An     O\n",
            "79  15968        81     O\n",
            "\n",
            "[80 rows x 3 columns]\n",
            "No promoter activity could be detected with various mouse Fli - 1 promoter - CAT constructs containing 600 bp of the 5 ' flanking region , the complete exon 1 , the 5 ' end of intron 1 and / or retroviral LTR sequence . The D . discoideum proteins were entirely conserved over the four regions known to be important for GTP binding and all contained the C - terminal CAAX aa motifs shared by other Rho proteins .\n",
            "['No', 'promoter', 'activity', 'could', 'be', 'detected', 'with', 'various', 'mouse', 'Fli', '-', '1', 'promoter', '-', 'CAT', 'constructs', 'containing', '600', 'bp', 'of', 'the', '5', \"'\", 'flanking', 'region', ',', 'the', 'complete', 'exon', '1', ',', 'the', '5', \"'\", 'end', 'of', 'intron', '1', 'and', '/', 'or', 'retroviral', 'LTR', 'sequence', '.', 'The', 'D', '.', 'discoideum', 'proteins', 'were', 'entirely', 'conserved', 'over', 'the', 'four', 'regions', 'known', 'to', 'be', 'important', 'for', 'GTP', 'binding', 'and', 'all', 'contained', 'the', 'C', '-', 'terminal', 'CAAX', 'aa', 'motifs', 'shared', 'by', 'other', 'Rho', 'proteins', '.']\n",
            "80 80\n",
            "['activity_0', 'could_1', 'be_2', 'detected_3', 'with_4', 'various_5', 'mouse_6', 'Fli_7', '-_8', '1_9', 'promoter_10', '-_11', 'CAT_12', 'constructs_13', 'containing_14', '600_15', 'bp_16', 'of_17', 'the_18', '5_19', \"'_20\", 'flanking_21', 'region_22', ',_23', 'the_24', 'complete_25', 'exon_26', '1_27', ',_28', 'the_29', '5_30', \"'_31\", 'end_32', 'of_33', 'intron_34', '1_35', 'and_36', '/_37', 'or_38', 'retroviral_39', 'LTR_40', 'sequence_41', '._42', 'The_43', 'D_44', '._45', 'discoideum_46', 'proteins_47', 'were_48', 'entirely_49', 'conserved_50', 'over_51', 'the_52', 'four_53', 'regions_54', 'known_55', 'to_56', 'be_57', 'important_58', 'for_59', 'GTP_60', 'binding_61', 'and_62', 'all_63', 'contained_64', 'the_65', 'C_66', '-_67', 'terminal_68', 'CAAX_69', 'aa_70', 'motifs_71', 'shared_72', 'by_73', 'other_74', 'Rho_75', 'proteins_76', '._77', 'An_78', '81_79']\n",
            "BAD:  15889\n",
            "rpws:      index        token label\n",
            "0   15969            -     O\n",
            "1   15970           nt     O\n",
            "2   15971       tandem     O\n",
            "3   15972  duplication     O\n",
            "4   15973           of     O\n",
            "..    ...          ...   ...\n",
            "68  16037          the     O\n",
            "69  16038        1980s     O\n",
            "70  16039            .     O\n",
            "71  16040          The     O\n",
            "72  16041       method     O\n",
            "\n",
            "[73 rows x 3 columns]\n",
            "An 81 - nt tandem duplication of the C - terminal coding region is located adjacent to the termination codon of the CyIIIb gene , a potential relic of a slipped mispairing and replication event . Only 53 % of all injuries and 12 . 5 % of serious injuries involved children , and in contrast to the 1950s and early 1960s , young adults appear at greatest risk in the 1980s .\n",
            "['An', '81', '-', 'nt', 'tandem', 'duplication', 'of', 'the', 'C', '-', 'terminal', 'coding', 'region', 'is', 'located', 'adjacent', 'to', 'the', 'termination', 'codon', 'of', 'the', 'CyIIIb', 'gene', ',', 'a', 'potential', 'relic', 'of', 'a', 'slipped', 'mispairing', 'and', 'replication', 'event', '.', 'Only', '53', '%', 'of', 'all', 'injuries', 'and', '12', '.', '5', '%', 'of', 'serious', 'injuries', 'involved', 'children', ',', 'and', 'in', 'contrast', 'to', 'the', '1950s', 'and', 'early', '1960s', ',', 'young', 'adults', 'appear', 'at', 'greatest', 'risk', 'in', 'the', '1980s', '.']\n",
            "73 73\n",
            "['-_0', 'nt_1', 'tandem_2', 'duplication_3', 'of_4', 'the_5', 'C_6', '-_7', 'terminal_8', 'coding_9', 'region_10', 'is_11', 'located_12', 'adjacent_13', 'to_14', 'the_15', 'termination_16', 'codon_17', 'of_18', 'the_19', 'CyIIIb_20', 'gene_21', ',_22', 'a_23', 'potential_24', 'relic_25', 'of_26', 'a_27', 'slipped_28', 'mispairing_29', 'and_30', 'replication_31', 'event_32', '._33', 'Only_34', '53_35', '%_36', 'of_37', 'all_38', 'injuries_39', 'and_40', '12_41', '._42', '5_43', '%_44', 'of_45', 'serious_46', 'injuries_47', 'involved_48', 'children_49', ',_50', 'and_51', 'in_52', 'contrast_53', 'to_54', 'the_55', '1950s_56', 'and_57', 'early_58', '1960s_59', ',_60', 'young_61', 'adults_62', 'appear_63', 'at_64', 'greatest_65', 'risk_66', 'in_67', 'the_68', '1980s_69', '._70', 'The_71', 'method_72']\n",
            "BAD:  15969\n",
            "rpws:      index       token label\n",
            "0   16042         was     O\n",
            "1   16043      tested     O\n",
            "2   16044          on     O\n",
            "3   16045           7     O\n",
            "4   16046        rats     O\n",
            "..    ...         ...   ...\n",
            "62  16104       human     O\n",
            "63  16105      dentin     O\n",
            "64  16106           .     O\n",
            "65  16107       Mouse     B\n",
            "66  16108  macrophage     I\n",
            "\n",
            "[67 rows x 3 columns]\n",
            "The method was tested on 7 rats with 48 - hr - old myocardial infarction and 4 sham - operated controls . Here we examine the population of gammaS transcripts in adult human lens and the structure of the human CRYGS genes . The slope of QD versus QEMF for the four tubes was near unity . The extent of the odontoblast process in human dentin .\n",
            "['The', 'method', 'was', 'tested', 'on', '7', 'rats', 'with', '48', '-', 'hr', '-', 'old', 'myocardial', 'infarction', 'and', '4', 'sham', '-', 'operated', 'controls', '.', 'Here', 'we', 'examine', 'the', 'population', 'of', 'gammaS', 'transcripts', 'in', 'adult', 'human', 'lens', 'and', 'the', 'structure', 'of', 'the', 'human', 'CRYGS', 'genes', '.', 'The', 'slope', 'of', 'QD', 'versus', 'QEMF', 'for', 'the', 'four', 'tubes', 'was', 'near', 'unity', '.', 'The', 'extent', 'of', 'the', 'odontoblast', 'process', 'in', 'human', 'dentin', '.']\n",
            "67 67\n",
            "['was_0', 'tested_1', 'on_2', '7_3', 'rats_4', 'with_5', '48_6', '-_7', 'hr_8', '-_9', 'old_10', 'myocardial_11', 'infarction_12', 'and_13', '4_14', 'sham_15', '-_16', 'operated_17', 'controls_18', '._19', 'Here_20', 'we_21', 'examine_22', 'the_23', 'population_24', 'of_25', 'gammaS_26', 'transcripts_27', 'in_28', 'adult_29', 'human_30', 'lens_31', 'and_32', 'the_33', 'structure_34', 'of_35', 'the_36', 'human_37', 'CRYGS_38', 'genes_39', '._40', 'The_41', 'slope_42', 'of_43', 'QD_44', 'versus_45', 'QEMF_46', 'for_47', 'the_48', 'four_49', 'tubes_50', 'was_51', 'near_52', 'unity_53', '._54', 'The_55', 'extent_56', 'of_57', 'the_58', 'odontoblast_59', 'process_60', 'in_61', 'human_62', 'dentin_63', '._64', 'Mouse_65', 'macrophage_66']\n",
            "BAD:  16042\n",
            "rpws:      index           token label\n",
            "0   16109            beta     I\n",
            "1   16110         subunit     I\n",
            "2   16111               (     I\n",
            "3   16112           CD11b     I\n",
            "4   16113               )     I\n",
            "5   16114            cDNA     I\n",
            "6   16115             for     O\n",
            "7   16116             the     O\n",
            "8   16117             CR3     B\n",
            "9   16118      complement     I\n",
            "10  16119        receptor     I\n",
            "11  16120               /     O\n",
            "12  16121             Mac     B\n",
            "13  16122               -     I\n",
            "14  16123               1     I\n",
            "15  16124         antigen     I\n",
            "16  16125               .     O\n",
            "17  16126        Enoxacin     O\n",
            "18  16127             has     O\n",
            "19  16128            been     O\n",
            "20  16129           shown     O\n",
            "21  16130              to     O\n",
            "22  16131              be     O\n",
            "23  16132              an     O\n",
            "24  16133       effective     O\n",
            "25  16134            well     O\n",
            "26  16135       tolerated     O\n",
            "27  16136             and     O\n",
            "28  16137      convenient     O\n",
            "29  16138       treatment     O\n",
            "30  16139             for     O\n",
            "31  16140      gonorrhoea     O\n",
            "32  16141               .     O\n",
            "33  16142              We     O\n",
            "34  16143        assessed     O\n",
            "35  16144  cardiovascular     O\n",
            "36  16145       variables     O\n",
            "37  16146             and     O\n",
            "38  16147           blood     O\n",
            "39  16148              O2     O\n",
            "40  16149        contents     O\n",
            "41  16150              in     O\n",
            "42  16151           order     O\n",
            "43  16152              to     O\n",
            "44  16153    characterize     O\n",
            "45  16154              O2     O\n",
            "46  16155       transport     O\n",
            "47  16156              in     O\n",
            "48  16157          ponies     O\n",
            "49  16158          during     O\n",
            "50  16159       treadmill     O\n",
            "51  16160        exercise     O\n",
            "52  16161               .     O\n",
            "53  16162             The     O\n",
            "54  16163        findings     O\n",
            "Mouse macrophage beta subunit ( CD11b ) cDNA for the CR3 complement receptor / Mac - 1 antigen . Enoxacin has been shown to be an effective well tolerated and convenient treatment for gonorrhoea . We assessed cardiovascular variables and blood O2 contents in order to characterize O2 transport in ponies during treadmill exercise .\n",
            "['Mouse', 'macrophage', 'beta', 'subunit', '(', 'CD11b', ')', 'cDNA', 'for', 'the', 'CR3', 'complement', 'receptor', '/', 'Mac', '-', '1', 'antigen', '.', 'Enoxacin', 'has', 'been', 'shown', 'to', 'be', 'an', 'effective', 'well', 'tolerated', 'and', 'convenient', 'treatment', 'for', 'gonorrhoea', '.', 'We', 'assessed', 'cardiovascular', 'variables', 'and', 'blood', 'O2', 'contents', 'in', 'order', 'to', 'characterize', 'O2', 'transport', 'in', 'ponies', 'during', 'treadmill', 'exercise', '.']\n",
            "55 55\n",
            "['beta_0', 'subunit_1', '(_2', 'CD11b_3', ')_4', 'cDNA_5', 'for_6', 'the_7', 'CR3_8', 'complement_9', 'receptor_10', '/_11', 'Mac_12', '-_13', '1_14', 'antigen_15', '._16', 'Enoxacin_17', 'has_18', 'been_19', 'shown_20', 'to_21', 'be_22', 'an_23', 'effective_24', 'well_25', 'tolerated_26', 'and_27', 'convenient_28', 'treatment_29', 'for_30', 'gonorrhoea_31', '._32', 'We_33', 'assessed_34', 'cardiovascular_35', 'variables_36', 'and_37', 'blood_38', 'O2_39', 'contents_40', 'in_41', 'order_42', 'to_43', 'characterize_44', 'O2_45', 'transport_46', 'in_47', 'ponies_48', 'during_49', 'treadmill_50', 'exercise_51', '._52', 'The_53', 'findings_54']\n",
            "BAD:  16109\n",
            "rpws:      index            token label\n",
            "0   16164              are     O\n",
            "1   16165        discussed     O\n",
            "2   16166             with     O\n",
            "3   16167        reference     O\n",
            "4   16168               to     O\n",
            "5   16169         possible     O\n",
            "6   16170       mechanisms     O\n",
            "7   16171               by     O\n",
            "8   16172            which     O\n",
            "9   16173         parasite     O\n",
            "10  16174      development     O\n",
            "11  16175            might     O\n",
            "12  16176               be     O\n",
            "13  16177       controlled     O\n",
            "14  16178                .     O\n",
            "15  16179              The     O\n",
            "16  16180               80     O\n",
            "17  16181               kb     O\n",
            "18  16182        preceding     O\n",
            "19  16183              the     O\n",
            "20  16184       expression     O\n",
            "21  16185             site     O\n",
            "22  16186              has     O\n",
            "23  16187              few     O\n",
            "24  16188                ,     O\n",
            "25  16189               if     O\n",
            "26  16190              any     O\n",
            "27  16191                ,     O\n",
            "28  16192       functional     O\n",
            "29  16193             ORFs     O\n",
            "30  16194                ,     O\n",
            "31  16195              but     O\n",
            "32  16196         contains     O\n",
            "33  16197               50     O\n",
            "34  16198               bp     O\n",
            "35  16199          repeats     O\n",
            "36  16200                ,     O\n",
            "37  16201             INGI     B\n",
            "38  16202  retrotransposon     I\n",
            "39  16203                -     I\n",
            "40  16204             like     I\n",
            "41  16205         elements     I\n",
            "42  16206                ,     O\n",
            "43  16207              and     O\n",
            "44  16208            novel     O\n",
            "45  16209                4     O\n",
            "46  16210                -     O\n",
            "47  16211               12     O\n",
            "48  16212               kb     O\n",
            "49  16213          repeats     O\n",
            "50  16214            found     O\n",
            "51  16215             near     O\n",
            "52  16216            other     O\n",
            "53  16217        telomeres     O\n",
            "54  16218                .     O\n",
            "55  16219              The     O\n",
            "56  16220        predicted     O\n",
            "The findings are discussed with reference to possible mechanisms by which parasite development might be controlled . The 80 kb preceding the expression site has few , if any , functional ORFs , but contains 50 bp repeats , INGI retrotransposon - like elements , and novel 4 - 12 kb repeats found near other telomeres .\n",
            "['The', 'findings', 'are', 'discussed', 'with', 'reference', 'to', 'possible', 'mechanisms', 'by', 'which', 'parasite', 'development', 'might', 'be', 'controlled', '.', 'The', '80', 'kb', 'preceding', 'the', 'expression', 'site', 'has', 'few', ',', 'if', 'any', ',', 'functional', 'ORFs', ',', 'but', 'contains', '50', 'bp', 'repeats', ',', 'INGI', 'retrotransposon', '-', 'like', 'elements', ',', 'and', 'novel', '4', '-', '12', 'kb', 'repeats', 'found', 'near', 'other', 'telomeres', '.']\n",
            "57 57\n",
            "['are_0', 'discussed_1', 'with_2', 'reference_3', 'to_4', 'possible_5', 'mechanisms_6', 'by_7', 'which_8', 'parasite_9', 'development_10', 'might_11', 'be_12', 'controlled_13', '._14', 'The_15', '80_16', 'kb_17', 'preceding_18', 'the_19', 'expression_20', 'site_21', 'has_22', 'few_23', ',_24', 'if_25', 'any_26', ',_27', 'functional_28', 'ORFs_29', ',_30', 'but_31', 'contains_32', '50_33', 'bp_34', 'repeats_35', ',_36', 'INGI_37', 'retrotransposon_38', '-_39', 'like_40', 'elements_41', ',_42', 'and_43', 'novel_44', '4_45', '-_46', '12_47', 'kb_48', 'repeats_49', 'found_50', 'near_51', 'other_52', 'telomeres_53', '._54', 'The_55', 'predicted_56']\n",
            "BAD:  16164\n",
            "rpws:      index              token label\n",
            "0   16221           proteins     O\n",
            "1   16222               show     O\n",
            "2   16223             strong     O\n",
            "3   16224           homology     O\n",
            "4   16225                 to     O\n",
            "5   16226                 an     O\n",
            "6   16227                ABA     B\n",
            "7   16228                  -     I\n",
            "8   16229          inducible     I\n",
            "9   16230            glycine     I\n",
            "10  16231                  -     I\n",
            "11  16232               rich     I\n",
            "12  16233            protein     I\n",
            "13  16234               from     O\n",
            "14  16235              maize     O\n",
            "15  16236            embryos     O\n",
            "16  16237                and     O\n",
            "17  16238                 to     O\n",
            "18  16239                the     O\n",
            "19  16240          mammalian     B\n",
            "20  16241                RNA     I\n",
            "21  16242                  -     I\n",
            "22  16243            binding     I\n",
            "23  16244            protein     I\n",
            "24  16245                 A1     I\n",
            "25  16246                 of     O\n",
            "26  16247                the     O\n",
            "27  16248      heterogeneous     O\n",
            "28  16249            nuclear     O\n",
            "29  16250  ribonucleoprotein     O\n",
            "30  16251            complex     O\n",
            "31  16252           involved     O\n",
            "32  16253                 in     O\n",
            "33  16254                pre     O\n",
            "34  16255                  -     O\n",
            "35  16256               mRNA     O\n",
            "36  16257           splicing     O\n",
            "37  16258                  .     O\n",
            "38  16259           Increase     O\n",
            "39  16260                 in     O\n",
            "40  16261                  [     O\n",
            "41  16262                 Ca     O\n",
            "42  16263                  +     O\n",
            "43  16264                  +     O\n",
            "44  16265                  ]     O\n",
            "45  16266       counteracted     O\n",
            "46  16267                the     O\n",
            "47  16268            effects     O\n",
            "48  16269                 of     O\n",
            "49  16270          verapamil     O\n",
            "50  16271                 on     O\n",
            "51  16272                RAD     O\n",
            "52  16273                  .     O\n",
            "53  16274                 We     O\n",
            "54  16275           compared     O\n",
            "The predicted proteins show strong homology to an ABA - inducible glycine - rich protein from maize embryos and to the mammalian RNA - binding protein A1 of the heterogeneous nuclear ribonucleoprotein complex involved in pre - mRNA splicing . Increase in [ Ca + + ] counteracted the effects of verapamil on RAD .\n",
            "['The', 'predicted', 'proteins', 'show', 'strong', 'homology', 'to', 'an', 'ABA', '-', 'inducible', 'glycine', '-', 'rich', 'protein', 'from', 'maize', 'embryos', 'and', 'to', 'the', 'mammalian', 'RNA', '-', 'binding', 'protein', 'A1', 'of', 'the', 'heterogeneous', 'nuclear', 'ribonucleoprotein', 'complex', 'involved', 'in', 'pre', '-', 'mRNA', 'splicing', '.', 'Increase', 'in', '[', 'Ca', '+', '+', ']', 'counteracted', 'the', 'effects', 'of', 'verapamil', 'on', 'RAD', '.']\n",
            "55 55\n",
            "['proteins_0', 'show_1', 'strong_2', 'homology_3', 'to_4', 'an_5', 'ABA_6', '-_7', 'inducible_8', 'glycine_9', '-_10', 'rich_11', 'protein_12', 'from_13', 'maize_14', 'embryos_15', 'and_16', 'to_17', 'the_18', 'mammalian_19', 'RNA_20', '-_21', 'binding_22', 'protein_23', 'A1_24', 'of_25', 'the_26', 'heterogeneous_27', 'nuclear_28', 'ribonucleoprotein_29', 'complex_30', 'involved_31', 'in_32', 'pre_33', '-_34', 'mRNA_35', 'splicing_36', '._37', 'Increase_38', 'in_39', '[_40', 'Ca_41', '+_42', '+_43', ']_44', 'counteracted_45', 'the_46', 'effects_47', 'of_48', 'verapamil_49', 'on_50', 'RAD_51', '._52', 'We_53', 'compared_54']\n",
            "BAD:  16221\n",
            "rpws:      index         token label\n",
            "0   16276           the     O\n",
            "1   16277    quantities     O\n",
            "2   16278     collected     O\n",
            "3   16279          with     O\n",
            "4   16280          both     O\n",
            "..    ...           ...   ...\n",
            "91  16367  localization     O\n",
            "92  16368        signal     O\n",
            "93  16369             .     O\n",
            "94  16370  Surprisingly     O\n",
            "95  16371             ,     O\n",
            "\n",
            "[96 rows x 3 columns]\n",
            "We compared the quantities collected with both pollen traps during February , March and April 1988 and 1989 . Cbl is a signaling molecule with multiple functional domains : an SH2 domain which binds phosphotyrosine residues , a RING finger domain which acts as a ubiquitin ligase , a proline - rich region which serves as a docking site for SH3 - containing proteins , phosphotyrosine residues which serve as docking sites for SH2 - containing proteins such a CrkL and the p85 subunit of PI 3 - kinase , and a nuclear localization signal .\n",
            "['We', 'compared', 'the', 'quantities', 'collected', 'with', 'both', 'pollen', 'traps', 'during', 'February', ',', 'March', 'and', 'April', '1988', 'and', '1989', '.', 'Cbl', 'is', 'a', 'signaling', 'molecule', 'with', 'multiple', 'functional', 'domains', ':', 'an', 'SH2', 'domain', 'which', 'binds', 'phosphotyrosine', 'residues', ',', 'a', 'RING', 'finger', 'domain', 'which', 'acts', 'as', 'a', 'ubiquitin', 'ligase', ',', 'a', 'proline', '-', 'rich', 'region', 'which', 'serves', 'as', 'a', 'docking', 'site', 'for', 'SH3', '-', 'containing', 'proteins', ',', 'phosphotyrosine', 'residues', 'which', 'serve', 'as', 'docking', 'sites', 'for', 'SH2', '-', 'containing', 'proteins', 'such', 'a', 'CrkL', 'and', 'the', 'p85', 'subunit', 'of', 'PI', '3', '-', 'kinase', ',', 'and', 'a', 'nuclear', 'localization', 'signal', '.']\n",
            "96 96\n",
            "['the_0', 'quantities_1', 'collected_2', 'with_3', 'both_4', 'pollen_5', 'traps_6', 'during_7', 'February_8', ',_9', 'March_10', 'and_11', 'April_12', '1988_13', 'and_14', '1989_15', '._16', 'Cbl_17', 'is_18', 'a_19', 'signaling_20', 'molecule_21', 'with_22', 'multiple_23', 'functional_24', 'domains_25', ':_26', 'an_27', 'SH2_28', 'domain_29', 'which_30', 'binds_31', 'phosphotyrosine_32', 'residues_33', ',_34', 'a_35', 'RING_36', 'finger_37', 'domain_38', 'which_39', 'acts_40', 'as_41', 'a_42', 'ubiquitin_43', 'ligase_44', ',_45', 'a_46', 'proline_47', '-_48', 'rich_49', 'region_50', 'which_51', 'serves_52', 'as_53', 'a_54', 'docking_55', 'site_56', 'for_57', 'SH3_58', '-_59', 'containing_60', 'proteins_61', ',_62', 'phosphotyrosine_63', 'residues_64', 'which_65', 'serve_66', 'as_67', 'docking_68', 'sites_69', 'for_70', 'SH2_71', '-_72', 'containing_73', 'proteins_74', 'such_75', 'a_76', 'CrkL_77', 'and_78', 'the_79', 'p85_80', 'subunit_81', 'of_82', 'PI_83', '3_84', '-_85', 'kinase_86', ',_87', 'and_88', 'a_89', 'nuclear_90', 'localization_91', 'signal_92', '._93', 'Surprisingly_94', ',_95']\n",
            "BAD:  16276\n",
            "rpws:      index        token label\n",
            "0   16372          apo     B\n",
            "1   16373            -     I\n",
            "2   16374          iso     I\n",
            "3   16375            -     I\n",
            "4   16376            1     I\n",
            "..    ...          ...   ...\n",
            "74  16446           XY     O\n",
            "75  16447        model     O\n",
            "76  16448            .     O\n",
            "77  16449          The     O\n",
            "78  16450  ventricular     O\n",
            "\n",
            "[79 rows x 3 columns]\n",
            "Surprisingly , apo - iso - 1 - cytochrome c is absent in cyc3 - strains , although apo - iso - 2 - cytochrome c is present at approximately the same level at which holo - iso - 2 - cytochrome c is found in CYC3 + strains . Here , we identify the analytical form of the PDF of one such measure , the order parameter in the low temperature phase of the 2D XY model .\n",
            "['Surprisingly', ',', 'apo', '-', 'iso', '-', '1', '-', 'cytochrome', 'c', 'is', 'absent', 'in', 'cyc3', '-', 'strains', ',', 'although', 'apo', '-', 'iso', '-', '2', '-', 'cytochrome', 'c', 'is', 'present', 'at', 'approximately', 'the', 'same', 'level', 'at', 'which', 'holo', '-', 'iso', '-', '2', '-', 'cytochrome', 'c', 'is', 'found', 'in', 'CYC3', '+', 'strains', '.', 'Here', ',', 'we', 'identify', 'the', 'analytical', 'form', 'of', 'the', 'PDF', 'of', 'one', 'such', 'measure', ',', 'the', 'order', 'parameter', 'in', 'the', 'low', 'temperature', 'phase', 'of', 'the', '2D', 'XY', 'model', '.']\n",
            "79 79\n",
            "['apo_0', '-_1', 'iso_2', '-_3', '1_4', '-_5', 'cytochrome_6', 'c_7', 'is_8', 'absent_9', 'in_10', 'cyc3_11', '-_12', 'strains_13', ',_14', 'although_15', 'apo_16', '-_17', 'iso_18', '-_19', '2_20', '-_21', 'cytochrome_22', 'c_23', 'is_24', 'present_25', 'at_26', 'approximately_27', 'the_28', 'same_29', 'level_30', 'at_31', 'which_32', 'holo_33', '-_34', 'iso_35', '-_36', '2_37', '-_38', 'cytochrome_39', 'c_40', 'is_41', 'found_42', 'in_43', 'CYC3_44', '+_45', 'strains_46', '._47', 'Here_48', ',_49', 'we_50', 'identify_51', 'the_52', 'analytical_53', 'form_54', 'of_55', 'the_56', 'PDF_57', 'of_58', 'one_59', 'such_60', 'measure_61', ',_62', 'the_63', 'order_64', 'parameter_65', 'in_66', 'the_67', 'low_68', 'temperature_69', 'phase_70', 'of_71', 'the_72', '2D_73', 'XY_74', 'model_75', '._76', 'The_77', 'ventricular_78']\n",
            "BAD:  16372\n",
            "rpws:      index      token label\n",
            "0   16451     demand     O\n",
            "1   16452  pacemaker     O\n",
            "2   16453          (     O\n",
            "3   16454        VVI     O\n",
            "4   16455          )     O\n",
            "..    ...        ...   ...\n",
            "70  16521          -     O\n",
            "71  16522     eaters     O\n",
            "72  16523          .     O\n",
            "73  16524        Our     O\n",
            "74  16525       data     O\n",
            "\n",
            "[75 rows x 3 columns]\n",
            "The ventricular demand pacemaker ( VVI ) was implanted three years ago and because of further impairment of cardiac performance an av - sequential pacemaker ( DDD ) was used to restore atrio - ventricular synchronisation . It was predicted that the Stroop task would trigger greater consumption of ice cream than a fearful film , and that this effect would be more pronounced for binge - eaters than non - binge - eaters .\n",
            "['The', 'ventricular', 'demand', 'pacemaker', '(', 'VVI', ')', 'was', 'implanted', 'three', 'years', 'ago', 'and', 'because', 'of', 'further', 'impairment', 'of', 'cardiac', 'performance', 'an', 'av', '-', 'sequential', 'pacemaker', '(', 'DDD', ')', 'was', 'used', 'to', 'restore', 'atrio', '-', 'ventricular', 'synchronisation', '.', 'It', 'was', 'predicted', 'that', 'the', 'Stroop', 'task', 'would', 'trigger', 'greater', 'consumption', 'of', 'ice', 'cream', 'than', 'a', 'fearful', 'film', ',', 'and', 'that', 'this', 'effect', 'would', 'be', 'more', 'pronounced', 'for', 'binge', '-', 'eaters', 'than', 'non', '-', 'binge', '-', 'eaters', '.']\n",
            "75 75\n",
            "['demand_0', 'pacemaker_1', '(_2', 'VVI_3', ')_4', 'was_5', 'implanted_6', 'three_7', 'years_8', 'ago_9', 'and_10', 'because_11', 'of_12', 'further_13', 'impairment_14', 'of_15', 'cardiac_16', 'performance_17', 'an_18', 'av_19', '-_20', 'sequential_21', 'pacemaker_22', '(_23', 'DDD_24', ')_25', 'was_26', 'used_27', 'to_28', 'restore_29', 'atrio_30', '-_31', 'ventricular_32', 'synchronisation_33', '._34', 'It_35', 'was_36', 'predicted_37', 'that_38', 'the_39', 'Stroop_40', 'task_41', 'would_42', 'trigger_43', 'greater_44', 'consumption_45', 'of_46', 'ice_47', 'cream_48', 'than_49', 'a_50', 'fearful_51', 'film_52', ',_53', 'and_54', 'that_55', 'this_56', 'effect_57', 'would_58', 'be_59', 'more_60', 'pronounced_61', 'for_62', 'binge_63', '-_64', 'eaters_65', 'than_66', 'non_67', '-_68', 'binge_69', '-_70', 'eaters_71', '._72', 'Our_73', 'data_74']\n",
            "BAD:  16451\n",
            "rpws:      index         token label\n",
            "0   16526           are     O\n",
            "1   16527            in     O\n",
            "2   16528          line     O\n",
            "3   16529          with     O\n",
            "4   16530           the     O\n",
            "..    ...           ...   ...\n",
            "68  16594     Cognitive     O\n",
            "69  16595      Subscale     O\n",
            "70  16596             .     O\n",
            "71  16597  Colonization     O\n",
            "72  16598     increased     O\n",
            "\n",
            "[73 rows x 3 columns]\n",
            "Our data are in line with the hypothesis that E2F functions as a growth - and cell cycle regulated tethering factor between Sp1 and the basic transcription machinery . There was a positive correlation between change from baseline in parietal lobe gray - matter cytosolic choline , expressed in terms of choline / creatine resonance ratios , and cognitive performance as measured with the Alzheimer ' s Disease Assessment Scale Cognitive Subscale .\n",
            "['Our', 'data', 'are', 'in', 'line', 'with', 'the', 'hypothesis', 'that', 'E2F', 'functions', 'as', 'a', 'growth', '-', 'and', 'cell', 'cycle', 'regulated', 'tethering', 'factor', 'between', 'Sp1', 'and', 'the', 'basic', 'transcription', 'machinery', '.', 'There', 'was', 'a', 'positive', 'correlation', 'between', 'change', 'from', 'baseline', 'in', 'parietal', 'lobe', 'gray', '-', 'matter', 'cytosolic', 'choline', ',', 'expressed', 'in', 'terms', 'of', 'choline', '/', 'creatine', 'resonance', 'ratios', ',', 'and', 'cognitive', 'performance', 'as', 'measured', 'with', 'the', 'Alzheimer', \"'\", 's', 'Disease', 'Assessment', 'Scale', 'Cognitive', 'Subscale', '.']\n",
            "73 73\n",
            "['are_0', 'in_1', 'line_2', 'with_3', 'the_4', 'hypothesis_5', 'that_6', 'E2F_7', 'functions_8', 'as_9', 'a_10', 'growth_11', '-_12', 'and_13', 'cell_14', 'cycle_15', 'regulated_16', 'tethering_17', 'factor_18', 'between_19', 'Sp1_20', 'and_21', 'the_22', 'basic_23', 'transcription_24', 'machinery_25', '._26', 'There_27', 'was_28', 'a_29', 'positive_30', 'correlation_31', 'between_32', 'change_33', 'from_34', 'baseline_35', 'in_36', 'parietal_37', 'lobe_38', 'gray_39', '-_40', 'matter_41', 'cytosolic_42', 'choline_43', ',_44', 'expressed_45', 'in_46', 'terms_47', 'of_48', 'choline_49', '/_50', 'creatine_51', 'resonance_52', 'ratios_53', ',_54', 'and_55', 'cognitive_56', 'performance_57', 'as_58', 'measured_59', 'with_60', 'the_61', 'Alzheimer_62', \"'_63\", 's_64', 'Disease_65', 'Assessment_66', 'Scale_67', 'Cognitive_68', 'Subscale_69', '._70', 'Colonization_71', 'increased_72']\n",
            "BAD:  16526\n",
            "rpws:      index          token label\n",
            "0   16599           with     O\n",
            "1   16600          level     O\n",
            "2   16601             of     O\n",
            "3   16602           care     O\n",
            "4   16603              :     O\n",
            "5   16604           from     O\n",
            "6   16605              9     O\n",
            "7   16606            per     O\n",
            "8   16607           cent     O\n",
            "9   16608             in     O\n",
            "10  16609    independent     O\n",
            "11  16610      residents     O\n",
            "12  16611             of     O\n",
            "13  16612     apartments     O\n",
            "14  16613             to     O\n",
            "15  16614             60     O\n",
            "16  16615            per     O\n",
            "17  16616           cent     O\n",
            "18  16617             in     O\n",
            "19  16618       patients     O\n",
            "20  16619             on     O\n",
            "21  16620             an     O\n",
            "22  16621          acute     O\n",
            "23  16622       hospital     O\n",
            "24  16623           ward     O\n",
            "25  16624              (     O\n",
            "26  16625              P     O\n",
            "27  16626           less     O\n",
            "28  16627           than     O\n",
            "29  16628              0     O\n",
            "30  16629              .     O\n",
            "31  16630           0001     O\n",
            "32  16631              )     O\n",
            "33  16632              .     O\n",
            "34  16633           IRA2     B\n",
            "35  16634              ,     O\n",
            "36  16635              a     O\n",
            "37  16636         second     O\n",
            "38  16637           gene     O\n",
            "39  16638             of     O\n",
            "40  16639  Saccharomyces     O\n",
            "41  16640     cerevisiae     O\n",
            "42  16641           that     O\n",
            "43  16642        encodes     O\n",
            "44  16643              a     O\n",
            "45  16644        protein     O\n",
            "46  16645           with     O\n",
            "47  16646              a     O\n",
            "48  16647         domain     O\n",
            "49  16648     homologous     O\n",
            "50  16649             to     O\n",
            "51  16650      mammalian     O\n",
            "52  16651            ras     B\n",
            "53  16652         GTPase     I\n",
            "54  16653              -     I\n",
            "55  16654     activating     I\n",
            "56  16655        protein     I\n",
            "57  16656              .     O\n",
            "58  16657           Exon     O\n",
            "59  16658            A1a     O\n",
            "Colonization increased with level of care : from 9 per cent in independent residents of apartments to 60 per cent in patients on an acute hospital ward ( P less than 0 . 0001 ) . IRA2 , a second gene of Saccharomyces cerevisiae that encodes a protein with a domain homologous to mammalian ras GTPase - activating protein .\n",
            "['Colonization', 'increased', 'with', 'level', 'of', 'care', ':', 'from', '9', 'per', 'cent', 'in', 'independent', 'residents', 'of', 'apartments', 'to', '60', 'per', 'cent', 'in', 'patients', 'on', 'an', 'acute', 'hospital', 'ward', '(', 'P', 'less', 'than', '0', '.', '0001', ')', '.', 'IRA2', ',', 'a', 'second', 'gene', 'of', 'Saccharomyces', 'cerevisiae', 'that', 'encodes', 'a', 'protein', 'with', 'a', 'domain', 'homologous', 'to', 'mammalian', 'ras', 'GTPase', '-', 'activating', 'protein', '.']\n",
            "60 60\n",
            "['with_0', 'level_1', 'of_2', 'care_3', ':_4', 'from_5', '9_6', 'per_7', 'cent_8', 'in_9', 'independent_10', 'residents_11', 'of_12', 'apartments_13', 'to_14', '60_15', 'per_16', 'cent_17', 'in_18', 'patients_19', 'on_20', 'an_21', 'acute_22', 'hospital_23', 'ward_24', '(_25', 'P_26', 'less_27', 'than_28', '0_29', '._30', '0001_31', ')_32', '._33', 'IRA2_34', ',_35', 'a_36', 'second_37', 'gene_38', 'of_39', 'Saccharomyces_40', 'cerevisiae_41', 'that_42', 'encodes_43', 'a_44', 'protein_45', 'with_46', 'a_47', 'domain_48', 'homologous_49', 'to_50', 'mammalian_51', 'ras_52', 'GTPase_53', '-_54', 'activating_55', 'protein_56', '._57', 'Exon_58', 'A1a_59']\n",
            "BAD:  16599\n",
            "rpws:      index        token label\n",
            "0   16659      encodes     O\n",
            "1   16660         most     O\n",
            "2   16661           of     O\n",
            "3   16662          the     O\n",
            "4   16663            5     O\n",
            "..    ...          ...   ...\n",
            "64  16723           G1     O\n",
            "65  16724            )     O\n",
            "66  16725            .     O\n",
            "67  16726        These     O\n",
            "68  16727  alterations     O\n",
            "\n",
            "[69 rows x 3 columns]\n",
            "Exon A1a encodes most of the 5 ' - untranslated region . Co - operativity of functional domains in the muscle - specific transcription factor Myf - 5 . Creatinine and creatine phosphokinase ( CPK ) . In our initial studies , we treated MCF - 7 cells with paclitaxel , which results in the arrest of cells in G1 with 4n DNA content ( pseudo G1 ) .\n",
            "['Exon', 'A1a', 'encodes', 'most', 'of', 'the', '5', \"'\", '-', 'untranslated', 'region', '.', 'Co', '-', 'operativity', 'of', 'functional', 'domains', 'in', 'the', 'muscle', '-', 'specific', 'transcription', 'factor', 'Myf', '-', '5', '.', 'Creatinine', 'and', 'creatine', 'phosphokinase', '(', 'CPK', ')', '.', 'In', 'our', 'initial', 'studies', ',', 'we', 'treated', 'MCF', '-', '7', 'cells', 'with', 'paclitaxel', ',', 'which', 'results', 'in', 'the', 'arrest', 'of', 'cells', 'in', 'G1', 'with', '4n', 'DNA', 'content', '(', 'pseudo', 'G1', ')', '.']\n",
            "69 69\n",
            "['encodes_0', 'most_1', 'of_2', 'the_3', '5_4', \"'_5\", '-_6', 'untranslated_7', 'region_8', '._9', 'Co_10', '-_11', 'operativity_12', 'of_13', 'functional_14', 'domains_15', 'in_16', 'the_17', 'muscle_18', '-_19', 'specific_20', 'transcription_21', 'factor_22', 'Myf_23', '-_24', '5_25', '._26', 'Creatinine_27', 'and_28', 'creatine_29', 'phosphokinase_30', '(_31', 'CPK_32', ')_33', '._34', 'In_35', 'our_36', 'initial_37', 'studies_38', ',_39', 'we_40', 'treated_41', 'MCF_42', '-_43', '7_44', 'cells_45', 'with_46', 'paclitaxel_47', ',_48', 'which_49', 'results_50', 'in_51', 'the_52', 'arrest_53', 'of_54', 'cells_55', 'in_56', 'G1_57', 'with_58', '4n_59', 'DNA_60', 'content_61', '(_62', 'pseudo_63', 'G1_64', ')_65', '._66', 'These_67', 'alterations_68']\n",
            "BAD:  16659\n",
            "rpws:       index           token label\n",
            "0    16728        occurred     O\n",
            "1    16729              at     O\n",
            "2    16730            both     O\n",
            "3    16731             the     O\n",
            "4    16732            mRNA     O\n",
            "..     ...             ...   ...\n",
            "99   16827            echo     O\n",
            "100  16828      techniques     O\n",
            "101  16829               .     O\n",
            "102  16830  Seroconversion     O\n",
            "103  16831           after     O\n",
            "\n",
            "[104 rows x 3 columns]\n",
            "These alterations occurred at both the mRNA and protein levels but did not significantly affect the subcellular distribution of any of the four isoforms . The glyoxysomal and plastid molecular chaperones ( 70 - kDa heat shock protein ) of watermelon cotyledons are encoded by a single gene . Suspected pelvic endometriosis was prospectively evaluated in 31 women with T1 - and T2 - weighted conventional spin - echo ( CSE ) magnetic resonance imaging alone and in combination with T1 - weighted fat - suppressed ( T1FS ) and gadolinium - enhanced T1FS ( Gd - T1FS ) spin - echo techniques .\n",
            "['These', 'alterations', 'occurred', 'at', 'both', 'the', 'mRNA', 'and', 'protein', 'levels', 'but', 'did', 'not', 'significantly', 'affect', 'the', 'subcellular', 'distribution', 'of', 'any', 'of', 'the', 'four', 'isoforms', '.', 'The', 'glyoxysomal', 'and', 'plastid', 'molecular', 'chaperones', '(', '70', '-', 'kDa', 'heat', 'shock', 'protein', ')', 'of', 'watermelon', 'cotyledons', 'are', 'encoded', 'by', 'a', 'single', 'gene', '.', 'Suspected', 'pelvic', 'endometriosis', 'was', 'prospectively', 'evaluated', 'in', '31', 'women', 'with', 'T1', '-', 'and', 'T2', '-', 'weighted', 'conventional', 'spin', '-', 'echo', '(', 'CSE', ')', 'magnetic', 'resonance', 'imaging', 'alone', 'and', 'in', 'combination', 'with', 'T1', '-', 'weighted', 'fat', '-', 'suppressed', '(', 'T1FS', ')', 'and', 'gadolinium', '-', 'enhanced', 'T1FS', '(', 'Gd', '-', 'T1FS', ')', 'spin', '-', 'echo', 'techniques', '.']\n",
            "104 104\n",
            "['occurred_0', 'at_1', 'both_2', 'the_3', 'mRNA_4', 'and_5', 'protein_6', 'levels_7', 'but_8', 'did_9', 'not_10', 'significantly_11', 'affect_12', 'the_13', 'subcellular_14', 'distribution_15', 'of_16', 'any_17', 'of_18', 'the_19', 'four_20', 'isoforms_21', '._22', 'The_23', 'glyoxysomal_24', 'and_25', 'plastid_26', 'molecular_27', 'chaperones_28', '(_29', '70_30', '-_31', 'kDa_32', 'heat_33', 'shock_34', 'protein_35', ')_36', 'of_37', 'watermelon_38', 'cotyledons_39', 'are_40', 'encoded_41', 'by_42', 'a_43', 'single_44', 'gene_45', '._46', 'Suspected_47', 'pelvic_48', 'endometriosis_49', 'was_50', 'prospectively_51', 'evaluated_52', 'in_53', '31_54', 'women_55', 'with_56', 'T1_57', '-_58', 'and_59', 'T2_60', '-_61', 'weighted_62', 'conventional_63', 'spin_64', '-_65', 'echo_66', '(_67', 'CSE_68', ')_69', 'magnetic_70', 'resonance_71', 'imaging_72', 'alone_73', 'and_74', 'in_75', 'combination_76', 'with_77', 'T1_78', '-_79', 'weighted_80', 'fat_81', '-_82', 'suppressed_83', '(_84', 'T1FS_85', ')_86', 'and_87', 'gadolinium_88', '-_89', 'enhanced_90', 'T1FS_91', '(_92', 'Gd_93', '-_94', 'T1FS_95', ')_96', 'spin_97', '-_98', 'echo_99', 'techniques_100', '._101', 'Seroconversion_102', 'after_103']\n",
            "BAD:  16728\n",
            "rpws:      index        token label\n",
            "0   16832    hepatitis     O\n",
            "1   16833            B     O\n",
            "2   16834  vaccination     O\n",
            "3   16835            .     O\n",
            "4   16836            A     O\n",
            "..    ...          ...   ...\n",
            "71  16903      bearing     O\n",
            "72  16904     activity     O\n",
            "73  16905            .     O\n",
            "74  16906          The     O\n",
            "75  16907      effects     O\n",
            "\n",
            "[76 rows x 3 columns]\n",
            "Seroconversion after hepatitis B vaccination . A case of bilateral testicular germ cell tumors in a 23 - year - old male is reported . Total body BMD ( r = 0 . 30 ; P = 0 . 02 ) and femoral neck BMD ( r = 0 . 39 ; P = 0 . 002 ) were positively correlated with weight - bearing activity but not with non - weight - bearing activity .\n",
            "['Seroconversion', 'after', 'hepatitis', 'B', 'vaccination', '.', 'A', 'case', 'of', 'bilateral', 'testicular', 'germ', 'cell', 'tumors', 'in', 'a', '23', '-', 'year', '-', 'old', 'male', 'is', 'reported', '.', 'Total', 'body', 'BMD', '(', 'r', '=', '0', '.', '30', ';', 'P', '=', '0', '.', '02', ')', 'and', 'femoral', 'neck', 'BMD', '(', 'r', '=', '0', '.', '39', ';', 'P', '=', '0', '.', '002', ')', 'were', 'positively', 'correlated', 'with', 'weight', '-', 'bearing', 'activity', 'but', 'not', 'with', 'non', '-', 'weight', '-', 'bearing', 'activity', '.']\n",
            "76 76\n",
            "['hepatitis_0', 'B_1', 'vaccination_2', '._3', 'A_4', 'case_5', 'of_6', 'bilateral_7', 'testicular_8', 'germ_9', 'cell_10', 'tumors_11', 'in_12', 'a_13', '23_14', '-_15', 'year_16', '-_17', 'old_18', 'male_19', 'is_20', 'reported_21', '._22', 'Total_23', 'body_24', 'BMD_25', '(_26', 'r_27', '=_28', '0_29', '._30', '30_31', ';_32', 'P_33', '=_34', '0_35', '._36', '02_37', ')_38', 'and_39', 'femoral_40', 'neck_41', 'BMD_42', '(_43', 'r_44', '=_45', '0_46', '._47', '39_48', ';_49', 'P_50', '=_51', '0_52', '._53', '002_54', ')_55', 'were_56', 'positively_57', 'correlated_58', 'with_59', 'weight_60', '-_61', 'bearing_62', 'activity_63', 'but_64', 'not_65', 'with_66', 'non_67', '-_68', 'weight_69', '-_70', 'bearing_71', 'activity_72', '._73', 'The_74', 'effects_75']\n",
            "BAD:  16832\n",
            "rpws:      index     token label\n",
            "0   16908        of     O\n",
            "1   16909    castor     O\n",
            "2   16910       oil     O\n",
            "3   16911         ,     O\n",
            "4   16912     alone     O\n",
            "..    ...       ...   ...\n",
            "69  16977       the     O\n",
            "70  16978     first     O\n",
            "71  16979         .     O\n",
            "72  16980       The     O\n",
            "73  16981  residual     O\n",
            "\n",
            "[74 rows x 3 columns]\n",
            "The effects of castor oil , alone , as well as in combination with PGI2 and indomethacin on gastrointestinal functions have been examined in rats . Checking for patient compliance , samples ( n = 10 ) with a FK I concentration of 0 ng / mL were re - analyzed . K . The hematopoietic form of PTPN6 transcript is initiated at a downstream promoter separated by 7 kb from the first .\n",
            "['The', 'effects', 'of', 'castor', 'oil', ',', 'alone', ',', 'as', 'well', 'as', 'in', 'combination', 'with', 'PGI2', 'and', 'indomethacin', 'on', 'gastrointestinal', 'functions', 'have', 'been', 'examined', 'in', 'rats', '.', 'Checking', 'for', 'patient', 'compliance', ',', 'samples', '(', 'n', '=', '10', ')', 'with', 'a', 'FK', 'I', 'concentration', 'of', '0', 'ng', '/', 'mL', 'were', 're', '-', 'analyzed', '.', 'K', '.', 'The', 'hematopoietic', 'form', 'of', 'PTPN6', 'transcript', 'is', 'initiated', 'at', 'a', 'downstream', 'promoter', 'separated', 'by', '7', 'kb', 'from', 'the', 'first', '.']\n",
            "74 74\n",
            "['of_0', 'castor_1', 'oil_2', ',_3', 'alone_4', ',_5', 'as_6', 'well_7', 'as_8', 'in_9', 'combination_10', 'with_11', 'PGI2_12', 'and_13', 'indomethacin_14', 'on_15', 'gastrointestinal_16', 'functions_17', 'have_18', 'been_19', 'examined_20', 'in_21', 'rats_22', '._23', 'Checking_24', 'for_25', 'patient_26', 'compliance_27', ',_28', 'samples_29', '(_30', 'n_31', '=_32', '10_33', ')_34', 'with_35', 'a_36', 'FK_37', 'I_38', 'concentration_39', 'of_40', '0_41', 'ng_42', '/_43', 'mL_44', 'were_45', 're_46', '-_47', 'analyzed_48', '._49', 'K_50', '._51', 'The_52', 'hematopoietic_53', 'form_54', 'of_55', 'PTPN6_56', 'transcript_57', 'is_58', 'initiated_59', 'at_60', 'a_61', 'downstream_62', 'promoter_63', 'separated_64', 'by_65', '7_66', 'kb_67', 'from_68', 'the_69', 'first_70', '._71', 'The_72', 'residual_73']\n",
            "BAD:  16908\n",
            "rpws:      index           token label\n",
            "0   16982           small     O\n",
            "1   16983           rRNAs     O\n",
            "2   16984              in     O\n",
            "3   16985               [     O\n",
            "4   16986            poky     O\n",
            "..    ...             ...   ...\n",
            "66  17048             TLD     O\n",
            "67  17049          system     O\n",
            "68  17050               .     O\n",
            "69  17051  Unconventional     O\n",
            "70  17052            mRNA     O\n",
            "\n",
            "[71 rows x 3 columns]\n",
            "The residual small rRNAs in [ poky ] appear to be synthesized via the upstream promoter ( s ) , but are missing 37 - 44 nucleotides from their 5 ' ends , indicating either that pre - rRNAs are processed abnormally or that abnormal 5 ' RNA ends are unstable . Extra dose due to extravehicular activity during the NASA4 mission measured by an on - board TLD system .\n",
            "['The', 'residual', 'small', 'rRNAs', 'in', '[', 'poky', ']', 'appear', 'to', 'be', 'synthesized', 'via', 'the', 'upstream', 'promoter', '(', 's', ')', ',', 'but', 'are', 'missing', '37', '-', '44', 'nucleotides', 'from', 'their', '5', \"'\", 'ends', ',', 'indicating', 'either', 'that', 'pre', '-', 'rRNAs', 'are', 'processed', 'abnormally', 'or', 'that', 'abnormal', '5', \"'\", 'RNA', 'ends', 'are', 'unstable', '.', 'Extra', 'dose', 'due', 'to', 'extravehicular', 'activity', 'during', 'the', 'NASA4', 'mission', 'measured', 'by', 'an', 'on', '-', 'board', 'TLD', 'system', '.']\n",
            "71 71\n",
            "['small_0', 'rRNAs_1', 'in_2', '[_3', 'poky_4', ']_5', 'appear_6', 'to_7', 'be_8', 'synthesized_9', 'via_10', 'the_11', 'upstream_12', 'promoter_13', '(_14', 's_15', ')_16', ',_17', 'but_18', 'are_19', 'missing_20', '37_21', '-_22', '44_23', 'nucleotides_24', 'from_25', 'their_26', '5_27', \"'_28\", 'ends_29', ',_30', 'indicating_31', 'either_32', 'that_33', 'pre_34', '-_35', 'rRNAs_36', 'are_37', 'processed_38', 'abnormally_39', 'or_40', 'that_41', 'abnormal_42', '5_43', \"'_44\", 'RNA_45', 'ends_46', 'are_47', 'unstable_48', '._49', 'Extra_50', 'dose_51', 'due_52', 'to_53', 'extravehicular_54', 'activity_55', 'during_56', 'the_57', 'NASA4_58', 'mission_59', 'measured_60', 'by_61', 'an_62', 'on_63', '-_64', 'board_65', 'TLD_66', 'system_67', '._68', 'Unconventional_69', 'mRNA_70']\n",
            "BAD:  16982\n",
            "rpws:      index           token label\n",
            "0   17053      processing     O\n",
            "1   17054              in     O\n",
            "2   17055             the     O\n",
            "3   17056      expression     O\n",
            "4   17057              of     O\n",
            "5   17058             two     O\n",
            "6   17059     calcineurin     B\n",
            "7   17060               B     I\n",
            "8   17061        isoforms     I\n",
            "9   17062              in     O\n",
            "10  17063   Dictyostelium     O\n",
            "11  17064               .     O\n",
            "12  17065             The     O\n",
            "13  17066         deduced     O\n",
            "14  17067           amino     O\n",
            "15  17068            acid     O\n",
            "16  17069        sequence     O\n",
            "17  17070              of     O\n",
            "18  17071            this     O\n",
            "19  17072            open     O\n",
            "20  17073         reading     O\n",
            "21  17074           frame     O\n",
            "22  17075              is     O\n",
            "23  17076   significantly     O\n",
            "24  17077      homologous     O\n",
            "25  17078              to     O\n",
            "26  17079             the     O\n",
            "27  17080             HSV     B\n",
            "28  17081               1     I\n",
            "29  17082            UL49     I\n",
            "30  17083               .     I\n",
            "31  17084               5     I\n",
            "32  17085            gene     I\n",
            "33  17086         product     I\n",
            "34  17087               ,     O\n",
            "35  17088             and     O\n",
            "36  17089              as     O\n",
            "37  17090            with     O\n",
            "38  17091            UL49     B\n",
            "39  17092               .     I\n",
            "40  17093               5     I\n",
            "41  17094               ,     O\n",
            "42  17095              it     O\n",
            "43  17096        contains     O\n",
            "44  17097               a     O\n",
            "45  17098       potential     O\n",
            "46  17099          signal     O\n",
            "47  17100        sequence     O\n",
            "48  17101             and     O\n",
            "49  17102   transmembrane     O\n",
            "50  17103          domain     O\n",
            "51  17104  characteristic     O\n",
            "52  17105              of     O\n",
            "53  17106        membrane     O\n",
            "54  17107               -     O\n",
            "55  17108      associated     O\n",
            "56  17109        proteins     O\n",
            "57  17110               .     O\n",
            "58  17111              It     O\n",
            "59  17112             has     O\n",
            "Unconventional mRNA processing in the expression of two calcineurin B isoforms in Dictyostelium . The deduced amino acid sequence of this open reading frame is significantly homologous to the HSV 1 UL49 . 5 gene product , and as with UL49 . 5 , it contains a potential signal sequence and transmembrane domain characteristic of membrane - associated proteins .\n",
            "['Unconventional', 'mRNA', 'processing', 'in', 'the', 'expression', 'of', 'two', 'calcineurin', 'B', 'isoforms', 'in', 'Dictyostelium', '.', 'The', 'deduced', 'amino', 'acid', 'sequence', 'of', 'this', 'open', 'reading', 'frame', 'is', 'significantly', 'homologous', 'to', 'the', 'HSV', '1', 'UL49', '.', '5', 'gene', 'product', ',', 'and', 'as', 'with', 'UL49', '.', '5', ',', 'it', 'contains', 'a', 'potential', 'signal', 'sequence', 'and', 'transmembrane', 'domain', 'characteristic', 'of', 'membrane', '-', 'associated', 'proteins', '.']\n",
            "60 60\n",
            "['processing_0', 'in_1', 'the_2', 'expression_3', 'of_4', 'two_5', 'calcineurin_6', 'B_7', 'isoforms_8', 'in_9', 'Dictyostelium_10', '._11', 'The_12', 'deduced_13', 'amino_14', 'acid_15', 'sequence_16', 'of_17', 'this_18', 'open_19', 'reading_20', 'frame_21', 'is_22', 'significantly_23', 'homologous_24', 'to_25', 'the_26', 'HSV_27', '1_28', 'UL49_29', '._30', '5_31', 'gene_32', 'product_33', ',_34', 'and_35', 'as_36', 'with_37', 'UL49_38', '._39', '5_40', ',_41', 'it_42', 'contains_43', 'a_44', 'potential_45', 'signal_46', 'sequence_47', 'and_48', 'transmembrane_49', 'domain_50', 'characteristic_51', 'of_52', 'membrane_53', '-_54', 'associated_55', 'proteins_56', '._57', 'It_58', 'has_59']\n",
            "BAD:  17053\n",
            "rpws:      index         token label\n",
            "0   17113          been     O\n",
            "1   17114  demonstrated     O\n",
            "2   17115          that     O\n",
            "3   17116   stimulation     O\n",
            "4   17117            of     O\n",
            "..    ...           ...   ...\n",
            "58  17171          S150     O\n",
            "59  17172       extract     O\n",
            "60  17173             .     O\n",
            "61  17174          Also     O\n",
            "62  17175             ,     O\n",
            "\n",
            "[63 rows x 3 columns]\n",
            "It has been demonstrated that stimulation of the pontomesencephalic parabrachial region ( PBR ) by microinjection of cholinergic drugs or electricity in the cat produces potent pain suppression which is not antagonized by the opiate antagonist , naloxone . GAGA factor is known to remodel the chromatin structure in concert with nucleosome - remodeling factor NURF in a Drosophila embryonic S150 extract .\n",
            "['It', 'has', 'been', 'demonstrated', 'that', 'stimulation', 'of', 'the', 'pontomesencephalic', 'parabrachial', 'region', '(', 'PBR', ')', 'by', 'microinjection', 'of', 'cholinergic', 'drugs', 'or', 'electricity', 'in', 'the', 'cat', 'produces', 'potent', 'pain', 'suppression', 'which', 'is', 'not', 'antagonized', 'by', 'the', 'opiate', 'antagonist', ',', 'naloxone', '.', 'GAGA', 'factor', 'is', 'known', 'to', 'remodel', 'the', 'chromatin', 'structure', 'in', 'concert', 'with', 'nucleosome', '-', 'remodeling', 'factor', 'NURF', 'in', 'a', 'Drosophila', 'embryonic', 'S150', 'extract', '.']\n",
            "63 63\n",
            "['been_0', 'demonstrated_1', 'that_2', 'stimulation_3', 'of_4', 'the_5', 'pontomesencephalic_6', 'parabrachial_7', 'region_8', '(_9', 'PBR_10', ')_11', 'by_12', 'microinjection_13', 'of_14', 'cholinergic_15', 'drugs_16', 'or_17', 'electricity_18', 'in_19', 'the_20', 'cat_21', 'produces_22', 'potent_23', 'pain_24', 'suppression_25', 'which_26', 'is_27', 'not_28', 'antagonized_29', 'by_30', 'the_31', 'opiate_32', 'antagonist_33', ',_34', 'naloxone_35', '._36', 'GAGA_37', 'factor_38', 'is_39', 'known_40', 'to_41', 'remodel_42', 'the_43', 'chromatin_44', 'structure_45', 'in_46', 'concert_47', 'with_48', 'nucleosome_49', '-_50', 'remodeling_51', 'factor_52', 'NURF_53', 'in_54', 'a_55', 'Drosophila_56', 'embryonic_57', 'S150_58', 'extract_59', '._60', 'Also_61', ',_62']\n",
            "BAD:  17113\n",
            "rpws:      index          token label\n",
            "0   17176           like     O\n",
            "1   17177            TUB     B\n",
            "2   17178              ,     O\n",
            "3   17179             it     O\n",
            "4   17180            has     O\n",
            "5   17181              a     O\n",
            "6   17182          wider     O\n",
            "7   17183        pattern     O\n",
            "8   17184             of     O\n",
            "9   17185         tissue     O\n",
            "10  17186     expression     O\n",
            "11  17187           than     O\n",
            "12  17188         either     O\n",
            "13  17189          TULP1     B\n",
            "14  17190             or     O\n",
            "15  17191          TULP2     B\n",
            "16  17192              .     O\n",
            "17  17193          After     O\n",
            "18  17194           nine     O\n",
            "19  17195         months     O\n",
            "20  17196            GFR     O\n",
            "21  17197       improved     O\n",
            "22  17198  spontaneously     O\n",
            "23  17199             to     O\n",
            "24  17200             32     O\n",
            "25  17201             ml     O\n",
            "26  17202              /     O\n",
            "27  17203            min     O\n",
            "28  17204              /     O\n",
            "29  17205              1     O\n",
            "30  17206              .     O\n",
            "31  17207             73     O\n",
            "32  17208             m2     O\n",
            "33  17209        despite     O\n",
            "34  17210             no     O\n",
            "35  17211    improvement     O\n",
            "36  17212             in     O\n",
            "37  17213            his     O\n",
            "38  17214   hypertension     O\n",
            "39  17215              .     O\n",
            "40  17216          First     O\n",
            "41  17217              ,     O\n",
            "42  17218     ste12Delta     B\n",
            "43  17219          cells     O\n",
            "44  17220         differ     O\n",
            "45  17221           from     O\n",
            "46  17222          cells     O\n",
            "47  17223           with     O\n",
            "48  17224    disruptions     O\n",
            "49  17225             of     O\n",
            "50  17226            the     O\n",
            "51  17227       upstream     O\n",
            "52  17228      signaling     O\n",
            "53  17229       elements     O\n",
            "54  17230              (     O\n",
            "55  17231              e     O\n",
            "56  17232              .     O\n",
            "57  17233              g     O\n",
            "58  17234              .     O\n",
            "Also , like TUB , it has a wider pattern of tissue expression than either TULP1 or TULP2 . After nine months GFR improved spontaneously to 32 ml / min / 1 . 73 m2 despite no improvement in his hypertension . First , ste12Delta cells differ from cells with disruptions of the upstream signaling elements ( e .\n",
            "['Also', ',', 'like', 'TUB', ',', 'it', 'has', 'a', 'wider', 'pattern', 'of', 'tissue', 'expression', 'than', 'either', 'TULP1', 'or', 'TULP2', '.', 'After', 'nine', 'months', 'GFR', 'improved', 'spontaneously', 'to', '32', 'ml', '/', 'min', '/', '1', '.', '73', 'm2', 'despite', 'no', 'improvement', 'in', 'his', 'hypertension', '.', 'First', ',', 'ste12Delta', 'cells', 'differ', 'from', 'cells', 'with', 'disruptions', 'of', 'the', 'upstream', 'signaling', 'elements', '(', 'e', '.']\n",
            "59 59\n",
            "['like_0', 'TUB_1', ',_2', 'it_3', 'has_4', 'a_5', 'wider_6', 'pattern_7', 'of_8', 'tissue_9', 'expression_10', 'than_11', 'either_12', 'TULP1_13', 'or_14', 'TULP2_15', '._16', 'After_17', 'nine_18', 'months_19', 'GFR_20', 'improved_21', 'spontaneously_22', 'to_23', '32_24', 'ml_25', '/_26', 'min_27', '/_28', '1_29', '._30', '73_31', 'm2_32', 'despite_33', 'no_34', 'improvement_35', 'in_36', 'his_37', 'hypertension_38', '._39', 'First_40', ',_41', 'ste12Delta_42', 'cells_43', 'differ_44', 'from_45', 'cells_46', 'with_47', 'disruptions_48', 'of_49', 'the_50', 'upstream_51', 'signaling_52', 'elements_53', '(_54', 'e_55', '._56', 'g_57', '._58']\n",
            "BAD:  17176\n",
            "rpws:      index       token label\n",
            "0   17235           ,     O\n",
            "1   17236   ste4Delta     B\n",
            "2   17237           ,     O\n",
            "3   17238  ste20Delta     B\n",
            "4   17239           ,     O\n",
            "..    ...         ...   ...\n",
            "59  17294        type     O\n",
            "60  17295         p53     B\n",
            "61  17296           .     O\n",
            "62  17297         The     O\n",
            "63  17298   selective     O\n",
            "\n",
            "[64 rows x 3 columns]\n",
            "g . , ste4Delta , ste20Delta , ste5Delta , ste11Delta , ste7Delta , or fus3Delta kss1Delta cells ) in that they clearly retain some capacity for inducing Ste3p phosphorylation . Herein , we report that CRE - decoy oligonucleotide treatment results in an increase in the p53 protein level in MCF - 7 human breast cancer cells that express wild - type p53 .\n",
            "['g', '.', ',', 'ste4Delta', ',', 'ste20Delta', ',', 'ste5Delta', ',', 'ste11Delta', ',', 'ste7Delta', ',', 'or', 'fus3Delta', 'kss1Delta', 'cells', ')', 'in', 'that', 'they', 'clearly', 'retain', 'some', 'capacity', 'for', 'inducing', 'Ste3p', 'phosphorylation', '.', 'Herein', ',', 'we', 'report', 'that', 'CRE', '-', 'decoy', 'oligonucleotide', 'treatment', 'results', 'in', 'an', 'increase', 'in', 'the', 'p53', 'protein', 'level', 'in', 'MCF', '-', '7', 'human', 'breast', 'cancer', 'cells', 'that', 'express', 'wild', '-', 'type', 'p53', '.']\n",
            "64 64\n",
            "[',_0', 'ste4Delta_1', ',_2', 'ste20Delta_3', ',_4', 'ste5Delta_5', ',_6', 'ste11Delta_7', ',_8', 'ste7Delta_9', ',_10', 'or_11', 'fus3Delta_12', 'kss1Delta_13', 'cells_14', ')_15', 'in_16', 'that_17', 'they_18', 'clearly_19', 'retain_20', 'some_21', 'capacity_22', 'for_23', 'inducing_24', 'Ste3p_25', 'phosphorylation_26', '._27', 'Herein_28', ',_29', 'we_30', 'report_31', 'that_32', 'CRE_33', '-_34', 'decoy_35', 'oligonucleotide_36', 'treatment_37', 'results_38', 'in_39', 'an_40', 'increase_41', 'in_42', 'the_43', 'p53_44', 'protein_45', 'level_46', 'in_47', 'MCF_48', '-_49', '7_50', 'human_51', 'breast_52', 'cancer_53', 'cells_54', 'that_55', 'express_56', 'wild_57', '-_58', 'type_59', 'p53_60', '._61', 'The_62', 'selective_63']\n",
            "BAD:  17235\n",
            "rpws:      index              token label\n",
            "0   17299                  5     B\n",
            "1   17300                  -     I\n",
            "2   17301                HT2     I\n",
            "3   17302           receptor     I\n",
            "4   17303            blocker     O\n",
            "5   17304         ketanserin     O\n",
            "6   17305                was     O\n",
            "7   17306              found     O\n",
            "8   17307                 to     O\n",
            "9   17308             reduce     O\n",
            "10  17309            maximal     O\n",
            "11  17310           urethral     O\n",
            "12  17311          pressures     O\n",
            "13  17312                 in     O\n",
            "14  17313            healthy     O\n",
            "15  17314            females     O\n",
            "16  17315                 by     O\n",
            "17  17316              about     O\n",
            "18  17317                 40     O\n",
            "19  17318                  %     O\n",
            "20  17319            without     O\n",
            "21  17320           reducing     O\n",
            "22  17321              blood     O\n",
            "23  17322           pressure     O\n",
            "24  17323                  .     O\n",
            "25  17324                 It     O\n",
            "26  17325               thus     O\n",
            "27  17326              seems     O\n",
            "28  17327             likely     O\n",
            "29  17328               that     O\n",
            "30  17329          genistein     O\n",
            "31  17330            affects     O\n",
            "32  17331                  a     O\n",
            "33  17332             common     O\n",
            "34  17333            pathway     O\n",
            "35  17334         downstream     O\n",
            "36  17335                 of     O\n",
            "37  17336              these     O\n",
            "38  17337            signals     O\n",
            "39  17338                  .     O\n",
            "40  17339            Studies     O\n",
            "41  17340                 in     O\n",
            "42  17341                  B     O\n",
            "43  17342                  .     O\n",
            "44  17343           glabrata     O\n",
            "45  17344                  ,     O\n",
            "46  17345                  B     O\n",
            "47  17346                  .     O\n",
            "48  17347          straminea     O\n",
            "49  17348                and     O\n",
            "50  17349                  B     O\n",
            "51  17350                  .     O\n",
            "52  17351        tenagophila     O\n",
            "53  17352              under     O\n",
            "54  17353            outdoor     O\n",
            "55  17354         conditions     O\n",
            "56  17355                  .     O\n",
            "57  17356                The     O\n",
            "58  17357  electrocardiogram     O\n",
            "The selective 5 - HT2 receptor blocker ketanserin was found to reduce maximal urethral pressures in healthy females by about 40 % without reducing blood pressure . It thus seems likely that genistein affects a common pathway downstream of these signals . Studies in B . glabrata , B . straminea and B . tenagophila under outdoor conditions .\n",
            "['The', 'selective', '5', '-', 'HT2', 'receptor', 'blocker', 'ketanserin', 'was', 'found', 'to', 'reduce', 'maximal', 'urethral', 'pressures', 'in', 'healthy', 'females', 'by', 'about', '40', '%', 'without', 'reducing', 'blood', 'pressure', '.', 'It', 'thus', 'seems', 'likely', 'that', 'genistein', 'affects', 'a', 'common', 'pathway', 'downstream', 'of', 'these', 'signals', '.', 'Studies', 'in', 'B', '.', 'glabrata', ',', 'B', '.', 'straminea', 'and', 'B', '.', 'tenagophila', 'under', 'outdoor', 'conditions', '.']\n",
            "59 59\n",
            "['5_0', '-_1', 'HT2_2', 'receptor_3', 'blocker_4', 'ketanserin_5', 'was_6', 'found_7', 'to_8', 'reduce_9', 'maximal_10', 'urethral_11', 'pressures_12', 'in_13', 'healthy_14', 'females_15', 'by_16', 'about_17', '40_18', '%_19', 'without_20', 'reducing_21', 'blood_22', 'pressure_23', '._24', 'It_25', 'thus_26', 'seems_27', 'likely_28', 'that_29', 'genistein_30', 'affects_31', 'a_32', 'common_33', 'pathway_34', 'downstream_35', 'of_36', 'these_37', 'signals_38', '._39', 'Studies_40', 'in_41', 'B_42', '._43', 'glabrata_44', ',_45', 'B_46', '._47', 'straminea_48', 'and_49', 'B_50', '._51', 'tenagophila_52', 'under_53', 'outdoor_54', 'conditions_55', '._56', 'The_57', 'electrocardiogram_58']\n",
            "BAD:  17299\n",
            "rpws:      index           token label\n",
            "0   17358              in     O\n",
            "1   17359     tachycardia     O\n",
            "2   17360               .     O\n",
            "3   17361  Identification     O\n",
            "4   17362              of     O\n",
            "..    ...             ...   ...\n",
            "58  17416        esterase     I\n",
            "59  17417        sequence     I\n",
            "60  17418               .     O\n",
            "61  17419             The     O\n",
            "62  17420       essential     O\n",
            "\n",
            "[63 rows x 3 columns]\n",
            "The electrocardiogram in tachycardia . Identification of a putative chromosomal replication origin from Helicobacter pylori and its interaction with the initiator protein DnaA . The primary structure of cholesterol esterase displayed no significant homology with other lipases , although the putative lipid interfacial recognition site of G - X - S - X - G is present in the cholesterol esterase sequence .\n",
            "['The', 'electrocardiogram', 'in', 'tachycardia', '.', 'Identification', 'of', 'a', 'putative', 'chromosomal', 'replication', 'origin', 'from', 'Helicobacter', 'pylori', 'and', 'its', 'interaction', 'with', 'the', 'initiator', 'protein', 'DnaA', '.', 'The', 'primary', 'structure', 'of', 'cholesterol', 'esterase', 'displayed', 'no', 'significant', 'homology', 'with', 'other', 'lipases', ',', 'although', 'the', 'putative', 'lipid', 'interfacial', 'recognition', 'site', 'of', 'G', '-', 'X', '-', 'S', '-', 'X', '-', 'G', 'is', 'present', 'in', 'the', 'cholesterol', 'esterase', 'sequence', '.']\n",
            "63 63\n",
            "['in_0', 'tachycardia_1', '._2', 'Identification_3', 'of_4', 'a_5', 'putative_6', 'chromosomal_7', 'replication_8', 'origin_9', 'from_10', 'Helicobacter_11', 'pylori_12', 'and_13', 'its_14', 'interaction_15', 'with_16', 'the_17', 'initiator_18', 'protein_19', 'DnaA_20', '._21', 'The_22', 'primary_23', 'structure_24', 'of_25', 'cholesterol_26', 'esterase_27', 'displayed_28', 'no_29', 'significant_30', 'homology_31', 'with_32', 'other_33', 'lipases_34', ',_35', 'although_36', 'the_37', 'putative_38', 'lipid_39', 'interfacial_40', 'recognition_41', 'site_42', 'of_43', 'G_44', '-_45', 'X_46', '-_47', 'S_48', '-_49', 'X_50', '-_51', 'G_52', 'is_53', 'present_54', 'in_55', 'the_56', 'cholesterol_57', 'esterase_58', 'sequence_59', '._60', 'The_61', 'essential_62']\n",
            "BAD:  17358\n",
            "rpws:      index               token label\n",
            "0   17421           questions     O\n",
            "1   17422               about     O\n",
            "2   17423           hepatitis     O\n",
            "3   17424                   C     O\n",
            "4   17425                 The     O\n",
            "5   17426     transcriptional     O\n",
            "6   17427          initiation     O\n",
            "7   17428                site     O\n",
            "8   17429                  of     O\n",
            "9   17430                RAG1     B\n",
            "10  17431                 was     O\n",
            "11  17432           localized     O\n",
            "12  17433                  at     O\n",
            "13  17434                   A     O\n",
            "14  17435                   ,     O\n",
            "15  17436                  26     O\n",
            "16  17437                  bp     O\n",
            "17  17438            upstream     O\n",
            "18  17439                  of     O\n",
            "19  17440                 the     O\n",
            "20  17441            putative     O\n",
            "21  17442       translational     O\n",
            "22  17443          initiation     O\n",
            "23  17444               codon     O\n",
            "24  17445                   ,     O\n",
            "25  17446                 ATG     O\n",
            "26  17447                   ,     O\n",
            "27  17448                  by     O\n",
            "28  17449                 the     O\n",
            "29  17450              primer     O\n",
            "30  17451           extension     O\n",
            "31  17452               assay     O\n",
            "32  17453                   .     O\n",
            "33  17454            Coronary     O\n",
            "34  17455     arteriographies     O\n",
            "35  17456                 and     O\n",
            "36  17457  ventriculographies     O\n",
            "37  17458       corresponding     O\n",
            "38  17459                  to     O\n",
            "39  17460                 274     O\n",
            "40  17461         consecutive     O\n",
            "41  17462            patients     O\n",
            "42  17463                   (     O\n",
            "43  17464             January     O\n",
            "44  17465                   ,     O\n",
            "45  17466                1975     O\n",
            "46  17467                   -     O\n",
            "47  17468             October     O\n",
            "48  17469                   ,     O\n",
            "49  17470                1978     O\n",
            "50  17471                   )     O\n",
            "51  17472                with     O\n",
            "52  17473         significant     O\n",
            "53  17474            coronary     O\n",
            "54  17475             lesions     O\n",
            "55  17476                 are     O\n",
            "56  17477            reviewed     O\n",
            "57  17478                   .     O\n",
            "58  17479       Visualization     O\n",
            "59  17480                  of     O\n",
            "The essential questions about hepatitis C The transcriptional initiation site of RAG1 was localized at A , 26 bp upstream of the putative translational initiation codon , ATG , by the primer extension assay . Coronary arteriographies and ventriculographies corresponding to 274 consecutive patients ( January , 1975 - October , 1978 ) with significant coronary lesions are reviewed .\n",
            "['The', 'essential', 'questions', 'about', 'hepatitis', 'C', 'The', 'transcriptional', 'initiation', 'site', 'of', 'RAG1', 'was', 'localized', 'at', 'A', ',', '26', 'bp', 'upstream', 'of', 'the', 'putative', 'translational', 'initiation', 'codon', ',', 'ATG', ',', 'by', 'the', 'primer', 'extension', 'assay', '.', 'Coronary', 'arteriographies', 'and', 'ventriculographies', 'corresponding', 'to', '274', 'consecutive', 'patients', '(', 'January', ',', '1975', '-', 'October', ',', '1978', ')', 'with', 'significant', 'coronary', 'lesions', 'are', 'reviewed', '.']\n",
            "60 60\n",
            "['questions_0', 'about_1', 'hepatitis_2', 'C_3', 'The_4', 'transcriptional_5', 'initiation_6', 'site_7', 'of_8', 'RAG1_9', 'was_10', 'localized_11', 'at_12', 'A_13', ',_14', '26_15', 'bp_16', 'upstream_17', 'of_18', 'the_19', 'putative_20', 'translational_21', 'initiation_22', 'codon_23', ',_24', 'ATG_25', ',_26', 'by_27', 'the_28', 'primer_29', 'extension_30', 'assay_31', '._32', 'Coronary_33', 'arteriographies_34', 'and_35', 'ventriculographies_36', 'corresponding_37', 'to_38', '274_39', 'consecutive_40', 'patients_41', '(_42', 'January_43', ',_44', '1975_45', '-_46', 'October_47', ',_48', '1978_49', ')_50', 'with_51', 'significant_52', 'coronary_53', 'lesions_54', 'are_55', 'reviewed_56', '._57', 'Visualization_58', 'of_59']\n",
            "BAD:  17421\n",
            "rpws:      index      token label\n",
            "0   17481        the     O\n",
            "1   17482      cells     O\n",
            "2   17483         by     O\n",
            "3   17484      phase     O\n",
            "4   17485   contrast     O\n",
            "..    ...        ...   ...\n",
            "56  17537          a     O\n",
            "57  17538  diagnosis     O\n",
            "58  17539          .     O\n",
            "59  17540    Protein     B\n",
            "60  17541     kinase     I\n",
            "\n",
            "[61 rows x 3 columns]\n",
            "Visualization of the cells by phase contrast microscopy indicated that murine PKCepsilon expression in the presence of glycerol resulted in a significant increase in the number of yeast cells exhibiting very small buds . Sonography depicted the true morphology of these cystic lesions more clearly than CT , and the sonographic findings virtually excluded uncomplicated hepatic cyst as a diagnosis .\n",
            "['Visualization', 'of', 'the', 'cells', 'by', 'phase', 'contrast', 'microscopy', 'indicated', 'that', 'murine', 'PKCepsilon', 'expression', 'in', 'the', 'presence', 'of', 'glycerol', 'resulted', 'in', 'a', 'significant', 'increase', 'in', 'the', 'number', 'of', 'yeast', 'cells', 'exhibiting', 'very', 'small', 'buds', '.', 'Sonography', 'depicted', 'the', 'true', 'morphology', 'of', 'these', 'cystic', 'lesions', 'more', 'clearly', 'than', 'CT', ',', 'and', 'the', 'sonographic', 'findings', 'virtually', 'excluded', 'uncomplicated', 'hepatic', 'cyst', 'as', 'a', 'diagnosis', '.']\n",
            "61 61\n",
            "['the_0', 'cells_1', 'by_2', 'phase_3', 'contrast_4', 'microscopy_5', 'indicated_6', 'that_7', 'murine_8', 'PKCepsilon_9', 'expression_10', 'in_11', 'the_12', 'presence_13', 'of_14', 'glycerol_15', 'resulted_16', 'in_17', 'a_18', 'significant_19', 'increase_20', 'in_21', 'the_22', 'number_23', 'of_24', 'yeast_25', 'cells_26', 'exhibiting_27', 'very_28', 'small_29', 'buds_30', '._31', 'Sonography_32', 'depicted_33', 'the_34', 'true_35', 'morphology_36', 'of_37', 'these_38', 'cystic_39', 'lesions_40', 'more_41', 'clearly_42', 'than_43', 'CT_44', ',_45', 'and_46', 'the_47', 'sonographic_48', 'findings_49', 'virtually_50', 'excluded_51', 'uncomplicated_52', 'hepatic_53', 'cyst_54', 'as_55', 'a_56', 'diagnosis_57', '._58', 'Protein_59', 'kinase_60']\n",
            "BAD:  17481\n",
            "rpws:       index        token label\n",
            "0    17542            C     I\n",
            "1    17543  transiently     O\n",
            "2    17544    activated     O\n",
            "3    17545  heteromeric     B\n",
            "4    17546            N     I\n",
            "..     ...          ...   ...\n",
            "96   17638     subtilis     O\n",
            "97   17639            )     O\n",
            "98   17640            .     O\n",
            "99   17641           In     O\n",
            "100  17642        vitro     O\n",
            "\n",
            "[101 rows x 3 columns]\n",
            "Protein kinase C transiently activated heteromeric N - methyl - D - aspartate receptor channels independent of the phosphorylatable C - terminal splice domain and of consensus phosphorylation sites . A 24 - year - old pregnant woman started to have hyperemesis gravidarum 6 weeks before admission . Highly significant similarities were detected between the N - terminal region of P30 and those of GENA [ the product of another unidentified gene ( geneA ) located upstream of the aceEF - lpd operon ] , and GNTR ( a putative transcriptional repressor of the gluconate operon of Bacillus subtilis ) .\n",
            "['Protein', 'kinase', 'C', 'transiently', 'activated', 'heteromeric', 'N', '-', 'methyl', '-', 'D', '-', 'aspartate', 'receptor', 'channels', 'independent', 'of', 'the', 'phosphorylatable', 'C', '-', 'terminal', 'splice', 'domain', 'and', 'of', 'consensus', 'phosphorylation', 'sites', '.', 'A', '24', '-', 'year', '-', 'old', 'pregnant', 'woman', 'started', 'to', 'have', 'hyperemesis', 'gravidarum', '6', 'weeks', 'before', 'admission', '.', 'Highly', 'significant', 'similarities', 'were', 'detected', 'between', 'the', 'N', '-', 'terminal', 'region', 'of', 'P30', 'and', 'those', 'of', 'GENA', '[', 'the', 'product', 'of', 'another', 'unidentified', 'gene', '(', 'geneA', ')', 'located', 'upstream', 'of', 'the', 'aceEF', '-', 'lpd', 'operon', ']', ',', 'and', 'GNTR', '(', 'a', 'putative', 'transcriptional', 'repressor', 'of', 'the', 'gluconate', 'operon', 'of', 'Bacillus', 'subtilis', ')', '.']\n",
            "101 101\n",
            "['C_0', 'transiently_1', 'activated_2', 'heteromeric_3', 'N_4', '-_5', 'methyl_6', '-_7', 'D_8', '-_9', 'aspartate_10', 'receptor_11', 'channels_12', 'independent_13', 'of_14', 'the_15', 'phosphorylatable_16', 'C_17', '-_18', 'terminal_19', 'splice_20', 'domain_21', 'and_22', 'of_23', 'consensus_24', 'phosphorylation_25', 'sites_26', '._27', 'A_28', '24_29', '-_30', 'year_31', '-_32', 'old_33', 'pregnant_34', 'woman_35', 'started_36', 'to_37', 'have_38', 'hyperemesis_39', 'gravidarum_40', '6_41', 'weeks_42', 'before_43', 'admission_44', '._45', 'Highly_46', 'significant_47', 'similarities_48', 'were_49', 'detected_50', 'between_51', 'the_52', 'N_53', '-_54', 'terminal_55', 'region_56', 'of_57', 'P30_58', 'and_59', 'those_60', 'of_61', 'GENA_62', '[_63', 'the_64', 'product_65', 'of_66', 'another_67', 'unidentified_68', 'gene_69', '(_70', 'geneA_71', ')_72', 'located_73', 'upstream_74', 'of_75', 'the_76', 'aceEF_77', '-_78', 'lpd_79', 'operon_80', ']_81', ',_82', 'and_83', 'GNTR_84', '(_85', 'a_86', 'putative_87', 'transcriptional_88', 'repressor_89', 'of_90', 'the_91', 'gluconate_92', 'operon_93', 'of_94', 'Bacillus_95', 'subtilis_96', ')_97', '._98', 'In_99', 'vitro_100']\n",
            "BAD:  17542\n",
            "rpws:      index        token label\n",
            "0   17643          DNA     O\n",
            "1   17644      binding     O\n",
            "2   17645       assays     O\n",
            "3   17646     indicate     O\n",
            "4   17647         that     O\n",
            "..    ...          ...   ...\n",
            "66  17709      spatial     O\n",
            "67  17710  information     O\n",
            "68  17711            .     O\n",
            "69  17712       Siglec     B\n",
            "70  17713            -     I\n",
            "\n",
            "[71 rows x 3 columns]\n",
            "In vitro DNA binding assays indicate that the elements identified can specifically interact with c - Ets - 1 protein . They are found to have several advantages over the conventional indices of skewness and kurtosis ( square root of b1 and b2 ) and no serious drawbacks . They elicit similar regional cerebral blood flow ( rCBF ) patterns , even though sign language is dependent on spatial information .\n",
            "['In', 'vitro', 'DNA', 'binding', 'assays', 'indicate', 'that', 'the', 'elements', 'identified', 'can', 'specifically', 'interact', 'with', 'c', '-', 'Ets', '-', '1', 'protein', '.', 'They', 'are', 'found', 'to', 'have', 'several', 'advantages', 'over', 'the', 'conventional', 'indices', 'of', 'skewness', 'and', 'kurtosis', '(', 'square', 'root', 'of', 'b1', 'and', 'b2', ')', 'and', 'no', 'serious', 'drawbacks', '.', 'They', 'elicit', 'similar', 'regional', 'cerebral', 'blood', 'flow', '(', 'rCBF', ')', 'patterns', ',', 'even', 'though', 'sign', 'language', 'is', 'dependent', 'on', 'spatial', 'information', '.']\n",
            "71 71\n",
            "['DNA_0', 'binding_1', 'assays_2', 'indicate_3', 'that_4', 'the_5', 'elements_6', 'identified_7', 'can_8', 'specifically_9', 'interact_10', 'with_11', 'c_12', '-_13', 'Ets_14', '-_15', '1_16', 'protein_17', '._18', 'They_19', 'are_20', 'found_21', 'to_22', 'have_23', 'several_24', 'advantages_25', 'over_26', 'the_27', 'conventional_28', 'indices_29', 'of_30', 'skewness_31', 'and_32', 'kurtosis_33', '(_34', 'square_35', 'root_36', 'of_37', 'b1_38', 'and_39', 'b2_40', ')_41', 'and_42', 'no_43', 'serious_44', 'drawbacks_45', '._46', 'They_47', 'elicit_48', 'similar_49', 'regional_50', 'cerebral_51', 'blood_52', 'flow_53', '(_54', 'rCBF_55', ')_56', 'patterns_57', ',_58', 'even_59', 'though_60', 'sign_61', 'language_62', 'is_63', 'dependent_64', 'on_65', 'spatial_66', 'information_67', '._68', 'Siglec_69', '-_70']\n",
            "BAD:  17643\n",
            "rpws:      index         token label\n",
            "0   17714             9     I\n",
            "1   17715            is     O\n",
            "2   17716     predicted     O\n",
            "3   17717            to     O\n",
            "4   17718       contain     O\n",
            "..    ...           ...   ...\n",
            "73  17787      handling     O\n",
            "74  17788  calculations     O\n",
            "75  17789             .     O\n",
            "76  17790           The     O\n",
            "77  17791  introduction     O\n",
            "\n",
            "[78 rows x 3 columns]\n",
            "Siglec - 9 is predicted to contain three extracellular immunoglobulin - like domains that comprise an N - terminal V - set domain and two C2 - set domains , a transmembrane region and a cytoplasmic tail containing two putative tyrosine - based signaling motifs . Determination of the LD 50 ( 30 ) during infant period and growth period The availability of a commercial program ( PCNONLIN ) is needed to carry out matrix handling calculations .\n",
            "['Siglec', '-', '9', 'is', 'predicted', 'to', 'contain', 'three', 'extracellular', 'immunoglobulin', '-', 'like', 'domains', 'that', 'comprise', 'an', 'N', '-', 'terminal', 'V', '-', 'set', 'domain', 'and', 'two', 'C2', '-', 'set', 'domains', ',', 'a', 'transmembrane', 'region', 'and', 'a', 'cytoplasmic', 'tail', 'containing', 'two', 'putative', 'tyrosine', '-', 'based', 'signaling', 'motifs', '.', 'Determination', 'of', 'the', 'LD', '50', '(', '30', ')', 'during', 'infant', 'period', 'and', 'growth', 'period', 'The', 'availability', 'of', 'a', 'commercial', 'program', '(', 'PCNONLIN', ')', 'is', 'needed', 'to', 'carry', 'out', 'matrix', 'handling', 'calculations', '.']\n",
            "78 78\n",
            "['9_0', 'is_1', 'predicted_2', 'to_3', 'contain_4', 'three_5', 'extracellular_6', 'immunoglobulin_7', '-_8', 'like_9', 'domains_10', 'that_11', 'comprise_12', 'an_13', 'N_14', '-_15', 'terminal_16', 'V_17', '-_18', 'set_19', 'domain_20', 'and_21', 'two_22', 'C2_23', '-_24', 'set_25', 'domains_26', ',_27', 'a_28', 'transmembrane_29', 'region_30', 'and_31', 'a_32', 'cytoplasmic_33', 'tail_34', 'containing_35', 'two_36', 'putative_37', 'tyrosine_38', '-_39', 'based_40', 'signaling_41', 'motifs_42', '._43', 'Determination_44', 'of_45', 'the_46', 'LD_47', '50_48', '(_49', '30_50', ')_51', 'during_52', 'infant_53', 'period_54', 'and_55', 'growth_56', 'period_57', 'The_58', 'availability_59', 'of_60', 'a_61', 'commercial_62', 'program_63', '(_64', 'PCNONLIN_65', ')_66', 'is_67', 'needed_68', 'to_69', 'carry_70', 'out_71', 'matrix_72', 'handling_73', 'calculations_74', '._75', 'The_76', 'introduction_77']\n",
            "BAD:  17714\n",
            "rpws:      index        token label\n",
            "0   17792        gives     O\n",
            "1   17793         also     O\n",
            "2   17794          the     O\n",
            "3   17795      history     O\n",
            "4   17796           of     O\n",
            "..    ...          ...   ...\n",
            "66  17858          COR     B\n",
            "67  17859    synthesis     O\n",
            "68  17860            .     O\n",
            "69  17861        Model     O\n",
            "70  17862  experiments     O\n",
            "\n",
            "[71 rows x 3 columns]\n",
            "The introduction gives also the history of Research Programme MZ - XVII and its implementation in several regions of Poland . The fecal excretion is found to be only one third of that measured in urine . The COR biosynthetic gene cluster in P . syringae pv . glycinea PG4180 is encoded by a 32 - kb region which contains both the structural and regulatory genes needed for COR synthesis .\n",
            "['The', 'introduction', 'gives', 'also', 'the', 'history', 'of', 'Research', 'Programme', 'MZ', '-', 'XVII', 'and', 'its', 'implementation', 'in', 'several', 'regions', 'of', 'Poland', '.', 'The', 'fecal', 'excretion', 'is', 'found', 'to', 'be', 'only', 'one', 'third', 'of', 'that', 'measured', 'in', 'urine', '.', 'The', 'COR', 'biosynthetic', 'gene', 'cluster', 'in', 'P', '.', 'syringae', 'pv', '.', 'glycinea', 'PG4180', 'is', 'encoded', 'by', 'a', '32', '-', 'kb', 'region', 'which', 'contains', 'both', 'the', 'structural', 'and', 'regulatory', 'genes', 'needed', 'for', 'COR', 'synthesis', '.']\n",
            "71 71\n",
            "['gives_0', 'also_1', 'the_2', 'history_3', 'of_4', 'Research_5', 'Programme_6', 'MZ_7', '-_8', 'XVII_9', 'and_10', 'its_11', 'implementation_12', 'in_13', 'several_14', 'regions_15', 'of_16', 'Poland_17', '._18', 'The_19', 'fecal_20', 'excretion_21', 'is_22', 'found_23', 'to_24', 'be_25', 'only_26', 'one_27', 'third_28', 'of_29', 'that_30', 'measured_31', 'in_32', 'urine_33', '._34', 'The_35', 'COR_36', 'biosynthetic_37', 'gene_38', 'cluster_39', 'in_40', 'P_41', '._42', 'syringae_43', 'pv_44', '._45', 'glycinea_46', 'PG4180_47', 'is_48', 'encoded_49', 'by_50', 'a_51', '32_52', '-_53', 'kb_54', 'region_55', 'which_56', 'contains_57', 'both_58', 'the_59', 'structural_60', 'and_61', 'regulatory_62', 'genes_63', 'needed_64', 'for_65', 'COR_66', 'synthesis_67', '._68', 'Model_69', 'experiments_70']\n",
            "BAD:  17792\n",
            "rpws:      index         token label\n",
            "0   17863            on     O\n",
            "1   17864     pheasants     O\n",
            "2   17865         using     O\n",
            "3   17866        single     O\n",
            "4   17867         doses     O\n",
            "5   17868            of     O\n",
            "6   17869           the     O\n",
            "7   17870   insecticide     O\n",
            "8   17871             (     O\n",
            "9   17872       Lindane     O\n",
            "10  17873             )     O\n",
            "11  17874           the     O\n",
            "12  17875     herbicide     O\n",
            "13  17876             (     O\n",
            "14  17877     Terbutryn     O\n",
            "15  17878             )     O\n",
            "16  17879             a     O\n",
            "17  17880       mineral     O\n",
            "18  17881    fertilizer     O\n",
            "19  17882             (     O\n",
            "20  17883       calcium     O\n",
            "21  17884      ammonium     O\n",
            "22  17885       nitrate     O\n",
            "23  17886             )     O\n",
            "24  17887           and     O\n",
            "25  17888           the     O\n",
            "26  17889     fungicide     O\n",
            "27  17890             (     O\n",
            "28  17891           HCB     O\n",
            "29  17892             )     O\n",
            "30  17893             .     O\n",
            "31  17894            In     O\n",
            "32  17895          this     O\n",
            "33  17896     technique     O\n",
            "34  17897             ,     O\n",
            "35  17898           the     O\n",
            "36  17899     posterior     O\n",
            "37  17900          wall     O\n",
            "38  17901            of     O\n",
            "39  17902           the     O\n",
            "40  17903    neopharynx     O\n",
            "41  17904      consists     O\n",
            "42  17905          only     O\n",
            "43  17906            of     O\n",
            "44  17907           the     O\n",
            "45  17908  prevertebral     O\n",
            "46  17909        tissue     O\n",
            "47  17910             ,     O\n",
            "48  17911         while     O\n",
            "49  17912           the     O\n",
            "50  17913          flap     O\n",
            "51  17914         forms     O\n",
            "52  17915           the     O\n",
            "53  17916      anterior     O\n",
            "54  17917           and     O\n",
            "55  17918       lateral     O\n",
            "56  17919         walls     O\n",
            "57  17920             .     O\n",
            "58  17921           The     O\n",
            "59  17922          most     O\n",
            "Model experiments on pheasants using single doses of the insecticide ( Lindane ) the herbicide ( Terbutryn ) a mineral fertilizer ( calcium ammonium nitrate ) and the fungicide ( HCB ) . In this technique , the posterior wall of the neopharynx consists only of the prevertebral tissue , while the flap forms the anterior and lateral walls .\n",
            "['Model', 'experiments', 'on', 'pheasants', 'using', 'single', 'doses', 'of', 'the', 'insecticide', '(', 'Lindane', ')', 'the', 'herbicide', '(', 'Terbutryn', ')', 'a', 'mineral', 'fertilizer', '(', 'calcium', 'ammonium', 'nitrate', ')', 'and', 'the', 'fungicide', '(', 'HCB', ')', '.', 'In', 'this', 'technique', ',', 'the', 'posterior', 'wall', 'of', 'the', 'neopharynx', 'consists', 'only', 'of', 'the', 'prevertebral', 'tissue', ',', 'while', 'the', 'flap', 'forms', 'the', 'anterior', 'and', 'lateral', 'walls', '.']\n",
            "60 60\n",
            "['on_0', 'pheasants_1', 'using_2', 'single_3', 'doses_4', 'of_5', 'the_6', 'insecticide_7', '(_8', 'Lindane_9', ')_10', 'the_11', 'herbicide_12', '(_13', 'Terbutryn_14', ')_15', 'a_16', 'mineral_17', 'fertilizer_18', '(_19', 'calcium_20', 'ammonium_21', 'nitrate_22', ')_23', 'and_24', 'the_25', 'fungicide_26', '(_27', 'HCB_28', ')_29', '._30', 'In_31', 'this_32', 'technique_33', ',_34', 'the_35', 'posterior_36', 'wall_37', 'of_38', 'the_39', 'neopharynx_40', 'consists_41', 'only_42', 'of_43', 'the_44', 'prevertebral_45', 'tissue_46', ',_47', 'while_48', 'the_49', 'flap_50', 'forms_51', 'the_52', 'anterior_53', 'and_54', 'lateral_55', 'walls_56', '._57', 'The_58', 'most_59']\n",
            "BAD:  17863\n",
            "rpws:      index    token label\n",
            "0   17923   common     O\n",
            "1   17924    hosts     O\n",
            "2   17925     were     O\n",
            "3   17926  equines     O\n",
            "4   17927        (     O\n",
            "..    ...      ...   ...\n",
            "84  18007    until     O\n",
            "85  18008      96h     O\n",
            "86  18009        .     O\n",
            "87  18010     claR     B\n",
            "88  18011      and     O\n",
            "\n",
            "[89 rows x 3 columns]\n",
            "The most common hosts were equines ( 31 % ) , bovines ( 25 % ) and raccoons ( 19 % ) . A single HD session using cellulose triacetate or polysulfone membrane significantly increased water content both at forearm and lower leg ( p < 0 . 05 ) . Both claR and car are expressed as monocistronic transcripts ; the level of transcript declined rapidly after 48h in complex media , but low sustained levels of both transcripts were observed in defined GSPG medium until 96h .\n",
            "['The', 'most', 'common', 'hosts', 'were', 'equines', '(', '31', '%', ')', ',', 'bovines', '(', '25', '%', ')', 'and', 'raccoons', '(', '19', '%', ')', '.', 'A', 'single', 'HD', 'session', 'using', 'cellulose', 'triacetate', 'or', 'polysulfone', 'membrane', 'significantly', 'increased', 'water', 'content', 'both', 'at', 'forearm', 'and', 'lower', 'leg', '(', 'p', '<', '0', '.', '05', ')', '.', 'Both', 'claR', 'and', 'car', 'are', 'expressed', 'as', 'monocistronic', 'transcripts', ';', 'the', 'level', 'of', 'transcript', 'declined', 'rapidly', 'after', '48h', 'in', 'complex', 'media', ',', 'but', 'low', 'sustained', 'levels', 'of', 'both', 'transcripts', 'were', 'observed', 'in', 'defined', 'GSPG', 'medium', 'until', '96h', '.']\n",
            "89 89\n",
            "['common_0', 'hosts_1', 'were_2', 'equines_3', '(_4', '31_5', '%_6', ')_7', ',_8', 'bovines_9', '(_10', '25_11', '%_12', ')_13', 'and_14', 'raccoons_15', '(_16', '19_17', '%_18', ')_19', '._20', 'A_21', 'single_22', 'HD_23', 'session_24', 'using_25', 'cellulose_26', 'triacetate_27', 'or_28', 'polysulfone_29', 'membrane_30', 'significantly_31', 'increased_32', 'water_33', 'content_34', 'both_35', 'at_36', 'forearm_37', 'and_38', 'lower_39', 'leg_40', '(_41', 'p_42', '<_43', '0_44', '._45', '05_46', ')_47', '._48', 'Both_49', 'claR_50', 'and_51', 'car_52', 'are_53', 'expressed_54', 'as_55', 'monocistronic_56', 'transcripts_57', ';_58', 'the_59', 'level_60', 'of_61', 'transcript_62', 'declined_63', 'rapidly_64', 'after_65', '48h_66', 'in_67', 'complex_68', 'media_69', ',_70', 'but_71', 'low_72', 'sustained_73', 'levels_74', 'of_75', 'both_76', 'transcripts_77', 'were_78', 'observed_79', 'in_80', 'defined_81', 'GSPG_82', 'medium_83', 'until_84', '96h_85', '._86', 'claR_87', 'and_88']\n",
            "BAD:  17923\n",
            "rpws:      index          token label\n",
            "0   18012            car     B\n",
            "1   18013           were     O\n",
            "2   18014            not     O\n",
            "3   18015  significantly     O\n",
            "4   18016      expressed     O\n",
            "5   18017             in     O\n",
            "6   18018        mutants     O\n",
            "7   18019      disrupted     O\n",
            "8   18020             in     O\n",
            "9   18021            the     O\n",
            "10  18022           ccaR     B\n",
            "11  18023           gene     I\n",
            "12  18024              ,     O\n",
            "13  18025              a     O\n",
            "14  18026     regulatory     O\n",
            "15  18027           gene     O\n",
            "16  18028           that     O\n",
            "17  18029       controls     O\n",
            "18  18030     positively     O\n",
            "19  18031     clavulanic     O\n",
            "20  18032           acid     O\n",
            "21  18033            and     O\n",
            "22  18034     cephamycin     O\n",
            "23  18035   biosynthesis     O\n",
            "24  18036              .     O\n",
            "25  18037        Whereas     O\n",
            "26  18038         mutant     B\n",
            "27  18039            6C4     I\n",
            "28  18040      specified     O\n",
            "29  18041              a     O\n",
            "30  18042           wild     B\n",
            "31  18043              -     I\n",
            "32  18044           type     I\n",
            "33  18045              -     I\n",
            "34  18046           size     I\n",
            "35  18047            Pol     I\n",
            "36  18048        protein     I\n",
            "37  18049              ,     O\n",
            "38  18050             we     O\n",
            "39  18051       detected     O\n",
            "40  18052             no     O\n",
            "41  18053           full     B\n",
            "42  18054              -     I\n",
            "43  18055         length     I\n",
            "44  18056            Pol     I\n",
            "45  18057        protein     I\n",
            "46  18058             in     O\n",
            "47  18059            7E4     O\n",
            "48  18060              -     O\n",
            "49  18061       infected     O\n",
            "50  18062           cell     O\n",
            "51  18063       extracts     O\n",
            "52  18064              .     O\n",
            "53  18065            The     O\n",
            "54  18066        hepatic     O\n",
            "claR and car were not significantly expressed in mutants disrupted in the ccaR gene , a regulatory gene that controls positively clavulanic acid and cephamycin biosynthesis . Whereas mutant 6C4 specified a wild - type - size Pol protein , we detected no full - length Pol protein in 7E4 - infected cell extracts .\n",
            "['claR', 'and', 'car', 'were', 'not', 'significantly', 'expressed', 'in', 'mutants', 'disrupted', 'in', 'the', 'ccaR', 'gene', ',', 'a', 'regulatory', 'gene', 'that', 'controls', 'positively', 'clavulanic', 'acid', 'and', 'cephamycin', 'biosynthesis', '.', 'Whereas', 'mutant', '6C4', 'specified', 'a', 'wild', '-', 'type', '-', 'size', 'Pol', 'protein', ',', 'we', 'detected', 'no', 'full', '-', 'length', 'Pol', 'protein', 'in', '7E4', '-', 'infected', 'cell', 'extracts', '.']\n",
            "55 55\n",
            "['car_0', 'were_1', 'not_2', 'significantly_3', 'expressed_4', 'in_5', 'mutants_6', 'disrupted_7', 'in_8', 'the_9', 'ccaR_10', 'gene_11', ',_12', 'a_13', 'regulatory_14', 'gene_15', 'that_16', 'controls_17', 'positively_18', 'clavulanic_19', 'acid_20', 'and_21', 'cephamycin_22', 'biosynthesis_23', '._24', 'Whereas_25', 'mutant_26', '6C4_27', 'specified_28', 'a_29', 'wild_30', '-_31', 'type_32', '-_33', 'size_34', 'Pol_35', 'protein_36', ',_37', 'we_38', 'detected_39', 'no_40', 'full_41', '-_42', 'length_43', 'Pol_44', 'protein_45', 'in_46', '7E4_47', '-_48', 'infected_49', 'cell_50', 'extracts_51', '._52', 'The_53', 'hepatic_54']\n",
            "rpws:      index           token label\n",
            "0   18067         isoform     O\n",
            "1   18068              of     O\n",
            "2   18069               6     B\n",
            "3   18070               -     I\n",
            "4   18071   phosphofructo     I\n",
            "..    ...             ...   ...\n",
            "76  18143            cell     O\n",
            "77  18144       functions     O\n",
            "78  18145               .     O\n",
            "79  18146             The     O\n",
            "80  18147  Caenorhabditis     B\n",
            "\n",
            "[81 rows x 3 columns]\n",
            "The hepatic isoform of 6 - phosphofructo - 2 - kinase / fructose - 2 , 6 - bisphosphatase ( PF2K / Fru - 2 , 6 - BPase ) is transcriptionally stimulated by glucocorticoids , whereas insulin blocks this stimulatory effect . Being implicated in insulin and GK gene regulations as a common transcription factor , IPF1 / STF - 1 / PDX - 1 is likely to play an essential role in maintaining normal beta - cell functions .\n",
            "['The', 'hepatic', 'isoform', 'of', '6', '-', 'phosphofructo', '-', '2', '-', 'kinase', '/', 'fructose', '-', '2', ',', '6', '-', 'bisphosphatase', '(', 'PF2K', '/', 'Fru', '-', '2', ',', '6', '-', 'BPase', ')', 'is', 'transcriptionally', 'stimulated', 'by', 'glucocorticoids', ',', 'whereas', 'insulin', 'blocks', 'this', 'stimulatory', 'effect', '.', 'Being', 'implicated', 'in', 'insulin', 'and', 'GK', 'gene', 'regulations', 'as', 'a', 'common', 'transcription', 'factor', ',', 'IPF1', '/', 'STF', '-', '1', '/', 'PDX', '-', '1', 'is', 'likely', 'to', 'play', 'an', 'essential', 'role', 'in', 'maintaining', 'normal', 'beta', '-', 'cell', 'functions', '.']\n",
            "81 81\n",
            "['isoform_0', 'of_1', '6_2', '-_3', 'phosphofructo_4', '-_5', '2_6', '-_7', 'kinase_8', '/_9', 'fructose_10', '-_11', '2_12', ',_13', '6_14', '-_15', 'bisphosphatase_16', '(_17', 'PF2K_18', '/_19', 'Fru_20', '-_21', '2_22', ',_23', '6_24', '-_25', 'BPase_26', ')_27', 'is_28', 'transcriptionally_29', 'stimulated_30', 'by_31', 'glucocorticoids_32', ',_33', 'whereas_34', 'insulin_35', 'blocks_36', 'this_37', 'stimulatory_38', 'effect_39', '._40', 'Being_41', 'implicated_42', 'in_43', 'insulin_44', 'and_45', 'GK_46', 'gene_47', 'regulations_48', 'as_49', 'a_50', 'common_51', 'transcription_52', 'factor_53', ',_54', 'IPF1_55', '/_56', 'STF_57', '-_58', '1_59', '/_60', 'PDX_61', '-_62', '1_63', 'is_64', 'likely_65', 'to_66', 'play_67', 'an_68', 'essential_69', 'role_70', 'in_71', 'maintaining_72', 'normal_73', 'beta_74', '-_75', 'cell_76', 'functions_77', '._78', 'The_79', 'Caenorhabditis_80']\n",
            "BAD:  18067\n",
            "rpws:      index    token label\n",
            "0   18148  elegans     I\n",
            "1   18149       NK     I\n",
            "2   18150        -     I\n",
            "3   18151        2     I\n",
            "4   18152    class     I\n",
            "..    ...      ...   ...\n",
            "73  18221        =     O\n",
            "74  18222        5     O\n",
            "75  18223        .     O\n",
            "76  18224        6     O\n",
            "77  18225        ,     O\n",
            "\n",
            "[78 rows x 3 columns]\n",
            "The Caenorhabditis elegans NK - 2 class homeoprotein CEH - 22 is involved in combinatorial activation of gene expression in pharyngeal muscle . Mouse embryo cells ( C57BL / 6 , H - 2b ) transformed by the E1A and E1B genes of adenovirus type 5 ( Ad5E1 MEC ) are highly immunogenic . The largest age adjusted differences between men with low and normal mood were for the AH4 ( 3 points , t = 5 .\n",
            "['The', 'Caenorhabditis', 'elegans', 'NK', '-', '2', 'class', 'homeoprotein', 'CEH', '-', '22', 'is', 'involved', 'in', 'combinatorial', 'activation', 'of', 'gene', 'expression', 'in', 'pharyngeal', 'muscle', '.', 'Mouse', 'embryo', 'cells', '(', 'C57BL', '/', '6', ',', 'H', '-', '2b', ')', 'transformed', 'by', 'the', 'E1A', 'and', 'E1B', 'genes', 'of', 'adenovirus', 'type', '5', '(', 'Ad5E1', 'MEC', ')', 'are', 'highly', 'immunogenic', '.', 'The', 'largest', 'age', 'adjusted', 'differences', 'between', 'men', 'with', 'low', 'and', 'normal', 'mood', 'were', 'for', 'the', 'AH4', '(', '3', 'points', ',', 't', '=', '5', '.']\n",
            "78 78\n",
            "['elegans_0', 'NK_1', '-_2', '2_3', 'class_4', 'homeoprotein_5', 'CEH_6', '-_7', '22_8', 'is_9', 'involved_10', 'in_11', 'combinatorial_12', 'activation_13', 'of_14', 'gene_15', 'expression_16', 'in_17', 'pharyngeal_18', 'muscle_19', '._20', 'Mouse_21', 'embryo_22', 'cells_23', '(_24', 'C57BL_25', '/_26', '6_27', ',_28', 'H_29', '-_30', '2b_31', ')_32', 'transformed_33', 'by_34', 'the_35', 'E1A_36', 'and_37', 'E1B_38', 'genes_39', 'of_40', 'adenovirus_41', 'type_42', '5_43', '(_44', 'Ad5E1_45', 'MEC_46', ')_47', 'are_48', 'highly_49', 'immunogenic_50', '._51', 'The_52', 'largest_53', 'age_54', 'adjusted_55', 'differences_56', 'between_57', 'men_58', 'with_59', 'low_60', 'and_61', 'normal_62', 'mood_63', 'were_64', 'for_65', 'the_66', 'AH4_67', '(_68', '3_69', 'points_70', ',_71', 't_72', '=_73', '5_74', '._75', '6_76', ',_77']\n",
            "BAD:  18148\n",
            "rpws:      index    token label\n",
            "0   18226        p     O\n",
            "1   18227        <     O\n",
            "2   18228        0     O\n",
            "3   18229        .     O\n",
            "4   18230     0001     O\n",
            "..    ...      ...   ...\n",
            "84  18310  culture     O\n",
            "85  18311    cells     O\n",
            "86  18312        .     O\n",
            "87  18313  RESULTS     O\n",
            "88  18314        :     O\n",
            "\n",
            "[89 rows x 3 columns]\n",
            "6 , p < 0 . 0001 ) and the CAMCOG ( 2 points , t = 5 . 8 , p < 0 . 0001 ) . Our findings suggest that resting Tl - 201 scintigraphy has limited value in the detection of coronary artery disease in patients with Hurler syndrome . We demonstrate that the WRM - 1 protein binds to LIT - 1 in vivo and that WRM - 1 can activate the LIT - 1 protein kinase when coexpressed in vertebrate tissue culture cells .\n",
            "['6', ',', 'p', '<', '0', '.', '0001', ')', 'and', 'the', 'CAMCOG', '(', '2', 'points', ',', 't', '=', '5', '.', '8', ',', 'p', '<', '0', '.', '0001', ')', '.', 'Our', 'findings', 'suggest', 'that', 'resting', 'Tl', '-', '201', 'scintigraphy', 'has', 'limited', 'value', 'in', 'the', 'detection', 'of', 'coronary', 'artery', 'disease', 'in', 'patients', 'with', 'Hurler', 'syndrome', '.', 'We', 'demonstrate', 'that', 'the', 'WRM', '-', '1', 'protein', 'binds', 'to', 'LIT', '-', '1', 'in', 'vivo', 'and', 'that', 'WRM', '-', '1', 'can', 'activate', 'the', 'LIT', '-', '1', 'protein', 'kinase', 'when', 'coexpressed', 'in', 'vertebrate', 'tissue', 'culture', 'cells', '.']\n",
            "89 89\n",
            "['p_0', '<_1', '0_2', '._3', '0001_4', ')_5', 'and_6', 'the_7', 'CAMCOG_8', '(_9', '2_10', 'points_11', ',_12', 't_13', '=_14', '5_15', '._16', '8_17', ',_18', 'p_19', '<_20', '0_21', '._22', '0001_23', ')_24', '._25', 'Our_26', 'findings_27', 'suggest_28', 'that_29', 'resting_30', 'Tl_31', '-_32', '201_33', 'scintigraphy_34', 'has_35', 'limited_36', 'value_37', 'in_38', 'the_39', 'detection_40', 'of_41', 'coronary_42', 'artery_43', 'disease_44', 'in_45', 'patients_46', 'with_47', 'Hurler_48', 'syndrome_49', '._50', 'We_51', 'demonstrate_52', 'that_53', 'the_54', 'WRM_55', '-_56', '1_57', 'protein_58', 'binds_59', 'to_60', 'LIT_61', '-_62', '1_63', 'in_64', 'vivo_65', 'and_66', 'that_67', 'WRM_68', '-_69', '1_70', 'can_71', 'activate_72', 'the_73', 'LIT_74', '-_75', '1_76', 'protein_77', 'kinase_78', 'when_79', 'coexpressed_80', 'in_81', 'vertebrate_82', 'tissue_83', 'culture_84', 'cells_85', '._86', 'RESULTS_87', ':_88']\n",
            "BAD:  18226\n",
            "rpws:      index      token label\n",
            "0   18315        All     O\n",
            "1   18316    primary     O\n",
            "2   18317     tumors     O\n",
            "3   18318       were     O\n",
            "4   18319   positive     O\n",
            "..    ...        ...   ...\n",
            "69  18384      still     O\n",
            "70  18385   retarded     O\n",
            "71  18386          .     O\n",
            "72  18387         To     O\n",
            "73  18388  determine     O\n",
            "\n",
            "[74 rows x 3 columns]\n",
            "RESULTS : All primary tumors were positive for CAM5 . 2 . Nuclear extracts from Sertoli cells were found to cause an E - box gel shift when the cells were stimulated to differentiate in culture , but not under basal conditions . On the sixth postirradiation day the absorption of phenobarbitone , sulphafurazole and mecamylamine had returned to the control level , but the absorption of quinidine and isoniazid was still retarded .\n",
            "['RESULTS', ':', 'All', 'primary', 'tumors', 'were', 'positive', 'for', 'CAM5', '.', '2', '.', 'Nuclear', 'extracts', 'from', 'Sertoli', 'cells', 'were', 'found', 'to', 'cause', 'an', 'E', '-', 'box', 'gel', 'shift', 'when', 'the', 'cells', 'were', 'stimulated', 'to', 'differentiate', 'in', 'culture', ',', 'but', 'not', 'under', 'basal', 'conditions', '.', 'On', 'the', 'sixth', 'postirradiation', 'day', 'the', 'absorption', 'of', 'phenobarbitone', ',', 'sulphafurazole', 'and', 'mecamylamine', 'had', 'returned', 'to', 'the', 'control', 'level', ',', 'but', 'the', 'absorption', 'of', 'quinidine', 'and', 'isoniazid', 'was', 'still', 'retarded', '.']\n",
            "74 74\n",
            "['All_0', 'primary_1', 'tumors_2', 'were_3', 'positive_4', 'for_5', 'CAM5_6', '._7', '2_8', '._9', 'Nuclear_10', 'extracts_11', 'from_12', 'Sertoli_13', 'cells_14', 'were_15', 'found_16', 'to_17', 'cause_18', 'an_19', 'E_20', '-_21', 'box_22', 'gel_23', 'shift_24', 'when_25', 'the_26', 'cells_27', 'were_28', 'stimulated_29', 'to_30', 'differentiate_31', 'in_32', 'culture_33', ',_34', 'but_35', 'not_36', 'under_37', 'basal_38', 'conditions_39', '._40', 'On_41', 'the_42', 'sixth_43', 'postirradiation_44', 'day_45', 'the_46', 'absorption_47', 'of_48', 'phenobarbitone_49', ',_50', 'sulphafurazole_51', 'and_52', 'mecamylamine_53', 'had_54', 'returned_55', 'to_56', 'the_57', 'control_58', 'level_59', ',_60', 'but_61', 'the_62', 'absorption_63', 'of_64', 'quinidine_65', 'and_66', 'isoniazid_67', 'was_68', 'still_69', 'retarded_70', '._71', 'To_72', 'determine_73']\n",
            "BAD:  18315\n",
            "rpws:      index            token label\n",
            "0   18389              the     O\n",
            "1   18390  transactivation     O\n",
            "2   18391        potential     O\n",
            "3   18392               of     O\n",
            "4   18393             each     O\n",
            "..    ...              ...   ...\n",
            "70  18459            E695K     O\n",
            "71  18460                )     O\n",
            "72  18461                .     O\n",
            "73  18462              The     O\n",
            "74  18463       organotins     O\n",
            "\n",
            "[75 rows x 3 columns]\n",
            "To determine the transactivation potential of each of the four Ahr Sp1 sites , we fused the Ahr promoter to a luciferase ( LUC ) reporter gene and transfected the construct into the Drosophila cell line Schneider - 2 , which contains no Sp1 or Sp1 - like factors . We determined that Hop ( T42 ) contains a glutamic acid - to - lysine substitution at amino acid residue 695 ( E695K ) .\n",
            "['To', 'determine', 'the', 'transactivation', 'potential', 'of', 'each', 'of', 'the', 'four', 'Ahr', 'Sp1', 'sites', ',', 'we', 'fused', 'the', 'Ahr', 'promoter', 'to', 'a', 'luciferase', '(', 'LUC', ')', 'reporter', 'gene', 'and', 'transfected', 'the', 'construct', 'into', 'the', 'Drosophila', 'cell', 'line', 'Schneider', '-', '2', ',', 'which', 'contains', 'no', 'Sp1', 'or', 'Sp1', '-', 'like', 'factors', '.', 'We', 'determined', 'that', 'Hop', '(', 'T42', ')', 'contains', 'a', 'glutamic', 'acid', '-', 'to', '-', 'lysine', 'substitution', 'at', 'amino', 'acid', 'residue', '695', '(', 'E695K', ')', '.']\n",
            "75 75\n",
            "['the_0', 'transactivation_1', 'potential_2', 'of_3', 'each_4', 'of_5', 'the_6', 'four_7', 'Ahr_8', 'Sp1_9', 'sites_10', ',_11', 'we_12', 'fused_13', 'the_14', 'Ahr_15', 'promoter_16', 'to_17', 'a_18', 'luciferase_19', '(_20', 'LUC_21', ')_22', 'reporter_23', 'gene_24', 'and_25', 'transfected_26', 'the_27', 'construct_28', 'into_29', 'the_30', 'Drosophila_31', 'cell_32', 'line_33', 'Schneider_34', '-_35', '2_36', ',_37', 'which_38', 'contains_39', 'no_40', 'Sp1_41', 'or_42', 'Sp1_43', '-_44', 'like_45', 'factors_46', '._47', 'We_48', 'determined_49', 'that_50', 'Hop_51', '(_52', 'T42_53', ')_54', 'contains_55', 'a_56', 'glutamic_57', 'acid_58', '-_59', 'to_60', '-_61', 'lysine_62', 'substitution_63', 'at_64', 'amino_65', 'acid_66', 'residue_67', '695_68', '(_69', 'E695K_70', ')_71', '._72', 'The_73', 'organotins_74']\n",
            "BAD:  18389\n",
            "rpws:      index       token label\n",
            "0   18464           .     O\n",
            "1   18465     Results     O\n",
            "2   18466          of     O\n",
            "3   18467       these     O\n",
            "4   18468     studies     O\n",
            "..    ...         ...   ...\n",
            "63  18527        with     O\n",
            "64  18528       Shr3p     B\n",
            "65  18529           .     O\n",
            "66  18530         The     O\n",
            "67  18531  impairment     O\n",
            "\n",
            "[68 rows x 3 columns]\n",
            "The organotins . Results of these studies indicate that binding of biotin to the protein results in protection of regions of the central domain in the vicinity of the active site and the C - terminal domain from chemical cleavage . The psh3 ( + ) gene encodes a protein of 215 amino acids , which shares a high degree of structural and functional similarity with Shr3p .\n",
            "['The', 'organotins', '.', 'Results', 'of', 'these', 'studies', 'indicate', 'that', 'binding', 'of', 'biotin', 'to', 'the', 'protein', 'results', 'in', 'protection', 'of', 'regions', 'of', 'the', 'central', 'domain', 'in', 'the', 'vicinity', 'of', 'the', 'active', 'site', 'and', 'the', 'C', '-', 'terminal', 'domain', 'from', 'chemical', 'cleavage', '.', 'The', 'psh3', '(', '+', ')', 'gene', 'encodes', 'a', 'protein', 'of', '215', 'amino', 'acids', ',', 'which', 'shares', 'a', 'high', 'degree', 'of', 'structural', 'and', 'functional', 'similarity', 'with', 'Shr3p', '.']\n",
            "68 68\n",
            "['._0', 'Results_1', 'of_2', 'these_3', 'studies_4', 'indicate_5', 'that_6', 'binding_7', 'of_8', 'biotin_9', 'to_10', 'the_11', 'protein_12', 'results_13', 'in_14', 'protection_15', 'of_16', 'regions_17', 'of_18', 'the_19', 'central_20', 'domain_21', 'in_22', 'the_23', 'vicinity_24', 'of_25', 'the_26', 'active_27', 'site_28', 'and_29', 'the_30', 'C_31', '-_32', 'terminal_33', 'domain_34', 'from_35', 'chemical_36', 'cleavage_37', '._38', 'The_39', 'psh3_40', '(_41', '+_42', ')_43', 'gene_44', 'encodes_45', 'a_46', 'protein_47', 'of_48', '215_49', 'amino_50', 'acids_51', ',_52', 'which_53', 'shares_54', 'a_55', 'high_56', 'degree_57', 'of_58', 'structural_59', 'and_60', 'functional_61', 'similarity_62', 'with_63', 'Shr3p_64', '._65', 'The_66', 'impairment_67']\n",
            "BAD:  18464\n",
            "rpws:      index      token label\n",
            "0   18532         of     O\n",
            "1   18533        the     O\n",
            "2   18534  nocturnal     O\n",
            "3   18535  secretion     O\n",
            "4   18536        was     O\n",
            "..    ...        ...   ...\n",
            "73  18605         to     O\n",
            "74  18606      cells     O\n",
            "75  18607          .     O\n",
            "76  18608        The     O\n",
            "77  18609    chicken     O\n",
            "\n",
            "[78 rows x 3 columns]\n",
            "The impairment of the nocturnal secretion was related to the subjects ' age and , for the GH secretory pattern only , also to the MMSE score . Thus , the antibody class switch appears to be directed by induction of accessibility , as assayed by transcription of germ line CH genes . This growth - inhibitory effect was suppressed by the mpk1 delta mutation , suggesting that hyperactivation of the Mpk1 pathway is toxic to cells .\n",
            "['The', 'impairment', 'of', 'the', 'nocturnal', 'secretion', 'was', 'related', 'to', 'the', 'subjects', \"'\", 'age', 'and', ',', 'for', 'the', 'GH', 'secretory', 'pattern', 'only', ',', 'also', 'to', 'the', 'MMSE', 'score', '.', 'Thus', ',', 'the', 'antibody', 'class', 'switch', 'appears', 'to', 'be', 'directed', 'by', 'induction', 'of', 'accessibility', ',', 'as', 'assayed', 'by', 'transcription', 'of', 'germ', 'line', 'CH', 'genes', '.', 'This', 'growth', '-', 'inhibitory', 'effect', 'was', 'suppressed', 'by', 'the', 'mpk1', 'delta', 'mutation', ',', 'suggesting', 'that', 'hyperactivation', 'of', 'the', 'Mpk1', 'pathway', 'is', 'toxic', 'to', 'cells', '.']\n",
            "78 78\n",
            "['of_0', 'the_1', 'nocturnal_2', 'secretion_3', 'was_4', 'related_5', 'to_6', 'the_7', 'subjects_8', \"'_9\", 'age_10', 'and_11', ',_12', 'for_13', 'the_14', 'GH_15', 'secretory_16', 'pattern_17', 'only_18', ',_19', 'also_20', 'to_21', 'the_22', 'MMSE_23', 'score_24', '._25', 'Thus_26', ',_27', 'the_28', 'antibody_29', 'class_30', 'switch_31', 'appears_32', 'to_33', 'be_34', 'directed_35', 'by_36', 'induction_37', 'of_38', 'accessibility_39', ',_40', 'as_41', 'assayed_42', 'by_43', 'transcription_44', 'of_45', 'germ_46', 'line_47', 'CH_48', 'genes_49', '._50', 'This_51', 'growth_52', '-_53', 'inhibitory_54', 'effect_55', 'was_56', 'suppressed_57', 'by_58', 'the_59', 'mpk1_60', 'delta_61', 'mutation_62', ',_63', 'suggesting_64', 'that_65', 'hyperactivation_66', 'of_67', 'the_68', 'Mpk1_69', 'pathway_70', 'is_71', 'toxic_72', 'to_73', 'cells_74', '._75', 'The_76', 'chicken_77']\n",
            "BAD:  18532\n",
            "rpws:      index       token label\n",
            "0   18610      genome     O\n",
            "1   18611    contains     O\n",
            "2   18612         two     O\n",
            "3   18613  functional     O\n",
            "4   18614  nonallelic     O\n",
            "..    ...         ...   ...\n",
            "59  18669          in     O\n",
            "60  18670       vitro     O\n",
            "61  18671           .     O\n",
            "62  18672  Endodontic     O\n",
            "63  18673   treatment     O\n",
            "\n",
            "[64 rows x 3 columns]\n",
            "The chicken genome contains two functional nonallelic beta1 , 4 - galactosyltransferase genes . Recombinant Human Erythropoietin and Platinum - Based Chemotherapy In Advanced Ovarian Cancer Homodimers of RIP60 ( replication initiation - region protein 60 kDA ) purified from nuclear extract bind two ATT - rich sites in oribeta and foster the formation of a twisted 720 bp DNA loop in vitro .\n",
            "['The', 'chicken', 'genome', 'contains', 'two', 'functional', 'nonallelic', 'beta1', ',', '4', '-', 'galactosyltransferase', 'genes', '.', 'Recombinant', 'Human', 'Erythropoietin', 'and', 'Platinum', '-', 'Based', 'Chemotherapy', 'In', 'Advanced', 'Ovarian', 'Cancer', 'Homodimers', 'of', 'RIP60', '(', 'replication', 'initiation', '-', 'region', 'protein', '60', 'kDA', ')', 'purified', 'from', 'nuclear', 'extract', 'bind', 'two', 'ATT', '-', 'rich', 'sites', 'in', 'oribeta', 'and', 'foster', 'the', 'formation', 'of', 'a', 'twisted', '720', 'bp', 'DNA', 'loop', 'in', 'vitro', '.']\n",
            "64 64\n",
            "['genome_0', 'contains_1', 'two_2', 'functional_3', 'nonallelic_4', 'beta1_5', ',_6', '4_7', '-_8', 'galactosyltransferase_9', 'genes_10', '._11', 'Recombinant_12', 'Human_13', 'Erythropoietin_14', 'and_15', 'Platinum_16', '-_17', 'Based_18', 'Chemotherapy_19', 'In_20', 'Advanced_21', 'Ovarian_22', 'Cancer_23', 'Homodimers_24', 'of_25', 'RIP60_26', '(_27', 'replication_28', 'initiation_29', '-_30', 'region_31', 'protein_32', '60_33', 'kDA_34', ')_35', 'purified_36', 'from_37', 'nuclear_38', 'extract_39', 'bind_40', 'two_41', 'ATT_42', '-_43', 'rich_44', 'sites_45', 'in_46', 'oribeta_47', 'and_48', 'foster_49', 'the_50', 'formation_51', 'of_52', 'a_53', 'twisted_54', '720_55', 'bp_56', 'DNA_57', 'loop_58', 'in_59', 'vitro_60', '._61', 'Endodontic_62', 'treatment_63']\n",
            "BAD:  18610\n",
            "rpws:      index      token label\n",
            "0   18674         of     O\n",
            "1   18675  deciduous     O\n",
            "2   18676      teeth     O\n",
            "3   18677      using     O\n",
            "4   18678        the     O\n",
            "..    ...        ...   ...\n",
            "59  18733  quiescent     O\n",
            "60  18734       SMCs     O\n",
            "61  18735          .     O\n",
            "62  18736    Overall     O\n",
            "63  18737      graft     O\n",
            "\n",
            "[64 rows x 3 columns]\n",
            "Endodontic treatment of deciduous teeth using the formocresol amputation method Some of these physiological responses are regulated via activation of transcription factors such as activator protein 1 ( AP - 1 ) . To test the role of myb family members in progression through the cell cycle , we comicroinjected c - myc and myb expression vectors into serum - deprived quiescent SMCs .\n",
            "['Endodontic', 'treatment', 'of', 'deciduous', 'teeth', 'using', 'the', 'formocresol', 'amputation', 'method', 'Some', 'of', 'these', 'physiological', 'responses', 'are', 'regulated', 'via', 'activation', 'of', 'transcription', 'factors', 'such', 'as', 'activator', 'protein', '1', '(', 'AP', '-', '1', ')', '.', 'To', 'test', 'the', 'role', 'of', 'myb', 'family', 'members', 'in', 'progression', 'through', 'the', 'cell', 'cycle', ',', 'we', 'comicroinjected', 'c', '-', 'myc', 'and', 'myb', 'expression', 'vectors', 'into', 'serum', '-', 'deprived', 'quiescent', 'SMCs', '.']\n",
            "64 64\n",
            "['of_0', 'deciduous_1', 'teeth_2', 'using_3', 'the_4', 'formocresol_5', 'amputation_6', 'method_7', 'Some_8', 'of_9', 'these_10', 'physiological_11', 'responses_12', 'are_13', 'regulated_14', 'via_15', 'activation_16', 'of_17', 'transcription_18', 'factors_19', 'such_20', 'as_21', 'activator_22', 'protein_23', '1_24', '(_25', 'AP_26', '-_27', '1_28', ')_29', '._30', 'To_31', 'test_32', 'the_33', 'role_34', 'of_35', 'myb_36', 'family_37', 'members_38', 'in_39', 'progression_40', 'through_41', 'the_42', 'cell_43', 'cycle_44', ',_45', 'we_46', 'comicroinjected_47', 'c_48', '-_49', 'myc_50', 'and_51', 'myb_52', 'expression_53', 'vectors_54', 'into_55', 'serum_56', '-_57', 'deprived_58', 'quiescent_59', 'SMCs_60', '._61', 'Overall_62', 'graft_63']\n",
            "BAD:  18674\n",
            "rpws:      index     token label\n",
            "0   18738       and     O\n",
            "1   18739   patient     O\n",
            "2   18740  survival     O\n",
            "3   18741     after     O\n",
            "4   18742       HAT     O\n",
            "..    ...       ...   ...\n",
            "56  18794        f1     O\n",
            "57  18795    allele     O\n",
            "58  18796         .     O\n",
            "59  18797       The     O\n",
            "60  18798      same     O\n",
            "\n",
            "[61 rows x 3 columns]\n",
            "Overall graft and patient survival after HAT were 33 . 3 % and 75 % , respectively . LAC9 is a DNA - binding protein that regulates transcription of the lactose - galactose regulon in Kluyveromyces lactis . We determined the complete nucleotide sequence of the gypsy element present at the forked locus of Drosophila melanogaster in the f1 allele .\n",
            "['Overall', 'graft', 'and', 'patient', 'survival', 'after', 'HAT', 'were', '33', '.', '3', '%', 'and', '75', '%', ',', 'respectively', '.', 'LAC9', 'is', 'a', 'DNA', '-', 'binding', 'protein', 'that', 'regulates', 'transcription', 'of', 'the', 'lactose', '-', 'galactose', 'regulon', 'in', 'Kluyveromyces', 'lactis', '.', 'We', 'determined', 'the', 'complete', 'nucleotide', 'sequence', 'of', 'the', 'gypsy', 'element', 'present', 'at', 'the', 'forked', 'locus', 'of', 'Drosophila', 'melanogaster', 'in', 'the', 'f1', 'allele', '.']\n",
            "61 61\n",
            "['and_0', 'patient_1', 'survival_2', 'after_3', 'HAT_4', 'were_5', '33_6', '._7', '3_8', '%_9', 'and_10', '75_11', '%_12', ',_13', 'respectively_14', '._15', 'LAC9_16', 'is_17', 'a_18', 'DNA_19', '-_20', 'binding_21', 'protein_22', 'that_23', 'regulates_24', 'transcription_25', 'of_26', 'the_27', 'lactose_28', '-_29', 'galactose_30', 'regulon_31', 'in_32', 'Kluyveromyces_33', 'lactis_34', '._35', 'We_36', 'determined_37', 'the_38', 'complete_39', 'nucleotide_40', 'sequence_41', 'of_42', 'the_43', 'gypsy_44', 'element_45', 'present_46', 'at_47', 'the_48', 'forked_49', 'locus_50', 'of_51', 'Drosophila_52', 'melanogaster_53', 'in_54', 'the_55', 'f1_56', 'allele_57', '._58', 'The_59', 'same_60']\n",
            "BAD:  18738\n",
            "rpws:      index     token label\n",
            "0   18799      high     O\n",
            "1   18800    degree     O\n",
            "2   18801        of     O\n",
            "3   18802  sequence     O\n",
            "4   18803  homology     O\n",
            "..    ...       ...   ...\n",
            "82  18881         .     O\n",
            "83  18882         A     O\n",
            "84  18883     group     O\n",
            "85  18884        of     O\n",
            "86  18885   factors     O\n",
            "\n",
            "[87 rows x 3 columns]\n",
            "The same high degree of sequence homology between the two F . diplosiphon PC alpha and PC beta sequences ( 85 and 77 % , respectively ) was found at both the nucleotide and amino acid levels , and similar results were obtained for interspecies comparisons . A wound model for decubitus and leg ulcers consisting of human dermal fibroblasts in type I collagen dermal \tO\n",
            "equivalent\tO\n",
            " matrix ( DEM ) was exposed in vitro to electric fields similar to postulated endogenous fields in wounds .\n",
            "['The', 'same', 'high', 'degree', 'of', 'sequence', 'homology', 'between', 'the', 'two', 'F', '.', 'diplosiphon', 'PC', 'alpha', 'and', 'PC', 'beta', 'sequences', '(', '85', 'and', '77', '%', ',', 'respectively', ')', 'was', 'found', 'at', 'both', 'the', 'nucleotide', 'and', 'amino', 'acid', 'levels', ',', 'and', 'similar', 'results', 'were', 'obtained', 'for', 'interspecies', 'comparisons', '.', 'A', 'wound', 'model', 'for', 'decubitus', 'and', 'leg', 'ulcers', 'consisting', 'of', 'human', 'dermal', 'fibroblasts', 'in', 'type', 'I', 'collagen', 'dermal', 'O', 'equivalent', 'O', 'matrix', '(', 'DEM', ')', 'was', 'exposed', 'in', 'vitro', 'to', 'electric', 'fields', 'similar', 'to', 'postulated', 'endogenous', 'fields', 'in', 'wounds', '.']\n",
            "87 87\n",
            "['high_0', 'degree_1', 'of_2', 'sequence_3', 'homology_4', 'between_5', 'the_6', 'two_7', 'F_8', '._9', 'diplosiphon_10', 'PC_11', 'alpha_12', 'and_13', 'PC_14', 'beta_15', 'sequences_16', '(_17', '85_18', 'and_19', '77_20', '%_21', ',_22', 'respectively_23', ')_24', 'was_25', 'found_26', 'at_27', 'both_28', 'the_29', 'nucleotide_30', 'and_31', 'amino_32', 'acid_33', 'levels_34', ',_35', 'and_36', 'similar_37', 'results_38', 'were_39', 'obtained_40', 'for_41', 'interspecies_42', 'comparisons_43', '._44', 'A_45', 'wound_46', 'model_47', 'for_48', 'decubitus_49', 'and_50', 'leg_51', 'ulcers_52', 'consisting_53', 'of_54', 'human_55', 'dermal_56', 'fibroblasts_57', 'in_58', 'type_59', 'I_60', 'collagen_61', 'dermal_62', '\\tO\\nequivalent\\tO\\n_63', 'matrix_64', '(_65', 'DEM_66', ')_67', 'was_68', 'exposed_69', 'in_70', 'vitro_71', 'to_72', 'electric_73', 'fields_74', 'similar_75', 'to_76', 'postulated_77', 'endogenous_78', 'fields_79', 'in_80', 'wounds_81', '._82', 'A_83', 'group_84', 'of_85', 'factors_86']\n",
            "BAD:  18799\n",
            "rpws:      index          token label\n",
            "0   18886          known     O\n",
            "1   18887             as     O\n",
            "2   18888     activating     B\n",
            "3   18889  transcription     I\n",
            "4   18890        factors     I\n",
            "..    ...            ...   ...\n",
            "68  18954              .     O\n",
            "69  18955            The     O\n",
            "70  18956     vertebrate     B\n",
            "71  18957  transcription     I\n",
            "72  18958        factors     I\n",
            "\n",
            "[73 rows x 3 columns]\n",
            "A group of factors known as activating transcription factors ( ATF ) have been found to bind to the latter and related sequences found upstream of early adenovirus promoters induced by E1A , and these factors are highly homologous to the CREB protein . In addition to these cases , patients w with high Loa microfilaremia also developed milder neurologic manifestations causing functional impairment lasting for at least one week after treatment .\n",
            "['A', 'group', 'of', 'factors', 'known', 'as', 'activating', 'transcription', 'factors', '(', 'ATF', ')', 'have', 'been', 'found', 'to', 'bind', 'to', 'the', 'latter', 'and', 'related', 'sequences', 'found', 'upstream', 'of', 'early', 'adenovirus', 'promoters', 'induced', 'by', 'E1A', ',', 'and', 'these', 'factors', 'are', 'highly', 'homologous', 'to', 'the', 'CREB', 'protein', '.', 'In', 'addition', 'to', 'these', 'cases', ',', 'patients', 'w', 'with', 'high', 'Loa', 'microfilaremia', 'also', 'developed', 'milder', 'neurologic', 'manifestations', 'causing', 'functional', 'impairment', 'lasting', 'for', 'at', 'least', 'one', 'week', 'after', 'treatment', '.']\n",
            "73 73\n",
            "['known_0', 'as_1', 'activating_2', 'transcription_3', 'factors_4', '(_5', 'ATF_6', ')_7', 'have_8', 'been_9', 'found_10', 'to_11', 'bind_12', 'to_13', 'the_14', 'latter_15', 'and_16', 'related_17', 'sequences_18', 'found_19', 'upstream_20', 'of_21', 'early_22', 'adenovirus_23', 'promoters_24', 'induced_25', 'by_26', 'E1A_27', ',_28', 'and_29', 'these_30', 'factors_31', 'are_32', 'highly_33', 'homologous_34', 'to_35', 'the_36', 'CREB_37', 'protein_38', '._39', 'In_40', 'addition_41', 'to_42', 'these_43', 'cases_44', ',_45', 'patients_46', 'w_47', 'with_48', 'high_49', 'Loa_50', 'microfilaremia_51', 'also_52', 'developed_53', 'milder_54', 'neurologic_55', 'manifestations_56', 'causing_57', 'functional_58', 'impairment_59', 'lasting_60', 'for_61', 'at_62', 'least_63', 'one_64', 'week_65', 'after_66', 'treatment_67', '._68', 'The_69', 'vertebrate_70', 'transcription_71', 'factors_72']\n",
            "BAD:  18886\n",
            "rpws:      index         token label\n",
            "0   18959           TCF     I\n",
            "1   18960             (     O\n",
            "2   18961             T     B\n",
            "3   18962          cell     I\n",
            "4   18963        factor     I\n",
            "..    ...           ...   ...\n",
            "70  19029             .     O\n",
            "71  19030  Hypocalcemia     O\n",
            "72  19031            is     O\n",
            "73  19032       usually     O\n",
            "74  19033           due     O\n",
            "\n",
            "[75 rows x 3 columns]\n",
            "The vertebrate transcription factors TCF ( T cell factor ) and LEF ( lymphocyte enhancer binding factor ) interact with beta - catenin and are hypothesized to mediate Wingless / Wnt signaling . Some of these DNA : protein complexes were also present , but at lower levels , in nuclear extracts from untransformed rat cells suggesting the possible involvement of cellular factors in the mechanism of down - regulation mediated by Ad12 E1A .\n",
            "['The', 'vertebrate', 'transcription', 'factors', 'TCF', '(', 'T', 'cell', 'factor', ')', 'and', 'LEF', '(', 'lymphocyte', 'enhancer', 'binding', 'factor', ')', 'interact', 'with', 'beta', '-', 'catenin', 'and', 'are', 'hypothesized', 'to', 'mediate', 'Wingless', '/', 'Wnt', 'signaling', '.', 'Some', 'of', 'these', 'DNA', ':', 'protein', 'complexes', 'were', 'also', 'present', ',', 'but', 'at', 'lower', 'levels', ',', 'in', 'nuclear', 'extracts', 'from', 'untransformed', 'rat', 'cells', 'suggesting', 'the', 'possible', 'involvement', 'of', 'cellular', 'factors', 'in', 'the', 'mechanism', 'of', 'down', '-', 'regulation', 'mediated', 'by', 'Ad12', 'E1A', '.']\n",
            "75 75\n",
            "['TCF_0', '(_1', 'T_2', 'cell_3', 'factor_4', ')_5', 'and_6', 'LEF_7', '(_8', 'lymphocyte_9', 'enhancer_10', 'binding_11', 'factor_12', ')_13', 'interact_14', 'with_15', 'beta_16', '-_17', 'catenin_18', 'and_19', 'are_20', 'hypothesized_21', 'to_22', 'mediate_23', 'Wingless_24', '/_25', 'Wnt_26', 'signaling_27', '._28', 'Some_29', 'of_30', 'these_31', 'DNA_32', ':_33', 'protein_34', 'complexes_35', 'were_36', 'also_37', 'present_38', ',_39', 'but_40', 'at_41', 'lower_42', 'levels_43', ',_44', 'in_45', 'nuclear_46', 'extracts_47', 'from_48', 'untransformed_49', 'rat_50', 'cells_51', 'suggesting_52', 'the_53', 'possible_54', 'involvement_55', 'of_56', 'cellular_57', 'factors_58', 'in_59', 'the_60', 'mechanism_61', 'of_62', 'down_63', '-_64', 'regulation_65', 'mediated_66', 'by_67', 'Ad12_68', 'E1A_69', '._70', 'Hypocalcemia_71', 'is_72', 'usually_73', 'due_74']\n",
            "rpws:      index        token label\n",
            "0   19034           to     O\n",
            "1   19035       either     O\n",
            "2   19036            a     O\n",
            "3   19037  disturbance     O\n",
            "4   19038           in     O\n",
            "..    ...          ...   ...\n",
            "62  19096            .     O\n",
            "63  19097           In     O\n",
            "64  19098        seven     O\n",
            "65  19099     patients     O\n",
            "66  19100            (     O\n",
            "\n",
            "[67 rows x 3 columns]\n",
            "Hypocalcemia is usually due to either a disturbance in the parathyroid hormone - adenylate cyclase system or a disturbance in vitamin D metabolism . Activity was reconstituted , however , by combining fractions that were enriched in the two components . If failure to solve invisible displacements was due to increased memory requirements , then the primates should perform at chance level on all 3 problems .\n",
            "['Hypocalcemia', 'is', 'usually', 'due', 'to', 'either', 'a', 'disturbance', 'in', 'the', 'parathyroid', 'hormone', '-', 'adenylate', 'cyclase', 'system', 'or', 'a', 'disturbance', 'in', 'vitamin', 'D', 'metabolism', '.', 'Activity', 'was', 'reconstituted', ',', 'however', ',', 'by', 'combining', 'fractions', 'that', 'were', 'enriched', 'in', 'the', 'two', 'components', '.', 'If', 'failure', 'to', 'solve', 'invisible', 'displacements', 'was', 'due', 'to', 'increased', 'memory', 'requirements', ',', 'then', 'the', 'primates', 'should', 'perform', 'at', 'chance', 'level', 'on', 'all', '3', 'problems', '.']\n",
            "67 67\n",
            "['to_0', 'either_1', 'a_2', 'disturbance_3', 'in_4', 'the_5', 'parathyroid_6', 'hormone_7', '-_8', 'adenylate_9', 'cyclase_10', 'system_11', 'or_12', 'a_13', 'disturbance_14', 'in_15', 'vitamin_16', 'D_17', 'metabolism_18', '._19', 'Activity_20', 'was_21', 'reconstituted_22', ',_23', 'however_24', ',_25', 'by_26', 'combining_27', 'fractions_28', 'that_29', 'were_30', 'enriched_31', 'in_32', 'the_33', 'two_34', 'components_35', '._36', 'If_37', 'failure_38', 'to_39', 'solve_40', 'invisible_41', 'displacements_42', 'was_43', 'due_44', 'to_45', 'increased_46', 'memory_47', 'requirements_48', ',_49', 'then_50', 'the_51', 'primates_52', 'should_53', 'perform_54', 'at_55', 'chance_56', 'level_57', 'on_58', 'all_59', '3_60', 'problems_61', '._62', 'In_63', 'seven_64', 'patients_65', '(_66']\n",
            "BAD:  19034\n",
            "rpws:      index  token label\n",
            "0   19101      7     O\n",
            "1   19102      .     O\n",
            "2   19103      8     O\n",
            "3   19104      %     O\n",
            "4   19105      )     O\n",
            "..    ...    ...   ...\n",
            "72  19173      .     O\n",
            "73  19174  Mouse     B\n",
            "74  19175   GRK6     I\n",
            "75  19176      -     I\n",
            "76  19177      C     I\n",
            "\n",
            "[77 rows x 3 columns]\n",
            "In seven patients ( 7 . 8 % ) with new Q - waves and a pathologic CK - MB profile ( group II ) troponin T reached median levels of 10 . 47 micrograms / l ( quartile 6 . 34 - 12 . 50 micrograms / l ) ( P < 0 . 001 I vs II ) . Mineralized bone nodule formation in vitro by cell populations from young adult rabbit alveolar bone .\n",
            "['In', 'seven', 'patients', '(', '7', '.', '8', '%', ')', 'with', 'new', 'Q', '-', 'waves', 'and', 'a', 'pathologic', 'CK', '-', 'MB', 'profile', '(', 'group', 'II', ')', 'troponin', 'T', 'reached', 'median', 'levels', 'of', '10', '.', '47', 'micrograms', '/', 'l', '(', 'quartile', '6', '.', '34', '-', '12', '.', '50', 'micrograms', '/', 'l', ')', '(', 'P', '<', '0', '.', '001', 'I', 'vs', 'II', ')', '.', 'Mineralized', 'bone', 'nodule', 'formation', 'in', 'vitro', 'by', 'cell', 'populations', 'from', 'young', 'adult', 'rabbit', 'alveolar', 'bone', '.']\n",
            "77 77\n",
            "['7_0', '._1', '8_2', '%_3', ')_4', 'with_5', 'new_6', 'Q_7', '-_8', 'waves_9', 'and_10', 'a_11', 'pathologic_12', 'CK_13', '-_14', 'MB_15', 'profile_16', '(_17', 'group_18', 'II_19', ')_20', 'troponin_21', 'T_22', 'reached_23', 'median_24', 'levels_25', 'of_26', '10_27', '._28', '47_29', 'micrograms_30', '/_31', 'l_32', '(_33', 'quartile_34', '6_35', '._36', '34_37', '-_38', '12_39', '._40', '50_41', 'micrograms_42', '/_43', 'l_44', ')_45', '(_46', 'P_47', '<_48', '0_49', '._50', '001_51', 'I_52', 'vs_53', 'II_54', ')_55', '._56', 'Mineralized_57', 'bone_58', 'nodule_59', 'formation_60', 'in_61', 'vitro_62', 'by_63', 'cell_64', 'populations_65', 'from_66', 'young_67', 'adult_68', 'rabbit_69', 'alveolar_70', 'bone_71', '._72', 'Mouse_73', 'GRK6_74', '-_75', 'C_76']\n",
            "BAD:  19101\n",
            "rpws:      index     token label\n",
            "0   19178  displays     O\n",
            "1   19179      none     O\n",
            "2   19180        of     O\n",
            "3   19181     these     O\n",
            "4   19182    motifs     O\n",
            "..    ...       ...   ...\n",
            "65  19243         .     O\n",
            "66  19244       The     O\n",
            "67  19245       132     B\n",
            "68  19246         -     I\n",
            "69  19247        bp     I\n",
            "\n",
            "[70 rows x 3 columns]\n",
            "Mouse GRK6 - C displays none of these motifs . Comparisons of the efficacy of benazepril and hydrochlorothiazide alone and in combination have shown that benazepril 20 mg once daily is as effective as or more effective in lowering diastolic blood pressure than hydrochlorothiazide 25 mg once daily and that the combination of benazepril 20 mg and hydrochlorothiazide 25 mg has a possibly synergistic effect on diastolic blood pressure .\n",
            "['Mouse', 'GRK6', '-', 'C', 'displays', 'none', 'of', 'these', 'motifs', '.', 'Comparisons', 'of', 'the', 'efficacy', 'of', 'benazepril', 'and', 'hydrochlorothiazide', 'alone', 'and', 'in', 'combination', 'have', 'shown', 'that', 'benazepril', '20', 'mg', 'once', 'daily', 'is', 'as', 'effective', 'as', 'or', 'more', 'effective', 'in', 'lowering', 'diastolic', 'blood', 'pressure', 'than', 'hydrochlorothiazide', '25', 'mg', 'once', 'daily', 'and', 'that', 'the', 'combination', 'of', 'benazepril', '20', 'mg', 'and', 'hydrochlorothiazide', '25', 'mg', 'has', 'a', 'possibly', 'synergistic', 'effect', 'on', 'diastolic', 'blood', 'pressure', '.']\n",
            "70 70\n",
            "['displays_0', 'none_1', 'of_2', 'these_3', 'motifs_4', '._5', 'Comparisons_6', 'of_7', 'the_8', 'efficacy_9', 'of_10', 'benazepril_11', 'and_12', 'hydrochlorothiazide_13', 'alone_14', 'and_15', 'in_16', 'combination_17', 'have_18', 'shown_19', 'that_20', 'benazepril_21', '20_22', 'mg_23', 'once_24', 'daily_25', 'is_26', 'as_27', 'effective_28', 'as_29', 'or_30', 'more_31', 'effective_32', 'in_33', 'lowering_34', 'diastolic_35', 'blood_36', 'pressure_37', 'than_38', 'hydrochlorothiazide_39', '25_40', 'mg_41', 'once_42', 'daily_43', 'and_44', 'that_45', 'the_46', 'combination_47', 'of_48', 'benazepril_49', '20_50', 'mg_51', 'and_52', 'hydrochlorothiazide_53', '25_54', 'mg_55', 'has_56', 'a_57', 'possibly_58', 'synergistic_59', 'effect_60', 'on_61', 'diastolic_62', 'blood_63', 'pressure_64', '._65', 'The_66', '132_67', '-_68', 'bp_69']\n",
            "BAD:  19178\n",
            "rpws:      index        token label\n",
            "0   19248        PHO8p     I\n",
            "1   19249     fragment     I\n",
            "2   19250            ,     O\n",
            "3   19251    connected     O\n",
            "4   19252           at     O\n",
            "..    ...          ...   ...\n",
            "85  19333            .     O\n",
            "86  19334      Plasmid     O\n",
            "87  19335    subclones     O\n",
            "88  19336           of     O\n",
            "89  19337  recombinant     O\n",
            "\n",
            "[90 rows x 3 columns]\n",
            "The 132 - bp PHO8p fragment , connected at position - 281 of the 5 ' upstream region of a HIS5 ' - ' lacZ fused gene , could sense Pi signals in vivo , but a 20 - bp synthetic oligonucleotide having the same sequence from - 544 to - 525 of the PHO8p DNA could not . 1988 ) . Sequences within the UAS2 element of the ENO2 gene bound a second protein which corresponded to the ABFI ( autonomously replicating sequence - binding factor ) protein .\n",
            "['The', '132', '-', 'bp', 'PHO8p', 'fragment', ',', 'connected', 'at', 'position', '-', '281', 'of', 'the', '5', \"'\", 'upstream', 'region', 'of', 'a', 'HIS5', \"'\", '-', \"'\", 'lacZ', 'fused', 'gene', ',', 'could', 'sense', 'Pi', 'signals', 'in', 'vivo', ',', 'but', 'a', '20', '-', 'bp', 'synthetic', 'oligonucleotide', 'having', 'the', 'same', 'sequence', 'from', '-', '544', 'to', '-', '525', 'of', 'the', 'PHO8p', 'DNA', 'could', 'not', '.', '1988', ')', '.', 'Sequences', 'within', 'the', 'UAS2', 'element', 'of', 'the', 'ENO2', 'gene', 'bound', 'a', 'second', 'protein', 'which', 'corresponded', 'to', 'the', 'ABFI', '(', 'autonomously', 'replicating', 'sequence', '-', 'binding', 'factor', ')', 'protein', '.']\n",
            "90 90\n",
            "['PHO8p_0', 'fragment_1', ',_2', 'connected_3', 'at_4', 'position_5', '-_6', '281_7', 'of_8', 'the_9', '5_10', \"'_11\", 'upstream_12', 'region_13', 'of_14', 'a_15', 'HIS5_16', \"'_17\", '-_18', \"'_19\", 'lacZ_20', 'fused_21', 'gene_22', ',_23', 'could_24', 'sense_25', 'Pi_26', 'signals_27', 'in_28', 'vivo_29', ',_30', 'but_31', 'a_32', '20_33', '-_34', 'bp_35', 'synthetic_36', 'oligonucleotide_37', 'having_38', 'the_39', 'same_40', 'sequence_41', 'from_42', '-_43', '544_44', 'to_45', '-_46', '525_47', 'of_48', 'the_49', 'PHO8p_50', 'DNA_51', 'could_52', 'not_53', '._54', '1988_55', ')_56', '._57', 'Sequences_58', 'within_59', 'the_60', 'UAS2_61', 'element_62', 'of_63', 'the_64', 'ENO2_65', 'gene_66', 'bound_67', 'a_68', 'second_69', 'protein_70', 'which_71', 'corresponded_72', 'to_73', 'the_74', 'ABFI_75', '(_76', 'autonomously_77', 'replicating_78', 'sequence_79', '-_80', 'binding_81', 'factor_82', ')_83', 'protein_84', '._85', 'Plasmid_86', 'subclones_87', 'of_88', 'recombinant_89']\n",
            "BAD:  19248\n",
            "rpws:      index   token label\n",
            "0   19338   phage     O\n",
            "1   19339  lambda     O\n",
            "2   19340  Asm152     O\n",
            "3   19341    were     O\n",
            "4   19342    able     O\n",
            "..    ...     ...   ...\n",
            "68  19406       .     O\n",
            "69  19407  Steady     O\n",
            "70  19408       -     O\n",
            "71  19409   state     O\n",
            "72  19410  levels     O\n",
            "\n",
            "[73 rows x 3 columns]\n",
            "Plasmid subclones of recombinant phage lambda Asm152 were able to complement both Escherichia coli gltB and A . sesbaniae Asm - Vi mutants ; NADPH - glutamate synthase activity was detected in all such strains complemented to Asm + . The sequence of LZ321 matched that of RREB1 , a transcription factor that bound to a Ras responsive element ( RRE ) very different from the sequence with which we isolated LZ321 .\n",
            "['Plasmid', 'subclones', 'of', 'recombinant', 'phage', 'lambda', 'Asm152', 'were', 'able', 'to', 'complement', 'both', 'Escherichia', 'coli', 'gltB', 'and', 'A', '.', 'sesbaniae', 'Asm', '-', 'Vi', 'mutants', ';', 'NADPH', '-', 'glutamate', 'synthase', 'activity', 'was', 'detected', 'in', 'all', 'such', 'strains', 'complemented', 'to', 'Asm', '+', '.', 'The', 'sequence', 'of', 'LZ321', 'matched', 'that', 'of', 'RREB1', ',', 'a', 'transcription', 'factor', 'that', 'bound', 'to', 'a', 'Ras', 'responsive', 'element', '(', 'RRE', ')', 'very', 'different', 'from', 'the', 'sequence', 'with', 'which', 'we', 'isolated', 'LZ321', '.']\n",
            "73 73\n",
            "['phage_0', 'lambda_1', 'Asm152_2', 'were_3', 'able_4', 'to_5', 'complement_6', 'both_7', 'Escherichia_8', 'coli_9', 'gltB_10', 'and_11', 'A_12', '._13', 'sesbaniae_14', 'Asm_15', '-_16', 'Vi_17', 'mutants_18', ';_19', 'NADPH_20', '-_21', 'glutamate_22', 'synthase_23', 'activity_24', 'was_25', 'detected_26', 'in_27', 'all_28', 'such_29', 'strains_30', 'complemented_31', 'to_32', 'Asm_33', '+_34', '._35', 'The_36', 'sequence_37', 'of_38', 'LZ321_39', 'matched_40', 'that_41', 'of_42', 'RREB1_43', ',_44', 'a_45', 'transcription_46', 'factor_47', 'that_48', 'bound_49', 'to_50', 'a_51', 'Ras_52', 'responsive_53', 'element_54', '(_55', 'RRE_56', ')_57', 'very_58', 'different_59', 'from_60', 'the_61', 'sequence_62', 'with_63', 'which_64', 'we_65', 'isolated_66', 'LZ321_67', '._68', 'Steady_69', '-_70', 'state_71', 'levels_72']\n",
            "BAD:  19338\n",
            "rpws:      index        token label\n",
            "0   19411           of     O\n",
            "1   19412       murine     B\n",
            "2   19413          MMR     I\n",
            "3   19414         mRNA     I\n",
            "4   19415         were     O\n",
            "..    ...          ...   ...\n",
            "56  19467            .     O\n",
            "57  19468         coli     O\n",
            "58  19469  translation     O\n",
            "59  19470   terminates     O\n",
            "60  19471         more     O\n",
            "\n",
            "[61 rows x 3 columns]\n",
            "Steady - state levels of murine MMR mRNA were measured in the macrophage cell line J774E , which is known to express the protein at the cell surface . Basal level expression was reduced to 20 and 50 % when the UGA stop codon was replaced by UAG or UAA , respectively , consistent with the finding that in E .\n",
            "['Steady', '-', 'state', 'levels', 'of', 'murine', 'MMR', 'mRNA', 'were', 'measured', 'in', 'the', 'macrophage', 'cell', 'line', 'J774E', ',', 'which', 'is', 'known', 'to', 'express', 'the', 'protein', 'at', 'the', 'cell', 'surface', '.', 'Basal', 'level', 'expression', 'was', 'reduced', 'to', '20', 'and', '50', '%', 'when', 'the', 'UGA', 'stop', 'codon', 'was', 'replaced', 'by', 'UAG', 'or', 'UAA', ',', 'respectively', ',', 'consistent', 'with', 'the', 'finding', 'that', 'in', 'E', '.']\n",
            "61 61\n",
            "['of_0', 'murine_1', 'MMR_2', 'mRNA_3', 'were_4', 'measured_5', 'in_6', 'the_7', 'macrophage_8', 'cell_9', 'line_10', 'J774E_11', ',_12', 'which_13', 'is_14', 'known_15', 'to_16', 'express_17', 'the_18', 'protein_19', 'at_20', 'the_21', 'cell_22', 'surface_23', '._24', 'Basal_25', 'level_26', 'expression_27', 'was_28', 'reduced_29', 'to_30', '20_31', 'and_32', '50_33', '%_34', 'when_35', 'the_36', 'UGA_37', 'stop_38', 'codon_39', 'was_40', 'replaced_41', 'by_42', 'UAG_43', 'or_44', 'UAA_45', ',_46', 'respectively_47', ',_48', 'consistent_49', 'with_50', 'the_51', 'finding_52', 'that_53', 'in_54', 'E_55', '._56', 'coli_57', 'translation_58', 'terminates_59', 'more_60']\n",
            "BAD:  19411\n",
            "rpws:      index         token label\n",
            "0   19472   efficiently     O\n",
            "1   19473            at     O\n",
            "2   19474           UAG     O\n",
            "3   19475           and     O\n",
            "4   19476           UAA     O\n",
            "5   19477          than     O\n",
            "6   19478            at     O\n",
            "7   19479           UGA     O\n",
            "8   19480             .     O\n",
            "9   19481  Discriminant     O\n",
            "10  19482      function     O\n",
            "11  19483      analysis     O\n",
            "12  19484      suggests     O\n",
            "13  19485          that     O\n",
            "14  19486         AUDIT     O\n",
            "15  19487        scores     O\n",
            "16  19488           can     O\n",
            "17  19489  successfully     O\n",
            "18  19490      identify     O\n",
            "19  19491           SAT     O\n",
            "20  19492             -     O\n",
            "21  19493      positive     O\n",
            "22  19494           and     O\n",
            "23  19495           SAT     O\n",
            "24  19496             -     O\n",
            "25  19497      negative     O\n",
            "26  19498      patients     O\n",
            "27  19499             ;     O\n",
            "28  19500           the     O\n",
            "29  19501      analysis     O\n",
            "30  19502     accounted     O\n",
            "31  19503           for     O\n",
            "32  19504            42     O\n",
            "33  19505             .     O\n",
            "34  19506             5     O\n",
            "35  19507             %     O\n",
            "36  19508            of     O\n",
            "37  19509           the     O\n",
            "38  19510      variance     O\n",
            "39  19511           and     O\n",
            "40  19512     correctly     O\n",
            "41  19513    classified     O\n",
            "42  19514            84     O\n",
            "43  19515             .     O\n",
            "44  19516             6     O\n",
            "45  19517             %     O\n",
            "46  19518            of     O\n",
            "47  19519           the     O\n",
            "48  19520        sample     O\n",
            "49  19521             .     O\n",
            "50  19522    Coronatine     B\n",
            "51  19523             (     O\n",
            "52  19524           COR     B\n",
            "53  19525             )     O\n",
            "coli translation terminates more efficiently at UAG and UAA than at UGA . Discriminant function analysis suggests that AUDIT scores can successfully identify SAT - positive and SAT - negative patients ; the analysis accounted for 42 . 5 % of the variance and correctly classified 84 . 6 % of the sample .\n",
            "['coli', 'translation', 'terminates', 'more', 'efficiently', 'at', 'UAG', 'and', 'UAA', 'than', 'at', 'UGA', '.', 'Discriminant', 'function', 'analysis', 'suggests', 'that', 'AUDIT', 'scores', 'can', 'successfully', 'identify', 'SAT', '-', 'positive', 'and', 'SAT', '-', 'negative', 'patients', ';', 'the', 'analysis', 'accounted', 'for', '42', '.', '5', '%', 'of', 'the', 'variance', 'and', 'correctly', 'classified', '84', '.', '6', '%', 'of', 'the', 'sample', '.']\n",
            "54 54\n",
            "['efficiently_0', 'at_1', 'UAG_2', 'and_3', 'UAA_4', 'than_5', 'at_6', 'UGA_7', '._8', 'Discriminant_9', 'function_10', 'analysis_11', 'suggests_12', 'that_13', 'AUDIT_14', 'scores_15', 'can_16', 'successfully_17', 'identify_18', 'SAT_19', '-_20', 'positive_21', 'and_22', 'SAT_23', '-_24', 'negative_25', 'patients_26', ';_27', 'the_28', 'analysis_29', 'accounted_30', 'for_31', '42_32', '._33', '5_34', '%_35', 'of_36', 'the_37', 'variance_38', 'and_39', 'correctly_40', 'classified_41', '84_42', '._43', '6_44', '%_45', 'of_46', 'the_47', 'sample_48', '._49', 'Coronatine_50', '(_51', 'COR_52', ')_53']\n",
            "BAD:  19472\n",
            "rpws:      index            token label\n",
            "0   19526               is     O\n",
            "1   19527                a     O\n",
            "2   19528          plasmid     O\n",
            "3   19529                -     O\n",
            "4   19530          encoded     O\n",
            "..    ...              ...   ...\n",
            "71  19597                .     O\n",
            "72  19598              The     O\n",
            "73  19599  transcriptional     O\n",
            "74  19600         activity     O\n",
            "75  19601               of     O\n",
            "\n",
            "[76 rows x 3 columns]\n",
            "Coronatine ( COR ) is a plasmid - encoded phytotoxin synthesized by several pathovars of phytopathogenic Pseudomonas syringae . Both the 5 ' and 3 ' untranslated regions also show significant similarity to the murine gene , with 79 and 70 % sequence identity , respectively . One patient exhibited a non - sense mutation ( codon 388 ) , which changed a glutamine codon ( CAG ) to a stop codon ( TAG ) .\n",
            "['Coronatine', '(', 'COR', ')', 'is', 'a', 'plasmid', '-', 'encoded', 'phytotoxin', 'synthesized', 'by', 'several', 'pathovars', 'of', 'phytopathogenic', 'Pseudomonas', 'syringae', '.', 'Both', 'the', '5', \"'\", 'and', '3', \"'\", 'untranslated', 'regions', 'also', 'show', 'significant', 'similarity', 'to', 'the', 'murine', 'gene', ',', 'with', '79', 'and', '70', '%', 'sequence', 'identity', ',', 'respectively', '.', 'One', 'patient', 'exhibited', 'a', 'non', '-', 'sense', 'mutation', '(', 'codon', '388', ')', ',', 'which', 'changed', 'a', 'glutamine', 'codon', '(', 'CAG', ')', 'to', 'a', 'stop', 'codon', '(', 'TAG', ')', '.']\n",
            "76 76\n",
            "['is_0', 'a_1', 'plasmid_2', '-_3', 'encoded_4', 'phytotoxin_5', 'synthesized_6', 'by_7', 'several_8', 'pathovars_9', 'of_10', 'phytopathogenic_11', 'Pseudomonas_12', 'syringae_13', '._14', 'Both_15', 'the_16', '5_17', \"'_18\", 'and_19', '3_20', \"'_21\", 'untranslated_22', 'regions_23', 'also_24', 'show_25', 'significant_26', 'similarity_27', 'to_28', 'the_29', 'murine_30', 'gene_31', ',_32', 'with_33', '79_34', 'and_35', '70_36', '%_37', 'sequence_38', 'identity_39', ',_40', 'respectively_41', '._42', 'One_43', 'patient_44', 'exhibited_45', 'a_46', 'non_47', '-_48', 'sense_49', 'mutation_50', '(_51', 'codon_52', '388_53', ')_54', ',_55', 'which_56', 'changed_57', 'a_58', 'glutamine_59', 'codon_60', '(_61', 'CAG_62', ')_63', 'to_64', 'a_65', 'stop_66', 'codon_67', '(_68', 'TAG_69', ')_70', '._71', 'The_72', 'transcriptional_73', 'activity_74', 'of_75']\n",
            "BAD:  19526\n",
            "rpws:      index   token label\n",
            "0   19602       P     B\n",
            "1   19603       -     I\n",
            "2   19604     450     I\n",
            "3   19605       (     I\n",
            "4   19606      11     I\n",
            "..    ...     ...   ...\n",
            "59  19661       .     O\n",
            "60  19662  Eleven     O\n",
            "61  19663      of     O\n",
            "62  19664     the     O\n",
            "63  19665      12     O\n",
            "\n",
            "[64 rows x 3 columns]\n",
            "The transcriptional activity of P - 450 ( 11 beta ) gene was studied with an in vitro transcription system using nuclear extracts prepared from bovine adrenal cortex . Renal dysplasia with multisystem malformation - - a study of 9 cases . This bud morphology results at least in part from a cell cycle delay imposed by the Cdc28p - inhibitory kinase Swe1p .\n",
            "['The', 'transcriptional', 'activity', 'of', 'P', '-', '450', '(', '11', 'beta', ')', 'gene', 'was', 'studied', 'with', 'an', 'in', 'vitro', 'transcription', 'system', 'using', 'nuclear', 'extracts', 'prepared', 'from', 'bovine', 'adrenal', 'cortex', '.', 'Renal', 'dysplasia', 'with', 'multisystem', 'malformation', '-', '-', 'a', 'study', 'of', '9', 'cases', '.', 'This', 'bud', 'morphology', 'results', 'at', 'least', 'in', 'part', 'from', 'a', 'cell', 'cycle', 'delay', 'imposed', 'by', 'the', 'Cdc28p', '-', 'inhibitory', 'kinase', 'Swe1p', '.']\n",
            "64 64\n",
            "['P_0', '-_1', '450_2', '(_3', '11_4', 'beta_5', ')_6', 'gene_7', 'was_8', 'studied_9', 'with_10', 'an_11', 'in_12', 'vitro_13', 'transcription_14', 'system_15', 'using_16', 'nuclear_17', 'extracts_18', 'prepared_19', 'from_20', 'bovine_21', 'adrenal_22', 'cortex_23', '._24', 'Renal_25', 'dysplasia_26', 'with_27', 'multisystem_28', 'malformation_29', '-_30', '-_31', 'a_32', 'study_33', 'of_34', '9_35', 'cases_36', '._37', 'This_38', 'bud_39', 'morphology_40', 'results_41', 'at_42', 'least_43', 'in_44', 'part_45', 'from_46', 'a_47', 'cell_48', 'cycle_49', 'delay_50', 'imposed_51', 'by_52', 'the_53', 'Cdc28p_54', '-_55', 'inhibitory_56', 'kinase_57', 'Swe1p_58', '._59', 'Eleven_60', 'of_61', 'the_62', '12_63']\n",
            "BAD:  19602\n",
            "rpws:      index          token label\n",
            "0   19666          exons     O\n",
            "1   19667           have     O\n",
            "2   19668       complete     O\n",
            "3   19669       sequence     O\n",
            "4   19670       homology     O\n",
            "..    ...            ...   ...\n",
            "67  19733              .     O\n",
            "68  19734       Cellular     O\n",
            "69  19735  fractionation     O\n",
            "70  19736            and     O\n",
            "71  19737        Percoll     O\n",
            "\n",
            "[72 rows x 3 columns]\n",
            "Eleven of the 12 exons have complete sequence homology with the RBM - 1 sequence . Copyright 1998 Academic Press . Laparoscopic examination 1 year after surgery revealed an enlarged , thin - walled , and fluid - filled uterine segment cranial to the midcornus occlusion sites in all animals . The three proteins copurified through several biochemical fractionation steps and could be coimmunoprecipitated by using antibodies against GCD1 or GCD2 .\n",
            "['Eleven', 'of', 'the', '12', 'exons', 'have', 'complete', 'sequence', 'homology', 'with', 'the', 'RBM', '-', '1', 'sequence', '.', 'Copyright', '1998', 'Academic', 'Press', '.', 'Laparoscopic', 'examination', '1', 'year', 'after', 'surgery', 'revealed', 'an', 'enlarged', ',', 'thin', '-', 'walled', ',', 'and', 'fluid', '-', 'filled', 'uterine', 'segment', 'cranial', 'to', 'the', 'midcornus', 'occlusion', 'sites', 'in', 'all', 'animals', '.', 'The', 'three', 'proteins', 'copurified', 'through', 'several', 'biochemical', 'fractionation', 'steps', 'and', 'could', 'be', 'coimmunoprecipitated', 'by', 'using', 'antibodies', 'against', 'GCD1', 'or', 'GCD2', '.']\n",
            "72 72\n",
            "['exons_0', 'have_1', 'complete_2', 'sequence_3', 'homology_4', 'with_5', 'the_6', 'RBM_7', '-_8', '1_9', 'sequence_10', '._11', 'Copyright_12', '1998_13', 'Academic_14', 'Press_15', '._16', 'Laparoscopic_17', 'examination_18', '1_19', 'year_20', 'after_21', 'surgery_22', 'revealed_23', 'an_24', 'enlarged_25', ',_26', 'thin_27', '-_28', 'walled_29', ',_30', 'and_31', 'fluid_32', '-_33', 'filled_34', 'uterine_35', 'segment_36', 'cranial_37', 'to_38', 'the_39', 'midcornus_40', 'occlusion_41', 'sites_42', 'in_43', 'all_44', 'animals_45', '._46', 'The_47', 'three_48', 'proteins_49', 'copurified_50', 'through_51', 'several_52', 'biochemical_53', 'fractionation_54', 'steps_55', 'and_56', 'could_57', 'be_58', 'coimmunoprecipitated_59', 'by_60', 'using_61', 'antibodies_62', 'against_63', 'GCD1_64', 'or_65', 'GCD2_66', '._67', 'Cellular_68', 'fractionation_69', 'and_70', 'Percoll_71']\n",
            "BAD:  19666\n",
            "rpws:      index           token label\n",
            "0   19738        gradient     O\n",
            "1   19739  centrifugation     O\n",
            "2   19740        combined     O\n",
            "3   19741            with     O\n",
            "4   19742  immunoblotting     O\n",
            "..    ...             ...   ...\n",
            "63  19801               .     O\n",
            "64  19802     Elimination     O\n",
            "65  19803              of     O\n",
            "66  19804           brush     O\n",
            "67  19805          border     O\n",
            "\n",
            "[68 rows x 3 columns]\n",
            "Cellular fractionation and Percoll gradient centrifugation combined with immunoblotting show that p67 cofractionates with nuclei and is enriched in resistant structure that is insoluble in 2 M NaCl , 25 mM lithium 3 , 5 ' - diiodosalicylate , and 1 % Triton but is soluble in 8 M urea . For the first time also a genomic sequence for a red algal lhc gene is presented .\n",
            "['Cellular', 'fractionation', 'and', 'Percoll', 'gradient', 'centrifugation', 'combined', 'with', 'immunoblotting', 'show', 'that', 'p67', 'cofractionates', 'with', 'nuclei', 'and', 'is', 'enriched', 'in', 'resistant', 'structure', 'that', 'is', 'insoluble', 'in', '2', 'M', 'NaCl', ',', '25', 'mM', 'lithium', '3', ',', '5', \"'\", '-', 'diiodosalicylate', ',', 'and', '1', '%', 'Triton', 'but', 'is', 'soluble', 'in', '8', 'M', 'urea', '.', 'For', 'the', 'first', 'time', 'also', 'a', 'genomic', 'sequence', 'for', 'a', 'red', 'algal', 'lhc', 'gene', 'is', 'presented', '.']\n",
            "68 68\n",
            "['gradient_0', 'centrifugation_1', 'combined_2', 'with_3', 'immunoblotting_4', 'show_5', 'that_6', 'p67_7', 'cofractionates_8', 'with_9', 'nuclei_10', 'and_11', 'is_12', 'enriched_13', 'in_14', 'resistant_15', 'structure_16', 'that_17', 'is_18', 'insoluble_19', 'in_20', '2_21', 'M_22', 'NaCl_23', ',_24', '25_25', 'mM_26', 'lithium_27', '3_28', ',_29', '5_30', \"'_31\", '-_32', 'diiodosalicylate_33', ',_34', 'and_35', '1_36', '%_37', 'Triton_38', 'but_39', 'is_40', 'soluble_41', 'in_42', '8_43', 'M_44', 'urea_45', '._46', 'For_47', 'the_48', 'first_49', 'time_50', 'also_51', 'a_52', 'genomic_53', 'sequence_54', 'for_55', 'a_56', 'red_57', 'algal_58', 'lhc_59', 'gene_60', 'is_61', 'presented_62', '._63', 'Elimination_64', 'of_65', 'brush_66', 'border_67']\n",
            "BAD:  19738\n",
            "rpws:      index          token label\n",
            "0   19806             as     O\n",
            "1   19807           well     O\n",
            "2   19808             as     O\n",
            "3   19809             of     O\n",
            "4   19810     glomerular     O\n",
            "5   19811         marker     O\n",
            "6   19812       proteins     O\n",
            "7   19813            was     O\n",
            "8   19814  significantly     O\n",
            "9   19815          lower     O\n",
            "10  19816          after     O\n",
            "11  19817    intravenous     O\n",
            "12  19818      injection     O\n",
            "13  19819             of     O\n",
            "14  19820            low     O\n",
            "15  19821              -     O\n",
            "16  19822        osmolar     O\n",
            "17  19823             CM     O\n",
            "18  19824      iopamidol     O\n",
            "19  19825            370     O\n",
            "20  19826              (     O\n",
            "21  19827            832     O\n",
            "22  19828           mOsm     O\n",
            "23  19829              /     O\n",
            "24  19830             kg     O\n",
            "25  19831              )     O\n",
            "26  19832           than     O\n",
            "27  19833          after     O\n",
            "28  19834      meglumine     O\n",
            "29  19835    diatrizoate     O\n",
            "30  19836             76     O\n",
            "31  19837              (     O\n",
            "32  19838           2100     O\n",
            "33  19839           mOsm     O\n",
            "34  19840              /     O\n",
            "35  19841             kg     O\n",
            "36  19842              )     O\n",
            "37  19843              .     O\n",
            "38  19844          Raman     O\n",
            "39  19845     scattering     O\n",
            "40  19846           from     O\n",
            "41  19847            VO2     O\n",
            "42  19848         single     O\n",
            "43  19849       crystals     O\n",
            "44  19850              :     O\n",
            "45  19851              A     O\n",
            "46  19852          study     O\n",
            "47  19853             of     O\n",
            "48  19854            the     O\n",
            "49  19855        effects     O\n",
            "50  19856             of     O\n",
            "51  19857        surface     O\n",
            "52  19858      oxidation     O\n",
            "53  19859              .     O\n",
            "54  19860    Combination     O\n",
            "55  19861   chemotherapy     O\n",
            "56  19862             in     O\n",
            "57  19863       advanced     O\n",
            "Elimination of brush border as well as of glomerular marker proteins was significantly lower after intravenous injection of low - osmolar CM iopamidol 370 ( 832 mOsm / kg ) than after meglumine diatrizoate 76 ( 2100 mOsm / kg ) . Raman scattering from VO2 single crystals : A study of the effects of surface oxidation .\n",
            "['Elimination', 'of', 'brush', 'border', 'as', 'well', 'as', 'of', 'glomerular', 'marker', 'proteins', 'was', 'significantly', 'lower', 'after', 'intravenous', 'injection', 'of', 'low', '-', 'osmolar', 'CM', 'iopamidol', '370', '(', '832', 'mOsm', '/', 'kg', ')', 'than', 'after', 'meglumine', 'diatrizoate', '76', '(', '2100', 'mOsm', '/', 'kg', ')', '.', 'Raman', 'scattering', 'from', 'VO2', 'single', 'crystals', ':', 'A', 'study', 'of', 'the', 'effects', 'of', 'surface', 'oxidation', '.']\n",
            "58 58\n",
            "['as_0', 'well_1', 'as_2', 'of_3', 'glomerular_4', 'marker_5', 'proteins_6', 'was_7', 'significantly_8', 'lower_9', 'after_10', 'intravenous_11', 'injection_12', 'of_13', 'low_14', '-_15', 'osmolar_16', 'CM_17', 'iopamidol_18', '370_19', '(_20', '832_21', 'mOsm_22', '/_23', 'kg_24', ')_25', 'than_26', 'after_27', 'meglumine_28', 'diatrizoate_29', '76_30', '(_31', '2100_32', 'mOsm_33', '/_34', 'kg_35', ')_36', '._37', 'Raman_38', 'scattering_39', 'from_40', 'VO2_41', 'single_42', 'crystals_43', ':_44', 'A_45', 'study_46', 'of_47', 'the_48', 'effects_49', 'of_50', 'surface_51', 'oxidation_52', '._53', 'Combination_54', 'chemotherapy_55', 'in_56', 'advanced_57']\n",
            "BAD:  19806\n",
            "rpws:      index         token label\n",
            "0   19864       ovarian     O\n",
            "1   19865        cancer     O\n",
            "2   19866             .     O\n",
            "3   19867        Making     O\n",
            "4   19868         sense     O\n",
            "5   19869           out     O\n",
            "6   19870            of     O\n",
            "7   19871        oxygen     O\n",
            "8   19872        sensor     O\n",
            "9   19873             .     O\n",
            "10  19874            In     O\n",
            "11  19875    comparison     O\n",
            "12  19876            to     O\n",
            "13  19877             a     O\n",
            "14  19878        silent     O\n",
            "15  19879      baseline     O\n",
            "16  19880             ,     O\n",
            "17  19881           CBF     O\n",
            "18  19882     increases     O\n",
            "19  19883          were     O\n",
            "20  19884      observed     O\n",
            "21  19885            in     O\n",
            "22  19886      auditory     O\n",
            "23  19887        cortex     O\n",
            "24  19888   bilaterally     O\n",
            "25  19889           and     O\n",
            "26  19890            in     O\n",
            "27  19891           the     O\n",
            "28  19892         right     O\n",
            "29  19893      superior     O\n",
            "30  19894      parietal     O\n",
            "31  19895             ,     O\n",
            "32  19896         right     O\n",
            "33  19897  dorsolateral     O\n",
            "34  19898       frontal     O\n",
            "35  19899             ,     O\n",
            "36  19900           and     O\n",
            "37  19901         right     O\n",
            "38  19902      premotor     O\n",
            "39  19903       regions     O\n",
            "40  19904             ,     O\n",
            "41  19905          with     O\n",
            "42  19906            no     O\n",
            "43  19907    modulation     O\n",
            "44  19908            as     O\n",
            "45  19909             a     O\n",
            "46  19910      function     O\n",
            "47  19911            of     O\n",
            "48  19912   attentional     O\n",
            "49  19913     condition     O\n",
            "50  19914             .     O\n",
            "51  19915            In     O\n",
            "52  19916         vitro     O\n",
            "53  19917             ,     O\n",
            "54  19918             c     B\n",
            "Combination chemotherapy in advanced ovarian cancer . Making sense out of oxygen sensor . In comparison to a silent baseline , CBF increases were observed in auditory cortex bilaterally and in the right superior parietal , right dorsolateral frontal , and right premotor regions , with no modulation as a function of attentional condition .\n",
            "['Combination', 'chemotherapy', 'in', 'advanced', 'ovarian', 'cancer', '.', 'Making', 'sense', 'out', 'of', 'oxygen', 'sensor', '.', 'In', 'comparison', 'to', 'a', 'silent', 'baseline', ',', 'CBF', 'increases', 'were', 'observed', 'in', 'auditory', 'cortex', 'bilaterally', 'and', 'in', 'the', 'right', 'superior', 'parietal', ',', 'right', 'dorsolateral', 'frontal', ',', 'and', 'right', 'premotor', 'regions', ',', 'with', 'no', 'modulation', 'as', 'a', 'function', 'of', 'attentional', 'condition', '.']\n",
            "55 55\n",
            "['ovarian_0', 'cancer_1', '._2', 'Making_3', 'sense_4', 'out_5', 'of_6', 'oxygen_7', 'sensor_8', '._9', 'In_10', 'comparison_11', 'to_12', 'a_13', 'silent_14', 'baseline_15', ',_16', 'CBF_17', 'increases_18', 'were_19', 'observed_20', 'in_21', 'auditory_22', 'cortex_23', 'bilaterally_24', 'and_25', 'in_26', 'the_27', 'right_28', 'superior_29', 'parietal_30', ',_31', 'right_32', 'dorsolateral_33', 'frontal_34', ',_35', 'and_36', 'right_37', 'premotor_38', 'regions_39', ',_40', 'with_41', 'no_42', 'modulation_43', 'as_44', 'a_45', 'function_46', 'of_47', 'attentional_48', 'condition_49', '._50', 'In_51', 'vitro_52', ',_53', 'c_54']\n",
            "BAD:  19864\n",
            "rpws:       index           token label\n",
            "0    19919               -     I\n",
            "1    19920             Src     I\n",
            "2    19921  phosphorylated     O\n",
            "3    19922             FAK     B\n",
            "4    19923             Tyr     I\n",
            "..     ...             ...   ...\n",
            "164  20083               .     O\n",
            "165  20084               6     O\n",
            "166  20085              mg     O\n",
            "167  20086              of     O\n",
            "168  20087       sustained     O\n",
            "\n",
            "[169 rows x 3 columns]\n",
            "In vitro , c - Src phosphorylated FAK Tyr - 925 in a glutathione S - transferase - FAK C - terminal domain fusion protein , whereas FAK did not . METHODS AND RESULTS : The effect of prolonged nitrate therapy between 2 days and 6 weeks during healing after infarction on serial parameters of ventricular remodeling ( scar expansion , scar thinning , ventricular dilation , and hypertrophy ) and function ( asynergy or akinesis plus dyskinesis and ejection fraction ) by serial two - dimensional echocardiography , hemodynamics , postmortem topography ( computerized planimetry , geometric maps , and radiographs ) , and collagen content ( hydroxyproline ) was studied in 64 instrumented dogs randomized 2 days after left anterior descending coronary artery ligation to various nitrate regimens ( n = 32 ) over the first 2 weeks ( subgroup 1 : 2 % transdermal nitroglycerin at 8 AM and 4 PM , n = 6 ; subgroup 2 : 2 % transdermal nitroglycerin plus 2 .\n",
            "['In', 'vitro', ',', 'c', '-', 'Src', 'phosphorylated', 'FAK', 'Tyr', '-', '925', 'in', 'a', 'glutathione', 'S', '-', 'transferase', '-', 'FAK', 'C', '-', 'terminal', 'domain', 'fusion', 'protein', ',', 'whereas', 'FAK', 'did', 'not', '.', 'METHODS', 'AND', 'RESULTS', ':', 'The', 'effect', 'of', 'prolonged', 'nitrate', 'therapy', 'between', '2', 'days', 'and', '6', 'weeks', 'during', 'healing', 'after', 'infarction', 'on', 'serial', 'parameters', 'of', 'ventricular', 'remodeling', '(', 'scar', 'expansion', ',', 'scar', 'thinning', ',', 'ventricular', 'dilation', ',', 'and', 'hypertrophy', ')', 'and', 'function', '(', 'asynergy', 'or', 'akinesis', 'plus', 'dyskinesis', 'and', 'ejection', 'fraction', ')', 'by', 'serial', 'two', '-', 'dimensional', 'echocardiography', ',', 'hemodynamics', ',', 'postmortem', 'topography', '(', 'computerized', 'planimetry', ',', 'geometric', 'maps', ',', 'and', 'radiographs', ')', ',', 'and', 'collagen', 'content', '(', 'hydroxyproline', ')', 'was', 'studied', 'in', '64', 'instrumented', 'dogs', 'randomized', '2', 'days', 'after', 'left', 'anterior', 'descending', 'coronary', 'artery', 'ligation', 'to', 'various', 'nitrate', 'regimens', '(', 'n', '=', '32', ')', 'over', 'the', 'first', '2', 'weeks', '(', 'subgroup', '1', ':', '2', '%', 'transdermal', 'nitroglycerin', 'at', '8', 'AM', 'and', '4', 'PM', ',', 'n', '=', '6', ';', 'subgroup', '2', ':', '2', '%', 'transdermal', 'nitroglycerin', 'plus', '2', '.']\n",
            "169 169\n",
            "['-_0', 'Src_1', 'phosphorylated_2', 'FAK_3', 'Tyr_4', '-_5', '925_6', 'in_7', 'a_8', 'glutathione_9', 'S_10', '-_11', 'transferase_12', '-_13', 'FAK_14', 'C_15', '-_16', 'terminal_17', 'domain_18', 'fusion_19', 'protein_20', ',_21', 'whereas_22', 'FAK_23', 'did_24', 'not_25', '._26', 'METHODS_27', 'AND_28', 'RESULTS_29', ':_30', 'The_31', 'effect_32', 'of_33', 'prolonged_34', 'nitrate_35', 'therapy_36', 'between_37', '2_38', 'days_39', 'and_40', '6_41', 'weeks_42', 'during_43', 'healing_44', 'after_45', 'infarction_46', 'on_47', 'serial_48', 'parameters_49', 'of_50', 'ventricular_51', 'remodeling_52', '(_53', 'scar_54', 'expansion_55', ',_56', 'scar_57', 'thinning_58', ',_59', 'ventricular_60', 'dilation_61', ',_62', 'and_63', 'hypertrophy_64', ')_65', 'and_66', 'function_67', '(_68', 'asynergy_69', 'or_70', 'akinesis_71', 'plus_72', 'dyskinesis_73', 'and_74', 'ejection_75', 'fraction_76', ')_77', 'by_78', 'serial_79', 'two_80', '-_81', 'dimensional_82', 'echocardiography_83', ',_84', 'hemodynamics_85', ',_86', 'postmortem_87', 'topography_88', '(_89', 'computerized_90', 'planimetry_91', ',_92', 'geometric_93', 'maps_94', ',_95', 'and_96', 'radiographs_97', ')_98', ',_99', 'and_100', 'collagen_101', 'content_102', '(_103', 'hydroxyproline_104', ')_105', 'was_106', 'studied_107', 'in_108', '64_109', 'instrumented_110', 'dogs_111', 'randomized_112', '2_113', 'days_114', 'after_115', 'left_116', 'anterior_117', 'descending_118', 'coronary_119', 'artery_120', 'ligation_121', 'to_122', 'various_123', 'nitrate_124', 'regimens_125', '(_126', 'n_127', '=_128', '32_129', ')_130', 'over_131', 'the_132', 'first_133', '2_134', 'weeks_135', '(_136', 'subgroup_137', '1_138', ':_139', '2_140', '%_141', 'transdermal_142', 'nitroglycerin_143', 'at_144', '8_145', 'AM_146', 'and_147', '4_148', 'PM_149', ',_150', 'n_151', '=_152', '6_153', ';_154', 'subgroup_155', '2_156', ':_157', '2_158', '%_159', 'transdermal_160', 'nitroglycerin_161', 'plus_162', '2_163', '._164', '6_165', 'mg_166', 'of_167', 'sustained_168']\n",
            "BAD:  19919\n",
            "rpws:      index          token label\n",
            "0   20088              -     O\n",
            "1   20089        release     O\n",
            "2   20090           oral     O\n",
            "3   20091  nitroglycerin     O\n",
            "4   20092             at     O\n",
            "..    ...            ...   ...\n",
            "78  20166              .     O\n",
            "79  20167            Gel     O\n",
            "80  20168          shift     O\n",
            "81  20169       analysis     O\n",
            "82  20170             of     O\n",
            "\n",
            "[83 rows x 3 columns]\n",
            "6 mg of sustained - release oral nitroglycerin at 8 AM , 3 PM , and 10 PM , n = 5 ; subgroup 3 : oral isosorbide dinitrate , 30 mg at 8 AM and 4 PM , n = 11 ) or 6 weeks ( subgroup 4 : isosorbide dinitrate , n = 10 ) and in matching controls ( n = 32 ) . The essential oil composition of three Zingiberaceae widely used as medicinal aromatic plants from S .\n",
            "['6', 'mg', 'of', 'sustained', '-', 'release', 'oral', 'nitroglycerin', 'at', '8', 'AM', ',', '3', 'PM', ',', 'and', '10', 'PM', ',', 'n', '=', '5', ';', 'subgroup', '3', ':', 'oral', 'isosorbide', 'dinitrate', ',', '30', 'mg', 'at', '8', 'AM', 'and', '4', 'PM', ',', 'n', '=', '11', ')', 'or', '6', 'weeks', '(', 'subgroup', '4', ':', 'isosorbide', 'dinitrate', ',', 'n', '=', '10', ')', 'and', 'in', 'matching', 'controls', '(', 'n', '=', '32', ')', '.', 'The', 'essential', 'oil', 'composition', 'of', 'three', 'Zingiberaceae', 'widely', 'used', 'as', 'medicinal', 'aromatic', 'plants', 'from', 'S', '.']\n",
            "83 83\n",
            "['-_0', 'release_1', 'oral_2', 'nitroglycerin_3', 'at_4', '8_5', 'AM_6', ',_7', '3_8', 'PM_9', ',_10', 'and_11', '10_12', 'PM_13', ',_14', 'n_15', '=_16', '5_17', ';_18', 'subgroup_19', '3_20', ':_21', 'oral_22', 'isosorbide_23', 'dinitrate_24', ',_25', '30_26', 'mg_27', 'at_28', '8_29', 'AM_30', 'and_31', '4_32', 'PM_33', ',_34', 'n_35', '=_36', '11_37', ')_38', 'or_39', '6_40', 'weeks_41', '(_42', 'subgroup_43', '4_44', ':_45', 'isosorbide_46', 'dinitrate_47', ',_48', 'n_49', '=_50', '10_51', ')_52', 'and_53', 'in_54', 'matching_55', 'controls_56', '(_57', 'n_58', '=_59', '32_60', ')_61', '._62', 'The_63', 'essential_64', 'oil_65', 'composition_66', 'of_67', 'three_68', 'Zingiberaceae_69', 'widely_70', 'used_71', 'as_72', 'medicinal_73', 'aromatic_74', 'plants_75', 'from_76', 'S_77', '._78', 'Gel_79', 'shift_80', 'analysis_81', 'of_82']\n",
            "BAD:  20088\n",
            "rpws:      index        token label\n",
            "0   20171      protein     O\n",
            "1   20172      binding     O\n",
            "2   20173         from     O\n",
            "3   20174      nuclear     O\n",
            "4   20175     extracts     O\n",
            "..    ...          ...   ...\n",
            "56  20227            .     O\n",
            "57  20228  CONCLUSIONS     O\n",
            "58  20229            :     O\n",
            "59  20230          The     O\n",
            "60  20231           MC     O\n",
            "\n",
            "[61 rows x 3 columns]\n",
            "Gel shift analysis of protein binding from nuclear extracts to these caveolin promoter DNA sequences , together with DNase I footprinting , confirmed nucleoprotein binding to the SRE - like elements as part of the transcriptional response to LDL - FC . A new model for objective assessment of cervical ripening : the effect of prostaglandin E2 and prelabor contractility .\n",
            "['Gel', 'shift', 'analysis', 'of', 'protein', 'binding', 'from', 'nuclear', 'extracts', 'to', 'these', 'caveolin', 'promoter', 'DNA', 'sequences', ',', 'together', 'with', 'DNase', 'I', 'footprinting', ',', 'confirmed', 'nucleoprotein', 'binding', 'to', 'the', 'SRE', '-', 'like', 'elements', 'as', 'part', 'of', 'the', 'transcriptional', 'response', 'to', 'LDL', '-', 'FC', '.', 'A', 'new', 'model', 'for', 'objective', 'assessment', 'of', 'cervical', 'ripening', ':', 'the', 'effect', 'of', 'prostaglandin', 'E2', 'and', 'prelabor', 'contractility', '.']\n",
            "61 61\n",
            "['protein_0', 'binding_1', 'from_2', 'nuclear_3', 'extracts_4', 'to_5', 'these_6', 'caveolin_7', 'promoter_8', 'DNA_9', 'sequences_10', ',_11', 'together_12', 'with_13', 'DNase_14', 'I_15', 'footprinting_16', ',_17', 'confirmed_18', 'nucleoprotein_19', 'binding_20', 'to_21', 'the_22', 'SRE_23', '-_24', 'like_25', 'elements_26', 'as_27', 'part_28', 'of_29', 'the_30', 'transcriptional_31', 'response_32', 'to_33', 'LDL_34', '-_35', 'FC_36', '._37', 'A_38', 'new_39', 'model_40', 'for_41', 'objective_42', 'assessment_43', 'of_44', 'cervical_45', 'ripening_46', ':_47', 'the_48', 'effect_49', 'of_50', 'prostaglandin_51', 'E2_52', 'and_53', 'prelabor_54', 'contractility_55', '._56', 'CONCLUSIONS_57', ':_58', 'The_59', 'MC_60']\n",
            "BAD:  20171\n",
            "rpws:      index      token label\n",
            "0   20232          /     O\n",
            "1   20233         UA     O\n",
            "2   20234      ratio     O\n",
            "3   20235   improves     O\n",
            "4   20236        the     O\n",
            "..    ...        ...   ...\n",
            "56  20288          .     O\n",
            "57  20289          (     O\n",
            "58  20290   ABSTRACT     O\n",
            "59  20291  TRUNCATED     O\n",
            "60  20292         AT     O\n",
            "\n",
            "[61 rows x 3 columns]\n",
            "CONCLUSIONS : The MC / UA ratio improves the sensitivity for the prediction of poor perinatal outcome when it is combined with the NST . Evaluation of materials and technics in vascular surgery using 111 - Indium The claustrocortical connection was investigated in 13 cats with selective injections of 30 % HRP in the three subdivisions of the auditory cortex .\n",
            "['CONCLUSIONS', ':', 'The', 'MC', '/', 'UA', 'ratio', 'improves', 'the', 'sensitivity', 'for', 'the', 'prediction', 'of', 'poor', 'perinatal', 'outcome', 'when', 'it', 'is', 'combined', 'with', 'the', 'NST', '.', 'Evaluation', 'of', 'materials', 'and', 'technics', 'in', 'vascular', 'surgery', 'using', '111', '-', 'Indium', 'The', 'claustrocortical', 'connection', 'was', 'investigated', 'in', '13', 'cats', 'with', 'selective', 'injections', 'of', '30', '%', 'HRP', 'in', 'the', 'three', 'subdivisions', 'of', 'the', 'auditory', 'cortex', '.']\n",
            "61 61\n",
            "['/_0', 'UA_1', 'ratio_2', 'improves_3', 'the_4', 'sensitivity_5', 'for_6', 'the_7', 'prediction_8', 'of_9', 'poor_10', 'perinatal_11', 'outcome_12', 'when_13', 'it_14', 'is_15', 'combined_16', 'with_17', 'the_18', 'NST_19', '._20', 'Evaluation_21', 'of_22', 'materials_23', 'and_24', 'technics_25', 'in_26', 'vascular_27', 'surgery_28', 'using_29', '111_30', '-_31', 'Indium_32', 'The_33', 'claustrocortical_34', 'connection_35', 'was_36', 'investigated_37', 'in_38', '13_39', 'cats_40', 'with_41', 'selective_42', 'injections_43', 'of_44', '30_45', '%_46', 'HRP_47', 'in_48', 'the_49', 'three_50', 'subdivisions_51', 'of_52', 'the_53', 'auditory_54', 'cortex_55', '._56', '(_57', 'ABSTRACT_58', 'TRUNCATED_59', 'AT_60']\n",
            "BAD:  20232\n",
            "rpws:      index        token label\n",
            "0   20293          400     O\n",
            "1   20294        WORDS     O\n",
            "2   20295            )     O\n",
            "3   20296     Applying     O\n",
            "4   20297          the     O\n",
            "..    ...          ...   ...\n",
            "60  20353            .     O\n",
            "61  20354       METHOD     O\n",
            "62  20355            :     O\n",
            "63  20356          Six     O\n",
            "64  20357  instruments     O\n",
            "\n",
            "[65 rows x 3 columns]\n",
            "( ABSTRACT TRUNCATED AT 400 WORDS ) Applying the experience mentioned above , 4 - META was used to bond a proprietary photocuring microfilled composite material to Class V cavities in freshly extracted human teeth . The phP1 mutation was induced by insertion of a 1 . 2 - kb P element into the 5 ' transcribed nontranslated region of the proximal polyhomeotic gene .\n",
            "['(', 'ABSTRACT', 'TRUNCATED', 'AT', '400', 'WORDS', ')', 'Applying', 'the', 'experience', 'mentioned', 'above', ',', '4', '-', 'META', 'was', 'used', 'to', 'bond', 'a', 'proprietary', 'photocuring', 'microfilled', 'composite', 'material', 'to', 'Class', 'V', 'cavities', 'in', 'freshly', 'extracted', 'human', 'teeth', '.', 'The', 'phP1', 'mutation', 'was', 'induced', 'by', 'insertion', 'of', 'a', '1', '.', '2', '-', 'kb', 'P', 'element', 'into', 'the', '5', \"'\", 'transcribed', 'nontranslated', 'region', 'of', 'the', 'proximal', 'polyhomeotic', 'gene', '.']\n",
            "65 65\n",
            "['400_0', 'WORDS_1', ')_2', 'Applying_3', 'the_4', 'experience_5', 'mentioned_6', 'above_7', ',_8', '4_9', '-_10', 'META_11', 'was_12', 'used_13', 'to_14', 'bond_15', 'a_16', 'proprietary_17', 'photocuring_18', 'microfilled_19', 'composite_20', 'material_21', 'to_22', 'Class_23', 'V_24', 'cavities_25', 'in_26', 'freshly_27', 'extracted_28', 'human_29', 'teeth_30', '._31', 'The_32', 'phP1_33', 'mutation_34', 'was_35', 'induced_36', 'by_37', 'insertion_38', 'of_39', 'a_40', '1_41', '._42', '2_43', '-_44', 'kb_45', 'P_46', 'element_47', 'into_48', 'the_49', '5_50', \"'_51\", 'transcribed_52', 'nontranslated_53', 'region_54', 'of_55', 'the_56', 'proximal_57', 'polyhomeotic_58', 'gene_59', '._60', 'METHOD_61', ':_62', 'Six_63', 'instruments_64']\n",
            "BAD:  20293\n",
            "rpws:      index       token label\n",
            "0   20358        were     O\n",
            "1   20359    reviewed     O\n",
            "2   20360           :     O\n",
            "3   20361         the     O\n",
            "4   20362        Berg     O\n",
            "..    ...         ...   ...\n",
            "69  20427       based     O\n",
            "70  20428  similarity     O\n",
            "71  20429    searches     O\n",
            "72  20430  identified     O\n",
            "73  20431       human     B\n",
            "\n",
            "[74 rows x 3 columns]\n",
            "METHOD : Six instruments were reviewed : the Berg Balance Scale ( Berg ) , the Clinical Test of Sensory Interaction and Balance ( CTSIB ) , the Functional Reach Test , the Tinetti Balance Test of the Performance - Oriented Assessment of Mobility Problems ( Tinetti ) , the Timed \tO\n",
            "Up\tO\n",
            "and\tO\n",
            "Go\tO\n",
            " Test ( TU & GT ) , and the Physical Performance Test ( PPT ) .\n",
            "['METHOD', ':', 'Six', 'instruments', 'were', 'reviewed', ':', 'the', 'Berg', 'Balance', 'Scale', '(', 'Berg', ')', ',', 'the', 'Clinical', 'Test', 'of', 'Sensory', 'Interaction', 'and', 'Balance', '(', 'CTSIB', ')', ',', 'the', 'Functional', 'Reach', 'Test', ',', 'the', 'Tinetti', 'Balance', 'Test', 'of', 'the', 'Performance', '-', 'Oriented', 'Assessment', 'of', 'Mobility', 'Problems', '(', 'Tinetti', ')', ',', 'the', 'Timed', 'O', 'Up', 'O', 'and', 'O', 'Go', 'O', 'Test', '(', 'TU', '&', 'GT', ')', ',', 'and', 'the', 'Physical', 'Performance', 'Test', '(', 'PPT', ')', '.']\n",
            "74 74\n",
            "['were_0', 'reviewed_1', ':_2', 'the_3', 'Berg_4', 'Balance_5', 'Scale_6', '(_7', 'Berg_8', ')_9', ',_10', 'the_11', 'Clinical_12', 'Test_13', 'of_14', 'Sensory_15', 'Interaction_16', 'and_17', 'Balance_18', '(_19', 'CTSIB_20', ')_21', ',_22', 'the_23', 'Functional_24', 'Reach_25', 'Test_26', ',_27', 'the_28', 'Tinetti_29', 'Balance_30', 'Test_31', 'of_32', 'the_33', 'Performance_34', '-_35', 'Oriented_36', 'Assessment_37', 'of_38', 'Mobility_39', 'Problems_40', '(_41', 'Tinetti_42', ')_43', ',_44', 'the_45', 'Timed_46', '\\tO\\nUp\\tO\\nand\\tO\\nGo\\tO\\n_47', 'Test_48', '(_49', 'TU_50', '&_51', 'GT_52', ')_53', ',_54', 'and_55', 'the_56', 'Physical_57', 'Performance_58', 'Test_59', '(_60', 'PPT_61', ')_62', '._63', 'Genome_64', 'timeline_65', '._66', 'Initial_67', 'computer_68', 'based_69', 'similarity_70', 'searches_71', 'identified_72', 'human_73']\n",
            "BAD:  20358\n",
            "rpws:      index           token label\n",
            "0   20432  retinoblastoma     I\n",
            "1   20433         binding     I\n",
            "2   20434         protein     I\n",
            "3   20435               1     I\n",
            "4   20436               (     O\n",
            "..    ...             ...   ...\n",
            "57  20489    measurements     O\n",
            "58  20490           under     O\n",
            "59  20491          steady     O\n",
            "60  20492            flow     O\n",
            "61  20493      conditions     O\n",
            "\n",
            "[62 rows x 3 columns]\n",
            "Genome timeline . Initial computer based similarity searches identified human retinoblastoma binding protein 1 ( RBP - 1 ) , Drosophila melanogaster male specific lethal - 3 ( Msl - 3 ) , S . pombe altered polarity - 13 ( Alp13 ) and S . cerevisiae Eaf3p , a component of the yeast NuA4 HAT complex ( Galarneau et al .\n",
            "['Genome', 'timeline', '.', 'Initial', 'computer', 'based', 'similarity', 'searches', 'identified', 'human', 'retinoblastoma', 'binding', 'protein', '1', '(', 'RBP', '-', '1', ')', ',', 'Drosophila', 'melanogaster', 'male', 'specific', 'lethal', '-', '3', '(', 'Msl', '-', '3', ')', ',', 'S', '.', 'pombe', 'altered', 'polarity', '-', '13', '(', 'Alp13', ')', 'and', 'S', '.', 'cerevisiae', 'Eaf3p', ',', 'a', 'component', 'of', 'the', 'yeast', 'NuA4', 'HAT', 'complex', '(', 'Galarneau', 'et', 'al', '.']\n",
            "62 62\n",
            "['retinoblastoma_0', 'binding_1', 'protein_2', '1_3', '(_4', 'RBP_5', '-_6', '1_7', ')_8', ',_9', 'Drosophila_10', 'melanogaster_11', 'male_12', 'specific_13', 'lethal_14', '-_15', '3_16', '(_17', 'Msl_18', '-_19', '3_20', ')_21', ',_22', 'S_23', '._24', 'pombe_25', 'altered_26', 'polarity_27', '-_28', '13_29', '(_30', 'Alp13_31', ')_32', 'and_33', 'S_34', '._35', 'cerevisiae_36', 'Eaf3p_37', ',_38', 'a_39', 'component_40', 'of_41', 'the_42', 'yeast_43', 'NuA4_44', 'HAT_45', 'complex_46', '(_47', 'Galarneau_48', 'et_49', 'al_50', '._51', ',_52', '2000_53', '._54', 'The_55', 'pressure_56', 'measurements_57', 'under_58', 'steady_59', 'flow_60', 'conditions_61']\n",
            "BAD:  20432\n",
            "rpws:      index        token label\n",
            "0   20494       showed     O\n",
            "1   20495         that     O\n",
            "2   20496          the     O\n",
            "3   20497  hemodynamic     O\n",
            "4   20498  performance     O\n",
            "..    ...          ...   ...\n",
            "61  20555    evaluable     O\n",
            "62  20556     patients     O\n",
            "63  20557   undergoing     O\n",
            "64  20558   allogeneic     O\n",
            "65  20559      sibling     O\n",
            "\n",
            "[66 rows x 3 columns]\n",
            ", 2000 . The pressure measurements under steady flow conditions showed that the hemodynamic performance ( including pressure gradient and effective orifice area ) of SPAB is superior to that of its stented counterpart , especially in the smaller sizes . Serum prolactin rapidly decreased after institution of treatment , with actual normalization ( less than 20 ng / ml ) by the 3rd month .\n",
            "[',', '2000', '.', 'The', 'pressure', 'measurements', 'under', 'steady', 'flow', 'conditions', 'showed', 'that', 'the', 'hemodynamic', 'performance', '(', 'including', 'pressure', 'gradient', 'and', 'effective', 'orifice', 'area', ')', 'of', 'SPAB', 'is', 'superior', 'to', 'that', 'of', 'its', 'stented', 'counterpart', ',', 'especially', 'in', 'the', 'smaller', 'sizes', '.', 'Serum', 'prolactin', 'rapidly', 'decreased', 'after', 'institution', 'of', 'treatment', ',', 'with', 'actual', 'normalization', '(', 'less', 'than', '20', 'ng', '/', 'ml', ')', 'by', 'the', '3rd', 'month', '.']\n",
            "66 66\n",
            "['showed_0', 'that_1', 'the_2', 'hemodynamic_3', 'performance_4', '(_5', 'including_6', 'pressure_7', 'gradient_8', 'and_9', 'effective_10', 'orifice_11', 'area_12', ')_13', 'of_14', 'SPAB_15', 'is_16', 'superior_17', 'to_18', 'that_19', 'of_20', 'its_21', 'stented_22', 'counterpart_23', ',_24', 'especially_25', 'in_26', 'the_27', 'smaller_28', 'sizes_29', '._30', 'Serum_31', 'prolactin_32', 'rapidly_33', 'decreased_34', 'after_35', 'institution_36', 'of_37', 'treatment_38', ',_39', 'with_40', 'actual_41', 'normalization_42', '(_43', 'less_44', 'than_45', '20_46', 'ng_47', '/_48', 'ml_49', ')_50', 'by_51', 'the_52', '3rd_53', 'month_54', '._55', 'We_56', 'report_57', 'five_58', 'of_59', '13_60', 'evaluable_61', 'patients_62', 'undergoing_63', 'allogeneic_64', 'sibling_65']\n",
            "BAD:  20494\n",
            "rpws:      index            token label\n",
            "0   20560               BM     O\n",
            "1   20561               or     O\n",
            "2   20562             PBSC     O\n",
            "3   20563  transplantation     O\n",
            "4   20564              for     O\n",
            "..    ...              ...   ...\n",
            "57  20617                -     O\n",
            "58  20618            power     O\n",
            "59  20619            laser     O\n",
            "60  20620      irradiation     O\n",
            "61  20621              can     O\n",
            "\n",
            "[62 rows x 3 columns]\n",
            "We report five of 13 evaluable patients undergoing allogeneic sibling BM or PBSC transplantation for MM between 1990 and 1997 who met the criteria for adjuvant alpha - IFN therapy . Moist healing versus wet - to - dry . In this report , we identify Tyr319 as a functionally important phosphorylation site in the ZAP - 70 interdomain B region .\n",
            "['We', 'report', 'five', 'of', '13', 'evaluable', 'patients', 'undergoing', 'allogeneic', 'sibling', 'BM', 'or', 'PBSC', 'transplantation', 'for', 'MM', 'between', '1990', 'and', '1997', 'who', 'met', 'the', 'criteria', 'for', 'adjuvant', 'alpha', '-', 'IFN', 'therapy', '.', 'Moist', 'healing', 'versus', 'wet', '-', 'to', '-', 'dry', '.', 'In', 'this', 'report', ',', 'we', 'identify', 'Tyr319', 'as', 'a', 'functionally', 'important', 'phosphorylation', 'site', 'in', 'the', 'ZAP', '-', '70', 'interdomain', 'B', 'region', '.']\n",
            "62 62\n",
            "['BM_0', 'or_1', 'PBSC_2', 'transplantation_3', 'for_4', 'MM_5', 'between_6', '1990_7', 'and_8', '1997_9', 'who_10', 'met_11', 'the_12', 'criteria_13', 'for_14', 'adjuvant_15', 'alpha_16', '-_17', 'IFN_18', 'therapy_19', '._20', 'Moist_21', 'healing_22', 'versus_23', 'wet_24', '-_25', 'to_26', '-_27', 'dry_28', '._29', 'In_30', 'this_31', 'report_32', ',_33', 'we_34', 'identify_35', 'Tyr319_36', 'as_37', 'a_38', 'functionally_39', 'important_40', 'phosphorylation_41', 'site_42', 'in_43', 'the_44', 'ZAP_45', '-_46', '70_47', 'interdomain_48', 'B_49', 'region_50', '._51', 'These_52', 'findings_53', 'suggest_54', 'that_55', 'low_56', '-_57', 'power_58', 'laser_59', 'irradiation_60', 'can_61']\n",
            "BAD:  20560\n",
            "rpws:      index      token label\n",
            "0   20622         be     O\n",
            "1   20623       used     O\n",
            "2   20624        for     O\n",
            "3   20625  promotion     O\n",
            "4   20626         of     O\n",
            "..    ...        ...   ...\n",
            "57  20679          .     O\n",
            "58  20680          1     O\n",
            "59  20681         kg     O\n",
            "60  20682          /     O\n",
            "61  20683         m2     O\n",
            "\n",
            "[62 rows x 3 columns]\n",
            "These findings suggest that low - power laser irradiation can be used for promotion of vascularization and take of tissue transplants . RESEARCH DESIGN AND METHODS : A total of 26 IDDM patients with normoalbuminuria were randomized into two groups , with one group receiving placebo ( n = 13 , age 36 + / - 3 years , BMI 24 .\n",
            "['These', 'findings', 'suggest', 'that', 'low', '-', 'power', 'laser', 'irradiation', 'can', 'be', 'used', 'for', 'promotion', 'of', 'vascularization', 'and', 'take', 'of', 'tissue', 'transplants', '.', 'RESEARCH', 'DESIGN', 'AND', 'METHODS', ':', 'A', 'total', 'of', '26', 'IDDM', 'patients', 'with', 'normoalbuminuria', 'were', 'randomized', 'into', 'two', 'groups', ',', 'with', 'one', 'group', 'receiving', 'placebo', '(', 'n', '=', '13', ',', 'age', '36', '+', '/', '-', '3', 'years', ',', 'BMI', '24', '.']\n",
            "62 62\n",
            "['be_0', 'used_1', 'for_2', 'promotion_3', 'of_4', 'vascularization_5', 'and_6', 'take_7', 'of_8', 'tissue_9', 'transplants_10', '._11', 'RESEARCH_12', 'DESIGN_13', 'AND_14', 'METHODS_15', ':_16', 'A_17', 'total_18', 'of_19', '26_20', 'IDDM_21', 'patients_22', 'with_23', 'normoalbuminuria_24', 'were_25', 'randomized_26', 'into_27', 'two_28', 'groups_29', ',_30', 'with_31', 'one_32', 'group_33', 'receiving_34', 'placebo_35', '(_36', 'n_37', '=_38', '13_39', ',_40', 'age_41', '36_42', '+_43', '/_44', '-_45', '3_46', 'years_47', ',_48', 'BMI_49', '24_50', '._51', '5_52', '+_53', '/_54', '-_55', '1_56', '._57', '1_58', 'kg_59', '/_60', 'm2_61']\n",
            "BAD:  20622\n",
            "rpws:      index     token label\n",
            "0   20684         )     O\n",
            "1   20685       and     O\n",
            "2   20686       the     O\n",
            "3   20687     other     O\n",
            "4   20688     group     O\n",
            "..    ...       ...   ...\n",
            "94  20778        kb     O\n",
            "95  20779   genomic     O\n",
            "96  20780  fragment     O\n",
            "97  20781  contains     O\n",
            "98  20782       the     O\n",
            "\n",
            "[99 rows x 3 columns]\n",
            "5 + / - 1 . 1 kg / m2 ) and the other group receiving an average of 15 mg lisinopril daily ( n = 13 , age 34 + / - 2 years , BMI 24 . 4 + / - 0 . 9 kg / m2 ) . Previously , a cDNA ( GT2 ) encoding this protein was isolated from a mouse 3T3 - L1 adipocyte library and was sequenced . Expression of human complement receptor type 2 ( CR2 / CD21 ) is primarily restricted to mature B cells and follicular dendritic cells .\n",
            "['5', '+', '/', '-', '1', '.', '1', 'kg', '/', 'm2', ')', 'and', 'the', 'other', 'group', 'receiving', 'an', 'average', 'of', '15', 'mg', 'lisinopril', 'daily', '(', 'n', '=', '13', ',', 'age', '34', '+', '/', '-', '2', 'years', ',', 'BMI', '24', '.', '4', '+', '/', '-', '0', '.', '9', 'kg', '/', 'm2', ')', '.', 'Previously', ',', 'a', 'cDNA', '(', 'GT2', ')', 'encoding', 'this', 'protein', 'was', 'isolated', 'from', 'a', 'mouse', '3T3', '-', 'L1', 'adipocyte', 'library', 'and', 'was', 'sequenced', '.', 'Expression', 'of', 'human', 'complement', 'receptor', 'type', '2', '(', 'CR2', '/', 'CD21', ')', 'is', 'primarily', 'restricted', 'to', 'mature', 'B', 'cells', 'and', 'follicular', 'dendritic', 'cells', '.']\n",
            "99 99\n",
            "[')_0', 'and_1', 'the_2', 'other_3', 'group_4', 'receiving_5', 'an_6', 'average_7', 'of_8', '15_9', 'mg_10', 'lisinopril_11', 'daily_12', '(_13', 'n_14', '=_15', '13_16', ',_17', 'age_18', '34_19', '+_20', '/_21', '-_22', '2_23', 'years_24', ',_25', 'BMI_26', '24_27', '._28', '4_29', '+_30', '/_31', '-_32', '0_33', '._34', '9_35', 'kg_36', '/_37', 'm2_38', ')_39', '._40', 'Previously_41', ',_42', 'a_43', 'cDNA_44', '(_45', 'GT2_46', ')_47', 'encoding_48', 'this_49', 'protein_50', 'was_51', 'isolated_52', 'from_53', 'a_54', 'mouse_55', '3T3_56', '-_57', 'L1_58', 'adipocyte_59', 'library_60', 'and_61', 'was_62', 'sequenced_63', '._64', 'Expression_65', 'of_66', 'human_67', 'complement_68', 'receptor_69', 'type_70', '2_71', '(_72', 'CR2_73', '/_74', 'CD21_75', ')_76', 'is_77', 'primarily_78', 'restricted_79', 'to_80', 'mature_81', 'B_82', 'cells_83', 'and_84', 'follicular_85', 'dendritic_86', 'cells_87', '._88', 'The_89', '6_90', '._91', '5_92', '-_93', 'kb_94', 'genomic_95', 'fragment_96', 'contains_97', 'the_98']\n",
            "BAD:  20684\n",
            "rpws:      index     token label\n",
            "0   20783  complete     O\n",
            "1   20784    coding     O\n",
            "2   20785    region     O\n",
            "3   20786        of     O\n",
            "4   20787      MyoD     B\n",
            "..    ...       ...   ...\n",
            "61  20844         )     O\n",
            "62  20845         M     O\n",
            "63  20846         -     O\n",
            "64  20847         1     O\n",
            "65  20848   whereas     O\n",
            "\n",
            "[66 rows x 3 columns]\n",
            "The 6 . 5 - kb genomic fragment contains the complete coding region of MyoD , distributed over three exons , plus 2 . 3 kb of 5 ' - noncoding sequence and 1 . 4 kb of 3 ' - noncoding sequence . Synthetic studies on furan derivatives by the wittig reaction The apparent binding constant of 6 to calf thymus DNA is 1 .\n",
            "['The', '6', '.', '5', '-', 'kb', 'genomic', 'fragment', 'contains', 'the', 'complete', 'coding', 'region', 'of', 'MyoD', ',', 'distributed', 'over', 'three', 'exons', ',', 'plus', '2', '.', '3', 'kb', 'of', '5', \"'\", '-', 'noncoding', 'sequence', 'and', '1', '.', '4', 'kb', 'of', '3', \"'\", '-', 'noncoding', 'sequence', '.', 'Synthetic', 'studies', 'on', 'furan', 'derivatives', 'by', 'the', 'wittig', 'reaction', 'The', 'apparent', 'binding', 'constant', 'of', '6', 'to', 'calf', 'thymus', 'DNA', 'is', '1', '.']\n",
            "66 66\n",
            "['complete_0', 'coding_1', 'region_2', 'of_3', 'MyoD_4', ',_5', 'distributed_6', 'over_7', 'three_8', 'exons_9', ',_10', 'plus_11', '2_12', '._13', '3_14', 'kb_15', 'of_16', '5_17', \"'_18\", '-_19', 'noncoding_20', 'sequence_21', 'and_22', '1_23', '._24', '4_25', 'kb_26', 'of_27', '3_28', \"'_29\", '-_30', 'noncoding_31', 'sequence_32', '._33', 'Synthetic_34', 'studies_35', 'on_36', 'furan_37', 'derivatives_38', 'by_39', 'the_40', 'wittig_41', 'reaction_42', 'The_43', 'apparent_44', 'binding_45', 'constant_46', 'of_47', '6_48', 'to_49', 'calf_50', 'thymus_51', 'DNA_52', 'is_53', '1_54', '._55', '68_56', 'X_57', '10_58', '(_59', '5_60', ')_61', 'M_62', '-_63', '1_64', 'whereas_65']\n",
            "BAD:  20783\n",
            "rpws:      index       token label\n",
            "0   20849   netropsin     B\n",
            "1   20850       under     O\n",
            "2   20851     similar     O\n",
            "3   20852  conditions     O\n",
            "4   20853       gives     O\n",
            "..    ...         ...   ...\n",
            "85  20934    promoter     I\n",
            "86  20935           (     O\n",
            "87  20936    pLovTATA     B\n",
            "88  20937           )     O\n",
            "89  20938    template     O\n",
            "\n",
            "[90 rows x 3 columns]\n",
            "68 X 10 ( 5 ) M - 1 whereas netropsin under similar conditions gives a value of 1 . 85 X 10 ( 7 ) M - 1 . Phenotypic screening of mutations in Pmr1 , the yeast secretory pathway Ca2 + / Mn2 + - ATPase , reveals residues critical for ion selectivity and transport . Molecular cloning of cDNAs encoding alpha - subunits of guanine nucleotide - binding regulatory proteins ( G - proteins ) has revealed the existence of nine species of alpha - subunits .\n",
            "['68', 'X', '10', '(', '5', ')', 'M', '-', '1', 'whereas', 'netropsin', 'under', 'similar', 'conditions', 'gives', 'a', 'value', 'of', '1', '.', '85', 'X', '10', '(', '7', ')', 'M', '-', '1', '.', 'Phenotypic', 'screening', 'of', 'mutations', 'in', 'Pmr1', ',', 'the', 'yeast', 'secretory', 'pathway', 'Ca2', '+', '/', 'Mn2', '+', '-', 'ATPase', ',', 'reveals', 'residues', 'critical', 'for', 'ion', 'selectivity', 'and', 'transport', '.', 'Molecular', 'cloning', 'of', 'cDNAs', 'encoding', 'alpha', '-', 'subunits', 'of', 'guanine', 'nucleotide', '-', 'binding', 'regulatory', 'proteins', '(', 'G', '-', 'proteins', ')', 'has', 'revealed', 'the', 'existence', 'of', 'nine', 'species', 'of', 'alpha', '-', 'subunits', '.']\n",
            "90 90\n",
            "['netropsin_0', 'under_1', 'similar_2', 'conditions_3', 'gives_4', 'a_5', 'value_6', 'of_7', '1_8', '._9', '85_10', 'X_11', '10_12', '(_13', '7_14', ')_15', 'M_16', '-_17', '1_18', '._19', 'Phenotypic_20', 'screening_21', 'of_22', 'mutations_23', 'in_24', 'Pmr1_25', ',_26', 'the_27', 'yeast_28', 'secretory_29', 'pathway_30', 'Ca2_31', '+_32', '/_33', 'Mn2_34', '+_35', '-_36', 'ATPase_37', ',_38', 'reveals_39', 'residues_40', 'critical_41', 'for_42', 'ion_43', 'selectivity_44', 'and_45', 'transport_46', '._47', 'Molecular_48', 'cloning_49', 'of_50', 'cDNAs_51', 'encoding_52', 'alpha_53', '-_54', 'subunits_55', 'of_56', 'guanine_57', 'nucleotide_58', '-_59', 'binding_60', 'regulatory_61', 'proteins_62', '(_63', 'G_64', '-_65', 'proteins_66', ')_67', 'has_68', 'revealed_69', 'the_70', 'existence_71', 'of_72', 'nine_73', 'species_74', 'of_75', 'alpha_76', '-_77', 'subunits_78', '._79', 'No_80', 'transactivation_81', 'of_82', 'the_83', 'ovalbumin_84', 'promoter_85', '(_86', 'pLovTATA_87', ')_88', 'template_89']\n",
            "BAD:  20849\n",
            "rpws:      index     token label\n",
            "0   20939   control     O\n",
            "1   20940       was     O\n",
            "2   20941  observed     O\n",
            "3   20942         .     O\n",
            "4   20943      Pros     O\n",
            "..    ...       ...   ...\n",
            "58  20997        73     O\n",
            "59  20998         (     O\n",
            "60  20999        76     O\n",
            "61  21000         .     O\n",
            "62  21001         8     O\n",
            "\n",
            "[63 rows x 3 columns]\n",
            "No transactivation of the ovalbumin promoter ( pLovTATA ) template control was observed . Pros and cons of selective inhibition of cyclooxygenase - 2 versus dual lipoxygenase / cyclooxygenase inhibition : is two better than one ? At 11 . 5 years of follow - up , 521 major coronary disease events had occurred , 261 fatal and 260 non - fatal .\n",
            "['No', 'transactivation', 'of', 'the', 'ovalbumin', 'promoter', '(', 'pLovTATA', ')', 'template', 'control', 'was', 'observed', '.', 'Pros', 'and', 'cons', 'of', 'selective', 'inhibition', 'of', 'cyclooxygenase', '-', '2', 'versus', 'dual', 'lipoxygenase', '/', 'cyclooxygenase', 'inhibition', ':', 'is', 'two', 'better', 'than', 'one', '?', 'At', '11', '.', '5', 'years', 'of', 'follow', '-', 'up', ',', '521', 'major', 'coronary', 'disease', 'events', 'had', 'occurred', ',', '261', 'fatal', 'and', '260', 'non', '-', 'fatal', '.']\n",
            "63 63\n",
            "['control_0', 'was_1', 'observed_2', '._3', 'Pros_4', 'and_5', 'cons_6', 'of_7', 'selective_8', 'inhibition_9', 'of_10', 'cyclooxygenase_11', '-_12', '2_13', 'versus_14', 'dual_15', 'lipoxygenase_16', '/_17', 'cyclooxygenase_18', 'inhibition_19', ':_20', 'is_21', 'two_22', 'better_23', 'than_24', 'one_25', '?_26', 'At_27', '11_28', '._29', '5_30', 'years_31', 'of_32', 'follow_33', '-_34', 'up_35', ',_36', '521_37', 'major_38', 'coronary_39', 'disease_40', 'events_41', 'had_42', 'occurred_43', ',_44', '261_45', 'fatal_46', 'and_47', '260_48', 'non_49', '-_50', 'fatal_51', '._52', 'The_53', 'etiology_54', 'was_55', 'established_56', 'in_57', '73_58', '(_59', '76_60', '._61', '8_62']\n",
            "BAD:  20939\n",
            "rpws:      index      token label\n",
            "0   21002          %     O\n",
            "1   21003          )     O\n",
            "2   21004        out     O\n",
            "3   21005         of     O\n",
            "4   21006         95     O\n",
            "..    ...        ...   ...\n",
            "72  21074  detecting     O\n",
            "73  21075    members     O\n",
            "74  21076         of     O\n",
            "75  21077    complex     O\n",
            "76  21078  metabolic     O\n",
            "\n",
            "[77 rows x 3 columns]\n",
            "The etiology was established in 73 ( 76 . 8 % ) out of 95 cases . Morphometry of the intestine of the pig . Serum zinc and copper levels and urine copper concentrations in men were significantly lower than in women , while there were no differences in serum or urinary zinc and copper levels with age . HDE was found to be exclusively targeted to and imported into peroxisomes in both heterologous expression systems .\n",
            "['The', 'etiology', 'was', 'established', 'in', '73', '(', '76', '.', '8', '%', ')', 'out', 'of', '95', 'cases', '.', 'Morphometry', 'of', 'the', 'intestine', 'of', 'the', 'pig', '.', 'Serum', 'zinc', 'and', 'copper', 'levels', 'and', 'urine', 'copper', 'concentrations', 'in', 'men', 'were', 'significantly', 'lower', 'than', 'in', 'women', ',', 'while', 'there', 'were', 'no', 'differences', 'in', 'serum', 'or', 'urinary', 'zinc', 'and', 'copper', 'levels', 'with', 'age', '.', 'HDE', 'was', 'found', 'to', 'be', 'exclusively', 'targeted', 'to', 'and', 'imported', 'into', 'peroxisomes', 'in', 'both', 'heterologous', 'expression', 'systems', '.']\n",
            "77 77\n",
            "['%_0', ')_1', 'out_2', 'of_3', '95_4', 'cases_5', '._6', 'Morphometry_7', 'of_8', 'the_9', 'intestine_10', 'of_11', 'the_12', 'pig_13', '._14', 'Serum_15', 'zinc_16', 'and_17', 'copper_18', 'levels_19', 'and_20', 'urine_21', 'copper_22', 'concentrations_23', 'in_24', 'men_25', 'were_26', 'significantly_27', 'lower_28', 'than_29', 'in_30', 'women_31', ',_32', 'while_33', 'there_34', 'were_35', 'no_36', 'differences_37', 'in_38', 'serum_39', 'or_40', 'urinary_41', 'zinc_42', 'and_43', 'copper_44', 'levels_45', 'with_46', 'age_47', '._48', 'HDE_49', 'was_50', 'found_51', 'to_52', 'be_53', 'exclusively_54', 'targeted_55', 'to_56', 'and_57', 'imported_58', 'into_59', 'peroxisomes_60', 'in_61', 'both_62', 'heterologous_63', 'expression_64', 'systems_65', '._66', 'GeneCalling_67', 'analysis_68', 'was_69', 'successful_70', 'in_71', 'detecting_72', 'members_73', 'of_74', 'complex_75', 'metabolic_76']\n",
            "BAD:  21002\n",
            "rpws:      index            token label\n",
            "0   21079         pathways     O\n",
            "1   21080              and     O\n",
            "2   21081       uncovering     O\n",
            "3   21082            novel     O\n",
            "4   21083            genes     O\n",
            "5   21084             that     O\n",
            "6   21085             were     O\n",
            "7   21086           either     O\n",
            "8   21087   coincidentally     O\n",
            "9   21088        regulated     O\n",
            "10  21089               or     O\n",
            "11  21090         directly     O\n",
            "12  21091         involved     O\n",
            "13  21092               in     O\n",
            "14  21093             such     O\n",
            "15  21094         pathways     O\n",
            "16  21095                .     O\n",
            "17  21096  Transformations     O\n",
            "18  21097             with     O\n",
            "19  21098         circular     O\n",
            "20  21099         plasmids     O\n",
            "21  21100          yielded     O\n",
            "22  21101           slowly     O\n",
            "23  21102              and     O\n",
            "24  21103      irregularly     O\n",
            "25  21104          growing     O\n",
            "26  21105        geneticin     O\n",
            "27  21106                -     O\n",
            "28  21107        resistant     O\n",
            "29  21108          mycelia     O\n",
            "30  21109               in     O\n",
            "31  21110            which     O\n",
            "32  21111                1     O\n",
            "33  21112                %     O\n",
            "34  21113               of     O\n",
            "35  21114           nuclei     O\n",
            "36  21115        contained     O\n",
            "37  21116          plasmid     O\n",
            "38  21117        sequences     O\n",
            "39  21118                .     O\n",
            "40  21119       CONCLUSION     O\n",
            "41  21120                :     O\n",
            "42  21121               In     O\n",
            "43  21122            these     O\n",
            "44  21123         patients     O\n",
            "45  21124          control     O\n",
            "46  21125               of     O\n",
            "47  21126      ventricular     O\n",
            "48  21127         response     O\n",
            "49  21128             rate     O\n",
            "GeneCalling analysis was successful in detecting members of complex metabolic pathways and uncovering novel genes that were either coincidentally regulated or directly involved in such pathways . Transformations with circular plasmids yielded slowly and irregularly growing geneticin - resistant mycelia in which 1 % of nuclei contained plasmid sequences .\n",
            "['GeneCalling', 'analysis', 'was', 'successful', 'in', 'detecting', 'members', 'of', 'complex', 'metabolic', 'pathways', 'and', 'uncovering', 'novel', 'genes', 'that', 'were', 'either', 'coincidentally', 'regulated', 'or', 'directly', 'involved', 'in', 'such', 'pathways', '.', 'Transformations', 'with', 'circular', 'plasmids', 'yielded', 'slowly', 'and', 'irregularly', 'growing', 'geneticin', '-', 'resistant', 'mycelia', 'in', 'which', '1', '%', 'of', 'nuclei', 'contained', 'plasmid', 'sequences', '.']\n",
            "50 50\n",
            "['pathways_0', 'and_1', 'uncovering_2', 'novel_3', 'genes_4', 'that_5', 'were_6', 'either_7', 'coincidentally_8', 'regulated_9', 'or_10', 'directly_11', 'involved_12', 'in_13', 'such_14', 'pathways_15', '._16', 'Transformations_17', 'with_18', 'circular_19', 'plasmids_20', 'yielded_21', 'slowly_22', 'and_23', 'irregularly_24', 'growing_25', 'geneticin_26', '-_27', 'resistant_28', 'mycelia_29', 'in_30', 'which_31', '1_32', '%_33', 'of_34', 'nuclei_35', 'contained_36', 'plasmid_37', 'sequences_38', '._39', 'CONCLUSION_40', ':_41', 'In_42', 'these_43', 'patients_44', 'control_45', 'of_46', 'ventricular_47', 'response_48', 'rate_49']\n",
            "BAD:  21079\n",
            "rpws:      index             token label\n",
            "0   21129              with     O\n",
            "1   21130            either     O\n",
            "2   21131               HBA     O\n",
            "3   21132                 +     O\n",
            "4   21133              VVIR     O\n",
            "5   21134         pacemaker     O\n",
            "6   21135                or     O\n",
            "7   21136  atrioventricular     O\n",
            "8   21137         modifying     O\n",
            "9   21138             drugs     O\n",
            "10  21139                 +     O\n",
            "11  21140               VVI     O\n",
            "12  21141         pacemaker     O\n",
            "13  21142              will     O\n",
            "14  21143              lead     O\n",
            "15  21144                to     O\n",
            "16  21145                 a     O\n",
            "17  21146       significant     O\n",
            "18  21147       improvement     O\n",
            "19  21148                in     O\n",
            "20  21149          exercise     O\n",
            "21  21150          duration     O\n",
            "22  21151               and     O\n",
            "23  21152           quality     O\n",
            "24  21153                of     O\n",
            "25  21154              life     O\n",
            "26  21155                 .     O\n",
            "27  21156               The     O\n",
            "28  21157             other     O\n",
            "29  21158           regions     O\n",
            "30  21159           include     O\n",
            "31  21160         potential     O\n",
            "32  21161           binding     O\n",
            "33  21162             sites     O\n",
            "34  21163               for     O\n",
            "35  21164     transcription     O\n",
            "36  21165           factors     O\n",
            "37  21166               ATF     B\n",
            "38  21167                 ,     O\n",
            "39  21168               NF1     B\n",
            "40  21169                 ,     O\n",
            "41  21170               and     O\n",
            "42  21171                 a     O\n",
            "43  21172             CCAAT     B\n",
            "44  21173                 -     I\n",
            "45  21174           binding     I\n",
            "46  21175           protein     I\n",
            "47  21176                 .     O\n",
            "48  21177              Both     O\n",
            "49  21178              purH     B\n",
            "50  21179               and     O\n",
            "51  21180              purD     B\n",
            "52  21181             genes     I\n",
            "53  21182        constitute     O\n",
            "54  21183                 a     O\n",
            "55  21184            single     O\n",
            "56  21185            operon     O\n",
            "57  21186               and     O\n",
            "CONCLUSION : In these patients control of ventricular response rate with either HBA + VVIR pacemaker or atrioventricular modifying drugs + VVI pacemaker will lead to a significant improvement in exercise duration and quality of life . The other regions include potential binding sites for transcription factors ATF , NF1 , and a CCAAT - binding protein .\n",
            "['CONCLUSION', ':', 'In', 'these', 'patients', 'control', 'of', 'ventricular', 'response', 'rate', 'with', 'either', 'HBA', '+', 'VVIR', 'pacemaker', 'or', 'atrioventricular', 'modifying', 'drugs', '+', 'VVI', 'pacemaker', 'will', 'lead', 'to', 'a', 'significant', 'improvement', 'in', 'exercise', 'duration', 'and', 'quality', 'of', 'life', '.', 'The', 'other', 'regions', 'include', 'potential', 'binding', 'sites', 'for', 'transcription', 'factors', 'ATF', ',', 'NF1', ',', 'and', 'a', 'CCAAT', '-', 'binding', 'protein', '.']\n",
            "58 58\n",
            "['with_0', 'either_1', 'HBA_2', '+_3', 'VVIR_4', 'pacemaker_5', 'or_6', 'atrioventricular_7', 'modifying_8', 'drugs_9', '+_10', 'VVI_11', 'pacemaker_12', 'will_13', 'lead_14', 'to_15', 'a_16', 'significant_17', 'improvement_18', 'in_19', 'exercise_20', 'duration_21', 'and_22', 'quality_23', 'of_24', 'life_25', '._26', 'The_27', 'other_28', 'regions_29', 'include_30', 'potential_31', 'binding_32', 'sites_33', 'for_34', 'transcription_35', 'factors_36', 'ATF_37', ',_38', 'NF1_39', ',_40', 'and_41', 'a_42', 'CCAAT_43', '-_44', 'binding_45', 'protein_46', '._47', 'Both_48', 'purH_49', 'and_50', 'purD_51', 'genes_52', 'constitute_53', 'a_54', 'single_55', 'operon_56', 'and_57']\n",
            "BAD:  21129\n",
            "rpws:      index        token label\n",
            "0   21187          are     O\n",
            "1   21188  coregulated     O\n",
            "2   21189           in     O\n",
            "3   21190   expression     O\n",
            "4   21191           by     O\n",
            "..    ...          ...   ...\n",
            "68  21255      signals     O\n",
            "69  21256          for     O\n",
            "70  21257          Zep     B\n",
            "71  21258         mRNA     I\n",
            "72  21259           in     O\n",
            "\n",
            "[73 rows x 3 columns]\n",
            "Both purH and purD genes constitute a single operon and are coregulated in expression by purines as other purine genes are . The patient demographics ( means + / - standard deviations ) were as follows : age , 57 + / - 12 years ; sex , nine males and three females ; APACHE II score , 15 + / - 3 ; diagnosis , 9 of 12 patients with pneumonia .\n",
            "['Both', 'purH', 'and', 'purD', 'genes', 'constitute', 'a', 'single', 'operon', 'and', 'are', 'coregulated', 'in', 'expression', 'by', 'purines', 'as', 'other', 'purine', 'genes', 'are', '.', 'The', 'patient', 'demographics', '(', 'means', '+', '/', '-', 'standard', 'deviations', ')', 'were', 'as', 'follows', ':', 'age', ',', '57', '+', '/', '-', '12', 'years', ';', 'sex', ',', 'nine', 'males', 'and', 'three', 'females', ';', 'APACHE', 'II', 'score', ',', '15', '+', '/', '-', '3', ';', 'diagnosis', ',', '9', 'of', '12', 'patients', 'with', 'pneumonia', '.']\n",
            "73 73\n",
            "['are_0', 'coregulated_1', 'in_2', 'expression_3', 'by_4', 'purines_5', 'as_6', 'other_7', 'purine_8', 'genes_9', 'are_10', '._11', 'The_12', 'patient_13', 'demographics_14', '(_15', 'means_16', '+_17', '/_18', '-_19', 'standard_20', 'deviations_21', ')_22', 'were_23', 'as_24', 'follows_25', ':_26', 'age_27', ',_28', '57_29', '+_30', '/_31', '-_32', '12_33', 'years_34', ';_35', 'sex_36', ',_37', 'nine_38', 'males_39', 'and_40', 'three_41', 'females_42', ';_43', 'APACHE_44', 'II_45', 'score_46', ',_47', '15_48', '+_49', '/_50', '-_51', '3_52', ';_53', 'diagnosis_54', ',_55', '9_56', 'of_57', '12_58', 'patients_59', 'with_60', 'pneumonia_61', '._62', 'In_63', 'situ_64', 'hybridization_65', 'reveals_66', 'strong_67', 'signals_68', 'for_69', 'Zep_70', 'mRNA_71', 'in_72']\n",
            "BAD:  21187\n",
            "rpws:      index       token label\n",
            "0   21260         the     O\n",
            "1   21261  cerebellum     O\n",
            "2   21262         and     O\n",
            "3   21263   olfactory     O\n",
            "4   21264        bulb     O\n",
            "..    ...         ...   ...\n",
            "73  21333       other     O\n",
            "74  21334       known     O\n",
            "75  21335   mammalian     O\n",
            "76  21336     members     O\n",
            "77  21337          of     O\n",
            "\n",
            "[78 rows x 3 columns]\n",
            "In situ hybridization reveals strong signals for Zep mRNA in the cerebellum and olfactory bulb with moderate signals detected in the hippocampus and cortex . TNF - alpha effect was eliminated by a 2 - bp substitution mutation in the NF - kappa B1 binding half site of the NF - kappa B cis element . UV - cross - linking studies suggested that these two complexes represent one and two molecules of ADR1 bound to DNA .\n",
            "['In', 'situ', 'hybridization', 'reveals', 'strong', 'signals', 'for', 'Zep', 'mRNA', 'in', 'the', 'cerebellum', 'and', 'olfactory', 'bulb', 'with', 'moderate', 'signals', 'detected', 'in', 'the', 'hippocampus', 'and', 'cortex', '.', 'TNF', '-', 'alpha', 'effect', 'was', 'eliminated', 'by', 'a', '2', '-', 'bp', 'substitution', 'mutation', 'in', 'the', 'NF', '-', 'kappa', 'B1', 'binding', 'half', 'site', 'of', 'the', 'NF', '-', 'kappa', 'B', 'cis', 'element', '.', 'UV', '-', 'cross', '-', 'linking', 'studies', 'suggested', 'that', 'these', 'two', 'complexes', 'represent', 'one', 'and', 'two', 'molecules', 'of', 'ADR1', 'bound', 'to', 'DNA', '.']\n",
            "78 78\n",
            "['the_0', 'cerebellum_1', 'and_2', 'olfactory_3', 'bulb_4', 'with_5', 'moderate_6', 'signals_7', 'detected_8', 'in_9', 'the_10', 'hippocampus_11', 'and_12', 'cortex_13', '._14', 'TNF_15', '-_16', 'alpha_17', 'effect_18', 'was_19', 'eliminated_20', 'by_21', 'a_22', '2_23', '-_24', 'bp_25', 'substitution_26', 'mutation_27', 'in_28', 'the_29', 'NF_30', '-_31', 'kappa_32', 'B1_33', 'binding_34', 'half_35', 'site_36', 'of_37', 'the_38', 'NF_39', '-_40', 'kappa_41', 'B_42', 'cis_43', 'element_44', '._45', 'UV_46', '-_47', 'cross_48', '-_49', 'linking_50', 'studies_51', 'suggested_52', 'that_53', 'these_54', 'two_55', 'complexes_56', 'represent_57', 'one_58', 'and_59', 'two_60', 'molecules_61', 'of_62', 'ADR1_63', 'bound_64', 'to_65', 'DNA_66', '._67', 'As_68', 'already_69', 'available_70', 'for_71', 'the_72', 'other_73', 'known_74', 'mammalian_75', 'members_76', 'of_77']\n",
            "BAD:  21260\n",
            "rpws:      index           token label\n",
            "0   21338            this     O\n",
            "1   21339          enzyme     O\n",
            "2   21340          family     O\n",
            "3   21341               ,     O\n",
            "4   21342              we     O\n",
            "5   21343            here     O\n",
            "6   21344          define     O\n",
            "7   21345      structural     O\n",
            "8   21346             and     O\n",
            "9   21347      functional     O\n",
            "10  21348        features     O\n",
            "11  21349              of     O\n",
            "12  21350           human     B\n",
            "13  21351        lymphoma     I\n",
            "14  21352      proprotein     I\n",
            "15  21353      convertase     I\n",
            "16  21354               (     O\n",
            "17  21355             LPC     B\n",
            "18  21356               )     O\n",
            "19  21357               .     O\n",
            "20  21358             The     O\n",
            "21  21359               M     O\n",
            "22  21360               -     O\n",
            "23  21361           phase     O\n",
            "24  21362       induction     O\n",
            "25  21363        activity     O\n",
            "26  21364              of     O\n",
            "27  21365            Cdc2     B\n",
            "28  21366               -     O\n",
            "29  21367           Cdc13     B\n",
            "30  21368              is     O\n",
            "31  21369       inhibited     O\n",
            "32  21370              by     O\n",
            "33  21371            Wee1     B\n",
            "34  21372        tyrosine     I\n",
            "35  21373          kinase     I\n",
            "36  21374               ,     O\n",
            "37  21375           which     O\n",
            "38  21376  phosphorylates     O\n",
            "39  21377            Cdc2     B\n",
            "40  21378              on     O\n",
            "41  21379        tyrosine     O\n",
            "42  21380               -     O\n",
            "43  21381              15     O\n",
            "44  21382               .     O\n",
            "45  21383      Nucleotide     O\n",
            "46  21384      sequencing     O\n",
            "47  21385        revealed     O\n",
            "48  21386               a     O\n",
            "49  21387            2076     O\n",
            "50  21388               -     O\n",
            "51  21389            base     O\n",
            "52  21390            pair     O\n",
            "53  21391            open     O\n",
            "54  21392         reading     O\n",
            "As already available for the other known mammalian members of this enzyme family , we here define structural and functional features of human lymphoma proprotein convertase ( LPC ) . The M - phase induction activity of Cdc2 - Cdc13 is inhibited by Wee1 tyrosine kinase , which phosphorylates Cdc2 on tyrosine - 15 .\n",
            "['As', 'already', 'available', 'for', 'the', 'other', 'known', 'mammalian', 'members', 'of', 'this', 'enzyme', 'family', ',', 'we', 'here', 'define', 'structural', 'and', 'functional', 'features', 'of', 'human', 'lymphoma', 'proprotein', 'convertase', '(', 'LPC', ')', '.', 'The', 'M', '-', 'phase', 'induction', 'activity', 'of', 'Cdc2', '-', 'Cdc13', 'is', 'inhibited', 'by', 'Wee1', 'tyrosine', 'kinase', ',', 'which', 'phosphorylates', 'Cdc2', 'on', 'tyrosine', '-', '15', '.']\n",
            "55 55\n",
            "['this_0', 'enzyme_1', 'family_2', ',_3', 'we_4', 'here_5', 'define_6', 'structural_7', 'and_8', 'functional_9', 'features_10', 'of_11', 'human_12', 'lymphoma_13', 'proprotein_14', 'convertase_15', '(_16', 'LPC_17', ')_18', '._19', 'The_20', 'M_21', '-_22', 'phase_23', 'induction_24', 'activity_25', 'of_26', 'Cdc2_27', '-_28', 'Cdc13_29', 'is_30', 'inhibited_31', 'by_32', 'Wee1_33', 'tyrosine_34', 'kinase_35', ',_36', 'which_37', 'phosphorylates_38', 'Cdc2_39', 'on_40', 'tyrosine_41', '-_42', '15_43', '._44', 'Nucleotide_45', 'sequencing_46', 'revealed_47', 'a_48', '2076_49', '-_50', 'base_51', 'pair_52', 'open_53', 'reading_54']\n",
            "BAD:  21338\n",
            "rpws:      index       token label\n",
            "0   21393       frame     O\n",
            "1   21394    encoding     O\n",
            "2   21395           a     O\n",
            "3   21396         692     O\n",
            "4   21397           -     O\n",
            "..    ...         ...   ...\n",
            "64  21457          in     O\n",
            "65  21458    patients     O\n",
            "66  21459        with     O\n",
            "67  21460  metastatic     O\n",
            "68  21461      breast     O\n",
            "\n",
            "[69 rows x 3 columns]\n",
            "Nucleotide sequencing revealed a 2076 - base pair open reading frame encoding a 692 - amino acid protein . But it is possible that the two hemispheres differ in the manner in which they make use of semantic structures : the left hemisphere in a selective manner , appropriate for the solution of a specific task , and the right hemisphere in a more diffuse and global manner .\n",
            "['Nucleotide', 'sequencing', 'revealed', 'a', '2076', '-', 'base', 'pair', 'open', 'reading', 'frame', 'encoding', 'a', '692', '-', 'amino', 'acid', 'protein', '.', 'But', 'it', 'is', 'possible', 'that', 'the', 'two', 'hemispheres', 'differ', 'in', 'the', 'manner', 'in', 'which', 'they', 'make', 'use', 'of', 'semantic', 'structures', ':', 'the', 'left', 'hemisphere', 'in', 'a', 'selective', 'manner', ',', 'appropriate', 'for', 'the', 'solution', 'of', 'a', 'specific', 'task', ',', 'and', 'the', 'right', 'hemisphere', 'in', 'a', 'more', 'diffuse', 'and', 'global', 'manner', '.']\n",
            "69 69\n",
            "['frame_0', 'encoding_1', 'a_2', '692_3', '-_4', 'amino_5', 'acid_6', 'protein_7', '._8', 'But_9', 'it_10', 'is_11', 'possible_12', 'that_13', 'the_14', 'two_15', 'hemispheres_16', 'differ_17', 'in_18', 'the_19', 'manner_20', 'in_21', 'which_22', 'they_23', 'make_24', 'use_25', 'of_26', 'semantic_27', 'structures_28', ':_29', 'the_30', 'left_31', 'hemisphere_32', 'in_33', 'a_34', 'selective_35', 'manner_36', ',_37', 'appropriate_38', 'for_39', 'the_40', 'solution_41', 'of_42', 'a_43', 'specific_44', 'task_45', ',_46', 'and_47', 'the_48', 'right_49', 'hemisphere_50', 'in_51', 'a_52', 'more_53', 'diffuse_54', 'and_55', 'global_56', 'manner_57', '._58', 'Supportive_59', 'therapy_60', 'with_61', 'xenogenous_62', 'peptides_63', 'in_64', 'patients_65', 'with_66', 'metastatic_67', 'breast_68']\n",
            "BAD:  21393\n",
            "rpws:       index         token label\n",
            "0    21462        cancer     O\n",
            "1    21463    undergoing     O\n",
            "2    21464    aggressive     O\n",
            "3    21465  chemotherapy     O\n",
            "4    21466             (     O\n",
            "..     ...           ...   ...\n",
            "101  21563            of     O\n",
            "102  21564          left     O\n",
            "103  21565   ventricular     O\n",
            "104  21566    activation     O\n",
            "105  21567            in     O\n",
            "\n",
            "[106 rows x 3 columns]\n",
            "Supportive therapy with xenogenous peptides in patients with metastatic breast cancer undergoing aggressive chemotherapy ( modified AC - protocol ) : a prospective , randomized double - blind study The pregnancies were terminated at 19 and 12 weeks of gestation , respectively . Single substitutions of three highly conserved phenylalanine residues ( Phe - 15 , Phe - 17 , Phe - 27 ) by alanine and substitution of one histidine ( His - 29 ) by glutamine , all located within the putative RNA - binding sites RNP - 1 and RNP - 2 , abolished the nucleic acid - binding activity of CspB .\n",
            "['Supportive', 'therapy', 'with', 'xenogenous', 'peptides', 'in', 'patients', 'with', 'metastatic', 'breast', 'cancer', 'undergoing', 'aggressive', 'chemotherapy', '(', 'modified', 'AC', '-', 'protocol', ')', ':', 'a', 'prospective', ',', 'randomized', 'double', '-', 'blind', 'study', 'The', 'pregnancies', 'were', 'terminated', 'at', '19', 'and', '12', 'weeks', 'of', 'gestation', ',', 'respectively', '.', 'Single', 'substitutions', 'of', 'three', 'highly', 'conserved', 'phenylalanine', 'residues', '(', 'Phe', '-', '15', ',', 'Phe', '-', '17', ',', 'Phe', '-', '27', ')', 'by', 'alanine', 'and', 'substitution', 'of', 'one', 'histidine', '(', 'His', '-', '29', ')', 'by', 'glutamine', ',', 'all', 'located', 'within', 'the', 'putative', 'RNA', '-', 'binding', 'sites', 'RNP', '-', '1', 'and', 'RNP', '-', '2', ',', 'abolished', 'the', 'nucleic', 'acid', '-', 'binding', 'activity', 'of', 'CspB', '.']\n",
            "106 106\n",
            "['cancer_0', 'undergoing_1', 'aggressive_2', 'chemotherapy_3', '(_4', 'modified_5', 'AC_6', '-_7', 'protocol_8', ')_9', ':_10', 'a_11', 'prospective_12', ',_13', 'randomized_14', 'double_15', '-_16', 'blind_17', 'study_18', 'The_19', 'pregnancies_20', 'were_21', 'terminated_22', 'at_23', '19_24', 'and_25', '12_26', 'weeks_27', 'of_28', 'gestation_29', ',_30', 'respectively_31', '._32', 'Single_33', 'substitutions_34', 'of_35', 'three_36', 'highly_37', 'conserved_38', 'phenylalanine_39', 'residues_40', '(_41', 'Phe_42', '-_43', '15_44', ',_45', 'Phe_46', '-_47', '17_48', ',_49', 'Phe_50', '-_51', '27_52', ')_53', 'by_54', 'alanine_55', 'and_56', 'substitution_57', 'of_58', 'one_59', 'histidine_60', '(_61', 'His_62', '-_63', '29_64', ')_65', 'by_66', 'glutamine_67', ',_68', 'all_69', 'located_70', 'within_71', 'the_72', 'putative_73', 'RNA_74', '-_75', 'binding_76', 'sites_77', 'RNP_78', '-_79', '1_80', 'and_81', 'RNP_82', '-_83', '2_84', ',_85', 'abolished_86', 'the_87', 'nucleic_88', 'acid_89', '-_90', 'binding_91', 'activity_92', 'of_93', 'CspB_94', '._95', 'The_96', 'cases_97', 'with_98', 'terminal_99', 'forces_100', 'of_101', 'left_102', 'ventricular_103', 'activation_104', 'in_105']\n",
            "BAD:  21462\n",
            "rpws:      index      token label\n",
            "0   21568        the     O\n",
            "1   21569       same     O\n",
            "2   21570  direction     O\n",
            "3   21571         as     O\n",
            "4   21572        the     O\n",
            "..    ...        ...   ...\n",
            "67  21635         in     O\n",
            "68  21636          E     O\n",
            "69  21637          .     O\n",
            "70  21638  herbicola     O\n",
            "71  21639      Eho13     O\n",
            "\n",
            "[72 rows x 3 columns]\n",
            "The cases with terminal forces of left ventricular activation in the same direction as the delta wave , superiorly and to the left at - 60 degrees or inferiorly and to the right at + 120 degrees , forming a single deflection of over 0 , 12 seconds ' duration , are the result of delayed activation of the anterior or posterior fascicle of the left bundle after a long delay .\n",
            "['The', 'cases', 'with', 'terminal', 'forces', 'of', 'left', 'ventricular', 'activation', 'in', 'the', 'same', 'direction', 'as', 'the', 'delta', 'wave', ',', 'superiorly', 'and', 'to', 'the', 'left', 'at', '-', '60', 'degrees', 'or', 'inferiorly', 'and', 'to', 'the', 'right', 'at', '+', '120', 'degrees', ',', 'forming', 'a', 'single', 'deflection', 'of', 'over', '0', ',', '12', 'seconds', \"'\", 'duration', ',', 'are', 'the', 'result', 'of', 'delayed', 'activation', 'of', 'the', 'anterior', 'or', 'posterior', 'fascicle', 'of', 'the', 'left', 'bundle', 'after', 'a', 'long', 'delay', '.']\n",
            "72 72\n",
            "['the_0', 'same_1', 'direction_2', 'as_3', 'the_4', 'delta_5', 'wave_6', ',_7', 'superiorly_8', 'and_9', 'to_10', 'the_11', 'left_12', 'at_13', '-_14', '60_15', 'degrees_16', 'or_17', 'inferiorly_18', 'and_19', 'to_20', 'the_21', 'right_22', 'at_23', '+_24', '120_25', 'degrees_26', ',_27', 'forming_28', 'a_29', 'single_30', 'deflection_31', 'of_32', 'over_33', '0_34', ',_35', '12_36', 'seconds_37', \"'_38\", 'duration_39', ',_40', 'are_41', 'the_42', 'result_43', 'of_44', 'delayed_45', 'activation_46', 'of_47', 'the_48', 'anterior_49', 'or_50', 'posterior_51', 'fascicle_52', 'of_53', 'the_54', 'left_55', 'bundle_56', 'after_57', 'a_58', 'long_59', 'delay_60', '._61', 'The_62', 'genes_63', 'encoding_64', 'these_65', 'carotenoids_66', 'in_67', 'E_68', '._69', 'herbicola_70', 'Eho13_71']\n",
            "BAD:  21568\n",
            "rpws:      index      token label\n",
            "0   21640        are     O\n",
            "1   21641  clustered     O\n",
            "2   21642         in     O\n",
            "3   21643          a     O\n",
            "4   21644          7     O\n",
            "..    ...        ...   ...\n",
            "60  21700   analysis     O\n",
            "61  21701         of     O\n",
            "62  21702        the     O\n",
            "63  21703       Hoxa     B\n",
            "64  21704          -     I\n",
            "\n",
            "[65 rows x 3 columns]\n",
            "The genes encoding these carotenoids in E . herbicola Eho13 are clustered in a 7 kb DNA fragment . The results show that while a larger polypeptide substrate carrying the HD1 / 3C site was processed more efficiently than a polypeptide substrate carrying the POL / Zn site , cleavage of the synthetic peptide substrates containing these two cleavage sites occurred at similar efficiencies .\n",
            "['The', 'genes', 'encoding', 'these', 'carotenoids', 'in', 'E', '.', 'herbicola', 'Eho13', 'are', 'clustered', 'in', 'a', '7', 'kb', 'DNA', 'fragment', '.', 'The', 'results', 'show', 'that', 'while', 'a', 'larger', 'polypeptide', 'substrate', 'carrying', 'the', 'HD1', '/', '3C', 'site', 'was', 'processed', 'more', 'efficiently', 'than', 'a', 'polypeptide', 'substrate', 'carrying', 'the', 'POL', '/', 'Zn', 'site', ',', 'cleavage', 'of', 'the', 'synthetic', 'peptide', 'substrates', 'containing', 'these', 'two', 'cleavage', 'sites', 'occurred', 'at', 'similar', 'efficiencies', '.']\n",
            "65 65\n",
            "['are_0', 'clustered_1', 'in_2', 'a_3', '7_4', 'kb_5', 'DNA_6', 'fragment_7', '._8', 'The_9', 'results_10', 'show_11', 'that_12', 'while_13', 'a_14', 'larger_15', 'polypeptide_16', 'substrate_17', 'carrying_18', 'the_19', 'HD1_20', '/_21', '3C_22', 'site_23', 'was_24', 'processed_25', 'more_26', 'efficiently_27', 'than_28', 'a_29', 'polypeptide_30', 'substrate_31', 'carrying_32', 'the_33', 'POL_34', '/_35', 'Zn_36', 'site_37', ',_38', 'cleavage_39', 'of_40', 'the_41', 'synthetic_42', 'peptide_43', 'substrates_44', 'containing_45', 'these_46', 'two_47', 'cleavage_48', 'sites_49', 'occurred_50', 'at_51', 'similar_52', 'efficiencies_53', '._54', 'Using_55', 'transgenic_56', 'lines_57', 'a_58', 'detailed_59', 'analysis_60', 'of_61', 'the_62', 'Hoxa_63', '-_64']\n",
            "BAD:  21640\n",
            "rpws:      index       token label\n",
            "0   21705           7     I\n",
            "1   21706    enhancer     O\n",
            "2   21707           -     O\n",
            "3   21708    directed     O\n",
            "4   21709  expression     O\n",
            "..    ...         ...   ...\n",
            "65  21770           +     O\n",
            "66  21771           1     O\n",
            "67  21772         GGG     O\n",
            "68  21773           ,     O\n",
            "69  21774          T7     B\n",
            "\n",
            "[70 rows x 3 columns]\n",
            "Using transgenic lines a detailed analysis of the Hoxa - 7 enhancer - directed expression during embryogenesis was performed . lacZ expression was first detected in the allantois at day 7 . 5 p . c . and in mesoderm and ectoderm at day 8 . 5 of gestation . An ERp60 - like protein from the filarial parasite Dirofilaria immitis has both transglutaminase and protein disulfide isomerase activity .\n",
            "['Using', 'transgenic', 'lines', 'a', 'detailed', 'analysis', 'of', 'the', 'Hoxa', '-', '7', 'enhancer', '-', 'directed', 'expression', 'during', 'embryogenesis', 'was', 'performed', '.', 'lacZ', 'expression', 'was', 'first', 'detected', 'in', 'the', 'allantois', 'at', 'day', '7', '.', '5', 'p', '.', 'c', '.', 'and', 'in', 'mesoderm', 'and', 'ectoderm', 'at', 'day', '8', '.', '5', 'of', 'gestation', '.', 'An', 'ERp60', '-', 'like', 'protein', 'from', 'the', 'filarial', 'parasite', 'Dirofilaria', 'immitis', 'has', 'both', 'transglutaminase', 'and', 'protein', 'disulfide', 'isomerase', 'activity', '.']\n",
            "70 70\n",
            "['7_0', 'enhancer_1', '-_2', 'directed_3', 'expression_4', 'during_5', 'embryogenesis_6', 'was_7', 'performed_8', '._9', 'lacZ_10', 'expression_11', 'was_12', 'first_13', 'detected_14', 'in_15', 'the_16', 'allantois_17', 'at_18', 'day_19', '7_20', '._21', '5_22', 'p_23', '._24', 'c_25', '._26', 'and_27', 'in_28', 'mesoderm_29', 'and_30', 'ectoderm_31', 'at_32', 'day_33', '8_34', '._35', '5_36', 'of_37', 'gestation_38', '._39', 'An_40', 'ERp60_41', '-_42', 'like_43', 'protein_44', 'from_45', 'the_46', 'filarial_47', 'parasite_48', 'Dirofilaria_49', 'immitis_50', 'has_51', 'both_52', 'transglutaminase_53', 'and_54', 'protein_55', 'disulfide_56', 'isomerase_57', 'activity_58', '._59', 'At_60', 'promoters_61', 'that_62', 'initiate_63', 'with_64', '+_65', '1_66', 'GGG_67', ',_68', 'T7_69']\n",
            "BAD:  21705\n",
            "rpws:      index        token label\n",
            "0   21775         RNAP     I\n",
            "1   21776  synthesizes     O\n",
            "2   21777            a     O\n",
            "3   21778       ladder     O\n",
            "4   21779           of     O\n",
            "..    ...          ...   ...\n",
            "80  21855         same     O\n",
            "81  21856   antibodies     O\n",
            "82  21857   recognized     O\n",
            "83  21858          the     O\n",
            "84  21859  recombinant     O\n",
            "\n",
            "[85 rows x 3 columns]\n",
            "At promoters that initiate with + 1 GGG , T7 RNAP synthesizes a ladder of poly ( G ) products as a result of slippage of the transcript on the three C residues in the template strand from + 1 to + 3 . Serum levels of angiotensin converting enzyme were well maintained . The UL3 ORF of 204 amino acids shows significant homology to UL3 ( nuclear phosphoprotein ) of HSV - 1 ( 62 % ) and PRV ( 53 % ) .\n",
            "['At', 'promoters', 'that', 'initiate', 'with', '+', '1', 'GGG', ',', 'T7', 'RNAP', 'synthesizes', 'a', 'ladder', 'of', 'poly', '(', 'G', ')', 'products', 'as', 'a', 'result', 'of', 'slippage', 'of', 'the', 'transcript', 'on', 'the', 'three', 'C', 'residues', 'in', 'the', 'template', 'strand', 'from', '+', '1', 'to', '+', '3', '.', 'Serum', 'levels', 'of', 'angiotensin', 'converting', 'enzyme', 'were', 'well', 'maintained', '.', 'The', 'UL3', 'ORF', 'of', '204', 'amino', 'acids', 'shows', 'significant', 'homology', 'to', 'UL3', '(', 'nuclear', 'phosphoprotein', ')', 'of', 'HSV', '-', '1', '(', '62', '%', ')', 'and', 'PRV', '(', '53', '%', ')', '.']\n",
            "85 85\n",
            "['RNAP_0', 'synthesizes_1', 'a_2', 'ladder_3', 'of_4', 'poly_5', '(_6', 'G_7', ')_8', 'products_9', 'as_10', 'a_11', 'result_12', 'of_13', 'slippage_14', 'of_15', 'the_16', 'transcript_17', 'on_18', 'the_19', 'three_20', 'C_21', 'residues_22', 'in_23', 'the_24', 'template_25', 'strand_26', 'from_27', '+_28', '1_29', 'to_30', '+_31', '3_32', '._33', 'Serum_34', 'levels_35', 'of_36', 'angiotensin_37', 'converting_38', 'enzyme_39', 'were_40', 'well_41', 'maintained_42', '._43', 'The_44', 'UL3_45', 'ORF_46', 'of_47', '204_48', 'amino_49', 'acids_50', 'shows_51', 'significant_52', 'homology_53', 'to_54', 'UL3_55', '(_56', 'nuclear_57', 'phosphoprotein_58', ')_59', 'of_60', 'HSV_61', '-_62', '1_63', '(_64', '62_65', '%_66', ')_67', 'and_68', 'PRV_69', '(_70', '53_71', '%_72', ')_73', '._74', 'On_75', 'Western_76', 'blots_77', ',_78', 'the_79', 'same_80', 'antibodies_81', 'recognized_82', 'the_83', 'recombinant_84']\n",
            "BAD:  21775\n",
            "rpws:      index       token label\n",
            "0   21860     protein     O\n",
            "1   21861   migrating     O\n",
            "2   21862    slightly     O\n",
            "3   21863      slower     O\n",
            "4   21864          on     O\n",
            "..    ...         ...   ...\n",
            "57  21917      effect     O\n",
            "58  21918          of     O\n",
            "59  21919  homozygous     O\n",
            "60  21920      sickle     O\n",
            "61  21921        cell     O\n",
            "\n",
            "[62 rows x 3 columns]\n",
            "On Western blots , the same antibodies recognized the recombinant protein migrating slightly slower on SDS / PAGE than chicken axonin - 1 . For routine medical applications , no sophisticated adjustment of the CHESS pulse is needed , as reported in previous methods . The 10 eyes injected with human RPE cells showed STAGE 2 or less in 4 weeks .\n",
            "['On', 'Western', 'blots', ',', 'the', 'same', 'antibodies', 'recognized', 'the', 'recombinant', 'protein', 'migrating', 'slightly', 'slower', 'on', 'SDS', '/', 'PAGE', 'than', 'chicken', 'axonin', '-', '1', '.', 'For', 'routine', 'medical', 'applications', ',', 'no', 'sophisticated', 'adjustment', 'of', 'the', 'CHESS', 'pulse', 'is', 'needed', ',', 'as', 'reported', 'in', 'previous', 'methods', '.', 'The', '10', 'eyes', 'injected', 'with', 'human', 'RPE', 'cells', 'showed', 'STAGE', '2', 'or', 'less', 'in', '4', 'weeks', '.']\n",
            "62 62\n",
            "['protein_0', 'migrating_1', 'slightly_2', 'slower_3', 'on_4', 'SDS_5', '/_6', 'PAGE_7', 'than_8', 'chicken_9', 'axonin_10', '-_11', '1_12', '._13', 'For_14', 'routine_15', 'medical_16', 'applications_17', ',_18', 'no_19', 'sophisticated_20', 'adjustment_21', 'of_22', 'the_23', 'CHESS_24', 'pulse_25', 'is_26', 'needed_27', ',_28', 'as_29', 'reported_30', 'in_31', 'previous_32', 'methods_33', '._34', 'The_35', '10_36', 'eyes_37', 'injected_38', 'with_39', 'human_40', 'RPE_41', 'cells_42', 'showed_43', 'STAGE_44', '2_45', 'or_46', 'less_47', 'in_48', '4_49', 'weeks_50', '._51', 'This_52', 'report_53', 'establishes_54', 'that_55', 'the_56', 'effect_57', 'of_58', 'homozygous_59', 'sickle_60', 'cell_61']\n",
            "BAD:  21860\n",
            "rpws:      index        token label\n",
            "0   21922      disease     O\n",
            "1   21923           on     O\n",
            "2   21924       growth     O\n",
            "3   21925     patterns     O\n",
            "4   21926           in     O\n",
            "5   21927    childhood     O\n",
            "6   21928           is     O\n",
            "7   21929     apparent     O\n",
            "8   21930       before     O\n",
            "9   21931          the     O\n",
            "10  21932          age     O\n",
            "11  21933           of     O\n",
            "12  21934            6     O\n",
            "13  21935        years     O\n",
            "14  21936            .     O\n",
            "15  21937          The     O\n",
            "16  21938    remaining     O\n",
            "17  21939           77     O\n",
            "18  21940  nucleotides     O\n",
            "19  21941           at     O\n",
            "20  21942          the     O\n",
            "21  21943            3     O\n",
            "22  21944            '     O\n",
            "23  21945          end     O\n",
            "24  21946           of     O\n",
            "25  21947       domain     O\n",
            "26  21948           II     O\n",
            "27  21949          and     O\n",
            "28  21950          all     O\n",
            "29  21951           of     O\n",
            "30  21952      domains     O\n",
            "31  21953          III     O\n",
            "32  21954            (     O\n",
            "33  21955          655     O\n",
            "34  21956  nucleotides     O\n",
            "35  21957            )     O\n",
            "36  21958          and     O\n",
            "37  21959           IV     O\n",
            "38  21960            (     O\n",
            "39  21961          770     O\n",
            "40  21962  nucleotides     O\n",
            "41  21963            )     O\n",
            "42  21964          are     O\n",
            "43  21965          not     O\n",
            "44  21966      present     O\n",
            "45  21967           in     O\n",
            "46  21968        DIssE     B\n",
            "47  21969          RNA     I\n",
            "48  21970            .     O\n",
            "49  21971          The     O\n",
            "50  21972   biological     O\n",
            "51  21973      actions     O\n",
            "52  21974           of     O\n",
            "53  21975          VHR     B\n",
            "54  21976           in     O\n",
            "55  21977      oocytes     O\n",
            "56  21978      clearly     O\n",
            "57  21979  distinguish     O\n",
            "58  21980           it     O\n",
            "This report establishes that the effect of homozygous sickle cell disease on growth patterns in childhood is apparent before the age of 6 years . The remaining 77 nucleotides at the 3 ' end of domain II and all of domains III ( 655 nucleotides ) and IV ( 770 nucleotides ) are not present in DIssE RNA .\n",
            "['This', 'report', 'establishes', 'that', 'the', 'effect', 'of', 'homozygous', 'sickle', 'cell', 'disease', 'on', 'growth', 'patterns', 'in', 'childhood', 'is', 'apparent', 'before', 'the', 'age', 'of', '6', 'years', '.', 'The', 'remaining', '77', 'nucleotides', 'at', 'the', '3', \"'\", 'end', 'of', 'domain', 'II', 'and', 'all', 'of', 'domains', 'III', '(', '655', 'nucleotides', ')', 'and', 'IV', '(', '770', 'nucleotides', ')', 'are', 'not', 'present', 'in', 'DIssE', 'RNA', '.']\n",
            "59 59\n",
            "['disease_0', 'on_1', 'growth_2', 'patterns_3', 'in_4', 'childhood_5', 'is_6', 'apparent_7', 'before_8', 'the_9', 'age_10', 'of_11', '6_12', 'years_13', '._14', 'The_15', 'remaining_16', '77_17', 'nucleotides_18', 'at_19', 'the_20', '3_21', \"'_22\", 'end_23', 'of_24', 'domain_25', 'II_26', 'and_27', 'all_28', 'of_29', 'domains_30', 'III_31', '(_32', '655_33', 'nucleotides_34', ')_35', 'and_36', 'IV_37', '(_38', '770_39', 'nucleotides_40', ')_41', 'are_42', 'not_43', 'present_44', 'in_45', 'DIssE_46', 'RNA_47', '._48', 'The_49', 'biological_50', 'actions_51', 'of_52', 'VHR_53', 'in_54', 'oocytes_55', 'clearly_56', 'distinguish_57', 'it_58']\n",
            "BAD:  21922\n",
            "rpws:      index            token label\n",
            "0   21981             from     O\n",
            "1   21982            other     O\n",
            "2   21983             dual     O\n",
            "3   21984      specificity     O\n",
            "4   21985     phosphatases     O\n",
            "5   21986                ,     O\n",
            "6   21987            which     O\n",
            "7   21988             have     O\n",
            "8   21989            shown     O\n",
            "9   21990       inhibitory     O\n",
            "10  21991          effects     O\n",
            "11  21992             when     O\n",
            "12  21993           tested     O\n",
            "13  21994               in     O\n",
            "14  21995          oocytes     O\n",
            "15  21996                .     O\n",
            "16  21997              The     O\n",
            "17  21998         presence     O\n",
            "18  21999               of     O\n",
            "19  22000      chloroquine     O\n",
            "20  22001               in     O\n",
            "21  22002           saliva     O\n",
            "22  22003             from     O\n",
            "23  22004            seven     O\n",
            "24  22005          healthy     O\n",
            "25  22006       volunteers     O\n",
            "26  22007              for     O\n",
            "27  22008               21     O\n",
            "28  22009             days     O\n",
            "29  22010            after     O\n",
            "30  22011                a     O\n",
            "31  22012           single     O\n",
            "32  22013              600     O\n",
            "33  22014               mg     O\n",
            "34  22015             oral     O\n",
            "35  22016             dose     O\n",
            "36  22017               of     O\n",
            "37  22018              the     O\n",
            "38  22019             drug     O\n",
            "39  22020              was     O\n",
            "40  22021      established     O\n",
            "41  22022               by     O\n",
            "42  22023  chromatographic     O\n",
            "43  22024              and     O\n",
            "44  22025    spectroscopic     O\n",
            "45  22026          methods     O\n",
            "46  22027                .     O\n",
            "47  22028              The     O\n",
            "48  22029             gp41     B\n",
            "49  22030          peptide     I\n",
            "50  22031                (     I\n",
            "51  22032              Glu     I\n",
            "52  22033                -     I\n",
            "53  22034              Leu     I\n",
            "54  22035                -     I\n",
            "55  22036              Asp     I\n",
            "56  22037                -     I\n",
            "The biological actions of VHR in oocytes clearly distinguish it from other dual specificity phosphatases , which have shown inhibitory effects when tested in oocytes . The presence of chloroquine in saliva from seven healthy volunteers for 21 days after a single 600 mg oral dose of the drug was established by chromatographic and spectroscopic methods .\n",
            "['The', 'biological', 'actions', 'of', 'VHR', 'in', 'oocytes', 'clearly', 'distinguish', 'it', 'from', 'other', 'dual', 'specificity', 'phosphatases', ',', 'which', 'have', 'shown', 'inhibitory', 'effects', 'when', 'tested', 'in', 'oocytes', '.', 'The', 'presence', 'of', 'chloroquine', 'in', 'saliva', 'from', 'seven', 'healthy', 'volunteers', 'for', '21', 'days', 'after', 'a', 'single', '600', 'mg', 'oral', 'dose', 'of', 'the', 'drug', 'was', 'established', 'by', 'chromatographic', 'and', 'spectroscopic', 'methods', '.']\n",
            "57 57\n",
            "['from_0', 'other_1', 'dual_2', 'specificity_3', 'phosphatases_4', ',_5', 'which_6', 'have_7', 'shown_8', 'inhibitory_9', 'effects_10', 'when_11', 'tested_12', 'in_13', 'oocytes_14', '._15', 'The_16', 'presence_17', 'of_18', 'chloroquine_19', 'in_20', 'saliva_21', 'from_22', 'seven_23', 'healthy_24', 'volunteers_25', 'for_26', '21_27', 'days_28', 'after_29', 'a_30', 'single_31', '600_32', 'mg_33', 'oral_34', 'dose_35', 'of_36', 'the_37', 'drug_38', 'was_39', 'established_40', 'by_41', 'chromatographic_42', 'and_43', 'spectroscopic_44', 'methods_45', '._46', 'The_47', 'gp41_48', 'peptide_49', '(_50', 'Glu_51', '-_52', 'Leu_53', '-_54', 'Asp_55', '-_56']\n",
            "BAD:  21981\n",
            "rpws:      index      token label\n",
            "0   22038        Lys     I\n",
            "1   22039          -     I\n",
            "2   22040        Trp     I\n",
            "3   22041          -     I\n",
            "4   22042        Ala     I\n",
            "..    ...        ...   ...\n",
            "66  22104      genes     O\n",
            "67  22105       that     O\n",
            "68  22106        are     O\n",
            "69  22107  repressed     O\n",
            "70  22108         in     O\n",
            "\n",
            "[71 rows x 3 columns]\n",
            "The gp41 peptide ( Glu - Leu - Asp - Lys - Trp - Ala ) fused to the C - terminus of Sj GST forms a loop stabilized by symmetry - related GSTs . Comparison of bioreactive and immunoreactive gastrin . Also , PTx had no effect on shear - dependent activation of JNK . A score ( APACHE II ) was calculated to assess the severity of disease .\n",
            "['The', 'gp41', 'peptide', '(', 'Glu', '-', 'Leu', '-', 'Asp', '-', 'Lys', '-', 'Trp', '-', 'Ala', ')', 'fused', 'to', 'the', 'C', '-', 'terminus', 'of', 'Sj', 'GST', 'forms', 'a', 'loop', 'stabilized', 'by', 'symmetry', '-', 'related', 'GSTs', '.', 'Comparison', 'of', 'bioreactive', 'and', 'immunoreactive', 'gastrin', '.', 'Also', ',', 'PTx', 'had', 'no', 'effect', 'on', 'shear', '-', 'dependent', 'activation', 'of', 'JNK', '.', 'A', 'score', '(', 'APACHE', 'II', ')', 'was', 'calculated', 'to', 'assess', 'the', 'severity', 'of', 'disease', '.']\n",
            "71 71\n",
            "['Lys_0', '-_1', 'Trp_2', '-_3', 'Ala_4', ')_5', 'fused_6', 'to_7', 'the_8', 'C_9', '-_10', 'terminus_11', 'of_12', 'Sj_13', 'GST_14', 'forms_15', 'a_16', 'loop_17', 'stabilized_18', 'by_19', 'symmetry_20', '-_21', 'related_22', 'GSTs_23', '._24', 'Comparison_25', 'of_26', 'bioreactive_27', 'and_28', 'immunoreactive_29', 'gastrin_30', '._31', 'Also_32', ',_33', 'PTx_34', 'had_35', 'no_36', 'effect_37', 'on_38', 'shear_39', '-_40', 'dependent_41', 'activation_42', 'of_43', 'JNK_44', '._45', 'A_46', 'score_47', '(_48', 'APACHE_49', 'II_50', ')_51', 'was_52', 'calculated_53', 'to_54', 'assess_55', 'the_56', 'severity_57', 'of_58', 'disease_59', '._60', 'We_61', 'tested_62', 'several_63', 'growth_64', 'regulatory_65', 'genes_66', 'that_67', 'are_68', 'repressed_69', 'in_70']\n",
            "BAD:  22038\n",
            "rpws:      index         token label\n",
            "0   22109     senescent     O\n",
            "1   22110         cells     O\n",
            "2   22111           for     O\n",
            "3   22112       ability     O\n",
            "4   22113            to     O\n",
            "..    ...           ...   ...\n",
            "60  22169  conservation     O\n",
            "61  22170      together     O\n",
            "62  22171          with     O\n",
            "63  22172           the     O\n",
            "64  22173       ability     O\n",
            "\n",
            "[65 rows x 3 columns]\n",
            "We tested several growth regulatory genes that are repressed in senescent cells for ability to restore activity to T [ K1 ] . We analyzed the contribution of the three different types of UV - inducible mitogen - activated protein ( MAP ) kinases ( ERK , JNK / SAPK , and p38 ) to the activation of the murine uPA promoter by UV .\n",
            "['We', 'tested', 'several', 'growth', 'regulatory', 'genes', 'that', 'are', 'repressed', 'in', 'senescent', 'cells', 'for', 'ability', 'to', 'restore', 'activity', 'to', 'T', '[', 'K1', ']', '.', 'We', 'analyzed', 'the', 'contribution', 'of', 'the', 'three', 'different', 'types', 'of', 'UV', '-', 'inducible', 'mitogen', '-', 'activated', 'protein', '(', 'MAP', ')', 'kinases', '(', 'ERK', ',', 'JNK', '/', 'SAPK', ',', 'and', 'p38', ')', 'to', 'the', 'activation', 'of', 'the', 'murine', 'uPA', 'promoter', 'by', 'UV', '.']\n",
            "65 65\n",
            "['senescent_0', 'cells_1', 'for_2', 'ability_3', 'to_4', 'restore_5', 'activity_6', 'to_7', 'T_8', '[_9', 'K1_10', ']_11', '._12', 'We_13', 'analyzed_14', 'the_15', 'contribution_16', 'of_17', 'the_18', 'three_19', 'different_20', 'types_21', 'of_22', 'UV_23', '-_24', 'inducible_25', 'mitogen_26', '-_27', 'activated_28', 'protein_29', '(_30', 'MAP_31', ')_32', 'kinases_33', '(_34', 'ERK_35', ',_36', 'JNK_37', '/_38', 'SAPK_39', ',_40', 'and_41', 'p38_42', ')_43', 'to_44', 'the_45', 'activation_46', 'of_47', 'the_48', 'murine_49', 'uPA_50', 'promoter_51', 'by_52', 'UV_53', '._54', 'The_55', 'high_56', 'degree_57', 'of_58', 'sequence_59', 'conservation_60', 'together_61', 'with_62', 'the_63', 'ability_64']\n",
            "BAD:  22109\n",
            "rpws:      index             token label\n",
            "0   22174                to     O\n",
            "1   22175            direct     O\n",
            "2   22176         nucleolar     O\n",
            "3   22177           protein     O\n",
            "4   22178         transport     O\n",
            "5   22179          supports     O\n",
            "6   22180               the     O\n",
            "7   22181        hypothesis     O\n",
            "8   22182              that     O\n",
            "9   22183             MAK16     B\n",
            "10  22184          proteins     I\n",
            "11  22185              play     O\n",
            "12  22186                 a     O\n",
            "13  22187               key     O\n",
            "14  22188              role     O\n",
            "15  22189                in     O\n",
            "16  22190               the     O\n",
            "17  22191        biogenesis     O\n",
            "18  22192                of     O\n",
            "19  22193               60S     B\n",
            "20  22194          subunits     I\n",
            "21  22195                 .     O\n",
            "22  22196  Characterization     O\n",
            "23  22197                of     O\n",
            "24  22198                 a     O\n",
            "25  22199           nuclear     B\n",
            "26  22200          deformed     I\n",
            "27  22201         epidermal     I\n",
            "28  22202    autoregulatory     I\n",
            "29  22203            factor     I\n",
            "30  22204                 -     I\n",
            "31  22205                 1     I\n",
            "32  22206                 (     O\n",
            "33  22207              DEAF     B\n",
            "34  22208                 -     I\n",
            "35  22209                 1     I\n",
            "36  22210                 )     O\n",
            "37  22211                 -     O\n",
            "38  22212           related     O\n",
            "39  22213                 (     O\n",
            "40  22214              NUDR     B\n",
            "41  22215                 )     O\n",
            "42  22216   transcriptional     O\n",
            "43  22217         regulator     O\n",
            "44  22218           protein     O\n",
            "45  22219                 .     O\n",
            "46  22220              Here     O\n",
            "47  22221                we     O\n",
            "48  22222              show     O\n",
            "49  22223              that     O\n",
            "50  22224               the     O\n",
            "51  22225              sck1     B\n",
            "52  22226              gene     I\n",
            "53  22227                 ,     O\n",
            "54  22228            cloned     O\n",
            "55  22229                as     O\n",
            "The high degree of sequence conservation together with the ability to direct nucleolar protein transport supports the hypothesis that MAK16 proteins play a key role in the biogenesis of 60S subunits . Characterization of a nuclear deformed epidermal autoregulatory factor - 1 ( DEAF - 1 ) - related ( NUDR ) transcriptional regulator protein .\n",
            "['The', 'high', 'degree', 'of', 'sequence', 'conservation', 'together', 'with', 'the', 'ability', 'to', 'direct', 'nucleolar', 'protein', 'transport', 'supports', 'the', 'hypothesis', 'that', 'MAK16', 'proteins', 'play', 'a', 'key', 'role', 'in', 'the', 'biogenesis', 'of', '60S', 'subunits', '.', 'Characterization', 'of', 'a', 'nuclear', 'deformed', 'epidermal', 'autoregulatory', 'factor', '-', '1', '(', 'DEAF', '-', '1', ')', '-', 'related', '(', 'NUDR', ')', 'transcriptional', 'regulator', 'protein', '.']\n",
            "56 56\n",
            "['to_0', 'direct_1', 'nucleolar_2', 'protein_3', 'transport_4', 'supports_5', 'the_6', 'hypothesis_7', 'that_8', 'MAK16_9', 'proteins_10', 'play_11', 'a_12', 'key_13', 'role_14', 'in_15', 'the_16', 'biogenesis_17', 'of_18', '60S_19', 'subunits_20', '._21', 'Characterization_22', 'of_23', 'a_24', 'nuclear_25', 'deformed_26', 'epidermal_27', 'autoregulatory_28', 'factor_29', '-_30', '1_31', '(_32', 'DEAF_33', '-_34', '1_35', ')_36', '-_37', 'related_38', '(_39', 'NUDR_40', ')_41', 'transcriptional_42', 'regulator_43', 'protein_44', '._45', 'Here_46', 'we_47', 'show_48', 'that_49', 'the_50', 'sck1_51', 'gene_52', ',_53', 'cloned_54', 'as_55']\n",
            "BAD:  22174\n",
            "rpws:       index        token label\n",
            "0    22230            a     O\n",
            "1    22231         high     O\n",
            "2    22232         copy     O\n",
            "3    22233       number     O\n",
            "4    22234   suppressor     O\n",
            "..     ...          ...   ...\n",
            "114  22344         acid     O\n",
            "115  22345  hydrophobic     O\n",
            "116  22346       signal     O\n",
            "117  22347      peptide     O\n",
            "118  22348            .     O\n",
            "\n",
            "[119 rows x 3 columns]\n",
            "Here we show that the sck1 gene , cloned as a high copy number suppressor of a mutation in git3 , is able to suppress the defects conferred by a mutation in any of these git genes . Hamdija Karamehmedovic ; Ibn al - Nefis , \tO\n",
            "Mudzez\tO\n",
            "al\tO\n",
            "-\tO\n",
            "Kanum\tO\n",
            " , Republicki zavod za zdravstvenu zastitu Sarajevo , 1961 , 1 - 219 ; Mr ph Samuel Elazar , Ajnija Omanic : \tO\n",
            "Bibliografija\tO\n",
            "medicinskih\tO\n",
            "djela\tO\n",
            "u\tO\n",
            "SR\tO\n",
            "BiH\tO\n",
            "do\tO\n",
            "1895\tO\n",
            ".\tO\n",
            " Medicinska knjiga Beograd - Zagreb 1984 ; Besides , the great contribution in bibliography was given by dr Lujo Taler , prof .\n",
            "['Here', 'we', 'show', 'that', 'the', 'sck1', 'gene', ',', 'cloned', 'as', 'a', 'high', 'copy', 'number', 'suppressor', 'of', 'a', 'mutation', 'in', 'git3', ',', 'is', 'able', 'to', 'suppress', 'the', 'defects', 'conferred', 'by', 'a', 'mutation', 'in', 'any', 'of', 'these', 'git', 'genes', '.', 'Hamdija', 'Karamehmedovic', ';', 'Ibn', 'al', '-', 'Nefis', ',', 'O', 'Mudzez', 'O', 'al', 'O', '-', 'O', 'Kanum', 'O', ',', 'Republicki', 'zavod', 'za', 'zdravstvenu', 'zastitu', 'Sarajevo', ',', '1961', ',', '1', '-', '219', ';', 'Mr', 'ph', 'Samuel', 'Elazar', ',', 'Ajnija', 'Omanic', ':', 'O', 'Bibliografija', 'O', 'medicinskih', 'O', 'djela', 'O', 'u', 'O', 'SR', 'O', 'BiH', 'O', 'do', 'O', '1895', 'O', '.', 'O', 'Medicinska', 'knjiga', 'Beograd', '-', 'Zagreb', '1984', ';', 'Besides', ',', 'the', 'great', 'contribution', 'in', 'bibliography', 'was', 'given', 'by', 'dr', 'Lujo', 'Taler', ',', 'prof', '.']\n",
            "119 119\n",
            "['a_0', 'high_1', 'copy_2', 'number_3', 'suppressor_4', 'of_5', 'a_6', 'mutation_7', 'in_8', 'git3_9', ',_10', 'is_11', 'able_12', 'to_13', 'suppress_14', 'the_15', 'defects_16', 'conferred_17', 'by_18', 'a_19', 'mutation_20', 'in_21', 'any_22', 'of_23', 'these_24', 'git_25', 'genes_26', '._27', 'Hamdija_28', 'Karamehmedovic_29', ';_30', 'Ibn_31', 'al_32', '-_33', 'Nefis_34', ',_35', '\\tO\\nMudzez\\tO\\nal\\tO\\n-\\tO\\nKanum\\tO\\n_36', ',_37', 'Republicki_38', 'zavod_39', 'za_40', 'zdravstvenu_41', 'zastitu_42', 'Sarajevo_43', ',_44', '1961_45', ',_46', '1_47', '-_48', '219_49', ';_50', 'Mr_51', 'ph_52', 'Samuel_53', 'Elazar_54', ',_55', 'Ajnija_56', 'Omanic_57', ':_58', '\\tO\\nBibliografija\\tO\\nmedicinskih\\tO\\ndjela\\tO\\nu\\tO\\nSR\\tO\\nBiH\\tO\\ndo\\tO\\n1895\\tO\\n.\\tO\\n_59', 'Medicinska_60', 'knjiga_61', 'Beograd_62', '-_63', 'Zagreb_64', '1984_65', ';_66', 'Besides_67', ',_68', 'the_69', 'great_70', 'contribution_71', 'in_72', 'bibliography_73', 'was_74', 'given_75', 'by_76', 'dr_77', 'Lujo_78', 'Taler_79', ',_80', 'prof_81', '._82', 'dr_83', '._84', 'The_85', 'longest_86', 'cDNA_87', 'insert_88', 'identified_89', 'was_90', '2_91', '._92', '2_93', 'kb_94', 'and_95', 'encoded_96', 'the_97', 'entire_98', '462_99', '-_100', 'amino_101', 'acid_102', 'open_103', 'reading_104', 'frame_105', 'of_106', 'rat_107', 'CgA_108', 'including_109', 'an_110', '18_111', '-_112', 'amino_113', 'acid_114', 'hydrophobic_115', 'signal_116', 'peptide_117', '._118']\n",
            "BAD:  22230\n",
            "rpws:      index         token label\n",
            "0   22349            We     O\n",
            "1   22350          have     O\n",
            "2   22351          used     O\n",
            "3   22352      systemic     O\n",
            "4   22353   application     O\n",
            "..    ...           ...   ...\n",
            "63  22412  individually     O\n",
            "64  22413             .     O\n",
            "65  22414         Among     O\n",
            "66  22415           the     O\n",
            "67  22416     different     O\n",
            "\n",
            "[68 rows x 3 columns]\n",
            "dr . The longest cDNA insert identified was 2 . 2 kb and encoded the entire 462 - amino acid open reading frame of rat CgA including an 18 - amino acid hydrophobic signal peptide . We have used systemic application of the ototoxic drug amikacin , to induce total cochlear haircell loss in the chinchilla , in order to create an animal model of profound deafness .\n",
            "['dr', '.', 'The', 'longest', 'cDNA', 'insert', 'identified', 'was', '2', '.', '2', 'kb', 'and', 'encoded', 'the', 'entire', '462', '-', 'amino', 'acid', 'open', 'reading', 'frame', 'of', 'rat', 'CgA', 'including', 'an', '18', '-', 'amino', 'acid', 'hydrophobic', 'signal', 'peptide', '.', 'We', 'have', 'used', 'systemic', 'application', 'of', 'the', 'ototoxic', 'drug', 'amikacin', ',', 'to', 'induce', 'total', 'cochlear', 'haircell', 'loss', 'in', 'the', 'chinchilla', ',', 'in', 'order', 'to', 'create', 'an', 'animal', 'model', 'of', 'profound', 'deafness', '.']\n",
            "68 68\n",
            "['We_0', 'have_1', 'used_2', 'systemic_3', 'application_4', 'of_5', 'the_6', 'ototoxic_7', 'drug_8', 'amikacin_9', ',_10', 'to_11', 'induce_12', 'total_13', 'cochlear_14', 'haircell_15', 'loss_16', 'in_17', 'the_18', 'chinchilla_19', ',_20', 'in_21', 'order_22', 'to_23', 'create_24', 'an_25', 'animal_26', 'model_27', 'of_28', 'profound_29', 'deafness_30', '._31', 'Cotransfection_32', 'with_33', 'C_34', '/_35', 'EBPbeta_36', 'and_37', 'GATA_38', '-_39', '1_40', 'expression_41', 'vectors_42', 'produced_43', 'a_44', '5_45', '-_46', 'fold_47', 'increase_48', 'compared_49', 'with_50', 'cotransfection_51', 'with_52', 'the_53', 'C_54', '/_55', 'EBPbeta_56', 'or_57', 'GATA_58', '-_59', '1_60', 'expression_61', 'vectors_62', 'individually_63', '._64', 'Among_65', 'the_66', 'different_67']\n",
            "BAD:  22349\n",
            "rpws:      index           token label\n",
            "0   22417      epithelial     O\n",
            "1   22418            cell     O\n",
            "2   22419           lines     O\n",
            "3   22420          tested     O\n",
            "4   22421               ,     O\n",
            "5   22422            only     O\n",
            "6   22423           RTS3b     O\n",
            "7   22424           cells     O\n",
            "8   22425         allowed     O\n",
            "9   22426              an     O\n",
            "10  22427      expression     O\n",
            "11  22428         pattern     O\n",
            "12  22429         similar     O\n",
            "13  22430              to     O\n",
            "14  22431            that     O\n",
            "15  22432        observed     O\n",
            "16  22433              in     O\n",
            "17  22434       naturally     O\n",
            "18  22435        infected     O\n",
            "19  22436          benign     O\n",
            "20  22437      condylomas     O\n",
            "21  22438               .     O\n",
            "22  22439              In     O\n",
            "23  22440        contrast     O\n",
            "24  22441              to     O\n",
            "25  22442             the     O\n",
            "26  22443             p16     B\n",
            "27  22444               -     O\n",
            "28  22445        mediated     O\n",
            "29  22446              G1     O\n",
            "30  22447           block     O\n",
            "31  22448               ,     O\n",
            "32  22449              G1     O\n",
            "33  22450          arrest     O\n",
            "34  22451        mediated     O\n",
            "35  22452              by     O\n",
            "36  22453             the     O\n",
            "37  22454             cdk     B\n",
            "38  22455      inhibitors     O\n",
            "39  22456         p21Cip1     B\n",
            "40  22457              or     O\n",
            "41  22458         p27Kip1     B\n",
            "42  22459          cannot     O\n",
            "43  22460              be     O\n",
            "44  22461        bypassed     O\n",
            "45  22462          either     O\n",
            "46  22463              by     O\n",
            "47  22464    inactivation     O\n",
            "48  22465              of     O\n",
            "49  22466             pRb     B\n",
            "50  22467              or     O\n",
            "51  22468  overexpression     O\n",
            "52  22469              of     O\n",
            "53  22470             E2F     B\n",
            "54  22471          family     I\n",
            "55  22472         members     I\n",
            "56  22473               .     O\n",
            "57  22474              In     O\n",
            "Cotransfection with C / EBPbeta and GATA - 1 expression vectors produced a 5 - fold increase compared with cotransfection with the C / EBPbeta or GATA - 1 expression vectors individually . Among the different epithelial cell lines tested , only RTS3b cells allowed an expression pattern similar to that observed in naturally infected benign condylomas .\n",
            "['Cotransfection', 'with', 'C', '/', 'EBPbeta', 'and', 'GATA', '-', '1', 'expression', 'vectors', 'produced', 'a', '5', '-', 'fold', 'increase', 'compared', 'with', 'cotransfection', 'with', 'the', 'C', '/', 'EBPbeta', 'or', 'GATA', '-', '1', 'expression', 'vectors', 'individually', '.', 'Among', 'the', 'different', 'epithelial', 'cell', 'lines', 'tested', ',', 'only', 'RTS3b', 'cells', 'allowed', 'an', 'expression', 'pattern', 'similar', 'to', 'that', 'observed', 'in', 'naturally', 'infected', 'benign', 'condylomas', '.']\n",
            "58 58\n",
            "['epithelial_0', 'cell_1', 'lines_2', 'tested_3', ',_4', 'only_5', 'RTS3b_6', 'cells_7', 'allowed_8', 'an_9', 'expression_10', 'pattern_11', 'similar_12', 'to_13', 'that_14', 'observed_15', 'in_16', 'naturally_17', 'infected_18', 'benign_19', 'condylomas_20', '._21', 'In_22', 'contrast_23', 'to_24', 'the_25', 'p16_26', '-_27', 'mediated_28', 'G1_29', 'block_30', ',_31', 'G1_32', 'arrest_33', 'mediated_34', 'by_35', 'the_36', 'cdk_37', 'inhibitors_38', 'p21Cip1_39', 'or_40', 'p27Kip1_41', 'cannot_42', 'be_43', 'bypassed_44', 'either_45', 'by_46', 'inactivation_47', 'of_48', 'pRb_49', 'or_50', 'overexpression_51', 'of_52', 'E2F_53', 'family_54', 'members_55', '._56', 'In_57']\n",
            "BAD:  22417\n",
            "rpws:      index            token label\n",
            "0   22475         addition     O\n",
            "1   22476                ,     O\n",
            "2   22477              the     O\n",
            "3   22478          authors     O\n",
            "4   22479            found     O\n",
            "..    ...              ...   ...\n",
            "63  22538              the     O\n",
            "64  22539  transcriptional     O\n",
            "65  22540          control     O\n",
            "66  22541               of     O\n",
            "67  22542      osteocalcin     B\n",
            "\n",
            "[68 rows x 3 columns]\n",
            "In contrast to the p16 - mediated G1 block , G1 arrest mediated by the cdk inhibitors p21Cip1 or p27Kip1 cannot be bypassed either by inactivation of pRb or overexpression of E2F family members . In addition , the authors found one an3 allele ( an3 - W138A ) in which a dTph1 element had inserted 30 bp upstream the translation start , without inactivating the gene .\n",
            "['In', 'contrast', 'to', 'the', 'p16', '-', 'mediated', 'G1', 'block', ',', 'G1', 'arrest', 'mediated', 'by', 'the', 'cdk', 'inhibitors', 'p21Cip1', 'or', 'p27Kip1', 'cannot', 'be', 'bypassed', 'either', 'by', 'inactivation', 'of', 'pRb', 'or', 'overexpression', 'of', 'E2F', 'family', 'members', '.', 'In', 'addition', ',', 'the', 'authors', 'found', 'one', 'an3', 'allele', '(', 'an3', '-', 'W138A', ')', 'in', 'which', 'a', 'dTph1', 'element', 'had', 'inserted', '30', 'bp', 'upstream', 'the', 'translation', 'start', ',', 'without', 'inactivating', 'the', 'gene', '.']\n",
            "68 68\n",
            "['addition_0', ',_1', 'the_2', 'authors_3', 'found_4', 'one_5', 'an3_6', 'allele_7', '(_8', 'an3_9', '-_10', 'W138A_11', ')_12', 'in_13', 'which_14', 'a_15', 'dTph1_16', 'element_17', 'had_18', 'inserted_19', '30_20', 'bp_21', 'upstream_22', 'the_23', 'translation_24', 'start_25', ',_26', 'without_27', 'inactivating_28', 'the_29', 'gene_30', '._31', 'Case_32', 'of_33', 'nephrotic_34', 'syndrome_35', 'caused_36', 'by_37', 'gold_38', 'preparations_39', 'To_40', 'investigate_41', 'the_42', 'role_43', 'of_44', 'bHLH_45', 'proteins_46', 'in_47', 'MC3T3_48', '-_49', 'E1_50', 'osteoblasts_51', ',_52', 'which_53', 'undergo_54', 'a_55', 'developmental_56', 'sequence_57', 'in_58', 'vitro_59', ',_60', 'we_61', 'analyzed_62', 'the_63', 'transcriptional_64', 'control_65', 'of_66', 'osteocalcin_67']\n",
            "BAD:  22475\n",
            "rpws:      index         token label\n",
            "0   22543          gene     I\n",
            "1   22544    expression     O\n",
            "2   22545            by     O\n",
            "3   22546        stable     O\n",
            "4   22547  transfection     O\n",
            "..    ...           ...   ...\n",
            "69  22612         study     O\n",
            "70  22613             ,     O\n",
            "71  22614            we     O\n",
            "72  22615          show     O\n",
            "73  22616          that     O\n",
            "\n",
            "[74 rows x 3 columns]\n",
            "Case of nephrotic syndrome caused by gold preparations To investigate the role of bHLH proteins in MC3T3 - E1 osteoblasts , which undergo a developmental sequence in vitro , we analyzed the transcriptional control of osteocalcin gene expression by stable transfection of an osteocalcin promoter - luciferase chimeric gene ( p637OC - luc ) and assessed the role of E - box cis - acting elements in osteocalcin promoter by DNA binding assays .\n",
            "['Case', 'of', 'nephrotic', 'syndrome', 'caused', 'by', 'gold', 'preparations', 'To', 'investigate', 'the', 'role', 'of', 'bHLH', 'proteins', 'in', 'MC3T3', '-', 'E1', 'osteoblasts', ',', 'which', 'undergo', 'a', 'developmental', 'sequence', 'in', 'vitro', ',', 'we', 'analyzed', 'the', 'transcriptional', 'control', 'of', 'osteocalcin', 'gene', 'expression', 'by', 'stable', 'transfection', 'of', 'an', 'osteocalcin', 'promoter', '-', 'luciferase', 'chimeric', 'gene', '(', 'p637OC', '-', 'luc', ')', 'and', 'assessed', 'the', 'role', 'of', 'E', '-', 'box', 'cis', '-', 'acting', 'elements', 'in', 'osteocalcin', 'promoter', 'by', 'DNA', 'binding', 'assays', '.']\n",
            "74 74\n",
            "['gene_0', 'expression_1', 'by_2', 'stable_3', 'transfection_4', 'of_5', 'an_6', 'osteocalcin_7', 'promoter_8', '-_9', 'luciferase_10', 'chimeric_11', 'gene_12', '(_13', 'p637OC_14', '-_15', 'luc_16', ')_17', 'and_18', 'assessed_19', 'the_20', 'role_21', 'of_22', 'E_23', '-_24', 'box_25', 'cis_26', '-_27', 'acting_28', 'elements_29', 'in_30', 'osteocalcin_31', 'promoter_32', 'by_33', 'DNA_34', 'binding_35', 'assays_36', '._37', 'In_38', 'summary_39', ',_40', 'the_41', 'only_42', 'treatment_43', '-_44', 'related_45', 'effect_46', 'noted_47', 'in_48', 'this_49', 'study_50', 'was_51', 'hydrocarbon_52', 'nephropathy_53', 'in_54', 'male_55', 'rats_56', ',_57', 'which_58', 'is_59', 'not_60', 'considered_61', 'relevant_62', 'for_63', 'human_64', 'health_65', '._66', 'In_67', 'this_68', 'study_69', ',_70', 'we_71', 'show_72', 'that_73']\n",
            "BAD:  22543\n",
            "rpws:      index        token label\n",
            "0   22617          HNF     B\n",
            "1   22618            -     I\n",
            "2   22619            4     I\n",
            "3   22620            ,     O\n",
            "4   22621            a     O\n",
            "..    ...          ...   ...\n",
            "79  22696       highly     O\n",
            "80  22697    expressed     O\n",
            "81  22698           in     O\n",
            "82  22699  postmeiotic     O\n",
            "83  22700        cells     O\n",
            "\n",
            "[84 rows x 3 columns]\n",
            "In summary , the only treatment - related effect noted in this study was hydrocarbon nephropathy in male rats , which is not considered relevant for human health . In this study , we show that HNF - 4 , a member of the steroid hormone receptor superfamily , binds the AF - 1 site on the apoB promoter and through it activates transcription in transient transfection assays in both liver and non - liver cell lines , HepG2 and HeLa , respectively .\n",
            "['In', 'summary', ',', 'the', 'only', 'treatment', '-', 'related', 'effect', 'noted', 'in', 'this', 'study', 'was', 'hydrocarbon', 'nephropathy', 'in', 'male', 'rats', ',', 'which', 'is', 'not', 'considered', 'relevant', 'for', 'human', 'health', '.', 'In', 'this', 'study', ',', 'we', 'show', 'that', 'HNF', '-', '4', ',', 'a', 'member', 'of', 'the', 'steroid', 'hormone', 'receptor', 'superfamily', ',', 'binds', 'the', 'AF', '-', '1', 'site', 'on', 'the', 'apoB', 'promoter', 'and', 'through', 'it', 'activates', 'transcription', 'in', 'transient', 'transfection', 'assays', 'in', 'both', 'liver', 'and', 'non', '-', 'liver', 'cell', 'lines', ',', 'HepG2', 'and', 'HeLa', ',', 'respectively', '.']\n",
            "84 84\n",
            "['HNF_0', '-_1', '4_2', ',_3', 'a_4', 'member_5', 'of_6', 'the_7', 'steroid_8', 'hormone_9', 'receptor_10', 'superfamily_11', ',_12', 'binds_13', 'the_14', 'AF_15', '-_16', '1_17', 'site_18', 'on_19', 'the_20', 'apoB_21', 'promoter_22', 'and_23', 'through_24', 'it_25', 'activates_26', 'transcription_27', 'in_28', 'transient_29', 'transfection_30', 'assays_31', 'in_32', 'both_33', 'liver_34', 'and_35', 'non_36', '-_37', 'liver_38', 'cell_39', 'lines_40', ',_41', 'HepG2_42', 'and_43', 'HeLa_44', ',_45', 'respectively_46', '._47', 'Gastrointestinal_48', 'accumulation_49', 'of_50', 'indium_51', '-_52', '111_53', 'labelled_54', 'granulocytes_55', 'in_56', 'reactive_57', 'arthritis_58', '._59', 'Acute_60', 'intoxication_61', 'with_62', 'cypermethrin_63', '(_64', 'NRDC_65', '149_66', ')_67', '._68', 'We_69', 'have_70', 'previously_71', 'shown_72', 'that_73', 'the_74', 'transcriptional_75', 'activator_76', 'CREM_77', 'is_78', 'highly_79', 'expressed_80', 'in_81', 'postmeiotic_82', 'cells_83']\n",
            "BAD:  22617\n",
            "rpws:      index           token label\n",
            "0   22701               .     O\n",
            "1   22702              No     O\n",
            "2   22703     interaction     O\n",
            "3   22704            with     O\n",
            "4   22705        epidural     O\n",
            "5   22706      anesthesia     O\n",
            "6   22707             was     O\n",
            "7   22708        observed     O\n",
            "8   22709               .     O\n",
            "9   22710              In     O\n",
            "10  22711            late     O\n",
            "11  22712         October     O\n",
            "12  22713               ,     O\n",
            "13  22714            1974     O\n",
            "14  22715               ,     O\n",
            "15  22716  Staphylococcus     O\n",
            "16  22717          aureus     O\n",
            "17  22718   postoperative     O\n",
            "18  22719           wound     O\n",
            "19  22720       infection     O\n",
            "20  22721             was     O\n",
            "21  22722        recorded     O\n",
            "22  22723              in     O\n",
            "23  22724               a     O\n",
            "24  22725        nonhuman     O\n",
            "25  22726         primate     O\n",
            "26  22727               (     O\n",
            "27  22728          Macaca     O\n",
            "28  22729         mulatta     O\n",
            "29  22730               )     O\n",
            "30  22731           which     O\n",
            "31  22732             had     O\n",
            "32  22733        recently     O\n",
            "33  22734       undergone     O\n",
            "34  22735        surgical     O\n",
            "35  22736       operation     O\n",
            "36  22737               .     O\n",
            "37  22738          Twenty     O\n",
            "38  22739               -     O\n",
            "39  22740             six     O\n",
            "40  22741          calves     O\n",
            "41  22742            were     O\n",
            "42  22743       subjected     O\n",
            "43  22744              to     O\n",
            "44  22745               a     O\n",
            "Gastrointestinal accumulation of indium - 111 labelled granulocytes in reactive arthritis . Acute intoxication with cypermethrin ( NRDC 149 ) . We have previously shown that the transcriptional activator CREM is highly expressed in postmeiotic cells . No interaction with epidural anesthesia was observed .\n",
            "['Gastrointestinal', 'accumulation', 'of', 'indium', '-', '111', 'labelled', 'granulocytes', 'in', 'reactive', 'arthritis', '.', 'Acute', 'intoxication', 'with', 'cypermethrin', '(', 'NRDC', '149', ')', '.', 'We', 'have', 'previously', 'shown', 'that', 'the', 'transcriptional', 'activator', 'CREM', 'is', 'highly', 'expressed', 'in', 'postmeiotic', 'cells', '.', 'No', 'interaction', 'with', 'epidural', 'anesthesia', 'was', 'observed', '.']\n",
            "45 45\n",
            "['._0', 'No_1', 'interaction_2', 'with_3', 'epidural_4', 'anesthesia_5', 'was_6', 'observed_7', '._8', 'In_9', 'late_10', 'October_11', ',_12', '1974_13', ',_14', 'Staphylococcus_15', 'aureus_16', 'postoperative_17', 'wound_18', 'infection_19', 'was_20', 'recorded_21', 'in_22', 'a_23', 'nonhuman_24', 'primate_25', '(_26', 'Macaca_27', 'mulatta_28', ')_29', 'which_30', 'had_31', 'recently_32', 'undergone_33', 'surgical_34', 'operation_35', '._36', 'Twenty_37', '-_38', 'six_39', 'calves_40', 'were_41', 'subjected_42', 'to_43', 'a_44']\n",
            "BAD:  22701\n",
            "rpws:      index           token label\n",
            "0   22746       technique     O\n",
            "1   22747              of     O\n",
            "2   22748    cryoablation     O\n",
            "3   22749              in     O\n",
            "4   22750           order     O\n",
            "5   22751              to     O\n",
            "6   22752       establish     O\n",
            "7   22753              an     O\n",
            "8   22754          animal     O\n",
            "9   22755           model     O\n",
            "10  22756              of     O\n",
            "11  22757        complete     O\n",
            "12  22758         cardiac     O\n",
            "13  22759     denervation     O\n",
            "14  22760               .     O\n",
            "15  22761             Our     O\n",
            "16  22762         current     O\n",
            "17  22763           study     O\n",
            "18  22764            aims     O\n",
            "19  22765              at     O\n",
            "20  22766      clarifying     O\n",
            "21  22767             the     O\n",
            "22  22768            role     O\n",
            "23  22769              of     O\n",
            "24  22770  myristoylation     O\n",
            "25  22771              in     O\n",
            "26  22772        caveolar     O\n",
            "27  22773       targeting     O\n",
            "28  22774           using     O\n",
            "29  22775            well     O\n",
            "30  22776               -     O\n",
            "31  22777   characterized     O\n",
            "32  22778       acylation     O\n",
            "33  22779         mutants     O\n",
            "34  22780              of     O\n",
            "35  22781             two     O\n",
            "36  22782           model     O\n",
            "37  22783        proteins     O\n",
            "38  22784               ,     O\n",
            "39  22785          namely     O\n",
            "40  22786             Gi1     B\n",
            "41  22787           alpha     I\n",
            "42  22788             and     O\n",
            "43  22789               c     B\n",
            "44  22790               -     I\n",
            "45  22791             Src     I\n",
            "46  22792               .     O\n",
            "47  22793             The     O\n",
            "48  22794          effect     O\n",
            "49  22795              of     O\n",
            "50  22796    deferoxamine     O\n",
            "In late October , 1974 , Staphylococcus aureus postoperative wound infection was recorded in a nonhuman primate ( Macaca mulatta ) which had recently undergone surgical operation . Twenty - six calves were subjected to a technique of cryoablation in order to establish an animal model of complete cardiac denervation .\n",
            "['In', 'late', 'October', ',', '1974', ',', 'Staphylococcus', 'aureus', 'postoperative', 'wound', 'infection', 'was', 'recorded', 'in', 'a', 'nonhuman', 'primate', '(', 'Macaca', 'mulatta', ')', 'which', 'had', 'recently', 'undergone', 'surgical', 'operation', '.', 'Twenty', '-', 'six', 'calves', 'were', 'subjected', 'to', 'a', 'technique', 'of', 'cryoablation', 'in', 'order', 'to', 'establish', 'an', 'animal', 'model', 'of', 'complete', 'cardiac', 'denervation', '.']\n",
            "51 51\n",
            "['technique_0', 'of_1', 'cryoablation_2', 'in_3', 'order_4', 'to_5', 'establish_6', 'an_7', 'animal_8', 'model_9', 'of_10', 'complete_11', 'cardiac_12', 'denervation_13', '._14', 'Our_15', 'current_16', 'study_17', 'aims_18', 'at_19', 'clarifying_20', 'the_21', 'role_22', 'of_23', 'myristoylation_24', 'in_25', 'caveolar_26', 'targeting_27', 'using_28', 'well_29', '-_30', 'characterized_31', 'acylation_32', 'mutants_33', 'of_34', 'two_35', 'model_36', 'proteins_37', ',_38', 'namely_39', 'Gi1_40', 'alpha_41', 'and_42', 'c_43', '-_44', 'Src_45', '._46', 'The_47', 'effect_48', 'of_49', 'deferoxamine_50']\n",
            "BAD:  22746\n",
            "rpws:      index    token label\n",
            "0   22797        B     O\n",
            "1   22798       on     O\n",
            "2   22799      the     O\n",
            "3   22800    blood     O\n",
            "4   22801    sugar     O\n",
            "..    ...      ...   ...\n",
            "70  22867        ,     O\n",
            "71  22868  defined     O\n",
            "72  22869       as     O\n",
            "73  22870       an     O\n",
            "74  22871     UAER     O\n",
            "\n",
            "[75 rows x 3 columns]\n",
            "Our current study aims at clarifying the role of myristoylation in caveolar targeting using well - characterized acylation mutants of two model proteins , namely Gi1 alpha and c - Src . The effect of deferoxamine B on the blood sugar behavior in intravenous glucose loading in healthy persons We infer that RecA - mediated cleavage of UmuD is another role for RecA in SOS mutagenesis , probably activating UmuD for its mutagenic function .\n",
            "['Our', 'current', 'study', 'aims', 'at', 'clarifying', 'the', 'role', 'of', 'myristoylation', 'in', 'caveolar', 'targeting', 'using', 'well', '-', 'characterized', 'acylation', 'mutants', 'of', 'two', 'model', 'proteins', ',', 'namely', 'Gi1', 'alpha', 'and', 'c', '-', 'Src', '.', 'The', 'effect', 'of', 'deferoxamine', 'B', 'on', 'the', 'blood', 'sugar', 'behavior', 'in', 'intravenous', 'glucose', 'loading', 'in', 'healthy', 'persons', 'We', 'infer', 'that', 'RecA', '-', 'mediated', 'cleavage', 'of', 'UmuD', 'is', 'another', 'role', 'for', 'RecA', 'in', 'SOS', 'mutagenesis', ',', 'probably', 'activating', 'UmuD', 'for', 'its', 'mutagenic', 'function', '.']\n",
            "75 75\n",
            "['B_0', 'on_1', 'the_2', 'blood_3', 'sugar_4', 'behavior_5', 'in_6', 'intravenous_7', 'glucose_8', 'loading_9', 'in_10', 'healthy_11', 'persons_12', 'We_13', 'infer_14', 'that_15', 'RecA_16', '-_17', 'mediated_18', 'cleavage_19', 'of_20', 'UmuD_21', 'is_22', 'another_23', 'role_24', 'for_25', 'RecA_26', 'in_27', 'SOS_28', 'mutagenesis_29', ',_30', 'probably_31', 'activating_32', 'UmuD_33', 'for_34', 'its_35', 'mutagenic_36', 'function_37', '._38', 'SETTING_39', ':_40', 'Participants_41', 'in_42', 'the_43', 'Physicians_44', \"'_45\", 'Health_46', 'Study_47', ',_48', 'a_49', 'randomized_50', 'trial_51', 'of_52', 'aspirin_53', 'and_54', 'beta_55', '-_56', 'carotene_57', 'among_58', 'U_59', '._60', 'S_61', '._62', 'male_63', 'physicians_64', '._65', 'The_66', 'prevalence_67', 'of_68', 'microalbuminuria_69', ',_70', 'defined_71', 'as_72', 'an_73', 'UAER_74']\n",
            "BAD:  22797\n",
            "rpws:      index     token label\n",
            "0   22872        in     O\n",
            "1   22873       the     O\n",
            "2   22874     range     O\n",
            "3   22875        of     O\n",
            "4   22876        15     O\n",
            "..    ...       ...   ...\n",
            "62  22934        in     O\n",
            "63  22935       the     O\n",
            "64  22936       AAV     O\n",
            "65  22937  inverted     O\n",
            "66  22938  terminal     O\n",
            "\n",
            "[67 rows x 3 columns]\n",
            "SETTING : Participants in the Physicians ' Health Study , a randomized trial of aspirin and beta - carotene among U . S . male physicians . The prevalence of microalbuminuria , defined as an UAER in the range of 15 - 150 micrograms min - 1 in an overnight urine sample , was 3 % ( 95 % C . I . interval : 1 .\n",
            "['SETTING', ':', 'Participants', 'in', 'the', 'Physicians', \"'\", 'Health', 'Study', ',', 'a', 'randomized', 'trial', 'of', 'aspirin', 'and', 'beta', '-', 'carotene', 'among', 'U', '.', 'S', '.', 'male', 'physicians', '.', 'The', 'prevalence', 'of', 'microalbuminuria', ',', 'defined', 'as', 'an', 'UAER', 'in', 'the', 'range', 'of', '15', '-', '150', 'micrograms', 'min', '-', '1', 'in', 'an', 'overnight', 'urine', 'sample', ',', 'was', '3', '%', '(', '95', '%', 'C', '.', 'I', '.', 'interval', ':', '1', '.']\n",
            "67 67\n",
            "['in_0', 'the_1', 'range_2', 'of_3', '15_4', '-_5', '150_6', 'micrograms_7', 'min_8', '-_9', '1_10', 'in_11', 'an_12', 'overnight_13', 'urine_14', 'sample_15', ',_16', 'was_17', '3_18', '%_19', '(_20', '95_21', '%_22', 'C_23', '._24', 'I_25', '._26', 'interval_27', ':_28', '1_29', '._30', '9_31', '-_32', '4_33', '._34', '0_35', ')_36', '._37', 'The_38', 'Rep_39', 'proteins_40', 'of_41', 'adeno_42', '-_43', 'associated_44', 'virus_45', 'type_46', '2_47', '(_48', 'AAV_49', ')_50', 'are_51', 'known_52', 'to_53', 'bind_54', 'to_55', 'Rep_56', 'recognition_57', 'sequences_58', '(_59', 'RRSs_60', ')_61', 'in_62', 'the_63', 'AAV_64', 'inverted_65', 'terminal_66']\n",
            "BAD:  22872\n",
            "rpws:      index        token label\n",
            "0   22939      repeats     O\n",
            "1   22940            (     O\n",
            "2   22941         ITRs     O\n",
            "3   22942            )     O\n",
            "4   22943            ,     O\n",
            "..    ...          ...   ...\n",
            "71  23010          may     O\n",
            "72  23011         have     O\n",
            "73  23012     enhanced     O\n",
            "74  23013  specificity     O\n",
            "75  23014          for     O\n",
            "\n",
            "[76 rows x 3 columns]\n",
            "9 - 4 . 0 ) . The Rep proteins of adeno - associated virus type 2 ( AAV ) are known to bind to Rep recognition sequences ( RRSs ) in the AAV inverted terminal repeats ( ITRs ) , the AAV p5 promoter , and the preferred AAV integration site in human chromosome 19 , called AAVS1 . LIP and DIP are less common IIPs , both characterized by ground - glass attenuation .\n",
            "['9', '-', '4', '.', '0', ')', '.', 'The', 'Rep', 'proteins', 'of', 'adeno', '-', 'associated', 'virus', 'type', '2', '(', 'AAV', ')', 'are', 'known', 'to', 'bind', 'to', 'Rep', 'recognition', 'sequences', '(', 'RRSs', ')', 'in', 'the', 'AAV', 'inverted', 'terminal', 'repeats', '(', 'ITRs', ')', ',', 'the', 'AAV', 'p5', 'promoter', ',', 'and', 'the', 'preferred', 'AAV', 'integration', 'site', 'in', 'human', 'chromosome', '19', ',', 'called', 'AAVS1', '.', 'LIP', 'and', 'DIP', 'are', 'less', 'common', 'IIPs', ',', 'both', 'characterized', 'by', 'ground', '-', 'glass', 'attenuation', '.']\n",
            "76 76\n",
            "['repeats_0', '(_1', 'ITRs_2', ')_3', ',_4', 'the_5', 'AAV_6', 'p5_7', 'promoter_8', ',_9', 'and_10', 'the_11', 'preferred_12', 'AAV_13', 'integration_14', 'site_15', 'in_16', 'human_17', 'chromosome_18', '19_19', ',_20', 'called_21', 'AAVS1_22', '._23', 'LIP_24', 'and_25', 'DIP_26', 'are_27', 'less_28', 'common_29', 'IIPs_30', ',_31', 'both_32', 'characterized_33', 'by_34', 'ground_35', '-_36', 'glass_37', 'attenuation_38', '._39', 'The_40', 'levels_41', 'of_42', 'TPAR1_43', 'mRNAs_44', 'were_45', 'dramatically_46', 'down_47', '-_48', 'regulated_49', 'in_50', 'regenerating_51', 'rat_52', 'liver_53', 'when_54', 'compared_55', 'to_56', 'normal_57', 'adult_58', 'liver_59', '._60', 'Copyright_61', '1999_62', 'Academic_63', 'Press_64', '._65', 'The_66', 'authors_67', 'suggest_68', 'that_69', 'alprazolam_70', 'may_71', 'have_72', 'enhanced_73', 'specificity_74', 'for_75']\n",
            "BAD:  22939\n",
            "rpws:      index           token label\n",
            "0   23015               a     O\n",
            "1   23016   subpopulation     O\n",
            "2   23017              of     O\n",
            "3   23018  benzodiazepine     B\n",
            "4   23019       receptors     I\n",
            "..    ...             ...   ...\n",
            "70  23085            edar     B\n",
            "71  23086             was     O\n",
            "72  23087         induced     O\n",
            "73  23088              by     O\n",
            "74  23089    activinbetaA     B\n",
            "\n",
            "[75 rows x 3 columns]\n",
            "The levels of TPAR1 mRNAs were dramatically down - regulated in regenerating rat liver when compared to normal adult liver . Copyright 1999 Academic Press . The authors suggest that alprazolam may have enhanced specificity for a subpopulation of benzodiazepine receptors . Southern hybridization analysis of BenR and BenS transformants suggested that plasmid integration occurred most frequently at the chromosomal bens locus , however evidence for gene conversion and heterologous recombination was also observed .\n",
            "['The', 'levels', 'of', 'TPAR1', 'mRNAs', 'were', 'dramatically', 'down', '-', 'regulated', 'in', 'regenerating', 'rat', 'liver', 'when', 'compared', 'to', 'normal', 'adult', 'liver', '.', 'Copyright', '1999', 'Academic', 'Press', '.', 'The', 'authors', 'suggest', 'that', 'alprazolam', 'may', 'have', 'enhanced', 'specificity', 'for', 'a', 'subpopulation', 'of', 'benzodiazepine', 'receptors', '.', 'Southern', 'hybridization', 'analysis', 'of', 'BenR', 'and', 'BenS', 'transformants', 'suggested', 'that', 'plasmid', 'integration', 'occurred', 'most', 'frequently', 'at', 'the', 'chromosomal', 'bens', 'locus', ',', 'however', 'evidence', 'for', 'gene', 'conversion', 'and', 'heterologous', 'recombination', 'was', 'also', 'observed', '.']\n",
            "75 75\n",
            "['a_0', 'subpopulation_1', 'of_2', 'benzodiazepine_3', 'receptors_4', '._5', 'Southern_6', 'hybridization_7', 'analysis_8', 'of_9', 'BenR_10', 'and_11', 'BenS_12', 'transformants_13', 'suggested_14', 'that_15', 'plasmid_16', 'integration_17', 'occurred_18', 'most_19', 'frequently_20', 'at_21', 'the_22', 'chromosomal_23', 'bens_24', 'locus_25', ',_26', 'however_27', 'evidence_28', 'for_29', 'gene_30', 'conversion_31', 'and_32', 'heterologous_33', 'recombination_34', 'was_35', 'also_36', 'observed_37', '._38', 'The_39', 'analysis_40', 'of_41', 'the_42', 'effects_43', 'of_44', 'eight_45', 'signaling_46', 'molecules_47', 'in_48', 'the_49', 'TGFbeta_50', ',_51', 'FGF_52', ',_53', 'Hh_54', ',_55', 'Wnt_56', ',_57', 'and_58', 'EGF_59', 'families_60', 'in_61', 'tooth_62', 'explant_63', 'cultures_64', 'revealed_65', 'that_66', 'the_67', 'expression_68', 'of_69', 'edar_70', 'was_71', 'induced_72', 'by_73', 'activinbetaA_74']\n",
            "BAD:  23015\n",
            "rpws:      index          token label\n",
            "0   23090              ,     O\n",
            "1   23091        whereas     O\n",
            "2   23092           Wnt6     B\n",
            "3   23093        induced     O\n",
            "4   23094  ectodysplasin     B\n",
            "..    ...            ...   ...\n",
            "78  23168             In     O\n",
            "79  23169           turn     O\n",
            "80  23170              ,     O\n",
            "81  23171     production     O\n",
            "82  23172             of     O\n",
            "\n",
            "[83 rows x 3 columns]\n",
            "The analysis of the effects of eight signaling molecules in the TGFbeta , FGF , Hh , Wnt , and EGF families in tooth explant cultures revealed that the expression of edar was induced by activinbetaA , whereas Wnt6 induced ectodysplasin expression . In mouse , two high - affinity binding sites with an apparent dissociation constant ( Kd ) of 50 to 100 nM have been mapped in the 5 ' ETS upstream from the early pre - rRNA processing site .\n",
            "['The', 'analysis', 'of', 'the', 'effects', 'of', 'eight', 'signaling', 'molecules', 'in', 'the', 'TGFbeta', ',', 'FGF', ',', 'Hh', ',', 'Wnt', ',', 'and', 'EGF', 'families', 'in', 'tooth', 'explant', 'cultures', 'revealed', 'that', 'the', 'expression', 'of', 'edar', 'was', 'induced', 'by', 'activinbetaA', ',', 'whereas', 'Wnt6', 'induced', 'ectodysplasin', 'expression', '.', 'In', 'mouse', ',', 'two', 'high', '-', 'affinity', 'binding', 'sites', 'with', 'an', 'apparent', 'dissociation', 'constant', '(', 'Kd', ')', 'of', '50', 'to', '100', 'nM', 'have', 'been', 'mapped', 'in', 'the', '5', \"'\", 'ETS', 'upstream', 'from', 'the', 'early', 'pre', '-', 'rRNA', 'processing', 'site', '.']\n",
            "83 83\n",
            "[',_0', 'whereas_1', 'Wnt6_2', 'induced_3', 'ectodysplasin_4', 'expression_5', '._6', 'In_7', 'mouse_8', ',_9', 'two_10', 'high_11', '-_12', 'affinity_13', 'binding_14', 'sites_15', 'with_16', 'an_17', 'apparent_18', 'dissociation_19', 'constant_20', '(_21', 'Kd_22', ')_23', 'of_24', '50_25', 'to_26', '100_27', 'nM_28', 'have_29', 'been_30', 'mapped_31', 'in_32', 'the_33', '5_34', \"'_35\", 'ETS_36', 'upstream_37', 'from_38', 'the_39', 'early_40', 'pre_41', '-_42', 'rRNA_43', 'processing_44', 'site_45', '._46', 'Single_47', 'oblique_48', '-_49', 'view_50', 'mammography_51', 'for_52', 'periodic_53', 'screening_54', 'for_55', 'breast_56', 'cancer_57', 'in_58', 'women_59', '._60', 'Radiative_61', 'corrections_62', 'to_63', 'pi_64', 'l2_65', 'decays_66', '._67', 'This_68', 'deletion_69', 'disrupts_70', 'the_71', 'PU_72', '._73', '1_74', 'Ets_75', 'domain_76', '._77', 'In_78', 'turn_79', ',_80', 'production_81', 'of_82']\n",
            "BAD:  23090\n",
            "rpws:      index       token label\n",
            "0   23173  sufficient     O\n",
            "1   23174     amounts     O\n",
            "2   23175          of     O\n",
            "3   23176        TraR     B\n",
            "4   23177  apparently     O\n",
            "..    ...         ...   ...\n",
            "66  23239          in     O\n",
            "67  23240     protein     O\n",
            "68  23241     storage     O\n",
            "69  23242    vacuoles     O\n",
            "70  23243          of     O\n",
            "\n",
            "[71 rows x 3 columns]\n",
            "Single oblique - view mammography for periodic screening for breast cancer in women . Radiative corrections to pi l2 decays . This deletion disrupts the PU . 1 Ets domain . In turn , production of sufficient amounts of TraR apparently is sensitive to a cellular function requiring RNase D . The predicted product exhibits 91 % amino acid identity to the murine voltage - gated potassium channel protein Kv1 .\n",
            "['Single', 'oblique', '-', 'view', 'mammography', 'for', 'periodic', 'screening', 'for', 'breast', 'cancer', 'in', 'women', '.', 'Radiative', 'corrections', 'to', 'pi', 'l2', 'decays', '.', 'This', 'deletion', 'disrupts', 'the', 'PU', '.', '1', 'Ets', 'domain', '.', 'In', 'turn', ',', 'production', 'of', 'sufficient', 'amounts', 'of', 'TraR', 'apparently', 'is', 'sensitive', 'to', 'a', 'cellular', 'function', 'requiring', 'RNase', 'D', '.', 'The', 'predicted', 'product', 'exhibits', '91', '%', 'amino', 'acid', 'identity', 'to', 'the', 'murine', 'voltage', '-', 'gated', 'potassium', 'channel', 'protein', 'Kv1', '.']\n",
            "71 71\n",
            "['sufficient_0', 'amounts_1', 'of_2', 'TraR_3', 'apparently_4', 'is_5', 'sensitive_6', 'to_7', 'a_8', 'cellular_9', 'function_10', 'requiring_11', 'RNase_12', 'D_13', '._14', 'The_15', 'predicted_16', 'product_17', 'exhibits_18', '91_19', '%_20', 'amino_21', 'acid_22', 'identity_23', 'to_24', 'the_25', 'murine_26', 'voltage_27', '-_28', 'gated_29', 'potassium_30', 'channel_31', 'protein_32', 'Kv1_33', '._34', '7_35', '(_36', 'Kcna7_37', ')_38', ',_39', 'which_40', 'plays_41', 'an_42', 'important_43', 'role_44', 'in_45', 'the_46', 'repolarization_47', 'of_48', 'cell_49', 'membranes_50', '._51', 'Phytohemagglutinin_52', '(_53', 'PHA_54', ')_55', ',_56', 'the_57', 'seed_58', 'lectin_59', 'of_60', 'the_61', 'common_62', 'bean_63', ',_64', 'accumulates_65', 'in_66', 'protein_67', 'storage_68', 'vacuoles_69', 'of_70']\n",
            "BAD:  23173\n",
            "rpws:      index       token label\n",
            "0   23244     storage     O\n",
            "1   23245  parenchyma     O\n",
            "2   23246       cells     O\n",
            "3   23247          in     O\n",
            "4   23248  cotyledons     O\n",
            "..    ...         ...   ...\n",
            "62  23306        free     O\n",
            "63  23307       fatty     O\n",
            "64  23308        acid     O\n",
            "65  23309           (     O\n",
            "66  23310         FFA     O\n",
            "\n",
            "[67 rows x 3 columns]\n",
            "7 ( Kcna7 ) , which plays an important role in the repolarization of cell membranes . Phytohemagglutinin ( PHA ) , the seed lectin of the common bean , accumulates in protein storage vacuoles of storage parenchyma cells in cotyledons . This dynamic nature may be relevant to the ability of E47 both to homodimerize and to heterodimerize with MyoD , Id , and Tal1 .\n",
            "['7', '(', 'Kcna7', ')', ',', 'which', 'plays', 'an', 'important', 'role', 'in', 'the', 'repolarization', 'of', 'cell', 'membranes', '.', 'Phytohemagglutinin', '(', 'PHA', ')', ',', 'the', 'seed', 'lectin', 'of', 'the', 'common', 'bean', ',', 'accumulates', 'in', 'protein', 'storage', 'vacuoles', 'of', 'storage', 'parenchyma', 'cells', 'in', 'cotyledons', '.', 'This', 'dynamic', 'nature', 'may', 'be', 'relevant', 'to', 'the', 'ability', 'of', 'E47', 'both', 'to', 'homodimerize', 'and', 'to', 'heterodimerize', 'with', 'MyoD', ',', 'Id', ',', 'and', 'Tal1', '.']\n",
            "67 67\n",
            "['storage_0', 'parenchyma_1', 'cells_2', 'in_3', 'cotyledons_4', '._5', 'This_6', 'dynamic_7', 'nature_8', 'may_9', 'be_10', 'relevant_11', 'to_12', 'the_13', 'ability_14', 'of_15', 'E47_16', 'both_17', 'to_18', 'homodimerize_19', 'and_20', 'to_21', 'heterodimerize_22', 'with_23', 'MyoD_24', ',_25', 'Id_26', ',_27', 'and_28', 'Tal1_29', '._30', 'Blood_31', 'samples_32', '(_33', '30_34', 'ml_35', ')_36', 'were_37', 'collected_38', 'via_39', 'an_40', 'indwelling_41', 'arm_42', 'catheter_43', 'at_44', 'rest_45', ',_46', 'and_47', 'at_48', 'minutes_49', '13_50', 'and_51', '28_52', 'of_53', 'exercise_54', 'for_55', 'determinations_56', 'of_57', 'plasma_58', 'EPI_59', ',_60', 'serum_61', 'free_62', 'fatty_63', 'acid_64', '(_65', 'FFA_66']\n",
            "BAD:  23244\n",
            "rpws:      index     token label\n",
            "0   23311         )     O\n",
            "1   23312         ,     O\n",
            "2   23313     serum     O\n",
            "3   23314  glycerol     O\n",
            "4   23315         (     O\n",
            "..    ...       ...   ...\n",
            "70  23381         (     O\n",
            "71  23382         -     O\n",
            "72  23383       125     O\n",
            "73  23384         /     O\n",
            "74  23385         +     O\n",
            "\n",
            "[75 rows x 3 columns]\n",
            "Blood samples ( 30 ml ) were collected via an indwelling arm catheter at rest , and at minutes 13 and 28 of exercise for determinations of plasma EPI , serum free fatty acid ( FFA ) , serum glycerol ( GLY ) , blood glucose ( GLU ) , and blood lactate ( LA ) concentrations . Among these , four patients had the AIDS syndrome , while 42 individuals were HIV carriers .\n",
            "['Blood', 'samples', '(', '30', 'ml', ')', 'were', 'collected', 'via', 'an', 'indwelling', 'arm', 'catheter', 'at', 'rest', ',', 'and', 'at', 'minutes', '13', 'and', '28', 'of', 'exercise', 'for', 'determinations', 'of', 'plasma', 'EPI', ',', 'serum', 'free', 'fatty', 'acid', '(', 'FFA', ')', ',', 'serum', 'glycerol', '(', 'GLY', ')', ',', 'blood', 'glucose', '(', 'GLU', ')', ',', 'and', 'blood', 'lactate', '(', 'LA', ')', 'concentrations', '.', 'Among', 'these', ',', 'four', 'patients', 'had', 'the', 'AIDS', 'syndrome', ',', 'while', '42', 'individuals', 'were', 'HIV', 'carriers', '.']\n",
            "75 75\n",
            "[')_0', ',_1', 'serum_2', 'glycerol_3', '(_4', 'GLY_5', ')_6', ',_7', 'blood_8', 'glucose_9', '(_10', 'GLU_11', ')_12', ',_13', 'and_14', 'blood_15', 'lactate_16', '(_17', 'LA_18', ')_19', 'concentrations_20', '._21', 'Among_22', 'these_23', ',_24', 'four_25', 'patients_26', 'had_27', 'the_28', 'AIDS_29', 'syndrome_30', ',_31', 'while_32', '42_33', 'individuals_34', 'were_35', 'HIV_36', 'carriers_37', '._38', 'Subjects_39', 'were_40', '73_41', 'male_42', 'and_43', 'female_44', 'employees_45', 'of_46', 'the_47', 'Xerox_48', 'Corporation_49', 'joining_50', 'a_51', 'newly_52', 'developed_53', 'health_54', 'fitness_55', 'program_56', '._57', 'Overexpression_58', 'of_59', 'p50_60', 'in_61', 'transient_62', 'cotransfection_63', 'studies_64', 'using_65', 'the_66', 'proximal_67', 'CRP_68', 'promoter_69', '(_70', '-_71', '125_72', '/_73', '+_74']\n",
            "BAD:  23311\n",
            "rpws:      index        token label\n",
            "0   23386            9     O\n",
            "1   23387            )     O\n",
            "2   23388       linked     O\n",
            "3   23389           to     O\n",
            "4   23390            a     O\n",
            "..    ...          ...   ...\n",
            "74  23460     promoter     O\n",
            "75  23461  controlling     O\n",
            "76  23462          LT1     B\n",
            "77  23463            /     O\n",
            "78  23464          LT2     B\n",
            "\n",
            "[79 rows x 3 columns]\n",
            "Subjects were 73 male and female employees of the Xerox Corporation joining a newly developed health fitness program . Overexpression of p50 in transient cotransfection studies using the proximal CRP promoter ( - 125 / + 9 ) linked to a luciferase reporter caused a 3 - fold increase of luciferase activity , while C / EBPbeta overexpression caused an 18 - fold increase ; simultaneous overexpression of both transcription factors increased luciferase activity approximately 600 - fold .\n",
            "['Subjects', 'were', '73', 'male', 'and', 'female', 'employees', 'of', 'the', 'Xerox', 'Corporation', 'joining', 'a', 'newly', 'developed', 'health', 'fitness', 'program', '.', 'Overexpression', 'of', 'p50', 'in', 'transient', 'cotransfection', 'studies', 'using', 'the', 'proximal', 'CRP', 'promoter', '(', '-', '125', '/', '+', '9', ')', 'linked', 'to', 'a', 'luciferase', 'reporter', 'caused', 'a', '3', '-', 'fold', 'increase', 'of', 'luciferase', 'activity', ',', 'while', 'C', '/', 'EBPbeta', 'overexpression', 'caused', 'an', '18', '-', 'fold', 'increase', ';', 'simultaneous', 'overexpression', 'of', 'both', 'transcription', 'factors', 'increased', 'luciferase', 'activity', 'approximately', '600', '-', 'fold', '.']\n",
            "79 79\n",
            "['9_0', ')_1', 'linked_2', 'to_3', 'a_4', 'luciferase_5', 'reporter_6', 'caused_7', 'a_8', '3_9', '-_10', 'fold_11', 'increase_12', 'of_13', 'luciferase_14', 'activity_15', ',_16', 'while_17', 'C_18', '/_19', 'EBPbeta_20', 'overexpression_21', 'caused_22', 'an_23', '18_24', '-_25', 'fold_26', 'increase_27', ';_28', 'simultaneous_29', 'overexpression_30', 'of_31', 'both_32', 'transcription_33', 'factors_34', 'increased_35', 'luciferase_36', 'activity_37', 'approximately_38', '600_39', '-_40', 'fold_41', '._42', 'Our_43', 'results_44', 'suggest_45', 'that_46', 'it_47', 'might_48', 'be_49', 'possible_50', 'to_51', 'augment_52', 'IN_53', 'function_54', 'in_55', 'vivo_56', 'through_57', 'a_58', 'heterologous_59', 'domain_60', '._61', 'While_62', 'differential_63', 'expression_64', 'of_65', 'the_66', 'two_67', 'transcripts_68', 'was_69', 'not_70', 'found_71', ',_72', 'the_73', 'promoter_74', 'controlling_75', 'LT1_76', '/_77', 'LT2_78']\n",
            "BAD:  23386\n",
            "rpws:      index          token label\n",
            "0   23465  transcription     O\n",
            "1   23466             is     O\n",
            "2   23467      regulated     O\n",
            "3   23468             in     O\n",
            "4   23469              a     O\n",
            "..    ...            ...   ...\n",
            "63  23528              (     O\n",
            "64  23529             NK     O\n",
            "65  23530              )     O\n",
            "66  23531           cell     O\n",
            "67  23532       activity     O\n",
            "\n",
            "[68 rows x 3 columns]\n",
            "Our results suggest that it might be possible to augment IN function in vivo through a heterologous domain . While differential expression of the two transcripts was not found , the promoter controlling LT1 / LT2 transcription is regulated in a cell cycle - dependent manner . Schottky - barrier heights of Ti and TiSi2 on n - type and p - type Si ( 100 ) .\n",
            "['Our', 'results', 'suggest', 'that', 'it', 'might', 'be', 'possible', 'to', 'augment', 'IN', 'function', 'in', 'vivo', 'through', 'a', 'heterologous', 'domain', '.', 'While', 'differential', 'expression', 'of', 'the', 'two', 'transcripts', 'was', 'not', 'found', ',', 'the', 'promoter', 'controlling', 'LT1', '/', 'LT2', 'transcription', 'is', 'regulated', 'in', 'a', 'cell', 'cycle', '-', 'dependent', 'manner', '.', 'Schottky', '-', 'barrier', 'heights', 'of', 'Ti', 'and', 'TiSi2', 'on', 'n', '-', 'type', 'and', 'p', '-', 'type', 'Si', '(', '100', ')', '.']\n",
            "68 68\n",
            "['transcription_0', 'is_1', 'regulated_2', 'in_3', 'a_4', 'cell_5', 'cycle_6', '-_7', 'dependent_8', 'manner_9', '._10', 'Schottky_11', '-_12', 'barrier_13', 'heights_14', 'of_15', 'Ti_16', 'and_17', 'TiSi2_18', 'on_19', 'n_20', '-_21', 'type_22', 'and_23', 'p_24', '-_25', 'type_26', 'Si_27', '(_28', '100_29', ')_30', '._31', 'A_32', 'survey_33', 'of_34', 'children_35', 'with_36', 'HBsAg_37', 'markers_38', 'related_39', 'to_40', 'their_41', 'parents_42', 'HBV_43', 'markers_44', '._45', 'Immunological_46', 'studies_47', 'also_48', 'failed_49', 'to_50', 'demonstrate_51', 'any_52', 'significant_53', 'change_54', 'except_55', 'for_56', 'a_57', 'significant_58', 'increase_59', 'of_60', 'natural_61', 'killer_62', '(_63', 'NK_64', ')_65', 'cell_66', 'activity_67']\n",
            "BAD:  23465\n",
            "rpws:      index            token label\n",
            "0   23533            after     O\n",
            "1   23534              IFN     B\n",
            "2   23535                -     I\n",
            "3   23536            gamma     I\n",
            "4   23537         infusion     O\n",
            "5   23538                .     O\n",
            "6   23539            Fetal     O\n",
            "7   23540             PaO2     O\n",
            "8   23541             fell     O\n",
            "9   23542         somewhat     O\n",
            "10  23543           during     O\n",
            "11  23544              the     O\n",
            "12  23545         recovery     O\n",
            "13  23546           stages     O\n",
            "14  23547               in     O\n",
            "15  23548             both     O\n",
            "16  23549              NIT     O\n",
            "17  23550              and     O\n",
            "18  23551          control     O\n",
            "19  23552           groups     O\n",
            "20  23553                .     O\n",
            "21  23554          Genetic     O\n",
            "22  23555         analyses     O\n",
            "23  23556         indicate     O\n",
            "24  23557             that     O\n",
            "25  23558             most     O\n",
            "26  23559               of     O\n",
            "27  23560              the     O\n",
            "28  23561         dominant     O\n",
            "29  23562          mutants     O\n",
            "30  23563              are     O\n",
            "31  23564              cis     O\n",
            "32  23565                -     O\n",
            "33  23566           acting     O\n",
            "34  23567              and     O\n",
            "35  23568             that     O\n",
            "36  23569              the     O\n",
            "37  23570        recessive     O\n",
            "38  23571          mutants     O\n",
            "39  23572           define     O\n",
            "40  23573                a     O\n",
            "41  23574          minimum     O\n",
            "42  23575               of     O\n",
            "43  23576            three     O\n",
            "44  23577  complementation     O\n",
            "45  23578           groups     O\n",
            "46  23579                ,     O\n",
            "47  23580       indicating     O\n",
            "48  23581             that     O\n",
            "49  23582          defects     O\n",
            "50  23583               in     O\n",
            "51  23584          several     O\n",
            "52  23585        different     O\n",
            "53  23586            genes     O\n",
            "54  23587              can     O\n",
            "55  23588          restore     O\n",
            "56  23589           higher     O\n",
            "A survey of children with HBsAg markers related to their parents HBV markers . Immunological studies also failed to demonstrate any significant change except for a significant increase of natural killer ( NK ) cell activity after IFN - gamma infusion . Fetal PaO2 fell somewhat during the recovery stages in both NIT and control groups .\n",
            "['A', 'survey', 'of', 'children', 'with', 'HBsAg', 'markers', 'related', 'to', 'their', 'parents', 'HBV', 'markers', '.', 'Immunological', 'studies', 'also', 'failed', 'to', 'demonstrate', 'any', 'significant', 'change', 'except', 'for', 'a', 'significant', 'increase', 'of', 'natural', 'killer', '(', 'NK', ')', 'cell', 'activity', 'after', 'IFN', '-', 'gamma', 'infusion', '.', 'Fetal', 'PaO2', 'fell', 'somewhat', 'during', 'the', 'recovery', 'stages', 'in', 'both', 'NIT', 'and', 'control', 'groups', '.']\n",
            "57 57\n",
            "['after_0', 'IFN_1', '-_2', 'gamma_3', 'infusion_4', '._5', 'Fetal_6', 'PaO2_7', 'fell_8', 'somewhat_9', 'during_10', 'the_11', 'recovery_12', 'stages_13', 'in_14', 'both_15', 'NIT_16', 'and_17', 'control_18', 'groups_19', '._20', 'Genetic_21', 'analyses_22', 'indicate_23', 'that_24', 'most_25', 'of_26', 'the_27', 'dominant_28', 'mutants_29', 'are_30', 'cis_31', '-_32', 'acting_33', 'and_34', 'that_35', 'the_36', 'recessive_37', 'mutants_38', 'define_39', 'a_40', 'minimum_41', 'of_42', 'three_43', 'complementation_44', 'groups_45', ',_46', 'indicating_47', 'that_48', 'defects_49', 'in_50', 'several_51', 'different_52', 'genes_53', 'can_54', 'restore_55', 'higher_56']\n",
            "BAD:  23533\n",
            "rpws:      index            token label\n",
            "0   23590           levels     O\n",
            "1   23591               of     O\n",
            "2   23592            HIS4C     B\n",
            "3   23593       expression     O\n",
            "4   23594                .     O\n",
            "5   23595              Rat     B\n",
            "6   23596      cholesterol     I\n",
            "7   23597             side     I\n",
            "8   23598                -     I\n",
            "9   23599            chain     I\n",
            "10  23600         cleavage     I\n",
            "11  23601       cytochrome     I\n",
            "12  23602                P     I\n",
            "13  23603                -     I\n",
            "14  23604              450     I\n",
            "15  23605                (     O\n",
            "16  23606                P     B\n",
            "17  23607                -     I\n",
            "18  23608           450scc     I\n",
            "19  23609                )     O\n",
            "20  23610             gene     O\n",
            "21  23611                .     O\n",
            "22  23612  Histopathologic     O\n",
            "23  23613          studies     O\n",
            "24  23614       displaying     O\n",
            "25  23615       spongiform     O\n",
            "26  23616          changes     O\n",
            "27  23617               in     O\n",
            "28  23618              the     O\n",
            "29  23619             gray     O\n",
            "30  23620           matter     O\n",
            "31  23621                ,     O\n",
            "32  23622         neuronal     O\n",
            "33  23623             loss     O\n",
            "34  23624                ,     O\n",
            "35  23625              and     O\n",
            "36  23626      atrogliosis     O\n",
            "37  23627        confirmed     O\n",
            "38  23628              the     O\n",
            "39  23629         clinical     O\n",
            "40  23630        diagnosis     O\n",
            "41  23631               of     O\n",
            "42  23632      Creutzfeldt     O\n",
            "43  23633                -     O\n",
            "44  23634            Jakob     O\n",
            "45  23635          disease     O\n",
            "46  23636                .     O\n",
            "47  23637              Pig     O\n",
            "48  23638              uPA     B\n",
            "49  23639         promoter     O\n",
            "50  23640                -     O\n",
            "51  23641              CAT     B\n",
            "52  23642       constructs     O\n",
            "53  23643             were     O\n",
            "54  23644             more     O\n",
            "55  23645           active     O\n",
            "56  23646             than     O\n",
            "57  23647            mouse     O\n",
            "Genetic analyses indicate that most of the dominant mutants are cis - acting and that the recessive mutants define a minimum of three complementation groups , indicating that defects in several different genes can restore higher levels of HIS4C expression . Rat cholesterol side - chain cleavage cytochrome P - 450 ( P - 450scc ) gene .\n",
            "['Genetic', 'analyses', 'indicate', 'that', 'most', 'of', 'the', 'dominant', 'mutants', 'are', 'cis', '-', 'acting', 'and', 'that', 'the', 'recessive', 'mutants', 'define', 'a', 'minimum', 'of', 'three', 'complementation', 'groups', ',', 'indicating', 'that', 'defects', 'in', 'several', 'different', 'genes', 'can', 'restore', 'higher', 'levels', 'of', 'HIS4C', 'expression', '.', 'Rat', 'cholesterol', 'side', '-', 'chain', 'cleavage', 'cytochrome', 'P', '-', '450', '(', 'P', '-', '450scc', ')', 'gene', '.']\n",
            "58 58\n",
            "['levels_0', 'of_1', 'HIS4C_2', 'expression_3', '._4', 'Rat_5', 'cholesterol_6', 'side_7', '-_8', 'chain_9', 'cleavage_10', 'cytochrome_11', 'P_12', '-_13', '450_14', '(_15', 'P_16', '-_17', '450scc_18', ')_19', 'gene_20', '._21', 'Histopathologic_22', 'studies_23', 'displaying_24', 'spongiform_25', 'changes_26', 'in_27', 'the_28', 'gray_29', 'matter_30', ',_31', 'neuronal_32', 'loss_33', ',_34', 'and_35', 'atrogliosis_36', 'confirmed_37', 'the_38', 'clinical_39', 'diagnosis_40', 'of_41', 'Creutzfeldt_42', '-_43', 'Jakob_44', 'disease_45', '._46', 'Pig_47', 'uPA_48', 'promoter_49', '-_50', 'CAT_51', 'constructs_52', 'were_53', 'more_54', 'active_55', 'than_56', 'mouse_57']\n",
            "BAD:  23590\n",
            "rpws:      index       token label\n",
            "0   23648  constructs     O\n",
            "1   23649          in     O\n",
            "2   23650        this     O\n",
            "3   23651       assay     O\n",
            "4   23652           .     O\n",
            "..    ...         ...   ...\n",
            "58  23706        mean     O\n",
            "59  23707      amount     O\n",
            "60  23708          of     O\n",
            "61  23709       blood     O\n",
            "62  23710        loss     O\n",
            "\n",
            "[63 rows x 3 columns]\n",
            "Histopathologic studies displaying spongiform changes in the gray matter , neuronal loss , and atrogliosis confirmed the clinical diagnosis of Creutzfeldt - Jakob disease . Pig uPA promoter - CAT constructs were more active than mouse constructs in this assay . Cloning the cDNA for a new human zinc finger protein defines a group of closely related Kruppel - like transcription factors .\n",
            "['Histopathologic', 'studies', 'displaying', 'spongiform', 'changes', 'in', 'the', 'gray', 'matter', ',', 'neuronal', 'loss', ',', 'and', 'atrogliosis', 'confirmed', 'the', 'clinical', 'diagnosis', 'of', 'Creutzfeldt', '-', 'Jakob', 'disease', '.', 'Pig', 'uPA', 'promoter', '-', 'CAT', 'constructs', 'were', 'more', 'active', 'than', 'mouse', 'constructs', 'in', 'this', 'assay', '.', 'Cloning', 'the', 'cDNA', 'for', 'a', 'new', 'human', 'zinc', 'finger', 'protein', 'defines', 'a', 'group', 'of', 'closely', 'related', 'Kruppel', '-', 'like', 'transcription', 'factors', '.']\n",
            "63 63\n",
            "['constructs_0', 'in_1', 'this_2', 'assay_3', '._4', 'Cloning_5', 'the_6', 'cDNA_7', 'for_8', 'a_9', 'new_10', 'human_11', 'zinc_12', 'finger_13', 'protein_14', 'defines_15', 'a_16', 'group_17', 'of_18', 'closely_19', 'related_20', 'Kruppel_21', '-_22', 'like_23', 'transcription_24', 'factors_25', '._26', 'By_27', 'computed_28', 'homology_29', 'search_30', ',_31', 'we_32', 'noticed_33', 'significant_34', 'similarities_35', 'between_36', 'US3_37', 'PK_38', 'and_39', 'p21_40', '-_41', 'activated_42', 'kinase_43', '(_44', 'PAK_45', ')_46', ',_47', 'which_48', 'is_49', 'activated_50', 'by_51', 'the_52', 'Cdc42_53', 'or_54', 'Rac_55', '._56', 'The_57', 'mean_58', 'amount_59', 'of_60', 'blood_61', 'loss_62']\n",
            "BAD:  23648\n",
            "rpws:      index      token label\n",
            "0   23711      prior     O\n",
            "1   23712         to     O\n",
            "2   23713       time     O\n",
            "3   23714         of     O\n",
            "4   23715  injection     O\n",
            "..    ...        ...   ...\n",
            "89  23800        the     O\n",
            "90  23801  consensus     O\n",
            "91  23802        ATP     O\n",
            "92  23803    binding     O\n",
            "93  23804        and     O\n",
            "\n",
            "[94 rows x 3 columns]\n",
            "By computed homology search , we noticed significant similarities between US3 PK and p21 - activated kinase ( PAK ) , which is activated by the Cdc42 or Rac . The mean amount of blood loss prior to time of injection was 4 . 5 units ( a range of 3 to 10 units ) . In response to acoustical stimulation the properties of response latency , discharge pattern , frequency tuning , binaural interaction , and habituation were examined to allow an appraisal of the differentiation of the MGB by electrophysiological means .\n",
            "['By', 'computed', 'homology', 'search', ',', 'we', 'noticed', 'significant', 'similarities', 'between', 'US3', 'PK', 'and', 'p21', '-', 'activated', 'kinase', '(', 'PAK', ')', ',', 'which', 'is', 'activated', 'by', 'the', 'Cdc42', 'or', 'Rac', '.', 'The', 'mean', 'amount', 'of', 'blood', 'loss', 'prior', 'to', 'time', 'of', 'injection', 'was', '4', '.', '5', 'units', '(', 'a', 'range', 'of', '3', 'to', '10', 'units', ')', '.', 'In', 'response', 'to', 'acoustical', 'stimulation', 'the', 'properties', 'of', 'response', 'latency', ',', 'discharge', 'pattern', ',', 'frequency', 'tuning', ',', 'binaural', 'interaction', ',', 'and', 'habituation', 'were', 'examined', 'to', 'allow', 'an', 'appraisal', 'of', 'the', 'differentiation', 'of', 'the', 'MGB', 'by', 'electrophysiological', 'means', '.']\n",
            "94 94\n",
            "['prior_0', 'to_1', 'time_2', 'of_3', 'injection_4', 'was_5', '4_6', '._7', '5_8', 'units_9', '(_10', 'a_11', 'range_12', 'of_13', '3_14', 'to_15', '10_16', 'units_17', ')_18', '._19', 'In_20', 'response_21', 'to_22', 'acoustical_23', 'stimulation_24', 'the_25', 'properties_26', 'of_27', 'response_28', 'latency_29', ',_30', 'discharge_31', 'pattern_32', ',_33', 'frequency_34', 'tuning_35', ',_36', 'binaural_37', 'interaction_38', ',_39', 'and_40', 'habituation_41', 'were_42', 'examined_43', 'to_44', 'allow_45', 'an_46', 'appraisal_47', 'of_48', 'the_49', 'differentiation_50', 'of_51', 'the_52', 'MGB_53', 'by_54', 'electrophysiological_55', 'means_56', '._57', 'These_58', '8_59', '-_60', 'oxoguanine_61', 'DNA_62', 'glycosylases_63', ',_64', 'hOgg1_65', '(_66', 'human_67', ')_68', 'and_69', 'mOgg1_70', '(_71', 'murine_72', ')_73', ',_74', 'are_75', 'homologous_76', 'to_77', 'each_78', 'other_79', 'and_80', 'to_81', 'yeast_82', 'Ogg1_83', '._84', 'The_85', 'CHED_86', 'protein_87', 'includes_88', 'the_89', 'consensus_90', 'ATP_91', 'binding_92', 'and_93']\n",
            "BAD:  23711\n",
            "rpws:      index            token label\n",
            "0   23805  phosphorylation     O\n",
            "1   23806          domains     O\n",
            "2   23807   characteristic     O\n",
            "3   23808               of     O\n",
            "4   23809          kinases     O\n",
            "..    ...              ...   ...\n",
            "65  23870              the     O\n",
            "66  23871           immune     O\n",
            "67  23872           system     O\n",
            "68  23873                .     O\n",
            "69  23874          However     O\n",
            "\n",
            "[70 rows x 3 columns]\n",
            "These 8 - oxoguanine DNA glycosylases , hOgg1 ( human ) and mOgg1 ( murine ) , are homologous to each other and to yeast Ogg1 . The CHED protein includes the consensus ATP binding and phosphorylation domains characteristic of kinases , displays 34 - 42 % identically aligned amino acid residues with other cdc2 - related kinases , and is considerably longer at its amino and carboxyl termini .\n",
            "['These', '8', '-', 'oxoguanine', 'DNA', 'glycosylases', ',', 'hOgg1', '(', 'human', ')', 'and', 'mOgg1', '(', 'murine', ')', ',', 'are', 'homologous', 'to', 'each', 'other', 'and', 'to', 'yeast', 'Ogg1', '.', 'The', 'CHED', 'protein', 'includes', 'the', 'consensus', 'ATP', 'binding', 'and', 'phosphorylation', 'domains', 'characteristic', 'of', 'kinases', ',', 'displays', '34', '-', '42', '%', 'identically', 'aligned', 'amino', 'acid', 'residues', 'with', 'other', 'cdc2', '-', 'related', 'kinases', ',', 'and', 'is', 'considerably', 'longer', 'at', 'its', 'amino', 'and', 'carboxyl', 'termini', '.']\n",
            "70 70\n",
            "['phosphorylation_0', 'domains_1', 'characteristic_2', 'of_3', 'kinases_4', ',_5', 'displays_6', '34_7', '-_8', '42_9', '%_10', 'identically_11', 'aligned_12', 'amino_13', 'acid_14', 'residues_15', 'with_16', 'other_17', 'cdc2_18', '-_19', 'related_20', 'kinases_21', ',_22', 'and_23', 'is_24', 'considerably_25', 'longer_26', 'at_27', 'its_28', 'amino_29', 'and_30', 'carboxyl_31', 'termini_32', '._33', 'CONCLUSIONS_34', ':_35', 'These_36', 'results_37', 'suggest_38', 'that_39', ',_40', 'by_41', 'interacting_42', 'with_43', 'cellular_44', 'transcription_45', 'factors_46', 'and_47', 'cofactors_48', ',_49', 'vIRF_50', '-_51', '2_52', 'may_53', 'modulate_54', 'the_55', 'expression_56', 'of_57', 'the_58', 'early_59', 'inflammatory_60', 'genes_61', 'and_62', 'potentially_63', 'deregulate_64', 'the_65', 'immune_66', 'system_67', '._68', 'However_69']\n",
            "BAD:  23805\n",
            "rpws:      index     token label\n",
            "0   23875         ,     O\n",
            "1   23876        in     O\n",
            "2   23877         7     O\n",
            "3   23878  patients     O\n",
            "4   23879      with     O\n",
            "..    ...       ...   ...\n",
            "56  23931       VP1     B\n",
            "57  23932         ,     O\n",
            "58  23933     which     O\n",
            "59  23934   contain     O\n",
            "60  23935      very     O\n",
            "\n",
            "[61 rows x 3 columns]\n",
            "CONCLUSIONS : These results suggest that , by interacting with cellular transcription factors and cofactors , vIRF - 2 may modulate the expression of the early inflammatory genes and potentially deregulate the immune system . However , in 7 patients with normal CT results the latter method showed areas of cerebral activity anatomically correlated with neurological signs or vascular lesions .\n",
            "['CONCLUSIONS', ':', 'These', 'results', 'suggest', 'that', ',', 'by', 'interacting', 'with', 'cellular', 'transcription', 'factors', 'and', 'cofactors', ',', 'vIRF', '-', '2', 'may', 'modulate', 'the', 'expression', 'of', 'the', 'early', 'inflammatory', 'genes', 'and', 'potentially', 'deregulate', 'the', 'immune', 'system', '.', 'However', ',', 'in', '7', 'patients', 'with', 'normal', 'CT', 'results', 'the', 'latter', 'method', 'showed', 'areas', 'of', 'cerebral', 'activity', 'anatomically', 'correlated', 'with', 'neurological', 'signs', 'or', 'vascular', 'lesions', '.']\n",
            "61 61\n",
            "[',_0', 'in_1', '7_2', 'patients_3', 'with_4', 'normal_5', 'CT_6', 'results_7', 'the_8', 'latter_9', 'method_10', 'showed_11', 'areas_12', 'of_13', 'cerebral_14', 'activity_15', 'anatomically_16', 'correlated_17', 'with_18', 'neurological_19', 'signs_20', 'or_21', 'vascular_22', 'lesions_23', '._24', 'We_25', 'report_26', 'that_27', 'Gcn5_28', ',_29', 'a_30', 'histone_31', 'H3_32', 'acetylase_33', ',_34', 'plays_35', 'a_36', 'central_37', 'role_38', 'in_39', 'initiation_40', 'of_41', 'meiosis_42', 'via_43', 'effects_44', 'on_45', 'IME2_46', 'expression_47', '._48', 'The_49', 'carboxy_50', '-_51', 'proximal_52', 'regions_53', 'of_54', 'the_55', 'VP1_56', ',_57', 'which_58', 'contain_59', 'very_60']\n",
            "BAD:  23875\n",
            "rpws:      index         token label\n",
            "0   23936           low     O\n",
            "1   23937         amino     O\n",
            "2   23938          acid     O\n",
            "3   23939      homology     O\n",
            "4   23940             ,     O\n",
            "5   23941     displayed     O\n",
            "6   23942      evidence     O\n",
            "7   23943            of     O\n",
            "8   23944  conservation     O\n",
            "9   23945            in     O\n",
            "10  23946    structural     O\n",
            "11  23947      features     O\n",
            "12  23948          such     O\n",
            "13  23949            as     O\n",
            "14  23950             a     O\n",
            "15  23951   hydrophilic     O\n",
            "16  23952             ,     O\n",
            "17  23953        highly     O\n",
            "18  23954         basic     O\n",
            "19  23955        domain     O\n",
            "20  23956             .     O\n",
            "21  23957    Interferon     B\n",
            "22  23958             -     I\n",
            "23  23959         alpha     I\n",
            "24  23960  inducibility     O\n",
            "25  23961            of     O\n",
            "26  23962         IFI16     B\n",
            "27  23963           may     O\n",
            "28  23964            be     O\n",
            "29  23965     regulated     O\n",
            "30  23966            by     O\n",
            "31  23967            an     O\n",
            "32  23968    interferon     B\n",
            "33  23969             -     I\n",
            "34  23970         alpha     I\n",
            "35  23971             /     I\n",
            "36  23972          beta     I\n",
            "37  23973             -     O\n",
            "38  23974    stimulated     O\n",
            "39  23975      response     O\n",
            "40  23976     consensus     O\n",
            "41  23977       element     O\n",
            "42  23978            in     O\n",
            "43  23979           the     O\n",
            "44  23980             5     O\n",
            "45  23981             '     O\n",
            "46  23982            UT     O\n",
            "47  23983          exon     O\n",
            "48  23984             ,     O\n",
            "49  23985            as     O\n",
            "50  23986             a     O\n",
            "51  23987       similar     O\n",
            "52  23988         motif     O\n",
            "53  23989            is     O\n",
            "54  23990     conserved     O\n",
            "55  23991            in     O\n",
            "56  23992           the     O\n",
            "We report that Gcn5 , a histone H3 acetylase , plays a central role in initiation of meiosis via effects on IME2 expression . The carboxy - proximal regions of the VP1 , which contain very low amino acid homology , displayed evidence of conservation in structural features such as a hydrophilic , highly basic domain .\n",
            "['We', 'report', 'that', 'Gcn5', ',', 'a', 'histone', 'H3', 'acetylase', ',', 'plays', 'a', 'central', 'role', 'in', 'initiation', 'of', 'meiosis', 'via', 'effects', 'on', 'IME2', 'expression', '.', 'The', 'carboxy', '-', 'proximal', 'regions', 'of', 'the', 'VP1', ',', 'which', 'contain', 'very', 'low', 'amino', 'acid', 'homology', ',', 'displayed', 'evidence', 'of', 'conservation', 'in', 'structural', 'features', 'such', 'as', 'a', 'hydrophilic', ',', 'highly', 'basic', 'domain', '.']\n",
            "57 57\n",
            "['low_0', 'amino_1', 'acid_2', 'homology_3', ',_4', 'displayed_5', 'evidence_6', 'of_7', 'conservation_8', 'in_9', 'structural_10', 'features_11', 'such_12', 'as_13', 'a_14', 'hydrophilic_15', ',_16', 'highly_17', 'basic_18', 'domain_19', '._20', 'Interferon_21', '-_22', 'alpha_23', 'inducibility_24', 'of_25', 'IFI16_26', 'may_27', 'be_28', 'regulated_29', 'by_30', 'an_31', 'interferon_32', '-_33', 'alpha_34', '/_35', 'beta_36', '-_37', 'stimulated_38', 'response_39', 'consensus_40', 'element_41', 'in_42', 'the_43', '5_44', \"'_45\", 'UT_46', 'exon_47', ',_48', 'as_49', 'a_50', 'similar_51', 'motif_52', 'is_53', 'conserved_54', 'in_55', 'the_56']\n",
            "BAD:  23936\n",
            "rpws:      index          token label\n",
            "0   23993  corresponding     O\n",
            "1   23994       position     O\n",
            "2   23995             in     O\n",
            "3   23996            the     O\n",
            "4   23997        related     O\n",
            "..    ...            ...   ...\n",
            "63  24056      potential     O\n",
            "64  24057             in     O\n",
            "65  24058       chickens     O\n",
            "66  24059              ,     O\n",
            "67  24060            and     O\n",
            "\n",
            "[68 rows x 3 columns]\n",
            "Interferon - alpha inducibility of IFI16 may be regulated by an interferon - alpha / beta - stimulated response consensus element in the 5 ' UT exon , as a similar motif is conserved in the corresponding position in the related myeloid cell nuclear differentiation antigen gene . ( ABSTRACT TRUNCATED AT 400 WORDS ) DESIGN - - A randomised double blind placebo controlled parallel arm trial .\n",
            "['Interferon', '-', 'alpha', 'inducibility', 'of', 'IFI16', 'may', 'be', 'regulated', 'by', 'an', 'interferon', '-', 'alpha', '/', 'beta', '-', 'stimulated', 'response', 'consensus', 'element', 'in', 'the', '5', \"'\", 'UT', 'exon', ',', 'as', 'a', 'similar', 'motif', 'is', 'conserved', 'in', 'the', 'corresponding', 'position', 'in', 'the', 'related', 'myeloid', 'cell', 'nuclear', 'differentiation', 'antigen', 'gene', '.', '(', 'ABSTRACT', 'TRUNCATED', 'AT', '400', 'WORDS', ')', 'DESIGN', '-', '-', 'A', 'randomised', 'double', 'blind', 'placebo', 'controlled', 'parallel', 'arm', 'trial', '.']\n",
            "68 68\n",
            "['corresponding_0', 'position_1', 'in_2', 'the_3', 'related_4', 'myeloid_5', 'cell_6', 'nuclear_7', 'differentiation_8', 'antigen_9', 'gene_10', '._11', '(_12', 'ABSTRACT_13', 'TRUNCATED_14', 'AT_15', '400_16', 'WORDS_17', ')_18', 'DESIGN_19', '-_20', '-_21', 'A_22', 'randomised_23', 'double_24', 'blind_25', 'placebo_26', 'controlled_27', 'parallel_28', 'arm_29', 'trial_30', '._31', 'Genealogical_32', 'analysis_33', 'suggested_34', 'mainly_35', 'a_36', 'mother_37', '-_38', 'to_39', '-_40', 'offspring_41', 'transmission_42', 'of_43', 'this_44', 'STLV_45', '-_46', '1_47', '._48', 'These_49', 'back_50', 'mutations_51', 'led_52', 'to_53', 'a_54', 'modest_55', 'decrease_56', 'in_57', 'kinase_58', 'activity_59', ',_60', 'decreased_61', 'tumorigenic_62', 'potential_63', 'in_64', 'chickens_65', ',_66', 'and_67']\n",
            "BAD:  23993\n",
            "rpws:      index         token label\n",
            "0   24061            an     O\n",
            "1   24062    unexpected     O\n",
            "2   24063      increase     O\n",
            "3   24064            in     O\n",
            "4   24065  transforming     O\n",
            "..    ...           ...   ...\n",
            "62  24123      feasible     O\n",
            "63  24124           and     O\n",
            "64  24125      reliable     O\n",
            "65  24126         under     O\n",
            "66  24127             a     O\n",
            "\n",
            "[67 rows x 3 columns]\n",
            "Genealogical analysis suggested mainly a mother - to - offspring transmission of this STLV - 1 . These back mutations led to a modest decrease in kinase activity , decreased tumorigenic potential in chickens , and an unexpected increase in transforming activity in rat cells . The quantification limit was 50 ng / ml for each of PHT , m - HPPH and p - HPPH .\n",
            "['Genealogical', 'analysis', 'suggested', 'mainly', 'a', 'mother', '-', 'to', '-', 'offspring', 'transmission', 'of', 'this', 'STLV', '-', '1', '.', 'These', 'back', 'mutations', 'led', 'to', 'a', 'modest', 'decrease', 'in', 'kinase', 'activity', ',', 'decreased', 'tumorigenic', 'potential', 'in', 'chickens', ',', 'and', 'an', 'unexpected', 'increase', 'in', 'transforming', 'activity', 'in', 'rat', 'cells', '.', 'The', 'quantification', 'limit', 'was', '50', 'ng', '/', 'ml', 'for', 'each', 'of', 'PHT', ',', 'm', '-', 'HPPH', 'and', 'p', '-', 'HPPH', '.']\n",
            "67 67\n",
            "['an_0', 'unexpected_1', 'increase_2', 'in_3', 'transforming_4', 'activity_5', 'in_6', 'rat_7', 'cells_8', '._9', 'The_10', 'quantification_11', 'limit_12', 'was_13', '50_14', 'ng_15', '/_16', 'ml_17', 'for_18', 'each_19', 'of_20', 'PHT_21', ',_22', 'm_23', '-_24', 'HPPH_25', 'and_26', 'p_27', '-_28', 'HPPH_29', '._30', 'Individual_31', 'intolerance_32', 'to_33', 'betaxolol_34', 'will_35', 'continue_36', 'to_37', 'occur_38', 'and_39', 'care_40', 'is_41', 'always_42', 'required_43', 'when_44', 'patients_45', 'with_46', 'cardiac_47', 'or_48', 'respiratory_49', 'dysfunction_50', 'are_51', 'exposed_52', 'to_53', 'any_54', 'beta_55', '-_56', 'antagonist_57', '._58', 'Pretoria_59', 'Pasteurisation_60', 'is_61', 'feasible_62', 'and_63', 'reliable_64', 'under_65', 'a_66']\n",
            "BAD:  24061\n",
            "rpws:      index         token label\n",
            "0   24128         range     O\n",
            "1   24129            of     O\n",
            "2   24130    conditions     O\n",
            "3   24131             .     O\n",
            "4   24132            In     O\n",
            "..    ...           ...   ...\n",
            "73  24201            in     O\n",
            "74  24202        guinea     O\n",
            "75  24203          pigs     O\n",
            "76  24204          were     O\n",
            "77  24205  investigated     O\n",
            "\n",
            "[78 rows x 3 columns]\n",
            "Individual intolerance to betaxolol will continue to occur and care is always required when patients with cardiac or respiratory dysfunction are exposed to any beta - antagonist . Pretoria Pasteurisation is feasible and reliable under a range of conditions . In the present study we use a tnaC - UGA - ' lacZ construct lacking the tnaC - tnaA spacer region to analyze the effect of TnaC synthesis on the behavior of the ribosome that translates tnaC .\n",
            "['Individual', 'intolerance', 'to', 'betaxolol', 'will', 'continue', 'to', 'occur', 'and', 'care', 'is', 'always', 'required', 'when', 'patients', 'with', 'cardiac', 'or', 'respiratory', 'dysfunction', 'are', 'exposed', 'to', 'any', 'beta', '-', 'antagonist', '.', 'Pretoria', 'Pasteurisation', 'is', 'feasible', 'and', 'reliable', 'under', 'a', 'range', 'of', 'conditions', '.', 'In', 'the', 'present', 'study', 'we', 'use', 'a', 'tnaC', '-', 'UGA', '-', \"'\", 'lacZ', 'construct', 'lacking', 'the', 'tnaC', '-', 'tnaA', 'spacer', 'region', 'to', 'analyze', 'the', 'effect', 'of', 'TnaC', 'synthesis', 'on', 'the', 'behavior', 'of', 'the', 'ribosome', 'that', 'translates', 'tnaC', '.']\n",
            "78 78\n",
            "['range_0', 'of_1', 'conditions_2', '._3', 'In_4', 'the_5', 'present_6', 'study_7', 'we_8', 'use_9', 'a_10', 'tnaC_11', '-_12', 'UGA_13', '-_14', \"'_15\", 'lacZ_16', 'construct_17', 'lacking_18', 'the_19', 'tnaC_20', '-_21', 'tnaA_22', 'spacer_23', 'region_24', 'to_25', 'analyze_26', 'the_27', 'effect_28', 'of_29', 'TnaC_30', 'synthesis_31', 'on_32', 'the_33', 'behavior_34', 'of_35', 'the_36', 'ribosome_37', 'that_38', 'translates_39', 'tnaC_40', '._41', 'The_42', 'in_43', 'vitro_44', 'activity_45', 'of_46', 'KP_47', '-_48', '103_49', ',_50', 'a_51', 'novel_52', 'triazole_53', 'derivative_54', ',_55', 'against_56', 'pathogenic_57', 'fungi_58', 'that_59', 'cause_60', 'dermatomycoses_61', 'and_62', 'its_63', 'therapeutic_64', 'efficacy_65', 'against_66', 'plantar_67', 'tinea_68', 'pedis_69', 'and_70', 'cutaneous_71', 'candidiasis_72', 'in_73', 'guinea_74', 'pigs_75', 'were_76', 'investigated_77']\n",
            "BAD:  24128\n",
            "rpws:      index        token label\n",
            "0   24206            .     O\n",
            "1   24207  Restriction     O\n",
            "2   24208     analysis     O\n",
            "3   24209           of     O\n",
            "4   24210          the     O\n",
            "..    ...          ...   ...\n",
            "76  24282            (     O\n",
            "77  24283            5     O\n",
            "78  24284            %     O\n",
            "79  24285            )     O\n",
            "80  24286           in     O\n",
            "\n",
            "[81 rows x 3 columns]\n",
            "The in vitro activity of KP - 103 , a novel triazole derivative , against pathogenic fungi that cause dermatomycoses and its therapeutic efficacy against plantar tinea pedis and cutaneous candidiasis in guinea pigs were investigated . Restriction analysis of the isolated genomic clones indicated that the endogenous sequences abutting the 3 ' ends of the 94 - A and 94 - K transgenes are separated by less than 20 kb , providing strong support for the single integration model .\n",
            "['The', 'in', 'vitro', 'activity', 'of', 'KP', '-', '103', ',', 'a', 'novel', 'triazole', 'derivative', ',', 'against', 'pathogenic', 'fungi', 'that', 'cause', 'dermatomycoses', 'and', 'its', 'therapeutic', 'efficacy', 'against', 'plantar', 'tinea', 'pedis', 'and', 'cutaneous', 'candidiasis', 'in', 'guinea', 'pigs', 'were', 'investigated', '.', 'Restriction', 'analysis', 'of', 'the', 'isolated', 'genomic', 'clones', 'indicated', 'that', 'the', 'endogenous', 'sequences', 'abutting', 'the', '3', \"'\", 'ends', 'of', 'the', '94', '-', 'A', 'and', '94', '-', 'K', 'transgenes', 'are', 'separated', 'by', 'less', 'than', '20', 'kb', ',', 'providing', 'strong', 'support', 'for', 'the', 'single', 'integration', 'model', '.']\n",
            "81 81\n",
            "['._0', 'Restriction_1', 'analysis_2', 'of_3', 'the_4', 'isolated_5', 'genomic_6', 'clones_7', 'indicated_8', 'that_9', 'the_10', 'endogenous_11', 'sequences_12', 'abutting_13', 'the_14', '3_15', \"'_16\", 'ends_17', 'of_18', 'the_19', '94_20', '-_21', 'A_22', 'and_23', '94_24', '-_25', 'K_26', 'transgenes_27', 'are_28', 'separated_29', 'by_30', 'less_31', 'than_32', '20_33', 'kb_34', ',_35', 'providing_36', 'strong_37', 'support_38', 'for_39', 'the_40', 'single_41', 'integration_42', 'model_43', '._44', 'Serum_45', 'triglycerides_46', 'also_47', 'showed_48', 'an_49', 'increase_50', 'from_51', 'birth_52', 'to_53', '6_54', 'months_55', 'of_56', 'age_57', ',_58', 'but_59', 'a_60', 'decrease_61', 'from_62', '6_63', 'months_64', 'to_65', '1_66', 'years_67', 'of_68', 'age_69', '._70', 'Female_71', 'Wistar_72', 'rats_73', 'receiving_74', 'alcohol_75', '(_76', '5_77', '%_78', ')_79', 'in_80']\n",
            "BAD:  24206\n",
            "rpws:      index       token label\n",
            "0   24287    drinking     O\n",
            "1   24288       water     O\n",
            "2   24289      during     O\n",
            "3   24290   lactation     O\n",
            "4   24291           (     O\n",
            "..    ...         ...   ...\n",
            "83  24370        that     O\n",
            "84  24371  conjugated     O\n",
            "85  24372   estrogens     O\n",
            "86  24373           ,     O\n",
            "87  24374       which     O\n",
            "\n",
            "[88 rows x 3 columns]\n",
            "Serum triglycerides also showed an increase from birth to 6 months of age , but a decrease from 6 months to 1 years of age . Female Wistar rats receiving alcohol ( 5 % ) in drinking water during lactation ( N = 7 ) were compared to normal controls fed ad libitum ( N = 6 ) . Antisense oligonucleotides complementary to the 5 ' end of PKC - zeta mRNA sequences significantly reduced the collagen lattice - stimulated alpha2 and MMP - 1 mRNA levels .\n",
            "['Serum', 'triglycerides', 'also', 'showed', 'an', 'increase', 'from', 'birth', 'to', '6', 'months', 'of', 'age', ',', 'but', 'a', 'decrease', 'from', '6', 'months', 'to', '1', 'years', 'of', 'age', '.', 'Female', 'Wistar', 'rats', 'receiving', 'alcohol', '(', '5', '%', ')', 'in', 'drinking', 'water', 'during', 'lactation', '(', 'N', '=', '7', ')', 'were', 'compared', 'to', 'normal', 'controls', 'fed', 'ad', 'libitum', '(', 'N', '=', '6', ')', '.', 'Antisense', 'oligonucleotides', 'complementary', 'to', 'the', '5', \"'\", 'end', 'of', 'PKC', '-', 'zeta', 'mRNA', 'sequences', 'significantly', 'reduced', 'the', 'collagen', 'lattice', '-', 'stimulated', 'alpha2', 'and', 'MMP', '-', '1', 'mRNA', 'levels', '.']\n",
            "88 88\n",
            "['drinking_0', 'water_1', 'during_2', 'lactation_3', '(_4', 'N_5', '=_6', '7_7', ')_8', 'were_9', 'compared_10', 'to_11', 'normal_12', 'controls_13', 'fed_14', 'ad_15', 'libitum_16', '(_17', 'N_18', '=_19', '6_20', ')_21', '._22', 'Antisense_23', 'oligonucleotides_24', 'complementary_25', 'to_26', 'the_27', '5_28', \"'_29\", 'end_30', 'of_31', 'PKC_32', '-_33', 'zeta_34', 'mRNA_35', 'sequences_36', 'significantly_37', 'reduced_38', 'the_39', 'collagen_40', 'lattice_41', '-_42', 'stimulated_43', 'alpha2_44', 'and_45', 'MMP_46', '-_47', '1_48', 'mRNA_49', 'levels_50', '._51', 'Overexpression_52', 'of_53', 'a_54', 'HIF_55', '-_56', '1alpha_57', 'construct_58', 'with_59', 'deletions_60', 'of_61', 'the_62', 'basic_63', 'domain_64', 'and_65', 'carboxy_66', 'terminus_67', 'blocked_68', 'reporter_69', 'gene_70', 'activation_71', 'by_72', 'endogenous_73', 'HIF_74', '-_75', '1_76', 'in_77', 'hypoxic_78', 'cells_79', '._80', 'We_81', 'hypothesized_82', 'that_83', 'conjugated_84', 'estrogens_85', ',_86', 'which_87']\n",
            "BAD:  24287\n",
            "rpws:      index          token label\n",
            "0   24375        contain     O\n",
            "1   24376        several     O\n",
            "2   24377     vasoactive     O\n",
            "3   24378     estrogenic     O\n",
            "4   24379      compounds     O\n",
            "5   24380              ,     O\n",
            "6   24381            may     O\n",
            "7   24382      favorably     O\n",
            "8   24383      influence     O\n",
            "9   24384            the     O\n",
            "10  24385      vasomotor     O\n",
            "11  24386       response     O\n",
            "12  24387             to     O\n",
            "13  24388  acetylcholine     O\n",
            "14  24389             in     O\n",
            "15  24390            men     O\n",
            "16  24391              .     O\n",
            "17  24392             In     O\n",
            "18  24393     Experiment     O\n",
            "19  24394              2     O\n",
            "20  24395              ,     O\n",
            "21  24396         levels     O\n",
            "22  24397           were     O\n",
            "23  24398              :     O\n",
            "24  24399      amprolium     O\n",
            "25  24400            and     O\n",
            "26  24401     ethopabate     O\n",
            "27  24402              ,     O\n",
            "28  24403              .     O\n",
            "29  24404             02     O\n",
            "30  24405              %     O\n",
            "31  24406              ;     O\n",
            "32  24407    salinomycin     O\n",
            "33  24408              ,     O\n",
            "34  24409             55     O\n",
            "35  24410             mg     O\n",
            "36  24411              /     O\n",
            "37  24412             kg     O\n",
            "38  24413              ;     O\n",
            "39  24414       monensin     O\n",
            "40  24415              ,     O\n",
            "41  24416             99     O\n",
            "42  24417             mg     O\n",
            "43  24418              /     O\n",
            "44  24419             kg     O\n",
            "45  24420              ;     O\n",
            "46  24421            and     O\n",
            "47  24422      lasalocid     O\n",
            "48  24423              ,     O\n",
            "49  24424            110     O\n",
            "50  24425             mg     O\n",
            "51  24426              /     O\n",
            "52  24427             kg     O\n",
            "Overexpression of a HIF - 1alpha construct with deletions of the basic domain and carboxy terminus blocked reporter gene activation by endogenous HIF - 1 in hypoxic cells . We hypothesized that conjugated estrogens , which contain several vasoactive estrogenic compounds , may favorably influence the vasomotor response to acetylcholine in men .\n",
            "['Overexpression', 'of', 'a', 'HIF', '-', '1alpha', 'construct', 'with', 'deletions', 'of', 'the', 'basic', 'domain', 'and', 'carboxy', 'terminus', 'blocked', 'reporter', 'gene', 'activation', 'by', 'endogenous', 'HIF', '-', '1', 'in', 'hypoxic', 'cells', '.', 'We', 'hypothesized', 'that', 'conjugated', 'estrogens', ',', 'which', 'contain', 'several', 'vasoactive', 'estrogenic', 'compounds', ',', 'may', 'favorably', 'influence', 'the', 'vasomotor', 'response', 'to', 'acetylcholine', 'in', 'men', '.']\n",
            "53 53\n",
            "['contain_0', 'several_1', 'vasoactive_2', 'estrogenic_3', 'compounds_4', ',_5', 'may_6', 'favorably_7', 'influence_8', 'the_9', 'vasomotor_10', 'response_11', 'to_12', 'acetylcholine_13', 'in_14', 'men_15', '._16', 'In_17', 'Experiment_18', '2_19', ',_20', 'levels_21', 'were_22', ':_23', 'amprolium_24', 'and_25', 'ethopabate_26', ',_27', '._28', '02_29', '%_30', ';_31', 'salinomycin_32', ',_33', '55_34', 'mg_35', '/_36', 'kg_37', ';_38', 'monensin_39', ',_40', '99_41', 'mg_42', '/_43', 'kg_44', ';_45', 'and_46', 'lasalocid_47', ',_48', '110_49', 'mg_50', '/_51', 'kg_52']\n",
            "BAD:  24375\n",
            "rpws:      index       token label\n",
            "0   24428           .     O\n",
            "1   24429    Residues     O\n",
            "2   24430     crucial     O\n",
            "3   24431         for     O\n",
            "4   24432         Ras     B\n",
            "..    ...         ...   ...\n",
            "75  24503    proteins     I\n",
            "76  24504           .     O\n",
            "77  24505       Under     O\n",
            "78  24506     control     O\n",
            "79  24507  conditions     O\n",
            "\n",
            "[80 rows x 3 columns]\n",
            "In Experiment 2 , levels were : amprolium and ethopabate , . 02 % ; salinomycin , 55 mg / kg ; monensin , 99 mg / kg ; and lasalocid , 110 mg / kg . Residues crucial for Ras interaction with GDP - GTP exchangers . 4 . In serum - starved NIH 3T3 cells , v - raf increased mdr1 promoter activity approximately 10 - fold compared to a v - raf frame - shift control .\n",
            "['In', 'Experiment', '2', ',', 'levels', 'were', ':', 'amprolium', 'and', 'ethopabate', ',', '.', '02', '%', ';', 'salinomycin', ',', '55', 'mg', '/', 'kg', ';', 'monensin', ',', '99', 'mg', '/', 'kg', ';', 'and', 'lasalocid', ',', '110', 'mg', '/', 'kg', '.', 'Residues', 'crucial', 'for', 'Ras', 'interaction', 'with', 'GDP', '-', 'GTP', 'exchangers', '.', '4', '.', 'In', 'serum', '-', 'starved', 'NIH', '3T3', 'cells', ',', 'v', '-', 'raf', 'increased', 'mdr1', 'promoter', 'activity', 'approximately', '10', '-', 'fold', 'compared', 'to', 'a', 'v', '-', 'raf', 'frame', '-', 'shift', 'control', '.']\n",
            "80 80\n",
            "['._0', 'Residues_1', 'crucial_2', 'for_3', 'Ras_4', 'interaction_5', 'with_6', 'GDP_7', '-_8', 'GTP_9', 'exchangers_10', '._11', '4_12', '._13', 'In_14', 'serum_15', '-_16', 'starved_17', 'NIH_18', '3T3_19', 'cells_20', ',_21', 'v_22', '-_23', 'raf_24', 'increased_25', 'mdr1_26', 'promoter_27', 'activity_28', 'approximately_29', '10_30', '-_31', 'fold_32', 'compared_33', 'to_34', 'a_35', 'v_36', '-_37', 'raf_38', 'frame_39', '-_40', 'shift_41', 'control_42', '._43', 'Searching_44', 'the_45', 'human_46', 'DNA_47', 'data_48', 'base_49', 'of_50', 'expressed_51', 'sequence_52', 'tags_53', '(_54', 'EST_55', ')_56', 'revealed_57', 'novel_58', 'partial_59', 'sequences_60', 'similar_61', 'to_62', ',_63', 'but_64', 'distinct_65', 'from_66', ',_67', 'the_68', 'sequences_69', 'of_70', 'the_71', 'previously_72', 'known_73', 'PEF_74', 'proteins_75', '._76', 'Under_77', 'control_78', 'conditions_79']\n",
            "BAD:  24428\n",
            "rpws:      index          token label\n",
            "0   24508            and     O\n",
            "1   24509             in     O\n",
            "2   24510            the     O\n",
            "3   24511       presence     O\n",
            "4   24512             of     O\n",
            "5   24513              3     O\n",
            "6   24514       micromol     O\n",
            "7   24515              /     O\n",
            "8   24516             kg     O\n",
            "9   24517            MIB     O\n",
            "10  24518            and     O\n",
            "11  24519            VER     O\n",
            "12  24520            the     O\n",
            "13  24521        maximal     O\n",
            "14  24522         effect     O\n",
            "15  24523             of     O\n",
            "16  24524  noradrenaline     O\n",
            "17  24525            was     O\n",
            "18  24526        reached     O\n",
            "19  24527             at     O\n",
            "20  24528              0     O\n",
            "21  24529              .     O\n",
            "22  24530              1     O\n",
            "23  24531       micromol     O\n",
            "24  24532              /     O\n",
            "25  24533             kg     O\n",
            "26  24534        whereas     O\n",
            "27  24535             in     O\n",
            "28  24536            the     O\n",
            "29  24537       presence     O\n",
            "30  24538             of     O\n",
            "31  24539             10     O\n",
            "32  24540       micromol     O\n",
            "33  24541              /     O\n",
            "34  24542             kg     O\n",
            "35  24543            MIB     O\n",
            "36  24544            and     O\n",
            "37  24545            VER     O\n",
            "38  24546             it     O\n",
            "39  24547            was     O\n",
            "40  24548        reached     O\n",
            "41  24549             at     O\n",
            "42  24550              a     O\n",
            "43  24551           dose     O\n",
            "44  24552             of     O\n",
            "45  24553              1     O\n",
            "46  24554       micromol     O\n",
            "47  24555              /     O\n",
            "48  24556             kg     O\n",
            "49  24557              .     O\n",
            "50  24558             It     O\n",
            "51  24559             is     O\n",
            "52  24560           also     O\n",
            "53  24561       affected     O\n",
            "54  24562             by     O\n",
            "55  24563            the     O\n",
            "56  24564           HAP2     B\n",
            "57  24565              /     I\n",
            "Searching the human DNA data base of expressed sequence tags ( EST ) revealed novel partial sequences similar to , but distinct from , the sequences of the previously known PEF proteins . Under control conditions and in the presence of 3 micromol / kg MIB and VER the maximal effect of noradrenaline was reached at 0 .\n",
            "['Searching', 'the', 'human', 'DNA', 'data', 'base', 'of', 'expressed', 'sequence', 'tags', '(', 'EST', ')', 'revealed', 'novel', 'partial', 'sequences', 'similar', 'to', ',', 'but', 'distinct', 'from', ',', 'the', 'sequences', 'of', 'the', 'previously', 'known', 'PEF', 'proteins', '.', 'Under', 'control', 'conditions', 'and', 'in', 'the', 'presence', 'of', '3', 'micromol', '/', 'kg', 'MIB', 'and', 'VER', 'the', 'maximal', 'effect', 'of', 'noradrenaline', 'was', 'reached', 'at', '0', '.']\n",
            "58 58\n",
            "['and_0', 'in_1', 'the_2', 'presence_3', 'of_4', '3_5', 'micromol_6', '/_7', 'kg_8', 'MIB_9', 'and_10', 'VER_11', 'the_12', 'maximal_13', 'effect_14', 'of_15', 'noradrenaline_16', 'was_17', 'reached_18', 'at_19', '0_20', '._21', '1_22', 'micromol_23', '/_24', 'kg_25', 'whereas_26', 'in_27', 'the_28', 'presence_29', 'of_30', '10_31', 'micromol_32', '/_33', 'kg_34', 'MIB_35', 'and_36', 'VER_37', 'it_38', 'was_39', 'reached_40', 'at_41', 'a_42', 'dose_43', 'of_44', '1_45', 'micromol_46', '/_47', 'kg_48', '._49', 'It_50', 'is_51', 'also_52', 'affected_53', 'by_54', 'the_55', 'HAP2_56', '/_57']\n",
            "BAD:  24508\n",
            "rpws:       index          token label\n",
            "0    24566              3     I\n",
            "1    24567              /     I\n",
            "2    24568              4     I\n",
            "3    24569  transcription     I\n",
            "4    24570         factor     I\n",
            "..     ...            ...   ...\n",
            "96   24662    luteinizing     B\n",
            "97   24663        hormone     I\n",
            "98   24664       receptor     I\n",
            "99   24665           gene     I\n",
            "100  24666              .     O\n",
            "\n",
            "[101 rows x 3 columns]\n",
            "1 micromol / kg whereas in the presence of 10 micromol / kg MIB and VER it was reached at a dose of 1 micromol / kg . It is also affected by the HAP2 / 3 / 4 transcription factor complex and by SNF1 and SSN6 . The effective use of superoxide dismutase from human erythrocytes in the late stages of experimental influenza infection Two different genes ( polA and polB ) encoding family B DNA polymerases were cloned from the organism by PCR using degenerated primers based on the two conserved motifs ( motif A and B ) .\n",
            "['1', 'micromol', '/', 'kg', 'whereas', 'in', 'the', 'presence', 'of', '10', 'micromol', '/', 'kg', 'MIB', 'and', 'VER', 'it', 'was', 'reached', 'at', 'a', 'dose', 'of', '1', 'micromol', '/', 'kg', '.', 'It', 'is', 'also', 'affected', 'by', 'the', 'HAP2', '/', '3', '/', '4', 'transcription', 'factor', 'complex', 'and', 'by', 'SNF1', 'and', 'SSN6', '.', 'The', 'effective', 'use', 'of', 'superoxide', 'dismutase', 'from', 'human', 'erythrocytes', 'in', 'the', 'late', 'stages', 'of', 'experimental', 'influenza', 'infection', 'Two', 'different', 'genes', '(', 'polA', 'and', 'polB', ')', 'encoding', 'family', 'B', 'DNA', 'polymerases', 'were', 'cloned', 'from', 'the', 'organism', 'by', 'PCR', 'using', 'degenerated', 'primers', 'based', 'on', 'the', 'two', 'conserved', 'motifs', '(', 'motif', 'A', 'and', 'B', ')', '.']\n",
            "101 101\n",
            "['3_0', '/_1', '4_2', 'transcription_3', 'factor_4', 'complex_5', 'and_6', 'by_7', 'SNF1_8', 'and_9', 'SSN6_10', '._11', 'The_12', 'effective_13', 'use_14', 'of_15', 'superoxide_16', 'dismutase_17', 'from_18', 'human_19', 'erythrocytes_20', 'in_21', 'the_22', 'late_23', 'stages_24', 'of_25', 'experimental_26', 'influenza_27', 'infection_28', 'Two_29', 'different_30', 'genes_31', '(_32', 'polA_33', 'and_34', 'polB_35', ')_36', 'encoding_37', 'family_38', 'B_39', 'DNA_40', 'polymerases_41', 'were_42', 'cloned_43', 'from_44', 'the_45', 'organism_46', 'by_47', 'PCR_48', 'using_49', 'degenerated_50', 'primers_51', 'based_52', 'on_53', 'the_54', 'two_55', 'conserved_56', 'motifs_57', '(_58', 'motif_59', 'A_60', 'and_61', 'B_62', ')_63', '._64', 'The_65', 'effects_66', 'on_67', 'survival_68', 'of_69', 'adjuvant_70', 'treatments_71', ',_72', 'including_73', 'pre_74', '-_75', 'or_76', 'postoperative_77', 'systemic_78', 'or_79', 'postoperative_80', 'intra_81', '-_82', 'arterial_83', 'chemotherapy_84', ',_85', 'are_86', 'currently_87', 'under_88', 'evaluation_89', '._90', 'Structure_91', 'and_92', 'regulation_93', 'of_94', 'the_95', 'luteinizing_96', 'hormone_97', 'receptor_98', 'gene_99', '._100']\n",
            "BAD:  24566\n",
            "rpws:      index             token label\n",
            "0   24667               The     O\n",
            "1   24668           disease     O\n",
            "2   24669         processes     O\n",
            "3   24670              that     O\n",
            "4   24671            affect     O\n",
            "5   24672        transplant     O\n",
            "6   24673          patients     O\n",
            "7   24674              both     O\n",
            "8   24675            before     O\n",
            "9   24676               and     O\n",
            "10  24677             after     O\n",
            "11  24678   transplantation     O\n",
            "12  24679               are     O\n",
            "13  24680               not     O\n",
            "14  24681              seen     O\n",
            "15  24682        frequently     O\n",
            "16  24683                in     O\n",
            "17  24684               the     O\n",
            "18  24685           general     O\n",
            "19  24686          practice     O\n",
            "20  24687                of     O\n",
            "21  24688  gastroenterology     O\n",
            "22  24689                 .     O\n",
            "23  24690           METHODS     O\n",
            "24  24691                 :     O\n",
            "25  24692            Single     O\n",
            "26  24693             white     O\n",
            "27  24694             flash     O\n",
            "28  24695               ERG     O\n",
            "29  24696                 ,     O\n",
            "30  24697          photopic     O\n",
            "31  24698               ERG     O\n",
            "32  24699                 ,     O\n",
            "33  24700          scotopic     O\n",
            "34  24701               ERG     O\n",
            "35  24702               and     O\n",
            "36  24703           flicker     O\n",
            "37  24704               ERG     O\n",
            "38  24705              were     O\n",
            "39  24706          recorded     O\n",
            "40  24707                in     O\n",
            "41  24708                30     O\n",
            "42  24709             cases     O\n",
            "43  24710                of     O\n",
            "44  24711        unilateral     O\n",
            "45  24712              CRVO     O\n",
            "46  24713                 .     O\n",
            "47  24714           However     O\n",
            "48  24715                 ,     O\n",
            "49  24716                to     O\n",
            "50  24717         encourage     O\n",
            "51  24718              good     O\n",
            "52  24719           contact     O\n",
            "53  24720           between     O\n",
            "54  24721               the     O\n",
            "55  24722           farmers     O\n",
            "56  24723               and     O\n",
            "57  24724               the     O\n",
            "58  24725      inseminating     O\n",
            "The effects on survival of adjuvant treatments , including pre - or postoperative systemic or postoperative intra - arterial chemotherapy , are currently under evaluation . Structure and regulation of the luteinizing hormone receptor gene . The disease processes that affect transplant patients both before and after transplantation are not seen frequently in the general practice of gastroenterology .\n",
            "['The', 'effects', 'on', 'survival', 'of', 'adjuvant', 'treatments', ',', 'including', 'pre', '-', 'or', 'postoperative', 'systemic', 'or', 'postoperative', 'intra', '-', 'arterial', 'chemotherapy', ',', 'are', 'currently', 'under', 'evaluation', '.', 'Structure', 'and', 'regulation', 'of', 'the', 'luteinizing', 'hormone', 'receptor', 'gene', '.', 'The', 'disease', 'processes', 'that', 'affect', 'transplant', 'patients', 'both', 'before', 'and', 'after', 'transplantation', 'are', 'not', 'seen', 'frequently', 'in', 'the', 'general', 'practice', 'of', 'gastroenterology', '.']\n",
            "59 59\n",
            "['The_0', 'disease_1', 'processes_2', 'that_3', 'affect_4', 'transplant_5', 'patients_6', 'both_7', 'before_8', 'and_9', 'after_10', 'transplantation_11', 'are_12', 'not_13', 'seen_14', 'frequently_15', 'in_16', 'the_17', 'general_18', 'practice_19', 'of_20', 'gastroenterology_21', '._22', 'METHODS_23', ':_24', 'Single_25', 'white_26', 'flash_27', 'ERG_28', ',_29', 'photopic_30', 'ERG_31', ',_32', 'scotopic_33', 'ERG_34', 'and_35', 'flicker_36', 'ERG_37', 'were_38', 'recorded_39', 'in_40', '30_41', 'cases_42', 'of_43', 'unilateral_44', 'CRVO_45', '._46', 'However_47', ',_48', 'to_49', 'encourage_50', 'good_51', 'contact_52', 'between_53', 'the_54', 'farmers_55', 'and_56', 'the_57', 'inseminating_58']\n",
            "rpws:      index       token label\n",
            "0   24726   personnel     O\n",
            "1   24727           ,     O\n",
            "2   24728          it     O\n",
            "3   24729          is     O\n",
            "4   24730  beneficial     O\n",
            "..    ...         ...   ...\n",
            "74  24800      tissue     O\n",
            "75  24801           .     O\n",
            "76  24802     Western     O\n",
            "77  24803  immunoblot     O\n",
            "78  24804    analysis     O\n",
            "\n",
            "[79 rows x 3 columns]\n",
            "METHODS : Single white flash ERG , photopic ERG , scotopic ERG and flicker ERG were recorded in 30 cases of unilateral CRVO . However , to encourage good contact between the farmers and the inseminating personnel , it is beneficial that herdsmen are present when cows are inseminated . Soft independent modelling of class analogy ( SIMCA ) is applied to identify near - infrared ( NIR ) spectra of ten excipients used in the pharmaceutical industry .\n",
            "['METHODS', ':', 'Single', 'white', 'flash', 'ERG', ',', 'photopic', 'ERG', ',', 'scotopic', 'ERG', 'and', 'flicker', 'ERG', 'were', 'recorded', 'in', '30', 'cases', 'of', 'unilateral', 'CRVO', '.', 'However', ',', 'to', 'encourage', 'good', 'contact', 'between', 'the', 'farmers', 'and', 'the', 'inseminating', 'personnel', ',', 'it', 'is', 'beneficial', 'that', 'herdsmen', 'are', 'present', 'when', 'cows', 'are', 'inseminated', '.', 'Soft', 'independent', 'modelling', 'of', 'class', 'analogy', '(', 'SIMCA', ')', 'is', 'applied', 'to', 'identify', 'near', '-', 'infrared', '(', 'NIR', ')', 'spectra', 'of', 'ten', 'excipients', 'used', 'in', 'the', 'pharmaceutical', 'industry', '.']\n",
            "79 79\n",
            "['personnel_0', ',_1', 'it_2', 'is_3', 'beneficial_4', 'that_5', 'herdsmen_6', 'are_7', 'present_8', 'when_9', 'cows_10', 'are_11', 'inseminated_12', '._13', 'Soft_14', 'independent_15', 'modelling_16', 'of_17', 'class_18', 'analogy_19', '(_20', 'SIMCA_21', ')_22', 'is_23', 'applied_24', 'to_25', 'identify_26', 'near_27', '-_28', 'infrared_29', '(_30', 'NIR_31', ')_32', 'spectra_33', 'of_34', 'ten_35', 'excipients_36', 'used_37', 'in_38', 'the_39', 'pharmaceutical_40', 'industry_41', '._42', 'Multiple_43', 'organ_44', 'failure_45', '(_46', 'MOF_47', ')_48', 'following_49', 'major_50', 'trauma_51', 'occurs_52', 'in_53', 'response_54', 'to_55', 'perfusion_56', 'deficits_57', ',_58', 'a_59', 'persistent_60', 'inflammatory_61', 'focus_62', ',_63', 'or_64', 'a_65', 'persistent_66', 'focus_67', 'of_68', 'dead_69', 'and_70', '/_71', 'or_72', 'injured_73', 'tissue_74', '._75', 'Western_76', 'immunoblot_77', 'analysis_78']\n",
            "BAD:  24726\n",
            "rpws:      index         token label\n",
            "0   24805      detected     O\n",
            "1   24806        p55gag     B\n",
            "2   24807           and     O\n",
            "3   24808           its     O\n",
            "4   24809      cleavage     O\n",
            "5   24810      products     O\n",
            "6   24811           p39     B\n",
            "7   24812           and     O\n",
            "8   24813           p27     B\n",
            "9   24814            in     O\n",
            "10  24815      purified     O\n",
            "11  24816     particles     O\n",
            "12  24817       derived     O\n",
            "13  24818            by     O\n",
            "14  24819    expression     O\n",
            "15  24820            of     O\n",
            "16  24821           gag     B\n",
            "17  24822           and     O\n",
            "18  24823           gag     B\n",
            "19  24824             -     O\n",
            "20  24825           pol     B\n",
            "21  24826             ,     O\n",
            "22  24827  respectively     O\n",
            "23  24828             .     O\n",
            "24  24829      Cefminox     O\n",
            "25  24830         shows     O\n",
            "26  24831       greater     O\n",
            "27  24832            in     O\n",
            "28  24833          vivo     O\n",
            "29  24834      activity     O\n",
            "30  24835          than     O\n",
            "31  24836          that     O\n",
            "32  24837      expected     O\n",
            "33  24838           for     O\n",
            "34  24839           the     O\n",
            "35  24840          MICs     O\n",
            "36  24841             ,     O\n",
            "37  24842     excellent     O\n",
            "38  24843      efficacy     O\n",
            "39  24844           and     O\n",
            "40  24845        safety     O\n",
            "41  24846             .     O\n",
            "42  24847    CONCLUSION     O\n",
            "43  24848             :     O\n",
            "44  24849          high     O\n",
            "45  24850    resolution     O\n",
            "46  24851            MF     O\n",
            "47  24852             -     O\n",
            "48  24853           ERG     O\n",
            "49  24854         seems     O\n",
            "50  24855          more     O\n",
            "51  24856     sensitive     O\n",
            "52  24857          than     O\n",
            "53  24858           low     O\n",
            "54  24859    resolution     O\n",
            "55  24860            MF     O\n",
            "56  24861             -     O\n",
            "57  24862           ERG     O\n",
            "58  24863             .     O\n",
            "59  24864             3     O\n",
            "Multiple organ failure ( MOF ) following major trauma occurs in response to perfusion deficits , a persistent inflammatory focus , or a persistent focus of dead and / or injured tissue . Western immunoblot analysis detected p55gag and its cleavage products p39 and p27 in purified particles derived by expression of gag and gag - pol , respectively .\n",
            "['Multiple', 'organ', 'failure', '(', 'MOF', ')', 'following', 'major', 'trauma', 'occurs', 'in', 'response', 'to', 'perfusion', 'deficits', ',', 'a', 'persistent', 'inflammatory', 'focus', ',', 'or', 'a', 'persistent', 'focus', 'of', 'dead', 'and', '/', 'or', 'injured', 'tissue', '.', 'Western', 'immunoblot', 'analysis', 'detected', 'p55gag', 'and', 'its', 'cleavage', 'products', 'p39', 'and', 'p27', 'in', 'purified', 'particles', 'derived', 'by', 'expression', 'of', 'gag', 'and', 'gag', '-', 'pol', ',', 'respectively', '.']\n",
            "60 60\n",
            "['detected_0', 'p55gag_1', 'and_2', 'its_3', 'cleavage_4', 'products_5', 'p39_6', 'and_7', 'p27_8', 'in_9', 'purified_10', 'particles_11', 'derived_12', 'by_13', 'expression_14', 'of_15', 'gag_16', 'and_17', 'gag_18', '-_19', 'pol_20', ',_21', 'respectively_22', '._23', 'Cefminox_24', 'shows_25', 'greater_26', 'in_27', 'vivo_28', 'activity_29', 'than_30', 'that_31', 'expected_32', 'for_33', 'the_34', 'MICs_35', ',_36', 'excellent_37', 'efficacy_38', 'and_39', 'safety_40', '._41', 'CONCLUSION_42', ':_43', 'high_44', 'resolution_45', 'MF_46', '-_47', 'ERG_48', 'seems_49', 'more_50', 'sensitive_51', 'than_52', 'low_53', 'resolution_54', 'MF_55', '-_56', 'ERG_57', '._58', '3_59']\n",
            "BAD:  24805\n",
            "rpws:      index               token label\n",
            "0   24865                   -     O\n",
            "1   24866                   5     O\n",
            "2   24867                   -     O\n",
            "3   24868                fold     O\n",
            "4   24869           increases     O\n",
            "5   24870                  in     O\n",
            "6   24871                 EGF     B\n",
            "7   24872            receptor     I\n",
            "8   24873                 but     O\n",
            "9   24874                 not     O\n",
            "10  24875                p185     B\n",
            "11  24876                   (     O\n",
            "12  24877                 neu     B\n",
            "13  24878                   )     O\n",
            "14  24879            tyrosine     O\n",
            "15  24880     phosphorylation     O\n",
            "16  24881               occur     O\n",
            "17  24882           following     O\n",
            "18  24883                  Gi     B\n",
            "19  24884                   -     I\n",
            "20  24885             coupled     I\n",
            "21  24886            receptor     I\n",
            "22  24887         stimulation     O\n",
            "23  24888                   .     O\n",
            "24  24889                Next     O\n",
            "25  24890                   ,     O\n",
            "26  24891                  to     O\n",
            "27  24892             examine     O\n",
            "28  24893                 the     O\n",
            "29  24894          regulation     O\n",
            "30  24895           mechanism     O\n",
            "31  24896                  of     O\n",
            "32  24897                CD44     B\n",
            "33  24898                   /     O\n",
            "34  24899                 ERM     B\n",
            "35  24900         interaction     O\n",
            "36  24901                  in     O\n",
            "37  24902                vivo     O\n",
            "38  24903                   ,     O\n",
            "39  24904                  we     O\n",
            "40  24905          reexamined     O\n",
            "41  24906                 the     O\n",
            "42  24907  immunoprecipitated     O\n",
            "43  24908                CD44     B\n",
            "44  24909                   /     O\n",
            "45  24910                 ERM     B\n",
            "46  24911             complex     O\n",
            "47  24912                from     O\n",
            "48  24913                 BHK     O\n",
            "49  24914               cells     O\n",
            "50  24915                 and     O\n",
            "51  24916               found     O\n",
            "52  24917                that     O\n",
            "53  24918                  it     O\n",
            "54  24919            contains     O\n",
            "55  24920                 Rho     B\n",
            "56  24921                   -     I\n",
            "57  24922                 GDP     I\n",
            "58  24923        dissociation     I\n",
            "59  24924           inhibitor     I\n",
            "Cefminox shows greater in vivo activity than that expected for the MICs , excellent efficacy and safety . CONCLUSION : high resolution MF - ERG seems more sensitive than low resolution MF - ERG . 3 - 5 - fold increases in EGF receptor but not p185 ( neu ) tyrosine phosphorylation occur following Gi - coupled receptor stimulation .\n",
            "['Cefminox', 'shows', 'greater', 'in', 'vivo', 'activity', 'than', 'that', 'expected', 'for', 'the', 'MICs', ',', 'excellent', 'efficacy', 'and', 'safety', '.', 'CONCLUSION', ':', 'high', 'resolution', 'MF', '-', 'ERG', 'seems', 'more', 'sensitive', 'than', 'low', 'resolution', 'MF', '-', 'ERG', '.', '3', '-', '5', '-', 'fold', 'increases', 'in', 'EGF', 'receptor', 'but', 'not', 'p185', '(', 'neu', ')', 'tyrosine', 'phosphorylation', 'occur', 'following', 'Gi', '-', 'coupled', 'receptor', 'stimulation', '.']\n",
            "60 60\n",
            "['-_0', '5_1', '-_2', 'fold_3', 'increases_4', 'in_5', 'EGF_6', 'receptor_7', 'but_8', 'not_9', 'p185_10', '(_11', 'neu_12', ')_13', 'tyrosine_14', 'phosphorylation_15', 'occur_16', 'following_17', 'Gi_18', '-_19', 'coupled_20', 'receptor_21', 'stimulation_22', '._23', 'Next_24', ',_25', 'to_26', 'examine_27', 'the_28', 'regulation_29', 'mechanism_30', 'of_31', 'CD44_32', '/_33', 'ERM_34', 'interaction_35', 'in_36', 'vivo_37', ',_38', 'we_39', 'reexamined_40', 'the_41', 'immunoprecipitated_42', 'CD44_43', '/_44', 'ERM_45', 'complex_46', 'from_47', 'BHK_48', 'cells_49', 'and_50', 'found_51', 'that_52', 'it_53', 'contains_54', 'Rho_55', '-_56', 'GDP_57', 'dissociation_58', 'inhibitor_59']\n",
            "BAD:  24865\n",
            "rpws:      index            token label\n",
            "0   24925                (     O\n",
            "1   24926              GDI     B\n",
            "2   24927                )     O\n",
            "3   24928                ,     O\n",
            "4   24929                a     O\n",
            "5   24930        regulator     O\n",
            "6   24931               of     O\n",
            "7   24932              Rho     B\n",
            "8   24933           GTPase     I\n",
            "9   24934                .     O\n",
            "10  24935              The     O\n",
            "11  24936          results     O\n",
            "12  24937           showed     O\n",
            "13  24938                a     O\n",
            "14  24939      significant     O\n",
            "15  24940            group     O\n",
            "16  24941           effect     O\n",
            "17  24942             only     O\n",
            "18  24943               on     O\n",
            "19  24944              the     O\n",
            "20  24945           active     O\n",
            "21  24946             task     O\n",
            "22  24947                .     O\n",
            "23  24948               On     O\n",
            "24  24949             days     O\n",
            "25  24950                7     O\n",
            "26  24951                ,     O\n",
            "27  24952               21     O\n",
            "28  24953                ,     O\n",
            "29  24954              and     O\n",
            "30  24955               35     O\n",
            "31  24956  postinoculation     O\n",
            "32  24957                (     O\n",
            "33  24958               PI     O\n",
            "34  24959                )     O\n",
            "35  24960                ,     O\n",
            "36  24961              all     O\n",
            "37  24962            birds     O\n",
            "38  24963         received     O\n",
            "39  24964            sheep     O\n",
            "40  24965     erythrocytes     O\n",
            "41  24966    intravenously     O\n",
            "42  24967                .     O\n",
            "43  24968               To     O\n",
            "44  24969        determine     O\n",
            "45  24970              cis     O\n",
            "46  24971                -     O\n",
            "47  24972           acting     O\n",
            "48  24973         elements     O\n",
            "49  24974      controlling     O\n",
            "50  24975              the     O\n",
            "51  24976              rat     O\n",
            "52  24977                B     B\n",
            "53  24978                -     I\n",
            "54  24979               50     I\n",
            "55  24980                /     O\n",
            "56  24981              GAP     B\n",
            "57  24982                -     I\n",
            "58  24983               43     I\n",
            "Next , to examine the regulation mechanism of CD44 / ERM interaction in vivo , we reexamined the immunoprecipitated CD44 / ERM complex from BHK cells and found that it contains Rho - GDP dissociation inhibitor ( GDI ) , a regulator of Rho GTPase . The results showed a significant group effect only on the active task .\n",
            "['Next', ',', 'to', 'examine', 'the', 'regulation', 'mechanism', 'of', 'CD44', '/', 'ERM', 'interaction', 'in', 'vivo', ',', 'we', 'reexamined', 'the', 'immunoprecipitated', 'CD44', '/', 'ERM', 'complex', 'from', 'BHK', 'cells', 'and', 'found', 'that', 'it', 'contains', 'Rho', '-', 'GDP', 'dissociation', 'inhibitor', '(', 'GDI', ')', ',', 'a', 'regulator', 'of', 'Rho', 'GTPase', '.', 'The', 'results', 'showed', 'a', 'significant', 'group', 'effect', 'only', 'on', 'the', 'active', 'task', '.']\n",
            "59 59\n",
            "['(_0', 'GDI_1', ')_2', ',_3', 'a_4', 'regulator_5', 'of_6', 'Rho_7', 'GTPase_8', '._9', 'The_10', 'results_11', 'showed_12', 'a_13', 'significant_14', 'group_15', 'effect_16', 'only_17', 'on_18', 'the_19', 'active_20', 'task_21', '._22', 'On_23', 'days_24', '7_25', ',_26', '21_27', ',_28', 'and_29', '35_30', 'postinoculation_31', '(_32', 'PI_33', ')_34', ',_35', 'all_36', 'birds_37', 'received_38', 'sheep_39', 'erythrocytes_40', 'intravenously_41', '._42', 'To_43', 'determine_44', 'cis_45', '-_46', 'acting_47', 'elements_48', 'controlling_49', 'the_50', 'rat_51', 'B_52', '-_53', '50_54', '/_55', 'GAP_56', '-_57', '43_58']\n",
            "BAD:  24925\n",
            "rpws:       index       token label\n",
            "0    24984        gene     O\n",
            "1    24985  expression     O\n",
            "2    24986           ,     O\n",
            "3    24987         the     O\n",
            "4    24988     genomic     O\n",
            "..     ...         ...   ...\n",
            "108  25092       eight     O\n",
            "109  25093     introns     O\n",
            "110  25094           .     O\n",
            "111  25095         TNF     B\n",
            "112  25096           -     I\n",
            "\n",
            "[113 rows x 3 columns]\n"
          ]
        },
        {
          "output_type": "error",
          "ename": "KeyboardInterrupt",
          "evalue": "ignored",
          "traceback": [
            "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
            "\u001b[0;31mKeyboardInterrupt\u001b[0m                         Traceback (most recent call last)",
            "\u001b[0;32m<ipython-input-38-e6cbc716baa1>\u001b[0m in \u001b[0;36m<cell line: 6>\u001b[0;34m()\u001b[0m\n\u001b[1;32m     11\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     12\u001b[0m     \u001b[0mtokens\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m[\u001b[0m\u001b[0mstr\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mrow\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m'token'\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m+\u001b[0m \u001b[0;34m'_'\u001b[0m \u001b[0;34m+\u001b[0m \u001b[0mstr\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mi\u001b[0m\u001b[0;34m)\u001b[0m \u001b[0;32mfor\u001b[0m \u001b[0mi\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mrow\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mrows\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0miterrows\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 13\u001b[0;31m     \u001b[0mlabels\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m[\u001b[0m\u001b[0mrow\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m'label'\u001b[0m\u001b[0;34m]\u001b[0m \u001b[0;32mfor\u001b[0m \u001b[0m_\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mrow\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mrows\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0miterrows\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     14\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     15\u001b[0m     \u001b[0mTEST_SENTENCE_LABELS\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mappend\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mdict\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mzip\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mtokens\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mlabels\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
            "\u001b[0;32m<ipython-input-38-e6cbc716baa1>\u001b[0m in \u001b[0;36m<listcomp>\u001b[0;34m(.0)\u001b[0m\n\u001b[1;32m     11\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     12\u001b[0m     \u001b[0mtokens\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m[\u001b[0m\u001b[0mstr\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mrow\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m'token'\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m+\u001b[0m \u001b[0;34m'_'\u001b[0m \u001b[0;34m+\u001b[0m \u001b[0mstr\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mi\u001b[0m\u001b[0;34m)\u001b[0m \u001b[0;32mfor\u001b[0m \u001b[0mi\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mrow\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mrows\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0miterrows\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 13\u001b[0;31m     \u001b[0mlabels\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m[\u001b[0m\u001b[0mrow\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m'label'\u001b[0m\u001b[0;34m]\u001b[0m \u001b[0;32mfor\u001b[0m \u001b[0m_\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mrow\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mrows\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0miterrows\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     14\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     15\u001b[0m     \u001b[0mTEST_SENTENCE_LABELS\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mappend\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mdict\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mzip\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mtokens\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mlabels\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
            "\u001b[0;32m/usr/local/lib/python3.10/dist-packages/pandas/core/frame.py\u001b[0m in \u001b[0;36miterrows\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m   1409\u001b[0m         \u001b[0mklass\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_constructor_sliced\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1410\u001b[0m         \u001b[0;32mfor\u001b[0m \u001b[0mk\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mv\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mzip\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mindex\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mvalues\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m-> 1411\u001b[0;31m             \u001b[0ms\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mklass\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mv\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mindex\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mcolumns\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mname\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mk\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m__finalize__\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m   1412\u001b[0m             \u001b[0;32myield\u001b[0m \u001b[0mk\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0ms\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1413\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
            "\u001b[0;32m/usr/local/lib/python3.10/dist-packages/pandas/core/series.py\u001b[0m in \u001b[0;36m__init__\u001b[0;34m(self, data, index, dtype, name, copy, fastpath)\u001b[0m\n\u001b[1;32m    468\u001b[0m                     \u001b[0mdata\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mdata\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mcopy\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    469\u001b[0m             \u001b[0;32melse\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 470\u001b[0;31m                 \u001b[0mdata\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0msanitize_array\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mdata\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mindex\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mdtype\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mcopy\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    471\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    472\u001b[0m                 \u001b[0mmanager\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mget_option\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"mode.data_manager\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
            "\u001b[0;32m/usr/local/lib/python3.10/dist-packages/pandas/core/construction.py\u001b[0m in \u001b[0;36msanitize_array\u001b[0;34m(data, index, dtype, copy, raise_cast_failure, allow_2d)\u001b[0m\n\u001b[1;32m    595\u001b[0m         \u001b[0;32melse\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    596\u001b[0m             \u001b[0;31m# we will try to copy by-definition here\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 597\u001b[0;31m             \u001b[0msubarr\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0m_try_cast\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mdata\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mdtype\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mcopy\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mraise_cast_failure\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    598\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    599\u001b[0m     \u001b[0;32melif\u001b[0m \u001b[0misinstance\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mdata\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mABCExtensionArray\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
            "\u001b[0;32m/usr/local/lib/python3.10/dist-packages/pandas/core/construction.py\u001b[0m in \u001b[0;36m_try_cast\u001b[0;34m(arr, dtype, copy, raise_cast_failure)\u001b[0m\n\u001b[1;32m    775\u001b[0m                 \u001b[0;32mreturn\u001b[0m \u001b[0msanitize_to_nanoseconds\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0marr\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mcopy\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mcopy\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    776\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 777\u001b[0;31m             \u001b[0mout\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mmaybe_infer_to_datetimelike\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0marr\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    778\u001b[0m             \u001b[0;32mif\u001b[0m \u001b[0mout\u001b[0m \u001b[0;32mis\u001b[0m \u001b[0marr\u001b[0m \u001b[0;32mand\u001b[0m \u001b[0mcopy\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    779\u001b[0m                 \u001b[0mout\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mout\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mcopy\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
            "\u001b[0;32m/usr/local/lib/python3.10/dist-packages/pandas/core/dtypes/cast.py\u001b[0m in \u001b[0;36mmaybe_infer_to_datetimelike\u001b[0;34m(value)\u001b[0m\n\u001b[1;32m   1206\u001b[0m         \u001b[0mv\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mv\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mravel\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1207\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m-> 1208\u001b[0;31m     \u001b[0;32mif\u001b[0m \u001b[0;32mnot\u001b[0m \u001b[0mlen\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mv\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m   1209\u001b[0m         \u001b[0;32mreturn\u001b[0m \u001b[0mvalue\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1210\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
            "\u001b[0;31mKeyboardInterrupt\u001b[0m: "
          ]
        }
      ],
      "source": [
        "n = len(test_df)\n",
        "i = 0\n",
        "\n",
        "TEST_SENTENCE_LABELS = []\n",
        "\n",
        "while i < n:\n",
        "  for chunk in TEST_SENTENCE_CHUNKS:\n",
        "    num_tokens = len(chunk.split())\n",
        "    rows = test_df.iloc[i:i+num_tokens].reset_index()\n",
        "    print('rpws: ', rows)\n",
        "\n",
        "    tokens = [str(row['token'])+ '_' + str(i) for i, row in rows.iterrows()]\n",
        "    labels = [row['label'] for _, row in rows.iterrows()]\n",
        "\n",
        "    TEST_SENTENCE_LABELS.append(dict(zip(tokens, labels)))\n",
        "\n",
        "    print(chunk)\n",
        "    print(chunk.split())\n",
        "    print(len(chunk.split()), len(tokens))\n",
        "    print(tokens)\n",
        "    if tokens[0][0] != chunk[0]:\n",
        "      print('BAD: ', i)\n",
        "\n",
        "    i += num_tokens"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "w826O6VJj4Cl"
      },
      "source": [
        "## NOTE: Play around until you find a test sentence you think is strong and use that as labeled example."
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "icAZGt4djRdP",
        "outputId": "3037f465-f41b-49ba-c2d7-cefc79444ae4"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Quinine is known to block voltage - , calcium - and ATP - sensitive K ( + ) - channels while 4 - aminopyridine is known to block voltage - sensitive K ( + ) - channels . 2 . In the counterbalanced method , quinine attenuated morphine - induced place preference , whereas 4 - aminopyridine was ineffective .\n",
            "\n",
            "Quinine B\n",
            "is O\n",
            "known O\n",
            "to O\n",
            "block O\n",
            "voltage O\n",
            "- O\n",
            ", O\n",
            "calcium B\n",
            "- O\n",
            "and O\n",
            "ATP B\n",
            "- O\n",
            "sensitive O\n",
            "K B\n",
            "( O\n",
            "+ O\n",
            ") O\n",
            "- O\n",
            "channels O\n",
            "while O\n",
            "4 B\n",
            "- I\n",
            "aminopyridine I\n",
            "is O\n",
            "known O\n",
            "to O\n",
            "block O\n",
            "voltage O\n",
            "- O\n",
            "sensitive O\n",
            "K B\n",
            "( O\n",
            "+ O\n",
            ") O\n",
            "- O\n",
            "channels O\n",
            ". O\n",
            "2 O\n",
            ". O\n",
            "In O\n",
            "the O\n",
            "counterbalanced O\n",
            "method O\n",
            ", O\n",
            "quinine B\n",
            "attenuated O\n",
            "morphine B\n",
            "- O\n",
            "induced O\n",
            "place O\n",
            "preference O\n",
            ", O\n",
            "whereas O\n",
            "4 B\n",
            "- I\n",
            "aminopyridine I\n",
            "was O\n",
            "ineffective O\n",
            ". O\n"
          ]
        }
      ],
      "source": [
        "i = 10\n",
        "\n",
        "print(TEST_SENTENCE_CHUNKS[i])\n",
        "print()\n",
        "for x,y in TEST_SENTENCE_LABELS[i].items():\n",
        "  print(x.split('_')[0],y)"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "xV6QHkW0hlAg"
      },
      "outputs": [],
      "source": [
        "example_text = TEST_SENTENCE_CHUNKS[i]\n",
        "example_response = TEST_SENTENCE_LABELS[i]"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "T1IBKwUsR-2H",
        "outputId": "af60cdab-72ac-4ab1-aa63-61194bfac87b"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Given a biomedical text, perform Named Entity Recognition analysis on this text, focusing on identifying ONLY CHEMICAL ENTITIES. Individual tokens are separated by white space and must be given exactly one label: ‘B’,  ‘I’, or ‘O’, according to the BIO labeling schema. Output a JSON object with key equal to token concatenated with '_'+(index of token) and value equal to label, where (index of token) starts from 0 for the first token, 1 for the second token, and so on. Here is a labeled example to help with this task:\n",
            "\n",
            "Example Text:\n",
            "Quinine is known to block voltage - , calcium - and ATP - sensitive K ( + ) - channels while 4 - aminopyridine is known to block voltage - sensitive K ( + ) - channels . 2 . In the counterbalanced method , quinine attenuated morphine - induced place preference , whereas 4 - aminopyridine was ineffective .\n",
            "\n",
            "Example Response:\n",
            "{'Quinine_0': 'B', 'is_1': 'O', 'known_2': 'O', 'to_3': 'O', 'block_4': 'O', 'voltage_5': 'O', '-_6': 'O', ',_7': 'O', 'calcium_8': 'B', '-_9': 'O', 'and_10': 'O', 'ATP_11': 'B', '-_12': 'O', 'sensitive_13': 'O', 'K_14': 'B', '(_15': 'O', '+_16': 'O', ')_17': 'O', '-_18': 'O', 'channels_19': 'O', 'while_20': 'O', '4_21': 'B', '-_22': 'I', 'aminopyridine_23': 'I', 'is_24': 'O', 'known_25': 'O', 'to_26': 'O', 'block_27': 'O', 'voltage_28': 'O', '-_29': 'O', 'sensitive_30': 'O', 'K_31': 'B', '(_32': 'O', '+_33': 'O', ')_34': 'O', '-_35': 'O', 'channels_36': 'O', '._37': 'O', '2_38': 'O', '._39': 'O', 'In_40': 'O', 'the_41': 'O', 'counterbalanced_42': 'O', 'method_43': 'O', ',_44': 'O', 'quinine_45': 'B', 'attenuated_46': 'O', 'morphine_47': 'B', '-_48': 'O', 'induced_49': 'O', 'place_50': 'O', 'preference_51': 'O', ',_52': 'O', 'whereas_53': 'O', '4_54': 'B', '-_55': 'I', 'aminopyridine_56': 'I', 'was_57': 'O', 'ineffective_58': 'O', '._59': 'O'}\n",
            "\n",
            "Now label the following text: \n",
            "22 - oxacalcitriol suppresses secondary hyperparathyroidism without inducing low bone turnover in dogs with renal failure . BACKGROUND : Calcitriol therapy suppresses serum levels of parathyroid hormone ( PTH ) in patients with renal failure but has several drawbacks , including hypercalcemia and / or marked suppression of bone turnover , which may lead to adynamic bone disease .\n"
          ]
        }
      ],
      "source": [
        "text = SENTENCE_CHUNKS[0] #text is the chunk to that is annotated by API call made in next cell\n",
        "\n",
        "# remember to change <entity type> based on dataset in \"focusing on identifying <entity type> entities\" of INSTRUCTION\n",
        "INSTRUCTION = f\"\"\"Given a biomedical text, perform Named Entity Recognition analysis on this text, focusing on identifying ONLY CHEMICAL ENTITIES. Individual tokens are separated by white space and must be given exactly one label: ‘B’,  ‘I’, or ‘O’, according to the BIO labeling schema. Output a JSON object with key equal to token concatenated with '_'+(index of token) and value equal to label, where (index of token) starts from 0 for the first token, 1 for the second token, and so on. Here is a labeled example to help with this task:\n",
        "\n",
        "Example Text:\n",
        "{example_text}\n",
        "\n",
        "Example Response:\n",
        "{example_response}\n",
        "\n",
        "Now label the following text: \"\"\"\n",
        "\n",
        "prompt = f\"\"\"{INSTRUCTION}\\n{text}\"\"\"\n",
        "print(prompt)"
      ]
    }
  ],
  "metadata": {
    "accelerator": "GPU",
    "colab": {
      "provenance": []
    },
    "kernelspec": {
      "display_name": "Python 3",
      "name": "python3"
    },
    "language_info": {
      "name": "python"
    }
  },
  "nbformat": 4,
  "nbformat_minor": 0
}